<!DOCTYPE html>
<html lang="en" class="no-js">
    <head>
        <meta charset="UTF-8"/>
        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
        <meta name="viewport" content="width=device-width, initial-scale=1"/>

        <title>What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map | Environmental Evidence | Full Text</title>

        

    <meta name="citation_abstract" content="Within the last decades, genome-editing techniques such as CRISPR/Cas, TALENs, Zinc-Finger Nucleases, Meganucleases, Oligonucleotide-Directed Mutagenesis and base editing have been developed enabling a precise modification of DNA sequences. Such techniques provide options for simple, time-saving and cost-effective applications compared to other breeding techniques and hence genome editing has already been promoted for a wide range of plant species. Although the application of genome-editing induces less unintended modifications (off-targets) in the genome compared to classical mutagenesis techniques, off-target effects are a prominent point of criticism as they are supposed to cause unintended effects, e.g. genomic instability or cell death. To address these aspects, this map aims to answer the following question: What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects? This primary question will be considered by two secondary questions: One aims to overview the market-oriented traits being modified by genome-editing in plants and the other explores the occurrence of off-target effects. A literature search in nine bibliographic databases, Google Scholar, and 47 web pages of companies and governmental agencies was conducted using predefined and tested search strings in English language. Articles were screened on title/abstract and full text level for relevance based on pre-defined inclusion criteria. The relevant information of included studies were mapped using a pre-defined data extraction strategy. Besides a descriptive summary of the relevant literature, a spreadsheet containing all extracted data is provided. Altogether, 555 relevant articles from journals, company web pages and web pages of governmental agencies were identified containing 1328 studies/applications of genome-editing in model plants and agricultural crops in the period January 1996 to May 2018. Most of the studies were conducted in China followed by the USA. Genome-editing was already applied in 68 different plants. Although most of the studies were basic research, 99 different market-oriented applications were identified in 28 different crops leading to plants with improved food and feed quality, agronomic value like growth characteristics or increased yield, tolerance to biotic and abiotic stress, herbicide tolerance or industrial benefits. 252 studies explored off-target effects. Most of the studies were conducted using CRISPR/Cas. Several studies firstly investigated whether sites in the genome show similarity to the target sequence and secondly analyzed these potential off-target sites by sequencing. In around 3% of the analyzed potential off-target sites, unintended mutations were detected. Only a few studies conducted off-target analyses using unbiased detection methods (e.g. whole genome sequencing). No off-target effects that could be correlated to the genome-editing process were identified in these studies. The rapid adoption in plant breeding was demonstrated by a considerable number of market oriented applications (crops and traits) described in publications worldwide. Studies investigating off-target effects are very heterogeneous in their structure and design. Therefore, an in-depth assessment regarding their weight of evidence is mandatory."/>

    <meta name="journal_id" content="13750"/>

    <meta name="dc.title" content="What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map"/>

    <meta name="dc.source" content="Environmental Evidence 2019 8:1"/>

    <meta name="dc.format" content="text/html"/>

    <meta name="dc.publisher" content="BioMed Central"/>

    <meta name="dc.date" content="2019-07-12"/>

    <meta name="dc.type" content="OriginalPaper"/>

    <meta name="dc.language" content="En"/>

    <meta name="dc.copyright" content="2019 The Author(s)"/>

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="dc.description" content="Within the last decades, genome-editing techniques such as CRISPR/Cas, TALENs, Zinc-Finger Nucleases, Meganucleases, Oligonucleotide-Directed Mutagenesis and base editing have been developed enabling a precise modification of DNA sequences. Such techniques provide options for simple, time-saving and cost-effective applications compared to other breeding techniques and hence genome editing has already been promoted for a wide range of plant species. Although the application of genome-editing induces less unintended modifications (off-targets) in the genome compared to classical mutagenesis techniques, off-target effects are a prominent point of criticism as they are supposed to cause unintended effects, e.g. genomic instability or cell death. To address these aspects, this map aims to answer the following question: What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects? This primary question will be considered by two secondary questions: One aims to overview the market-oriented traits being modified by genome-editing in plants and the other explores the occurrence of off-target effects. A literature search in nine bibliographic databases, Google Scholar, and 47 web pages of companies and governmental agencies was conducted using predefined and tested search strings in English language. Articles were screened on title/abstract and full text level for relevance based on pre-defined inclusion criteria. The relevant information of included studies were mapped using a pre-defined data extraction strategy. Besides a descriptive summary of the relevant literature, a spreadsheet containing all extracted data is provided. Altogether, 555 relevant articles from journals, company web pages and web pages of governmental agencies were identified containing 1328 studies/applications of genome-editing in model plants and agricultural crops in the period January 1996 to May 2018. Most of the studies were conducted in China followed by the USA. Genome-editing was already applied in 68 different plants. Although most of the studies were basic research, 99 different market-oriented applications were identified in 28 different crops leading to plants with improved food and feed quality, agronomic value like growth characteristics or increased yield, tolerance to biotic and abiotic stress, herbicide tolerance or industrial benefits. 252 studies explored off-target effects. Most of the studies were conducted using CRISPR/Cas. Several studies firstly investigated whether sites in the genome show similarity to the target sequence and secondly analyzed these potential off-target sites by sequencing. In around 3% of the analyzed potential off-target sites, unintended mutations were detected. Only a few studies conducted off-target analyses using unbiased detection methods (e.g. whole genome sequencing). No off-target effects that could be correlated to the genome-editing process were identified in these studies. The rapid adoption in plant breeding was demonstrated by a considerable number of market oriented applications (crops and traits) described in publications worldwide. Studies investigating off-target effects are very heterogeneous in their structure and design. Therefore, an in-depth assessment regarding their weight of evidence is mandatory."/>

    <meta name="prism.issn" content="2047-2382"/>

    <meta name="prism.publicationName" content="Environmental Evidence"/>

    <meta name="prism.publicationDate" content="2019-07-12"/>

    <meta name="prism.volume" content="8"/>

    <meta name="prism.number" content="1"/>

    <meta name="prism.section" content="OriginalPaper"/>

    <meta name="prism.startingPage" content="1"/>

    <meta name="prism.endingPage" content="33"/>

    <meta name="prism.copyright" content="2019 The Author(s)"/>

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com"/>

    <meta name="prism.url" content="https://environmentalevidencejournal.biomedcentral.com/articles/10.1186/s13750-019-0171-5"/>

    <meta name="prism.doi" content="doi:10.1186/s13750-019-0171-5"/>

    <meta name="citation_pdf_url" content="https://environmentalevidencejournal.biomedcentral.com/track/pdf/10.1186/s13750-019-0171-5"/>

    <meta name="citation_fulltext_html_url" content="https://environmentalevidencejournal.biomedcentral.com/articles/10.1186/s13750-019-0171-5"/>

    <meta name="citation_journal_title" content="Environmental Evidence"/>

    <meta name="citation_journal_abbrev" content="Environ Evid"/>

    <meta name="citation_publisher" content="BioMed Central"/>

    <meta name="citation_issn" content="2047-2382"/>

    <meta name="citation_title" content="What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map"/>

    <meta name="citation_volume" content="8"/>

    <meta name="citation_issue" content="1"/>

    <meta name="citation_publication_date" content="2019/12"/>

    <meta name="citation_online_date" content="2019/07/12"/>

    <meta name="citation_firstpage" content="1"/>

    <meta name="citation_lastpage" content="33"/>

    <meta name="citation_article_type" content="Systematic Map"/>

    <meta name="citation_fulltext_world_readable" content=""/>

    <meta name="citation_language" content="en"/>

    <meta name="dc.identifier" content="doi:10.1186/s13750-019-0171-5"/>

    <meta name="DOI" content="10.1186/s13750-019-0171-5"/>

    <meta name="citation_doi" content="10.1186/s13750-019-0171-5"/>

    <meta name="description" content="Within the last decades, genome-editing techniques such as CRISPR/Cas, TALENs, Zinc-Finger Nucleases, Meganucleases, Oligonucleotide-Directed Mutagenesis and base editing have been developed enabling a precise modification of DNA sequences. Such techniques provide options for simple, time-saving and cost-effective applications compared to other breeding techniques and hence genome editing has already been promoted for a wide range of plant species. Although the application of genome-editing induces less unintended modifications (off-targets) in the genome compared to classical mutagenesis techniques, off-target effects are a prominent point of criticism as they are supposed to cause unintended effects, e.g. genomic instability or cell death. To address these aspects, this map aims to answer the following question: What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects? This primary question will be considered by two secondary questions: One aims to overview the market-oriented traits being modified by genome-editing in plants and the other explores the occurrence of off-target effects. A literature search in nine bibliographic databases, Google Scholar, and 47 web pages of companies and governmental agencies was conducted using predefined and tested search strings in English language. Articles were screened on title/abstract and full text level for relevance based on pre-defined inclusion criteria. The relevant information of included studies were mapped using a pre-defined data extraction strategy. Besides a descriptive summary of the relevant literature, a spreadsheet containing all extracted data is provided. Altogether, 555 relevant articles from journals, company web pages and web pages of governmental agencies were identified containing 1328 studies/applications of genome-editing in model plants and agricultural crops in the period January 1996 to May 2018. Most of the studies were conducted in China followed by the USA. Genome-editing was already applied in 68 different plants. Although most of the studies were basic research, 99 different market-oriented applications were identified in 28 different crops leading to plants with improved food and feed quality, agronomic value like growth characteristics or increased yield, tolerance to biotic and abiotic stress, herbicide tolerance or industrial benefits. 252 studies explored off-target effects. Most of the studies were conducted using CRISPR/Cas. Several studies firstly investigated whether sites in the genome show similarity to the target sequence and secondly analyzed these potential off-target sites by sequencing. In around 3% of the analyzed potential off-target sites, unintended mutations were detected. Only a few studies conducted off-target analyses using unbiased detection methods (e.g. whole genome sequencing). No off-target effects that could be correlated to the genome-editing process were identified in these studies. The rapid adoption in plant breeding was demonstrated by a considerable number of market oriented applications (crops and traits) described in publications worldwide. Studies investigating off-target effects are very heterogeneous in their structure and design. Therefore, an in-depth assessment regarding their weight of evidence is mandatory."/>

    <meta name="dc.creator" content="Dominik Modrzejewski"/>

    <meta name="dc.creator" content="Frank Hartung"/>

    <meta name="dc.creator" content="Thorben Sprink"/>

    <meta name="dc.creator" content="D&#246;rthe Krause"/>

    <meta name="dc.creator" content="Christian Kohl"/>

    <meta name="dc.creator" content="Ralf Wilhelm"/>

    <meta name="dc.subject" content="Environmental Management"/>

    <meta name="dc.subject" content="Ecosystems"/>

    <meta name="dc.subject" content="Environmental Law/Policy/Ecojustice"/>

    <meta name="dc.subject" content="Environmental Policy"/>

    <meta name="dc.subject" content="Forestry Management"/>

    <meta name="citation_reference" content="citation_title=Pflanzenz&#252;chtung; citation_publication_date=2011; citation_id=CR1; citation_author=H Becker; citation_publisher=UTB"/>

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Plant genetics, sustainable agriculture and global food security; citation_author=P Ronald; citation_volume=188; citation_publication_date=2011; citation_pages=11-20; citation_doi=10.1534/genetics.111.128553; citation_id=CR2"/>

    <meta name="citation_reference" content="FAO&#8212;Committee on Agriculture (17th session). 06.03.2003. 
http://www.fao.org/docrep/meeting/006/y8704e.htm

. Accessed 25 Sept 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Theor Appl Genet; citation_title=Map-based cloning and characterization of Zea mays male sterility33 (ZmMs33) gene, encoding a glycerol-3-phosphate acyltransferase; citation_author=K Xie, S Wu, Z Li, Y Zhou, D Zhang, Z Dong; citation_volume=131; citation_publication_date=2018; citation_pages=1363-1378; citation_doi=10.1007/s00122-018-3083-9; citation_id=CR4"/>

    <meta name="citation_reference" content="citation_journal_title=Sustainability; citation_title=What have we learned from the land sparing-sharing model?; citation_author=B Phalan; citation_volume=10; citation_publication_date=2018; citation_pages=1760; citation_doi=10.3390/su10061760; citation_id=CR5"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Green Revolution research saved an estimated 18 to 27 million hectares from being brought into agricultural production; citation_author=JR Stevenson, N Villoria, D Byerlee, T Kelley, M Maredia; citation_volume=110; citation_publication_date=2013; citation_pages=8363-8368; citation_doi=10.1073/pnas.1208065110; citation_id=CR6"/>

    <meta name="citation_reference" content="citation_journal_title=Crit Rev Plant Sci; citation_title=Genome editing of plants; citation_author=DD Songstad, JF Petolino, DF Voytas, NA Reichert; citation_volume=36; citation_publication_date=2017; citation_pages=1-23; citation_doi=10.1080/07352689.2017.1281663; citation_id=CR7"/>

    <meta name="citation_reference" content="International Atomic Energy Agency (IAEA). Mutant variety database. 
https://mvd.iaea.org/#!Search?page=1&amp;size=15&amp;sortby=Name&amp;sort=ASC

. Accessed 25 Sept 2018."/>

    <meta name="citation_reference" content="citation_title=Dictionary of plant breeding; citation_publication_date=2009; citation_id=CR9; citation_author=RHJ Schlegel; citation_publisher=CRC Press"/>

    <meta name="citation_reference" content="SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017."/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Regeneration of intact tobacco plants containing full length copies of genetically engineered T-DNA, and transmission of T-DNA to R1 progeny; citation_author=KA Barton, AN Binns, AJM Matzke, M-D Chilton; citation_volume=32; citation_publication_date=1983; citation_pages=1033-1043; citation_doi=10.1016/0092-8674(83)90288-X; citation_id=CR11"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Expression of chimaeric genes transferred into plant cells using a Ti-plasmid-derived vector; citation_author=L Herrera-Estrella, A Depicker, M Montagu, J Schell; citation_volume=303; citation_publication_date=1983; citation_pages=209-213; citation_doi=10.1038/303209a0; citation_id=CR12"/>

    <meta name="citation_reference" content="citation_journal_title=J Cell Physiol; citation_title=Programmable site-specific nucleases for targeted genome engineering in higher eukaryotes; citation_author=G Govindan, S Ramalingam; citation_volume=231; citation_publication_date=2016; citation_pages=2380-2392; citation_doi=10.1002/jcp.25367; citation_id=CR13"/>

    <meta name="citation_reference" content="citation_journal_title=EFSA J; citation_title=Scientific opinion addressing the safety assessment of plants developed using Zinc Finger Nuclease 3 and other Site-Directed Nucleases with similar function; citation_author=; citation_volume=10; citation_publication_date=2012; citation_pages=2943; citation_doi=10.2903/j.efsa.2012.2943; citation_id=CR14"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol Rep; citation_title=New traits in crops produced by genome editing techniques based on deletions; citation_author=CCM Wiel, JG Schaart, LAP Lotz, MJM Smulders; citation_volume=11; citation_publication_date=2017; citation_pages=1-8; citation_doi=10.1007/s11816-017-0425-z; citation_id=CR15"/>

    <meta name="citation_reference" content="citation_journal_title=Trends Biotechnol; citation_title=Site-directed nucleases: a paradigm shift in predictable, knowledge-based plant breeding; citation_author=N Podevin, HV Davies, F Hartung, F Nogu&#233;, JM Casacuberta; citation_volume=31; citation_publication_date=2013; citation_pages=375-383; citation_doi=10.1016/j.tibtech.2013.03.004; citation_id=CR16"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage; citation_author=AC Komor, YB Kim, MS Packer, JA Zuris, DR Liu; citation_volume=533; citation_publication_date=2016; citation_pages=420-424; citation_doi=10.1038/nature17946; citation_id=CR17"/>

    <meta name="citation_reference" content="Nationale Akademie der Wissenschaften Leopoldina, Deutsche Forschungsgemeinschaft, acatech &#8210; Deutsche Akademie der Technikwissenschaften, Union der deutschen Akademien der Wissenschaften. The opportunities and limits of genome editing. 2015."/>

    <meta name="citation_reference" content="B&#246;meke O, Kahrmann J, Matthies A. Detaillierte &#220;bersicht zum regulatorischen Status der neuen molekularbiologischen Techniken (NMT) in ausgew&#228;hlten Drittstaaten. 
https://www.bmel.de/DE/Landwirtschaft/Pflanzenbau/Gentechnik/_Texte/Neue_molekularbiologische_Techniken.html

. Accessed 21 Feb 2019."/>

    <meta name="citation_reference" content="citation_journal_title=Plant Mol Biol; citation_title=Identification of a regulatory element responsible for salt induction of rice OsRAV2 through ex situ and in situ promoter analysis; citation_author=Y-B Duan, J Li, R-Y Qin, R-F Xu, H Li, Y-C Yang; citation_volume=90; citation_publication_date=2016; citation_pages=49-62; citation_doi=10.1007/s11103-015-0393-z; citation_id=CR20"/>

    <meta name="citation_reference" content="citation_journal_title=BMC Plant Biol; citation_title=Direct stacking of sequence-specific nuclease-induced mutations to produce high oleic and low linolenic soybean oil; citation_author=ZL Demorest, A Coffman, NJ Baltes, TJ Stoddard, BM Clasen, S Luo; citation_volume=16; citation_publication_date=2016; citation_pages=225; citation_doi=10.1186/s12870-016-0906-1; citation_id=CR21"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant Pathol; citation_title=Development of broad virus resistance in non-transgenic cucumber using CRISPR/Cas9 technology; citation_author=J Chandrasekaran, M Brumin, D Wolf, D Leibman, C Klap, M Pearlsman; citation_volume=17; citation_publication_date=2016; citation_pages=1140-1153; citation_doi=10.1111/mpp.12375; citation_id=CR22"/>

    <meta name="citation_reference" content="citation_title=Oligo-mediated targeted gene editing; citation_inbook_title=Advances in new technology for targeted modification of plant genomes; citation_publication_date=2015; citation_pages=73-89; citation_id=CR23; citation_author=GFW Gocal; citation_author=C Sch&#246;pke; citation_author=PR Beetham; citation_publisher=Springer"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=ARGOS8 variants generated by CRISPR&#8211;Cas9 improve maize grain yield under field drought stress conditions; citation_author=J Shi, H Gao, H Wang, HR Lafitte, RL Archibald, M Yang; citation_volume=15; citation_publication_date=2017; citation_pages=207-216; citation_doi=10.1111/pbi.12603; citation_id=CR24"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Simultaneous editing of three homoeoalleles in hexaploid bread wheat confers heritable resistance to powdery mildew; citation_author=Y Wang, X Cheng, Q Shan, Y Zhang, J Liu, C Gao, J-L Qiu; citation_volume=32; citation_publication_date=2014; citation_pages=947-951; citation_doi=10.1038/nbt.2969; citation_id=CR25"/>

    <meta name="citation_reference" content="citation_journal_title=Int J Mol Sci; citation_title=Biased and unbiased methods for the detection of off-target cleavage by CRISPR/Cas9: an overview; citation_author=F Martin, S S&#225;nchez-Hern&#225;ndez, A Guti&#233;rrez-Guerrero, J Pinedo-Gomez, K Benabdellah; citation_publication_date=2016; citation_doi=10.3390/ijms17091507; citation_id=CR26"/>

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Basic local alignment search tool; citation_author=S Altschul, W Gish, W Miller, EW Myers, DJ Lipman; citation_volume=215; citation_publication_date=1990; citation_pages=403-410; citation_doi=10.1006/jmbi.1990.9999; citation_id=CR27"/>

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases; citation_author=S Bae, J Park, J-S Kim; citation_volume=30; citation_publication_date=2014; citation_pages=1473-1475; citation_doi=10.1093/bioinformatics/btu048; citation_id=CR28"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant; citation_title=CRISPR-P: a web tool for synthetic single-guide RNA design of CRISPR-system in plants; citation_author=Y Lei, L Lu, H-Y Liu, S Li, F Xing, L-L Chen; citation_volume=7; citation_publication_date=2014; citation_pages=1494-1496; citation_doi=10.1093/mp/ssu044; citation_id=CR29"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant; citation_title=CRISPR-P 2.0: an improved CRISPR&#8211;Cas9 tool for genome editing in plants; citation_author=H Liu, Y Ding, Y Zhou, W Jin, K Xie, L-L Chen; citation_volume=10; citation_publication_date=2017; citation_pages=530-532; citation_doi=10.1016/j.molp.2017.01.003; citation_id=CR30"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Cell; citation_title=Methods for optimizing CRISPR&#8211;Cas9 genome editing specificity; citation_author=J Tycko, VE Myer, PD Hsu; citation_volume=63; citation_publication_date=2016; citation_pages=355-370; citation_doi=10.1016/j.molcel.2016.07.004; citation_id=CR31"/>

    <meta name="citation_reference" content="citation_journal_title=Biotechnol Adv; citation_title=Detection of on-target and off-target mutations generated by CRISPR/Cas9 and other sequence-specific nucleases; citation_author=J Zischewski, R Fischer, L Bortesi; citation_volume=35; citation_publication_date=2017; citation_pages=95-104; citation_doi=10.1016/j.biotechadv.2016.12.003; citation_id=CR32"/>

    <meta name="citation_reference" content="citation_title=Current status of emerging technologies for plant breeding. Biosafety and knowledge gaps of site directed nucleases and oligonucleotide-directed mutagenesis; citation_publication_date=2015; citation_id=CR33; citation_author=SZ Agapito-Tenfen; citation_author=O-G Wikmark; citation_publisher=Troms&#248;"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Ther Nucleic Acids; citation_title=Off-target effects in CRISPR/Cas9-mediated genome engineering; citation_author=X-H Zhang, LY Tee, X-G Wang, Q-S Huang, S-H Yang; citation_volume=4; citation_publication_date=2015; citation_pages=e264; citation_doi=10.1038/mtna.2015.37; citation_id=CR34"/>

    <meta name="citation_reference" content="citation_journal_title=Trends Biotechnol; citation_title=Fine-tuning next-generation genome editing tools; citation_author=CN Kanchiswamy, M Maffei, M Malnoy, R Velasco, J-S Kim; citation_volume=34; citation_publication_date=2016; citation_pages=562-574; citation_doi=10.1016/j.tibtech.2016.03.007; citation_id=CR35"/>

    <meta name="citation_reference" content="Collaboration for Environmental Evidence. Guidelines for systematic review and evidence synthesis in environmental management, Version 4.2. Environmental Evidence. 2013. 
http://www.environmentalevidence.org/Documents/Guidelines/Guidelines4.2.pd

. Accessed 13 Sept 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Environ Evid; citation_title=What is the available evidence for the application of genome editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map protocol; citation_author=D Modrzejewski, F Hartung, T Sprink, D Krause, C Kohl, J Schiemann, R Wilhelm; citation_volume=7; citation_publication_date=2018; citation_pages=11; citation_doi=10.1186/s13750-018-0130-6; citation_id=CR37"/>

    <meta name="citation_reference" content="citation_journal_title=Educ Psychol Meas; citation_title=A coefficient of agreement for nominal scales; citation_author=J Cohen; citation_volume=20; citation_publication_date=1960; citation_pages=37-46; citation_doi=10.1177/001316446002000104; citation_id=CR38"/>

    <meta name="citation_reference" content="citation_journal_title=Environ Evid; citation_title=Online tools supporting the conduct and reporting of systematic reviews and systematic maps: a case study on CADIMA and review of existing tools; citation_author=C Kohl, EJ McIntosh, S Unger, NR Haddaway, S Kecke, J Schiemann, R Wilhelm; citation_volume=7; citation_publication_date=2018; citation_pages=2420; citation_doi=10.1186/s13750-018-0115-5; citation_id=CR39"/>

    <meta name="citation_reference" content="Haddaway NR, Macura B, Whaley P, Pullin AS. ROSES for Systematic Map Reports. Version 1.0. 2017. 
https://doi.org/10.6084/m9.figshare.5897299

."/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=CRISPR&#8211;Cas9 targeted mutagenesis leads to simultaneous modification of different homoeologous gene copies in polyploid oilseed rape (Brassica napus); citation_author=J Braatz, H-J Harloff, M Mascher, N Stein, A Himmelbach, C Jung; citation_volume=174; citation_publication_date=2017; citation_pages=9; citation_doi=10.1104/pp.17.00426; citation_id=CR41"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Precise editing of CLAVATA genes in Brassica napus L. regulates multilocular silique development; citation_author=Y Yang, K Zhu, H Li, S Han, Q Meng, SU Khan; citation_volume=16; citation_publication_date=2018; citation_pages=1322-1335; citation_doi=10.1111/pbi.12872; citation_id=CR42"/>

    <meta name="citation_reference" content="citation_journal_title=Sci China Life Sci; citation_title=Increased lateral root formation by CRISPR/Cas9-mediated editing of arginase genes in cotton; citation_author=Y Wang, Z Meng, C Liang, Z Meng, Y Wang, G Sun; citation_volume=60; citation_publication_date=2017; citation_pages=524-527; citation_doi=10.1007/s11427-017-9031-y; citation_id=CR43"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant; citation_title=Engineering non-transgenic gynoecious cucumber using an improved transformation protocol and optimized CRISPR/Cas9 system; citation_author=B Hu, D Li, X Liu, J Qi, D Gao, S Zhao; citation_volume=10; citation_publication_date=2017; citation_pages=1575-1578; citation_doi=10.1016/j.molp.2017.09.005; citation_id=CR44"/>

    <meta name="citation_reference" content="citation_journal_title=G3; citation_title=High-resolution analysis of the efficiency, heritability, and editing outcomes of CRISPR/Cas9-induced modifications of NCED4 in lettuce (Lactuca sativa); citation_author=LD Bertier, M Ron, H Huo, KJ Bradford, AB Britt, RW Michelmore; citation_volume=8; citation_publication_date=2018; citation_pages=1513-1521; citation_doi=10.1534/g3.117.300396; citation_id=CR45"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2015. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-013-01air.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=ZmCCT9 enhances maize adaptation to higher latitudes; citation_author=C Huang, H Sun, D Xu, Q Chen, Y Liang, X Wang; citation_volume=115; citation_publication_date=2018; citation_pages=E334-E341; citation_doi=10.1073/pnas.1718058115; citation_id=CR47"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Targeted mutagenesis, precise gene editing, and site-specific gene insertion in maize using Cas9 and guide RNA; citation_author=S Svitashev, JK Young, C Schwartz, H Gao, SC Falco, AM Cigan; citation_volume=169; citation_publication_date=2015; citation_pages=931-945; citation_doi=10.1104/pp.15.00793; citation_id=CR48"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Commun; citation_title=Genome editing in maize directed by CRISPR&#8211;Cas9 ribonucleoprotein complexes; citation_author=S Svitashev, C Schwartz, B Lenderts, JK Young, Cigan A Mark; citation_volume=7; citation_publication_date=2016; citation_pages=1-7; citation_doi=10.1038/ncomms13274; citation_id=CR49"/>

    <meta name="citation_reference" content="citation_journal_title=J Genet Genom; citation_title=Generation of thermosensitive male-sterile maize by targeted knockout of the ZmTMS5 gene; citation_author=J Li, H Zhang, X Si, Y Tian, K Chen, J Liu; citation_volume=44; citation_publication_date=2017; citation_pages=465-468; citation_doi=10.1016/j.jgg.2017.02.002; citation_id=CR50"/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=MATRILINEAL, a sperm-specific phospholipase, triggers maize haploid induction; citation_author=T Kelliher, D Starr, L Richbourg, S Chintamanani, B Delzer, ML Nuccio; citation_volume=542; citation_publication_date=2017; citation_pages=105-109; citation_doi=10.1038/nature20827; citation_id=CR51"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Improving cold storage and processing traits in potato through targeted gene knockout; citation_author=BM Clasen, TJ Stoddard, S Luo, ZL Demorest, J Li, F Cedrone; citation_volume=14; citation_publication_date=2016; citation_pages=169-176; citation_doi=10.1111/pbi.12370; citation_id=CR52"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Reassessment of the four yield-related genes Gn1a, DEP1, GS3, and IPA1 in rice using a CRISPR/Cas9 system; citation_author=M Li, X Li, Z Zhou, P Wu, M Fang, X Pan; citation_volume=7; citation_publication_date=2016; citation_pages=1-13; citation_doi=10.3389/fpls.2016.00377; citation_id=CR53"/>

    <meta name="citation_reference" content="citation_journal_title=J Integr Plant Biol; citation_title=QTL editing confers opposing yield performance in different rice varieties; citation_author=L Shen, C Wang, Y Fu, J Wang, Q Liu, X Zhang; citation_publication_date=2016; citation_doi=10.1111/jipb.12501; citation_id=CR54"/>

    <meta name="citation_reference" content="citation_journal_title=J Genet Genom; citation_title=Rapid improvement of grain weight via highly efficient CRISPR/Cas9-mediated multiplex genome editing in rice; citation_author=R Xu, Y Yang, R Qin, H Li, C Qiu, L Li; citation_volume=43; citation_publication_date=2016; citation_pages=529-532; citation_doi=10.1016/j.jgg.2016.07.003; citation_id=CR55"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant; citation_title=A novel QTL q TGW3 encodes the GSK3/SHAGGY-like kinase OsGSK5/OsSK41 that interacts with OsARF4 to negatively regulate grain size and weight in rice; citation_author=Z Hu, S-J Lu, M-J Wang, H He, L Sun, H Wang; citation_volume=11; citation_publication_date=2018; citation_pages=736-749; citation_doi=10.1016/j.molp.2018.03.005; citation_id=CR56"/>

    <meta name="citation_reference" content="citation_journal_title=Chin J Rice Sci; citation_title=Orientation improvement of grain length and grain number in rice by using CRISPR/Cas9 system; citation_author=L Shen, J Li, Y Fu, J Wang, Y Hua, X Jiao, C Yan, K Wang; citation_publication_date=2017; citation_doi=10.16819/j.1001-7216.2017.7029; citation_id=CR57"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Agric Sin; citation_title=CRISPR/Cas9 system-based editing of phytochrome-interacting factor OsPIL15; citation_author=X Ji, F Li, Y Yan, HZ Sun, J Zhang, JZ Li; citation_publication_date=2017; citation_doi=10.3864/j.issn.0578-1752.2017.15.002; citation_id=CR58"/>

    <meta name="citation_reference" content="citation_journal_title=Sci China Life Sci; citation_title=Rapid generation of genetic diversity by multiplex CRISPR/Cas9 genome editing in rice; citation_author=L Shen, Y Hua, Y Fu, J Li, Q Liu, X Jiao; citation_volume=60; citation_publication_date=2017; citation_pages=506-515; citation_doi=10.1007/s11427-017-9008-8; citation_id=CR59"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Blocking amino acid transporter OsAAP3 improves grain yield by promoting outgrowth buds and increasing tiller number in rice; citation_author=K Lu, B Wu, J Wang, W Zhu, H Nie, J Qian; citation_volume=50; citation_publication_date=2018; citation_pages=1416; citation_doi=10.1111/pbi.12907; citation_id=CR60"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Mutation in rice abscisic acid2 results in cell death, enhanced disease-resistance, altered seed dormancy and development; citation_author=Y Liao, Q Bai, P Xu, T Wu, D Guo, Y Peng; citation_volume=9; citation_publication_date=2018; citation_pages=1248; citation_doi=10.3389/fpls.2018.00405; citation_id=CR61"/>

    <meta name="citation_reference" content="citation_journal_title=J Genet Genom; citation_title=High-efficiency breeding of early-maturing rice cultivars via CRISPR/Cas9-mediated genome editing; citation_author=X Li, W Zhou, Y Ren, X Tian, T Lv, Z Wang; citation_volume=44; citation_publication_date=2017; citation_pages=175-178; citation_doi=10.1016/j.jgg.2017.02.001; citation_id=CR62"/>

    <meta name="citation_reference" content="citation_journal_title=J Exp Bot; citation_title=Plastidic phosphoglucomutase and ADP-glucose pyrophosphorylase mutants impair starch synthesis in rice pollen grains and cause male sterility; citation_author=S-K Lee, J-S Eom, S-K Hwang, D Shin, G An, TW Okita, J-S Jeon; citation_volume=67; citation_publication_date=2016; citation_pages=5557-5569; citation_doi=10.1093/jxb/erw324; citation_id=CR63"/>

    <meta name="citation_reference" content="citation_journal_title=J Genet Genom; citation_title=Development of japonica photo-sensitive genic male sterile rice lines by editing carbon starved anther using CRISPR/Cas9; citation_author=Q Li, D Zhang, M Chen, W Liang, J Wei, Y Qi, Z Yuan; citation_volume=43; citation_publication_date=2016; citation_pages=415-419; citation_doi=10.1016/j.jgg.2016.04.011; citation_id=CR64"/>

    <meta name="citation_reference" content="citation_journal_title=J Integr Plant Biol; citation_title=Suppression or knockout of SaF/SaM overcomes the Sa-mediated hybrid male sterility in rice; citation_author=Y Xie, B Niu, Y Long, G Li, J Tang, Y Zhang; citation_volume=59; citation_publication_date=2017; citation_pages=669-679; citation_doi=10.1111/jipb.12564; citation_id=CR65"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Development of commercial thermo-sensitive genic male sterile rice accelerates hybrid rice breeding using the CRISPR/Cas9-mediated TMS5 editing system; citation_author=H Zhou, M He, J Li, L Chen, Z Huang, S Zheng; citation_volume=6; citation_publication_date=2016; citation_pages=1-12; citation_doi=10.1038/srep37395; citation_id=CR66"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Breed; citation_title=Knockout of OsACOS12 caused male sterility in rice; citation_author=T Zou, Z He, L Qu, M Liu, J Zeng, Y Liang; citation_volume=37; citation_publication_date=2017; citation_pages=437; citation_doi=10.1007/s11032-017-0722-9; citation_id=CR67"/>

    <meta name="citation_reference" content="citation_journal_title=Rice; citation_title=OsLAP6/OsPKS1, an orthologue of Arabidopsis PKSA/LAP6, is critical for proper pollen exine formation; citation_author=T Zou, Q Xiao, W Li, T Luo, G Yuan, Z He; citation_volume=10; citation_publication_date=2017; citation_pages=615; citation_doi=10.1186/s12284-017-0191-0; citation_id=CR68"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS Genet; citation_title=Receptor-like kinase RUPO interacts with potassium transporters to regulate pollen tube growth and integrity in rice; citation_author=L Liu, C Zheng, B Kuang, L Wei, L Yan, T Wang; citation_volume=12; citation_publication_date=2016; citation_pages=e1006085; citation_doi=10.1371/journal.pgen.1006085; citation_id=CR69"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=TALEN-based mutagenesis of lipoxygenase LOX3 enhances the storage tolerance of rice (Oryza sativa) seeds; citation_author=L Ma, F Zhu, Z Li, J Zhang, X Li, J Dong, T Wang; citation_volume=10; citation_publication_date=2015; citation_pages=e0143877; citation_doi=10.1371/journal.pone.0143877; citation_id=CR70"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Mol Biol; citation_title=Novel rice mutants overexpressing the brassinosteroid catabolic gene CYP734A4; citation_author=W Qian, C Wu, Y Fu, G Hu, Z He, W Liu; citation_volume=93; citation_publication_date=2017; citation_pages=197-208; citation_doi=10.1007/s11103-016-0558-4; citation_id=CR71"/>

    <meta name="citation_reference" content="citation_journal_title=New Phytol; citation_title=Both maternally and paternally imprinted genes regulate seed development in rice; citation_author=J Yuan, S Chen, W Jiao, L Wang, L Wang, W Ye; citation_volume=216; citation_publication_date=2017; citation_pages=373-387; citation_doi=10.1111/nph.14510; citation_id=CR72"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant; citation_title=Precise editing of a target base in the rice genome using a modified CRISPR/Cas9 system; citation_author=Y Lu, J-K Zhu; citation_volume=10; citation_publication_date=2017; citation_pages=523-525; citation_doi=10.1016/j.molp.2016.11.013; citation_id=CR73"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell Rep; citation_title=Deletion of a target gene in Indica rice via CRISPR/Cas9; citation_author=Y Wang, L Geng, M Yuan, J Wei, C Jin, M Li; citation_volume=36; citation_publication_date=2017; citation_pages=1333-1343; citation_doi=10.1007/s00299-017-2158-4; citation_id=CR74"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=9-cis-Epoxycarotenoid dioxygenase 3 regulates plant growth and enhances multi-abiotic stress tolerance in rice; citation_author=Y Huang, Y Guo, Y Liu, F Zhang, Z Wang, H Wang; citation_volume=9; citation_publication_date=2018; citation_pages=1248; citation_doi=10.3389/fpls.2018.00162; citation_id=CR75"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=CRISPR/Cas9-mediated targeted mutagenesis of GmFT2a delays flowering time in soya bean; citation_author=Y Cai, L Chen, X Liu, C Guo, S Sun, C Wu; citation_publication_date=2017; citation_doi=10.1111/pbi.12758; citation_id=CR76"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Targeted mutagenesis in tetraploid switchgrass (Panicum virgatum L.) using CRISPR/Cas9; citation_author=Y Liu, P Merrick, Z Zhang, C Ji, B Yang, S-Z Fei; citation_volume=16; citation_publication_date=2018; citation_pages=381-393; citation_doi=10.1111/pbi.12778; citation_id=CR77"/>

    <meta name="citation_reference" content="citation_journal_title=Biochem Biophys Res Commun; citation_title=CRISPR/Cas9-mediated mutagenesis of the RIN locus that regulates tomato fruit ripening; citation_author=Y Ito, A Nishizawa-Yokoi, M Endo, M Mikami, S Toki; citation_volume=467; citation_publication_date=2015; citation_pages=76-82; citation_doi=10.1016/j.bbrc.2015.09.117; citation_id=CR78"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Targeted mutagenesis of the tomato PROCERA gene using transcription activator-like effector nucleases; citation_author=VS Lor, CG Starker, DF Voytas, D Weiss, NE Olszewski; citation_volume=166; citation_publication_date=2014; citation_pages=1288-1291; citation_doi=10.1104/pp.114.247593; citation_id=CR79"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Genet; citation_title=Variation in the flowering gene SELF PRUNING 5G promotes day-neutrality and early yield in tomato; citation_author=S Soyk, NA M&#252;ller, SJ Park, I Schmalenbach, K Jiang, R Hayama; citation_volume=49; citation_publication_date=2017; citation_pages=162-168; citation_doi=10.1038/ng.3733; citation_id=CR80"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2018. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/18-051-01_air_response_signed.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Engineering quantitative trait variation for crop improvement by genome editing; citation_author=D Rodr&#237;guez-Leal, ZH Lemmon, J Man, ME Bartlett, ZB Lippman; citation_volume=171; citation_issue=470&#8211;480; citation_publication_date=2017; citation_pages=e8; citation_doi=10.1016/j.cell.2017.08.030; citation_id=CR82"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Commun; citation_title=Targeted recombination between homologous chromosomes for precise breeding in tomato; citation_author=S Filler Hayut, C Melamed Bessudo, AA Levy; citation_volume=8; citation_publication_date=2017; citation_pages=15605; citation_doi=10.1038/ncomms15605; citation_id=CR83"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=Efficient in planta gene targeting in tomato using geminiviral replicons and the CRISPR/Cas9 system; citation_author=T Dahan-Meir, S Filler-Hayut, C Melamed-Bessudo, S Bocobza, H Czosnek, A Aharoni, AA Levy; citation_volume=95; citation_publication_date=2018; citation_pages=5-16; citation_doi=10.1111/tpj.13932; citation_id=CR84"/>

    <meta name="citation_reference" content="citation_journal_title=J Genet Genom; citation_title=Efficient generation of pink-fruited tomatoes using CRISPR/Cas9 system; citation_author=L Deng, H Wang, C Sun, Q Li, H Jiang, M Du; citation_volume=45; citation_publication_date=2018; citation_pages=51-54; citation_doi=10.1016/j.jgg.2017.10.002; citation_id=CR85"/>

    <meta name="citation_reference" content="citation_journal_title=CRISPR J; citation_title=Transgenerational CRISPR&#8211;Cas9 activity facilitates multiplex gene editing in allopolyploid wheat; citation_author=W Wang, Q Pan, F He, A Akhunova, S Chao, H Trick, E Akhunov; citation_volume=1; citation_publication_date=2018; citation_pages=65-74; citation_doi=10.1089/crispr.2017.0010; citation_id=CR86"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=Analysis of the functions of TaGW2 homoeologs in wheat grain weight and protein content traits; citation_author=Y Zhang, D Li, D Zhang, X Zhao, X Cao, L Dong; citation_volume=94; citation_publication_date=2018; citation_pages=857-866; citation_doi=10.1111/tpj.13903; citation_id=CR87"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Efficient genome editing of wild strawberry genes, vector development and validation; citation_author=J Zhou, G Wang, Z Liu; citation_volume=166; citation_publication_date=2018; citation_pages=1292; citation_doi=10.1111/pbi.12922; citation_id=CR88"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2017. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-038-02_air_inquiry_cbidel.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol Biochem; citation_title=Mutagenesis of the FAE1 genes significantly changes fatty acid composition in seeds of Camelina sativa; citation_author=ME Ozseyhan, J Kang, X Mu, C Lu; citation_volume=123; citation_publication_date=2018; citation_pages=1-7; citation_doi=10.1016/j.plaphy.2017.11.021; citation_id=CR90"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Significant enhancement of fatty acid composition in seeds of the allohexaploid, Camelina sativa, using CRISPR/Cas9 gene editing; citation_author=WZ Jiang, IM Henry, PG Lynagh, L Comai, EB Cahoon, DP Weeks; citation_volume=15; citation_publication_date=2017; citation_pages=648-657; citation_doi=10.1111/pbi.12663; citation_id=CR91"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Selective gene dosage by CRISPR&#8211;Cas9 genome editing in hexaploid Camelina sativa; citation_author=C Morineau, Y Bellec, F Tellier, L Gissot, Z Kelemen, F Nogu&#233;, J-D Faure; citation_volume=15; citation_publication_date=2017; citation_pages=729-739; citation_doi=10.1111/pbi.12671; citation_id=CR92"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol; citation_title=Simultaneous targeting of multiple gene homeologs to alter seed oil production in Camelina sativa; citation_author=JA Aznar-Moreno, TP Durrett; citation_volume=58; citation_publication_date=2017; citation_pages=1260-1267; citation_doi=10.1093/pcp/pcx058; citation_id=CR93"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol Biochem; citation_title=CRISPR/Cas9-mediated genome editing of the fatty acid desaturase 2 gene in Brassica napus; citation_author=A Okuzaki, T Ogawa, C Koizuka, K Kaneko, M Inaba, J Imamura, N Koizuka; citation_publication_date=2018; citation_doi=10.1016/j.plaphy.2018.04.025; citation_id=CR94"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2015. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-352-01_air_inquiry_cbidel.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Crop J; citation_title=Systematic identification of endogenous RNA polymerase III promoters for efficient RNA guide-based genome editing technologies in maize; citation_author=X Qi, L Dong, C Liu, L Mao, F Liu, X Zhang; citation_volume=6; citation_publication_date=2018; citation_pages=314-320; citation_doi=10.1016/j.cj.2018.02.005; citation_id=CR96"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2015. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-078-02_air_inquiry.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2010. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/DOW_Email_%20to_Susan_%20Kohler_032010.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Precise genome modification in the crop species Zea mays using zinc-finger nucleases; citation_author=VK Shukla, Y Doyon, JC Miller, RC DeKelver, EA Moehle, SE Worden; citation_volume=459; citation_publication_date=2009; citation_pages=437-441; citation_doi=10.1038/nature07992; citation_id=CR99"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2015. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-321-01_air_inquiry.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Manipulating the biosynthesis of bioactive compound alkaloids for next-generation metabolic engineering in opium poppy using CRISPR&#8211;Cas 9 genome editing technology; citation_author=Y Alagoz, T Gurkok, B Zhang, T Unver; citation_volume=6; citation_publication_date=2016; citation_pages=30910-30918; citation_doi=10.1038/srep30910; citation_id=CR101"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Mol Biol; citation_title=TALEN-mediated targeted mutagenesis of fatty acid desaturase 2 (FAD2) in peanut (Arachis hypogaea L.) promotes the accumulation of oleic acid; citation_author=S Wen, H Liu, X Li, X Chen, Y Hong, H Li; citation_volume=97; citation_publication_date=2018; citation_pages=177-185; citation_doi=10.1007/s11103-018-0731-z; citation_id=CR102"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2016. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/16-090-01_air_inquiry_cbidel.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2016. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/16-320-01_air_inquiry.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell; citation_title=Sterol side chain reductase 2 is a key enzyme in the biosynthesis of cholesterol, the common precursor of toxic steroidal glycoalkaloids in potato; citation_author=S Sawai, K Ohyama, S Yasumoto, H Seki, T Sakuma, T Yamamoto; citation_volume=26; citation_publication_date=2014; citation_pages=3763-3774; citation_doi=10.1105/tpc.114.130096; citation_id=CR105"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol Biochem; citation_title=Generation of &#945;-solanine-free hairy roots of potato by CRISPR/Cas9 mediated genome editing of the St16DOX gene; citation_author=M Nakayasu, R Akiyama, HJ Lee, K Osakabe, Y Osakabe, B Watanabe; citation_publication_date=2018; citation_doi=10.1016/j.plaphy.2018.04.026; citation_id=CR106"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Creation of fragrant rice by targeted knockout of the OsBADH2 gene using TALEN technology; citation_author=Q Shan, Y Zhang, K Chen, K Zhang, C Gao; citation_volume=13; citation_publication_date=2015; citation_pages=791-800; citation_doi=10.1111/pbi.12312; citation_id=CR107"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Generation of high-amylose rice through CRISPR/Cas9-mediated targeted mutagenesis of starch branching enzymes; citation_author=Y Sun, G Jiao, Z Liu, X Zhang, J Li, X Guo; citation_volume=8; citation_publication_date=2017; citation_pages=1-15; citation_doi=10.3389/fpls.2017.00298; citation_id=CR108"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=OsPT4 contributes to arsenate uptake and transport in rice; citation_author=Y Ye, P Li, T Xu, L Zeng, D Cheng, M Yang; citation_volume=8; citation_publication_date=2017; citation_pages=311; citation_doi=10.3389/fpls.2017.02197; citation_id=CR109"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=OsARM1, an R2R3 MYB transcription factor, is involved in regulation of the response to arsenic stress in rice; citation_author=F-Z Wang, M-X Chen, L-J Yu, L-J Xie, L-B Yuan, H Qi; citation_volume=8; citation_publication_date=2017; citation_pages=1868; citation_doi=10.3389/fpls.2017.01868; citation_id=CR110"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol Biochem; citation_title=Production of high oleic/low linoleic rice by genome editing; citation_author=K Abe, E Araki, Y Suzuki, S Toki, H Saika; citation_publication_date=2018; citation_doi=10.1016/j.plaphy.2018.04.033; citation_id=CR111"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=Production of low-Cs+ rice plants by inactivation of the K+ transporter OsHAK1 with the CRISPR&#8211;Cas system; citation_author=M Nieves-Cordones, S Mohamed, K Tanoi, NI Kobayashi, K Takagi, A Vernet; citation_volume=92; citation_publication_date=2017; citation_pages=43-56; citation_doi=10.1111/tpj.13632; citation_id=CR112"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Knockout of OsNramp5 using the CRISPR/Cas9 system produces low Cd-accumulating indica rice without compromising yield; citation_author=L Tang, B Mao, Y Li, Q Lv, L Zhang, C Chen; citation_volume=7; citation_publication_date=2017; citation_pages=14438; citation_doi=10.1038/s41598-017-14832-9; citation_id=CR113"/>

    <meta name="citation_reference" content="citation_journal_title=J Integr Plant Biol; citation_title=Generation of new glutinous rice by CRISPR/Cas9-targeted mutagenesis of the Waxy gene in elite rice varieties; citation_author=J Zhang, H Zhang, JR Botella, J-K Zhu; citation_volume=60; citation_publication_date=2018; citation_pages=369-375; citation_doi=10.1111/jipb.12620; citation_id=CR114"/>

    <meta name="citation_reference" content="citation_journal_title=Phytochemistry; citation_title=CRISPR/Cas9-mediated efficient targeted mutagenesis of RAS in Salvia miltiorrhiza; citation_author=Z Zhou, H Tan, Q Li, J Chen, S Gao, Y Wang; citation_volume=148; citation_publication_date=2018; citation_pages=63-70; citation_doi=10.1016/j.phytochem.2018.01.015; citation_id=CR115"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2014. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/cellectis_air_fad2k0_soy_cbidel.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2015. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-071-01air.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Improved soybean oil quality by targeted mutagenesis of the fatty acid desaturase 2 gene family; citation_author=W Haun, A Coffman, BM Clasen, ZL Demorest, A Lowy, E Ray; citation_volume=12; citation_publication_date=2014; citation_pages=934-940; citation_doi=10.1111/pbi.12201; citation_id=CR118"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Tomato facultative parthenocarpy results from SlAGAMOUS-LIKE 6 loss of function; citation_author=C Klap, E Yeshayahou, AM Bolger, T Arazi, SK Gupta, S Shabtai; citation_volume=15; citation_publication_date=2017; citation_pages=634-647; citation_doi=10.1111/pbi.12662; citation_id=CR119"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Rapid breeding of parthenocarpic tomato plants using CRISPR/Cas9; citation_author=R Ueta, C Abe, T Watanabe, SS Sugano, R Ishihara, H Ezura; citation_volume=7; citation_publication_date=2017; citation_pages=507; citation_doi=10.1038/s41598-017-00501-4; citation_id=CR120"/>

    <meta name="citation_reference" content="citation_journal_title=J Agric Food Chem; citation_title=Utilization of a genome-edited tomato (Solanum lycopersicum) with high gamma aminobutyric acid content in hybrid breeding; citation_author=J Lee, S Nonaka, M Takayama, H Ezura; citation_volume=66; citation_publication_date=2018; citation_pages=963-971; citation_doi=10.1021/acs.jafc.7b05171; citation_id=CR121"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Efficient increase of &#611;-aminobutyric acid (GABA) content in tomato fruits by targeted mutagenesis; citation_author=S Nonaka, C Arai, M Takayama, C Matsukura, H Ezura; citation_volume=7; citation_publication_date=2017; citation_pages=7057; citation_doi=10.1038/s41598-017-06400-y; citation_id=CR122"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Lycopene is enriched in tomato fruit by CRISPR/Cas9-mediated multiplex genome editing; citation_author=X Li, Y Wang, S Chen, H Tian, D Fu, B Zhu; citation_volume=9; citation_publication_date=2018; citation_pages=179; citation_doi=10.3389/fpls.2018.00559; citation_id=CR123"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=CRISPR/Cas9-induced targeted mutagenesis and gene replacement to generate long-shelf life tomato lines; citation_author=Q-H Yu, B Wang, N Li, Y Tang, S Yang, T Yang; citation_volume=7; citation_publication_date=2017; citation_pages=818; citation_doi=10.1038/s41598-017-12262-1; citation_id=CR124"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2017. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-038-01_air_inquiry_cbidel.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Low-gluten, nontransgenic wheat engineered with CRISPR/Cas9; citation_author=S S&#225;nchez-Le&#243;n, J Gil-Humanes, CV Ozuna, MJ Gim&#233;nez, C Sousa, DF Voytas, F Barro; citation_publication_date=2017; citation_doi=10.1111/pbi.12837; citation_id=CR126"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Transient expression of CRISPR/Cas9 machinery targeting TcNPR3 enhances defense response in Theobroma cacao; citation_author=AS Fister, L Landherr, SN Maximova, MJ Guiltinan; citation_volume=9; citation_publication_date=2018; citation_pages=47; citation_doi=10.3389/fpls.2018.00268; citation_id=CR127"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Modification of the PthA4 effector binding elements in Type I CsLOB1 promoter using Cas9/sgRNA to produce transgenic Duncan grapefruit alleviating Xcc&#916;pthA4:dCsLOB1.3 infection; citation_author=H Jia, V Orbovic, JB Jones, N Wang; citation_volume=14; citation_publication_date=2016; citation_pages=1291-1301; citation_doi=10.1111/pbi.12495; citation_id=CR128"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Genome editing of the disease susceptibility gene CsLOB1 in citrus confers resistance to citrus canker; citation_author=H Jia, Y Zhang, V Orbovi&#263;, J Xu, FF White, JB Jones, N Wang; citation_volume=15; citation_publication_date=2017; citation_pages=817-823; citation_doi=10.1111/pbi.12677; citation_id=CR129"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=CRISPR/Cas9-mediated efficient targeted mutagenesis in grape in the first generation; citation_author=X Wang, M Tu, D Wang, J Liu, Y Li, Z Li; citation_volume=16; citation_publication_date=2018; citation_pages=844-855; citation_doi=10.1111/pbi.12832; citation_id=CR130"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2017. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-076-01_air_inquiry_cbidel.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Engineering canker-resistant plants through CRISPR/Cas9-targeted editing of the susceptibility gene CsLOB1 promoter in citrus; citation_author=A Peng, S Chen, T Lei, L Xu, Y He, L Wu; citation_volume=15; citation_publication_date=2017; citation_pages=1509-1519; citation_doi=10.1111/pbi.12733; citation_id=CR132"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Enhanced rice blast resistance by CRISPR/Cas9-targeted mutagenesis of the ERF transcription factor gene OsERF922; citation_author=F Wang, C Wang, P Liu, C Lei, W Hao, Y Gao; citation_volume=11; citation_publication_date=2016; citation_pages=e0154027; citation_doi=10.1371/journal.pone.0154027; citation_id=CR133"/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=Gene targeting by the TAL effector PthXo2 reveals cryptic resistance gene for bacterial blight of rice; citation_author=J Zhou, Z Peng, J Long, D Sosso, B Liu, J-S Eom; citation_volume=82; citation_publication_date=2015; citation_pages=632-643; citation_doi=10.1111/tpj.12838; citation_id=CR134"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Targeted promoter editing for rice resistance to Xanthomonas oryzae pv. oryzae reveals differential activities for SWEET14-inducing TAL effectors; citation_author=S Blanvillain-Baufum&#233;, M Reschke, M Sol&#233;, F Auguy, H Doucoure, B Szurek; citation_volume=15; citation_publication_date=2017; citation_pages=306-317; citation_doi=10.1111/pbi.12613; citation_id=CR135"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=High-efficiency TALEN-based gene editing produces disease-resistant rice; citation_author=T Li, B Liu, MH Spalding, DP Weeks, B Yang; citation_volume=30; citation_publication_date=2012; citation_pages=390-392; citation_doi=10.1038/nbt.2199; citation_id=CR136"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant Microbe Interact; citation_title=Induction of Xa10-like genes in rice cultivar nipponbare confers disease resistance to rice bacterial blight; citation_author=J Wang, D Tian, K Gu, X Yang, L Wang, X Zeng, Z Yin; citation_volume=30; citation_publication_date=2017; citation_pages=466-477; citation_doi=10.1094/MPMI-11-16-0229-R; citation_id=CR137"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Bull; citation_title=SRWD1, a novel target gene of DELLA and WRKY proteins, participates in the development and immune response of rice (Oryza sativa L.); citation_author=C Xie, G Zhang, Y Zhang, X Song, H Guo, X Chen, R Fang; citation_volume=62; citation_publication_date=2017; citation_pages=1639-1648; citation_doi=10.1016/j.scib.2017.12.002; citation_id=CR138"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Loss of function of a rice TPR-domain RNA-binding protein confers broad-spectrum disease resistance; citation_author=X Zhou, H Liao, M Chern, J Yin, Y Chen, J Wang; citation_volume=115; citation_publication_date=2018; citation_pages=3174-3179; citation_doi=10.1073/pnas.1705927115; citation_id=CR139"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2014. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/air_isu_ting_rice.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=A transcription activator-like effector Tal7 of Xanthomonas oryzae pv. oryzicola activates rice gene Os09g29100 to suppress rice immunity; citation_author=L Cai, Y Cao, Z Xu, W Ma, M Zakria, L Zou; citation_volume=7; citation_publication_date=2017; citation_pages=5089; citation_doi=10.1038/s41598-017-04800-8; citation_id=CR141"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Novel alleles of rice eIF4G generated by CRISPR/Cas9-targeted mutagenesis confer resistance to Rice tungro spherical virus; citation_author=A Macovei, NR Sevilla, C Cantos, GB Jonson, I Slamet-Loedin, T &#268;erm&#225;k; citation_volume=47; citation_publication_date=2018; citation_pages=417; citation_doi=10.1111/pbi.12927; citation_id=CR142"/>

    <meta name="citation_reference" content="citation_journal_title=Sci Rep; citation_title=Rapid generation of a transgene-free powdery mildew resistant tomato by genome deletion; citation_author=V Nekrasov, C Wang, J Win, C Lanz, D Weigel, S Kamoun; citation_volume=7; citation_publication_date=2017; citation_pages=482; citation_doi=10.1038/s41598-017-00578-x; citation_id=CR143"/>

    <meta name="citation_reference" content="Mahfouz M, Tashkandi M, Ali Z, Aljedaani F, Shami A. Engineering resistance against tomato yellow leaf curl virus via the CRISPR/Cas9 system in tomato. 2017. 
https://doi.org/10.1101/237735

."/>

    <meta name="citation_reference" content="Toledo Thomazella DP de, Brail Q, Dahlbeck D, Staskawicz BJ. CRISPR&#8211;Cas9 mediated mutagenesis of a DMR6 ortholog in tomato confers broad-spectrum disease resistance. 2016:1&#8211;23. 
https://doi.org/10.1101/064824

."/>

    <meta name="citation_reference" content="citation_journal_title=Plant J; citation_title=Simultaneous modification of three homoeologs of TaEDR1 by genome editing enhances powdery mildew resistance in wheat; citation_author=Y Zhang, Y Bai, G Wu, S Zou, Y Chen, C Gao, D Tang; citation_volume=91; citation_publication_date=2017; citation_pages=714-724; citation_doi=10.1111/tpj.13599; citation_id=CR146"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2015. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-238-01_air_inquiry_cbidel.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Plant Mol Biol; citation_title=Spontaneous mutation frequency in plants obscures the effect of chimeraplasty; citation_author=R Ruiter, I Brande, E Stals, S Delaur&#233;, M Cornelissen, K D&#8217;Halluin; citation_volume=53; citation_publication_date=2003; citation_pages=675-689; citation_doi=10.1023/b:plan.0000019111.96107.01; citation_id=CR148"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Allele exchange at the EPSPS locus confers glyphosate tolerance in cassava; citation_author=AW Hummel, RD Chauhan, T Cermak, AM Mutka, A Vijayaraghavan, A Boyher; citation_volume=16; citation_publication_date=2018; citation_pages=1275-1282; citation_doi=10.1111/pbi.12868; citation_id=CR149"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Targeted molecular trait stacking in cotton through targeted double-strand break induction; citation_author=K D&#8217;Halluin, C Vanderstraeten, J Hulle, J Rosolowska, I Brande, A Pennewaert; citation_volume=11; citation_publication_date=2013; citation_pages=933-941; citation_doi=10.1111/pbi.12085; citation_id=CR150"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Oligonucleotide-mediated genome editing provides precision and function to engineered nucleases and antibiotics in plants; citation_author=NJ Sauer, J Narv&#225;ez-V&#225;squez, J Mozoruk, RB Miller, ZJ Warburg, MJ Woodward; citation_volume=170; citation_publication_date=2016; citation_pages=1917-1928; citation_doi=10.1104/pp.15.01696; citation_id=CR151"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=Trait stacking via targeted genome editing; citation_author=WM Ainley, L Sastry-Dent, ME Welter, MG Murray, B Zeitler, R Amora; citation_volume=11; citation_publication_date=2013; citation_pages=1126-1134; citation_doi=10.1111/pbi.12107; citation_id=CR152"/>

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci; citation_title=Targeted manipulation of maize genes in vivo using chimeric RNA/DNA oligonucleotides; citation_author=T Zhu, DJ Peterson, L Tagliani, G Clair, CL Baszczynski, B Bowen; citation_volume=96; citation_publication_date=1999; citation_pages=8768-8773; citation_doi=10.1073/pnas.96.15.8768; citation_id=CR153"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Engineering herbicide-resistant maize using chimeric RNA/DNA oligonucleotides; citation_author=T Zhu, K Mettenburg, DJ Peterson, L Tagliani, CL Baszczynski; citation_volume=18; citation_publication_date=2000; citation_pages=555-558; citation_doi=10.1038/75435; citation_id=CR154"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Geminivirus-mediated genome editing in potato (Solanum tuberosum L.) using sequence-specific nucleases; citation_author=NM Butler, NJ Baltes, DF Voytas, DS Douches; citation_volume=7; citation_publication_date=2016; citation_pages=1-13; citation_doi=10.3389/fpls.2016.01045; citation_id=CR155"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Plants; citation_title=Gene replacements and insertions in rice by intron targeting using CRISPR&#8211;Cas9; citation_author=J Li, X Meng, Y Zong, K Chen, H Zhang, J Liu; citation_volume=2; citation_publication_date=2016; citation_pages=1-6; citation_doi=10.1038/nplants.2016.139; citation_id=CR156"/>

    <meta name="citation_reference" content="citation_journal_title=Mol Plant; citation_title=Engineering herbicide-resistant rice plants through CRISPR/Cas9-mediated homologous recombination of acetolactate synthase; citation_author=Y Sun, X Zhang, C Wu, Y He, Y Ma, H Hou; citation_volume=9; citation_publication_date=2016; citation_pages=628-631; citation_doi=10.1016/j.molp.2016.01.001; citation_id=CR157"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Gene editing by co-transformation of TALEN and chimeric RNA/DNA oligonucleotides on the rice OsEPSPS gene and the inheritance of mutations; citation_author=M Wang, Y Liu, C Zhang, J Liu, X Liu, L Wang; citation_volume=10; citation_publication_date=2015; citation_pages=e0122755; citation_doi=10.1371/journal.pone.0122755; citation_id=CR158"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell Rep; citation_title=Chimeric RNA/DNA oligonucleotide-directed gene targeting in rice; citation_author=A Okuzaki, K Toriyama; citation_volume=22; citation_publication_date=2004; citation_pages=509-512; citation_doi=10.1007/s00299-003-0698-2; citation_id=CR159"/>

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Targeted base editing in rice and tomato using a CRISPR&#8211;Cas9 cytidine deaminase fusion; citation_author=Z Shimatani, S Kashojiya, M Takayama, R Terada, T Arazoe, H Ishii; citation_volume=35; citation_publication_date=2017; citation_pages=441-443; citation_doi=10.1038/nbt.3833; citation_id=CR160"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol Biochem; citation_title=Inheritance of co-edited genes by CRISPR-based targeted nucleotide substitutions in rice; citation_author=Z Shimatani, U Fujikura, H Ishii, Y Matsui, M Suzuki, Y Ueke; citation_publication_date=2018; citation_doi=10.1016/j.plaphy.2018.04.028; citation_id=CR161"/>

    <meta name="citation_reference" content="citation_journal_title=Front Plant Sci; citation_title=Efficient CRISPR/Cas9-mediated genome editing using a chimeric single-guide RNA molecule; citation_author=H Butt, A Eid, Z Ali, MAM Atia, MM Mokhtar, N Hassan; citation_volume=8; citation_publication_date=2017; citation_pages=1441; citation_doi=10.3389/fpls.2017.01441; citation_id=CR162"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Physiol; citation_title=Cas9-guide RNA directed genome editing in soybean; citation_author=Z Li, Z-B Liu, A Xing, BP Moon, JP Koellhoffer, L Huang; citation_volume=169; citation_publication_date=2015; citation_pages=960-970; citation_doi=10.1104/pp.15.00783; citation_id=CR163"/>

    <meta name="citation_reference" content="citation_journal_title=Prog Mol Biol Transl Sci; citation_title=Use of CRISPR/Cas9 for crop improvement in maize and soybean; citation_author=D Chilcoat, Z-B Liu, J Sander; citation_volume=149; citation_publication_date=2017; citation_pages=27-46; citation_doi=10.1016/bs.pmbts.2017.04.005; citation_id=CR164"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2018. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/18-036-01_a1_air_inquiry_cbidel.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=New Phytol; citation_title=Exploiting SNPs for biallelic CRISPR mutations in the outcrossing woody perennial Populus reveals 4-coumarate:CoA ligase specificity and redundancy; citation_author=X Zhou, TB Jacobs, L-J Xue, SA Harding, C-J Tsai; citation_volume=208; citation_publication_date=2015; citation_pages=298-301; citation_doi=10.1111/nph.13470; citation_id=CR166"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell Rep; citation_title=Efficient targeted multiallelic mutagenesis in tetraploid potato (Solanum tuberosum) by transient CRISPR&#8211;Cas9 expression in protoplasts; citation_author=M Andersson, H Turesson, A Nicolia, A-S F&#228;lt, M Samuelsson, P Hofvander; citation_volume=36; citation_publication_date=2017; citation_pages=117-128; citation_doi=10.1007/s00299-016-2062-3; citation_id=CR167"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Mol Biol; citation_title=TALEN mediated targeted mutagenesis of the caffeic acid O-methyltransferase in highly polyploid sugarcane improves cell wall composition for production of bioethanol; citation_author=JH Jung, F Altpeter; citation_volume=92; citation_publication_date=2016; citation_pages=131-142; citation_doi=10.1007/s11103-016-0499-y; citation_id=CR168"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=TALEN-mediated targeted mutagenesis of more than 100 COMT copies/alleles in highly polyploid sugarcane improves saccharification efficiency without compromising biomass yield; citation_author=B Kannan, JH Jung, GW Moxley, S-M Lee, F Altpeter; citation_volume=16; citation_publication_date=2018; citation_pages=856-866; citation_doi=10.1111/pbi.12833; citation_id=CR169"/>

    <meta name="citation_reference" content="citation_journal_title=Biotechnol Biofuels; citation_title=Defined tetra-allelic gene disruption of the 4-coumarate: coenzyme A ligase 1 (Pv4CL1) gene by CRISPR/Cas9 in switchgrass results in lignin reduction and improved sugar release; citation_author=J-J Park, CG Yoo, A Flanagan, Y Pu, S Debnath, Y Ge; citation_volume=10; citation_publication_date=2017; citation_pages=284; citation_doi=10.1186/s13068-017-0972-0; citation_id=CR170"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2011. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-126-01_air_inquiry.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Afrika Focus; citation_title=Modulation of energy homeostasis in maize and Arabidopsis to develop lines tolerant to drought, genotoxic and oxidative stresses; citation_author=E Njuguna, G Coussens, S Aesaert, P Neyt, S Anami, M Lijsebettens; citation_publication_date=2018; citation_doi=10.21825/af.v30i2.8080; citation_id=CR172"/>

    <meta name="citation_reference" content="United States Department of Agriculture (USDA). 2017. 
https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-219-01_air_inquiry.pdf

. Accessed 25 Aug 2018."/>

    <meta name="citation_reference" content="citation_journal_title=Funct Integr Genom; citation_title=CRISPR/Cas9 genome editing in wheat; citation_author=D Kim, B Alptekin, H Budak; citation_volume=18; citation_publication_date=2018; citation_pages=31-41; citation_doi=10.1007/s10142-017-0572-x; citation_id=CR174"/>

    <meta name="citation_reference" content="citation_journal_title=Sci China Life Sci; citation_title=Robust genome editing of CRISPR&#8211;Cas9 at NAG PAMs in rice; citation_author=X Meng, X Hu, Q Liu, X Song, C Gao, J Li, K Wang; citation_volume=61; citation_publication_date=2018; citation_pages=122-125; citation_doi=10.1007/s11427-017-9247-9; citation_id=CR175"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Signal Behav; citation_title=Activity and specificity of TRV-mediated gene editing in plants; citation_author=Z Ali, A Abul-faraj, M Piatek, MM Mahfouz; citation_volume=10; citation_publication_date=2015; citation_pages=e1044191; citation_doi=10.1080/15592324.2015.1044191; citation_id=CR176"/>

    <meta name="citation_reference" content="citation_journal_title=J Plant Biol; citation_title=Knock out of the annexin gene OsAnn3 via CRISPR/Cas9-mediated genome editing decreased cold tolerance in rice; citation_author=C Shen, Z Que, Y Xia, N Tang, D Li, R He, M Cao; citation_volume=60; citation_publication_date=2017; citation_pages=539-547; citation_doi=10.1007/s12374-016-0400-1; citation_id=CR177"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Cell Rep; citation_title=Efficient CRISPR/Cas9-based gene knockout in watermelon; citation_author=S Tian, L Jiang, Q Gao, J Zhang, M Zong, H Zhang; citation_volume=36; citation_publication_date=2017; citation_pages=399-406; citation_doi=10.1007/s00299-016-2089-5; citation_id=CR178"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol J; citation_title=The CRISPR/Cas9 system produces specific and homozygous targeted gene editing in rice in one generation; citation_author=H Zhang, J Zhang, P Wei, B Zhang, F Gou, Z Feng; citation_volume=12; citation_publication_date=2014; citation_pages=797-807; citation_doi=10.1111/pbi.12200; citation_id=CR179"/>

    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Germline-transmitted genome editing in Arabidopsis thaliana using TAL-effector-nucleases; citation_author=J Forner, A Pfeiffer, T Langenecker, PA Manavella, P Manavella, JU Lohmann; citation_volume=10; citation_publication_date=2015; citation_pages=e0121056; citation_doi=10.1371/journal.pone.0121056; citation_id=CR180"/>

    <meta name="citation_reference" content="citation_journal_title=Chin J Biotechnol; citation_title=Specificity and inheritance of rBE3 and rBE4 endonuclease-induced gene modifications in rice; citation_author=B Ren, F Yan, Y Kuang, N Li, D Zhang, H Lin, H Zhou; citation_volume=33; citation_issue=10; citation_publication_date=2007; citation_pages=1776-85; citation_id=CR181"/>

    <meta name="citation_reference" content="citation_journal_title=Plant Biotechnol Rep; citation_title=Genome editing for targeted improvement of plants; citation_author=K Khandagale, A Nadaf; citation_volume=10; citation_publication_date=2016; citation_pages=327-343; citation_doi=10.1007/s11816-016-0417-4; citation_id=CR182"/>

    <meta name="citation_author" content="Dominik Modrzejewski"/>

    <meta name="citation_author_institution" content="Institute for Biosafety in Plant Biotechnology, Julius K&#252;hn-Institute, Federal Research Centre for Cultivated Plants, Quedlinburg, Germany"/>

    <meta name="citation_author" content="Frank Hartung"/>

    <meta name="citation_author_institution" content="Institute for Biosafety in Plant Biotechnology, Julius K&#252;hn-Institute, Federal Research Centre for Cultivated Plants, Quedlinburg, Germany"/>

    <meta name="citation_author" content="Thorben Sprink"/>

    <meta name="citation_author_institution" content="Institute for Biosafety in Plant Biotechnology, Julius K&#252;hn-Institute, Federal Research Centre for Cultivated Plants, Quedlinburg, Germany"/>

    <meta name="citation_author" content="D&#246;rthe Krause"/>

    <meta name="citation_author_institution" content="Institute for Biosafety in Plant Biotechnology, Julius K&#252;hn-Institute, Federal Research Centre for Cultivated Plants, Quedlinburg, Germany"/>

    <meta name="citation_author" content="Christian Kohl"/>

    <meta name="citation_author_institution" content="Institute for Biosafety in Plant Biotechnology, Julius K&#252;hn-Institute, Federal Research Centre for Cultivated Plants, Quedlinburg, Germany"/>

    <meta name="citation_author" content="Ralf Wilhelm"/>

    <meta name="citation_author_institution" content="Institute for Biosafety in Plant Biotechnology, Julius K&#252;hn-Institute, Federal Research Centre for Cultivated Plants, Quedlinburg, Germany"/>



        <meta name="format-detection" content="telephone=no">

        
<link rel="apple-touch-icon" sizes="180x180" href=/static/img/favicons/bmc/apple-touch-icon-582ef1d0f5.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/img/favicons/bmc/favicon-32x32-5d7879efe1.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/img/favicons/bmc/favicon-16x16-c241ac1a2f.png>
<link rel="manifest" href=/static/app-bmc/manifest-3bb9ad383b.json>
<link rel="mask-icon" href=/static/img/favicons/bmc/safari-pinned-tab-427338756f.svg color="#e6e6e6">
<link rel="shortcut icon" data-test="shortcut-icon" href=/static/img/favicons/bmc/favicon-1886637b78.ico>
<meta name="msapplication-config" content=/static/app-bmc/browserconfig-ecd9aa8205.xml>
<meta name="msapplication-TileColor"
      content="#1b3051">
<meta name="msapplication-TileImage"
      content=/static/img/favicons/bmc/app-icon-144x144.png>
<meta name="theme-color"
      content="#1b3051">


        
        <script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>

        
    <link rel="stylesheet" media="screen" href=/static/app-bmc/css/core-article-c36432aacc.css>


<link rel="stylesheet" media="screen" href=/static/app-bmc/css/core-7156a8fb40.css>
<link rel="stylesheet" media="print" href=/static/app-bmc/css/print-7680b1a80b.css>

<link rel="stylesheet" id="js-mustard" href=/static/app-bmc/css/enhanced-2a2a463ce1.css
      media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">


    <link rel="stylesheet" href=/static/app-bmc/css/enhanced-article-f5eef3a445.css
          media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">



        

        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'environmentalevidencejournal.biomedcentral.com',
            siteWithPath: 'environmentalevidencejournal.biomedcentral.com' + window.location.pathname,
            twitterHashtag: 'EnvironmentalEvidence',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/s13750-019-0171-5',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            hasFigshareInvoked: false,
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script>

        
    <script type="text/javascript" data-test="dataLayer">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/s13750-019-0171-5","articleType":"Systematic Map","peerReviewType":"Closed","supplement":null,"keywords":"New plant breeding techniques;Gene editing;Targeted genome modification;Mutagenesis;Unintended effects;Off-target mutation;Evidence map;Evidence synthesis"},"contentInfo":{"imprint":"BioMed Central","title":"What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map","publishedAt":1562889600000,"publishedAtDate":"2019-07-12","author":["Dominik Modrzejewski","Frank Hartung","Thorben Sprink","Drthe Krause","Christian Kohl","Ralf Wilhelm"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY + CC0","openAccess":true},"environment":"live"},"journal":{"siteKey":"environmentalevidencejournal.biomedcentral.com","volume":"8","issue":"1","title":"Environmental Evidence","type":"MLS 2","journalID":13750,"section":[]},"category":{"pmc":{"primarySubject":"Environment"},"contentType":"Systematic Map","publishingSegment":"LS 18"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"},"attributes":{"featureFlags":[],"environment":"live"}},"event":"dataLayerCreated"}];
    </script>

        

        <script class="js-entry">
    (function(w, d) {
        window.config = window.config || {};
        window.config.mustardcut = false;

        var ctmLinkEl = d.getElementById('js-mustard');

        if (ctmLinkEl && w.matchMedia && w.matchMedia(ctmLinkEl.media).matches) {
            window.config.mustardcut = true;

            
            
            
                window.Component = {};
            

            var currentScript = d.currentScript || d.head.querySelector('script.js-entry');

            
            function catchNoModuleSupport() {
                var scriptEl = d.createElement('script');
                return (!('noModule' in scriptEl) && 'onbeforeload' in scriptEl)
            }

            var headScripts = [
                { 'src' : '/static/js/polyfill-es5-bundle-ab77bcf8ba.js', 'async': false, 'module': false},
                { 'src' : '/static/js/airbrake-es5-bundle-4b7cd08b58.js', 'async': false, 'module': false},
                { 'src' : '/static/js/airbrake-es6-bundle-2473643371.js', 'async': false, 'module': true}
            ];

            var bodyScripts = [
                { 'src' : '/static/js/app-es5-bundle-f19b73132c.js', 'async': false, 'module': false},
                { 'src' : '/static/js/app-es6-bundle-b3be01973f.js', 'async': false, 'module': true}
                
                
                    , { 'src' : '/static/js/global-article-es5-bundle-bb21cea78d.js', 'async': false, 'module': false},
                    { 'src' : '/static/js/global-article-es6-bundle-53cefd7005.js', 'async': false, 'module': true}
                
            ];

            function createScript(script) {
                var scriptEl = d.createElement('script');
                scriptEl.src = script.src;
                scriptEl.async = script.async;
                if (script.module === true) {
                    scriptEl.type = "module";
                    if (catchNoModuleSupport()) {
                        scriptEl.src = '';
                    }
                } else if (script.module === false) {
                    scriptEl.setAttribute('nomodule', true)
                }
                if (script.charset) {
                    scriptEl.setAttribute('charset', script.charset);
                }

                return scriptEl;
            }

            for (var i=0; i<headScripts.length; ++i) {
                var scriptEl = createScript(headScripts[i]);
                currentScript.parentNode.insertBefore(scriptEl, currentScript.nextSibling);
            }

            d.addEventListener('DOMContentLoaded', function() {
                for (var i=0; i<bodyScripts.length; ++i) {
                    var scriptEl = createScript(bodyScripts[i]);
                    d.body.appendChild(scriptEl);
                }
            });

            // Webfont repeat view
            var config = w.config;
            if (config && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                d.documentElement.className += ' webfonts-loaded';
            }
        }
    })(window, document);
</script>


        
    
        
            <script src="/static/js/jquery-220afd743d.js"></script>
        
    
    
        <script src="https://cdn.cookielaw.org/scripttemplates/otSDKStub.js"
                type="text/javascript" charset="UTF-8" id="onetrust-bmc-control"
                data-domain-script="6fe96aee-7031-457e-8d5c-5b01fa7ec2e0">
        </script>
        <script type="text/javascript">
            function OptanonWrapper() { window.dataLayer.push( { event: 'OneTrustGroupsUpdated' } ) }
        </script>
    

        
    
        
    

        
    
        
            <!-- Google Tag Manager -->
            <script>
                (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
            </script>
            <!-- End Google Tag Manager -->
        
    


        
    
    <link rel="canonical" href="https://environmentalevidencejournal.biomedcentral.com/articles/10.1186/s13750-019-0171-5"/>
    

        
        
        
        <meta property="og:url" content="https://environmentalevidencejournal.biomedcentral.com/articles/10.1186/s13750-019-0171-5"/>
        <meta property="og:type" content="article"/>
        <meta property="og:site_name" content="Environmental Evidence"/>
        <meta property="og:title" content="What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map"/>
        <meta property="og:description" content="Within the last decades, genome-editing techniques such as CRISPR/Cas, TALENs, Zinc-Finger Nucleases, Meganucleases, Oligonucleotide-Directed Mutagenesis and base editing have been developed enabling a precise modification of DNA sequences. Such techniques provide options for simple, time-saving and cost-effective applications compared to other breeding techniques and hence genome editing has already been promoted for a wide range of plant species. Although the application of genome-editing induces less unintended modifications (off-targets) in the genome compared to classical mutagenesis techniques, off-target effects are a prominent point of criticism as they are supposed to cause unintended effects, e.g. genomic instability or cell death. To address these aspects, this map aims to answer the following question: What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects? This primary question will be considered by two secondary questions: One aims to overview the market-oriented traits being modified by genome-editing in plants and the other explores the occurrence of off-target effects. A literature search in nine bibliographic databases, Google Scholar, and 47 web pages of companies and governmental agencies was conducted using predefined and tested search strings in English language. Articles were screened on title/abstract and full text level for relevance based on pre-defined inclusion criteria. The relevant information of included studies were mapped using a pre-defined data extraction strategy. Besides a descriptive summary of the relevant literature, a spreadsheet containing all extracted data is provided. Altogether, 555 relevant articles from journals, company web pages and web pages of governmental agencies were identified containing 1328 studies/applications of genome-editing in model plants and agricultural crops in the period January 1996 to May 2018. Most of the studies were conducted in China followed by the USA. Genome-editing was already applied in 68 different plants. Although most of the studies were basic research, 99 different market-oriented applications were identified in 28 different crops leading to plants with improved food and feed quality, agronomic value like growth characteristics or increased yield, tolerance to biotic and abiotic stress, herbicide tolerance or industrial benefits. 252 studies explored off-target effects. Most of the studies were conducted using CRISPR/Cas. Several studies firstly investigated whether sites in the genome show similarity to the target sequence and secondly analyzed these potential off-target sites by sequencing. In around 3% of the analyzed potential off-target sites, unintended mutations were detected. Only a few studies conducted off-target analyses using unbiased detection methods (e.g. whole genome sequencing). No off-target effects that could be correlated to the genome-editing process were identified in these studies. The rapid adoption in plant breeding was demonstrated by a considerable number of market oriented applications (crops and traits) described in publications worldwide. Studies investigating off-target effects are very heterogeneous in their structure and design. Therefore, an in-depth assessment regarding their weight of evidence is mandatory."/>
        <meta property="og:image" content="https://media.springernature.com/w110/springer-static/cover/journal/13750.jpg"/>
    
        
    </head>

    <body class="journal journal-fulltext"
    
          >
        
    
        
            <!-- Google Tag Manager (noscript) -->
            <noscript>
                <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                        height="0" width="0" style="display:none;visibility:hidden"></iframe>
            </noscript>
            <!-- End Google Tag Manager (noscript) -->
        
    

        <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg"><symbol id="icon-access" viewBox="0 0 18 18"><path d="M14 8a1 1 0 011 1v7h2.5a.5.5 0 01.5.5V18H0v-1.5a.5.5 0 01.5-.5H3V9a1 1 0 112 0v7h8V9a1 1 0 011-1zM6 8l2 1v5l-2 1zm6 0v7l-2-1V9zM9.573.401l7.036 4.925A.92.92 0 0116.081 7H1.92a.92.92 0 01-.528-1.674L8.427.401a1 1 0 011.146 0zM9 2.441L5.345 5h7.31z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="M10.238 16.905a7.96 7.96 0 003.53-1.48c-.874-2.514-2.065-3.936-3.768-4.319V9.83a3.001 3.001 0 10-2 0v1.277c-1.703.383-2.894 1.805-3.767 4.319A7.96 7.96 0 009 17c.419 0 .832-.032 1.238-.095zm4.342-2.172a8 8 0 10-11.16 0c.757-2.017 1.84-3.608 3.49-4.322a4 4 0 114.182 0c1.649.714 2.731 2.305 3.488 4.322zM9 18A9 9 0 119 0a9 9 0 010 18z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="M4 10h2.5a.5.5 0 110 1H3.414l-1.121 1.121a1 1 0 00-.293.707V13h14v-.172a1 1 0 00-.293-.707L14 10.414V7A5 5 0 004 7zm3 4a2 2 0 104 0zm-5 0a1 1 0 01-1-1v-.172a2 2 0 01.586-1.414L3 10V7a6 6 0 1112 0v3l1.414 1.414A2 2 0 0117 12.828V13a1 1 0 01-1 1h-4a3 3 0 01-6 0z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="M7.897 12.028v-7.69l-2.45 2.446a.981.981 0 01-1.385.02.986.986 0 010-1.397l4.123-4.118a.99.99 0 011.398 0l4.123 4.118a.976.976 0 01.016 1.38.989.989 0 01-1.401-.002l-2.45-2.447v8.676c0 .541-.437.98-.985.982L.987 14a.987.987 0 110-1.972z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="M3.283 11.53l4.031 4.176a.943.943 0 001.367 0l4.031-4.176c.378-.391.384-1.02.016-1.4a.942.942 0 00-1.37.003l-2.395 2.482V1L8.956.883A.977.977 0 007.998 0c-.495 0-.903.386-.959.883L7.033 1v11.615l-2.396-2.482a.936.936 0 00-1.354-.02 1.027 1.027 0 000 1.417z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="M4.47 3.283L.294 7.314a.943.943 0 000 1.367l4.176 4.031c.391.378 1.02.384 1.4.016a.942.942 0 00-.003-1.37L3.385 8.963H15l.117-.007A.977.977 0 0016 7.998c0-.495-.386-.903-.883-.959L15 7.033H3.385l2.482-2.396a.936.936 0 00.02-1.354 1.027 1.027 0 00-1.417 0z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="M11.53 12.717l4.176-4.031a.943.943 0 000-1.367L11.53 3.288c-.391-.378-1.02-.384-1.4-.016a.942.942 0 00.003 1.37l2.482 2.395H1l-.117.007A.977.977 0 000 8.002c0 .495.386.903.883.959L1 8.967h11.615l-2.482 2.396a.936.936 0 00-.02 1.354 1.027 1.027 0 001.417 0z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="M7.897 4.972v7.69l-2.45-2.446a.981.981 0 00-1.385-.02.986.986 0 000 1.397l4.123 4.118a.99.99 0 001.398 0l4.123-4.118a.976.976 0 00.016-1.38.989.989 0 00-1.401.002l-2.45 2.447V3.986a.982.982 0 00-.985-.982L.987 3a.987.987 0 100 1.972z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="M12.717 4.47L8.686.294a.943.943 0 00-1.367 0L3.288 4.47c-.378.391-.384 1.02-.016 1.4a.942.942 0 001.37-.003l2.395-2.482V15l.007.117a.977.977 0 00.958.883c.495 0 .903-.386.959-.883L8.967 15V3.385l2.396 2.482c.379.393.986.4 1.354.02a1.027 1.027 0 000-1.417z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="M13 15V2.01c0-.008-.002-.01.001-.01H2v13.006c0 .548.446.994.994.994h10.274A1.99 1.99 0 0113 15zm-12 .006V2c0-.555.447-1 .999-1H13c.552 0 .999.452.999 1.01V5h3v9.991A2.003 2.003 0 0115.006 17H2.994C1.894 17 1 16.107 1 15.006zM14 6v9a1 1 0 002 0V6zM4 4h7v4H4zm1 1v2h5V5zM4 9h7v1H4zm0 2h7v1H4zm0 2h7v1H4z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="M13.096 13.559a.503.503 0 01-.707 0 .493.493 0 010-.702A5.414 5.414 0 0013.998 9a5.414 5.414 0 00-1.609-3.856.493.493 0 010-.701.503.503 0 01.708 0A6.401 6.401 0 0115 9a6.401 6.401 0 01-1.904 4.559zm2.123 2.102a.503.503 0 01-.707 0 .493.493 0 010-.7A8.367 8.367 0 0017 9a8.367 8.367 0 00-2.488-5.96c-.195-.193-.195-.507 0-.7s.513-.194.708 0A9.355 9.355 0 0118 9a9.355 9.355 0 01-2.78 6.661zM1 11.976c-.552 0-1-.444-1-.992V7.016a.996.996 0 011-.992h2l5.42-3.84a1.005 1.005 0 011.395.231c.122.169.187.37.187.577v12.016a.996.996 0 01-1 .992c-.209 0-.412-.065-.582-.185L3 11.975zm0-.992h2.321l5.68 4.024V2.992l-5.68 4.024h-2.32z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="M0 0h24v24H0z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="M4 13V2h1v11h11V2H3a1 1 0 00-1 1v10.268A1.99 1.99 0 013 13zm12 1H3a1 1 0 000 2h13zm0 3H3a2 2 0 01-2-2V3a2 2 0 012-2h13a1 1 0 011 1v14a1 1 0 01-1 1zM7.5 4h6a.5.5 0 110 1h-6a.5.5 0 010-1zm1 2h4a.5.5 0 110 1h-4a.5.5 0 010-1z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="M11 16.995v-7.8l-2.482 2.482a.994.994 0 01-1.403.02 1.001 1.001 0 010-1.417l4.177-4.176a1.001 1.001 0 011.416 0l4.177 4.176a.991.991 0 01.016 1.4 1 1 0 01-1.42-.003L13 9.195v8.8c0 .55-.443.995-.998.996L4 18.995a1 1 0 010-2z" fill-rule="evenodd"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="M12.5 0c.276 0 .5.215.5.49V1h2c1.107 0 2 .895 2 2v12c0 1.107-.895 2-2 2H3c-1.107 0-2-.895-2-2V3c0-1.107.895-2 2-2h1v1H3c-.554 0-1 .446-1 1v3h14V3c0-.554-.446-1-1-1h-2v1.51a.5.5 0 01-.5.49.492.492 0 01-.5-.49V.49a.5.5 0 01.5-.49zM16 7H2v8c0 .554.446 1 1 1h12c.554 0 1-.446 1-1zM5 13v1H4v-1zm3 0v1H7v-1zm3 0v1h-1v-1zm-6-2v1H4v-1zm3 0v1H7v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zM8 9v1H7V9zm6 0v1h-1V9zm-3 0v1h-1V9zM5.5 0c.276 0 .5.215.5.49V1h5v1H6v1.51a.5.5 0 01-.5.49.492.492 0 01-.5-.49V.49A.5.5 0 015.5 0z" fill-rule="evenodd"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="M5 3.414l3.293 3.293a1 1 0 001.414-1.414l-4-4a1 1 0 00-1.414 0l-4 4a1 1 0 001.414 1.414z" fill-rule="evenodd"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="M5 6.586l3.293-3.293a1 1 0 011.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 111.414-1.414z" fill-rule="evenodd"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="M4.706 4.033L2.292 1.637A.953.953 0 012.273.283a.98.98 0 011.378 0l4.063 4.031a.96.96 0 010 1.367L3.651 9.712a.97.97 0 01-1.362.016.96.96 0 01.003-1.37l2.414-2.395L5.693 5z" fill-rule="evenodd"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="M8 14A6 6 0 118 2a6 6 0 010 12z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="M8 12a4 4 0 100-8 4 4 0 000 8zm0 2A6 6 0 118 2a6 6 0 010 12z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="M8.636 6a4 4 0 012.174 7.358 6.227 6.227 0 013.462 3.641h-1.077a5.218 5.218 0 00-2.769-2.718l-.79-.33v-1.023l.63-.41a3 3 0 10-3.26 0l.63.41v1.024l-.79.329a5.218 5.218 0 00-2.77 2.718H3a6.227 6.227 0 013.461-3.641A4 4 0 018.636 6zm7.369-5C17.107 1 18 1.894 18 3.003v5.994A1.999 1.999 0 0116.005 11L13 10.999v-1h3.005A.998.998 0 0017 8.997V3.003A.997.997 0 0016.005 2H1.995A.999.999 0 001 3.003V9c0 .667.333 1 1 1l2-.001v1H2c-1 0-2-1-2-2V3.004A2 2 0 011.995 1zM7.5 4a.5.5 0 110 1h-4a.5.5 0 010-1zm3 0a.5.5 0 110 1h-1a.5.5 0 010-1zm4 0a.5.5 0 110 1h-2a.5.5 0 010-1z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="M2.297 12.277a1.007 1.007 0 000 1.427 1.006 1.006 0 001.426 0L8 9.425l4.277 4.277a1.007 1.007 0 001.427 0 1.006 1.006 0 000-1.426L9.425 8l4.277-4.277a1.007 1.007 0 000-1.427 1.006 1.006 0 00-1.426 0L8 6.575 3.723 2.297a1.007 1.007 0 00-1.427 0 1.006 1.006 0 000 1.426L6.575 8z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="M15 4a2 2 0 012 2v9a2 2 0 01-2 2H7a2 2 0 01-2-2h1a1 1 0 00.883.993L7 16h8a1 1 0 00.993-.883L16 15V6a1 1 0 00-.883-.993L15 5h-1V4zm-4-3a2 2 0 012 2v9a2 2 0 01-2 2H3a2 2 0 01-2-2V3a2 2 0 012-2zm0 1H3a1 1 0 00-.993.883L2 3v9a1 1 0 00.883.993L3 13h8a1 1 0 00.993-.883L12 12V3a1 1 0 00-.883-.993zM9.5 9a.5.5 0 010 1h-5a.5.5 0 010-1zm0-2a.5.5 0 010 1h-5a.5.5 0 010-1zm0-2a.5.5 0 010 1h-5a.5.5 0 010-1z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="M12 3a6 6 0 11-3 11.197A6 6 0 119 3.803 5.961 5.961 0 0112 3zM6 4a5 5 0 102.085 9.546C6.808 12.446 6 10.817 6 9s.808-3.446 2.084-4.546A4.983 4.983 0 006 4zm6 0c-.744 0-1.45.162-2.085.454C11.192 5.554 12 7.183 12 9s-.808 3.446-2.084 4.546A5 5 0 1012 4zm-1.416 7H7.416c.156.358.353.693.585 1.001h1.998a4.97 4.97 0 00.585-1zM11 9H7c0 .343.035.678.1 1.001h3.8c.065-.323.1-.658.1-1.001zm-.416-2H7.416A4.962 4.962 0 007.1 8h3.8a4.962 4.962 0 00-.316-1zM9.001 5l-.083.063A5.03 5.03 0 008 6h2a5.03 5.03 0 00-.999-1z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="M10.005 0c.55 0 1.318.323 1.707.712l4.576 4.576c.393.393.712 1.168.712 1.714v8.99A2.003 2.003 0 0115.006 18H2.994A1.994 1.994 0 011 16.005V1.995C1 .893 1.887 0 3 0zm0 1H3c-.557 0-1 .443-1 .995v14.01c0 .55.445.995.994.995h12.012c.546 0 .994-.45.994-1.009V7.002c0-.283-.215-.803-.419-1.007L11.005 1.42c-.204-.204-.719-.419-1-.419zM8.5 5c.276 0 .5.216.5.492v6.148l1.746-1.78a.497.497 0 01.7-.006.498.498 0 01-.002.702l-2.591 2.591a.5.5 0 01-.706 0l-2.59-2.59a.502.502 0 01-.004-.704.49.49 0 01.7.007L8 11.64V5.492C8 5.22 8.232 5 8.5 5z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="M12.998 13a.999.999 0 110 2H3.002A1.006 1.006 0 012 14c0-.557.449-1 1.002-1zM8 1c.552 0 1 .445 1 .996v6.802l2.482-2.482c.392-.393 1.022-.401 1.403-.02a1.002 1.002 0 010 1.417l-4.177 4.178a1.001 1.001 0 01-1.416 0L3.115 7.713a.992.992 0 01-.016-1.4 1 1 0 011.42.002L7 8.798V1.996C7 1.446 7.444 1 8 1z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="M8.726 2.546A3 3 0 105.37 7.519l.63.409v1.024l-.79.329A5.221 5.221 0 002 14.099V15H1v-.901a6.221 6.221 0 013.825-5.741 4 4 0 114.976-6.213 4.965 4.965 0 00-1.075.4zM6 17H5v-.901a6.221 6.221 0 013.825-5.741 4 4 0 114.349 0A6.221 6.221 0 0117 16.099V17h-1v-.901a5.221 5.221 0 00-3.21-4.818l-.79-.33V9.929l.63-.409a3 3 0 10-3.26 0l.63.409v1.024l-.79.329A5.221 5.221 0 006 16.099z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="M16.005 2A2 2 0 0118 4.006v9.988A2 2 0 0116.005 16H1.995A2 2 0 010 13.994V4.006A2 2 0 011.995 2zm0 1H1.995A1 1 0 001 4.006v9.988A1 1 0 001.995 15h14.01A1 1 0 0017 13.994V4.006A1 1 0 0016.005 3zM16 5.557V7l-7 4-7-4V5.557l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="M9 0a9 9 0 110 18A9 9 0 019 0zm2.863 4.711L9 7.574 6.137 4.711a1.007 1.007 0 00-1.427 0 1.006 1.006 0 00.001 1.426L7.574 9l-2.863 2.863a1.007 1.007 0 000 1.427 1.006 1.006 0 001.426-.001L9 10.426l2.863 2.863a1.007 1.007 0 001.427 0 1.006 1.006 0 00-.001-1.426L10.426 9l2.863-2.863a1.007 1.007 0 000-1.427 1.006 1.006 0 00-1.426.001z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="M6.764 1.414l.833-.833a1.984 1.984 0 012.806 0l.833.833A2 2 0 0012.65 2H14a2 2 0 012 2v1.35a2 2 0 00.586 1.414l.833.833a1.984 1.984 0 010 2.806l-.833.833A2 2 0 0016 12.65V14a2 2 0 01-2 2h-1.35a2 2 0 00-1.414.586l-.833.833a1.984 1.984 0 01-2.806 0l-.833-.833A2 2 0 005.35 16H4a2 2 0 01-2-2v-1.35a2 2 0 00-.586-1.414l-.833-.833a1.984 1.984 0 010-2.806l.833-.833A2 2 0 002 5.35V4a2 2 0 012-2h1.35a2 2 0 001.414-.586zM5.35 3H4a1 1 0 00-1 1v1.35a3 3 0 01-.879 2.121l-.833.833a.984.984 0 000 1.392l.833.833A3 3 0 013 12.65V14a1 1 0 001 1h1.35a3 3 0 012.121.879l.833.833a.984.984 0 001.392 0l.833-.833A3 3 0 0112.65 15H14a1 1 0 001-1v-1.35a3 3 0 01.879-2.121l.833-.833a.984.984 0 000-1.392l-.833-.833A3 3 0 0115 5.35V4a1 1 0 00-1-1h-1.35a3 3 0 01-2.121-.879l-.833-.833a.984.984 0 00-1.392 0l-.833.833A3 3 0 015.35 3zm3.587 11.496a.505.505 0 01-.064.003 5.474 5.474 0 01-2.759-.816.5.5 0 01.526-.85 4.474 4.474 0 002.256.666c.017 0 .035.002.052.004A.506.506 0 019 13.5a4.5 4.5 0 003.493-7.338.5.5 0 11.775-.631A5.5 5.5 0 019 14.5a.505.505 0 01-.063-.004zm1.58-10.784a.5.5 0 01-.276.961 4.485 4.485 0 00-1.156-.172C6.527 4.5 4.5 6.503 4.5 9c0 .627.128 1.235.373 1.796a.5.5 0 11-.917.4A5.483 5.483 0 013.5 9c0-3.052 2.477-5.5 5.594-5.5.486.009.963.08 1.422.212zm-1.977 6.41l2.698-2.946a.5.5 0 01.762.648l-3.016 3.343a.5.5 0 01-.693.047L6.36 9.592A.5.5 0 117 8.824z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="M9 17A8 8 0 109 1a8 8 0 000 16zm0 1A9 9 0 119 0a9 9 0 010 18zm0-2.5a.5.5 0 110-1 5.5 5.5 0 005.497-5.317.5.5 0 111 .032A6.5 6.5 0 019 15.5zM8.71 2.506a.5.5 0 11.043 1A5.5 5.5 0 003.5 9a.5.5 0 01-1 0 6.5 6.5 0 016.21-6.494zm1.137 8.048l-1.2-1.2a.5.5 0 11.707-.708l1.2 1.2 1.6-4-4.506 1.802-1.803 4.507zM13.95 4.05l-2.67 6.673a1 1 0 01-.557.557L4.05 13.95l2.67-6.673a1 1 0 01.557-.557z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="M14.974 0C15.54 0 16 .448 16 1c0 .242-.09.476-.254.658L10.052 8v5.5a.992.992 0 01-.41.8L7.59 15.8a1.044 1.044 0 01-1.437-.2.983.983 0 01-.205-.6V8L.254 1.658A.982.982 0 01.35.248C.537.087.778 0 1.026 0zM9.052 8.5H6.947v6.491l.012.007.02.002.02-.007 2.054-1.5zM1.026 1l-.015.002L6.843 7.5h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="M9 5l-6 6v5h4v-4h4v4h4v-5zm7 6.586V16a1 1 0 01-1 1h-5v-4H8v4H3a1 1 0 01-1-1v-4.414a.997.997 0 01-.707-.293l-.586-.586a1 1 0 010-1.414L9 1l8.293 8.293a1 1 0 010 1.414l-.586.586a.997.997 0 01-.707.293zM9 2.414L1.414 10l.586.586 7-7 7 7 .586-.586z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="M10.005 0c.55 0 1.318.323 1.707.712l4.576 4.576c.393.393.712 1.168.712 1.714v8.99A2.003 2.003 0 0115.006 18H2.994A1.994 1.994 0 011 16.005V1.995C1 .893 1.887 0 3 0zM6.508 10.175l-3.894 6.75a.991.991 0 00.38.075h10.603l-2.433-4.215-1.575 2.728zM15.006 17c.546 0 .994-.45.994-1.009V7.002c0-.283-.215-.803-.419-1.007L11.005 1.42c-.204-.204-.719-.419-1-.419H3c-.557 0-1 .443-1 .995v13.99l4.508-7.81 3.081 5.338 1.575-2.729L14.752 17zM12 6a2 2 0 110 4 2 2 0 010-4zm0 1a1 1 0 100 2 1 1 0 000-2z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="M9 0a9 9 0 110 18A9 9 0 019 0zm0 7H7.5l-.117.007a1 1 0 00-.112 1.967l.112.02L7.5 9H8v3h-.5l-.117.007a1 1 0 00-.857.764l-.02.112L6.5 13l.007.117a1 1 0 00.764.857l.112.02L7.5 14h3l.117-.007a1 1 0 00.112-1.967l-.112-.02L10.5 12H10V8l-.007-.117a1 1 0 00-.764-.857l-.112-.02zm0-3.25a1.25 1.25 0 100 2.5 1.25 1.25 0 000-2.5z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="M7 17v-2h4v2h2v-3H5v3zM4 6.998H2.474a.5.5 0 01-.339-.868l1.232-1.132H1V17h3v-2H1v-1h3v-1h1V8.997H4zm10 0v2h-1V13h1v1h3v1h-3v2h3V4.999h-2.37l1.232 1.132a.5.5 0 01-.338.868zm3-3a1 1 0 011 1V18H0V4.998a1 1 0 011-1h3.455L8.66.131a.5.5 0 01.677 0l4.205 3.865zm-4 3H5v1h8zm-2 6.001h1V9h-1zm-1 0V9H8v4zm-3 0V9H6v4zm8-4a1 1 0 011 1v2h-2v-2a1 1 0 011-1zm-12 0a1 1 0 011 1v2H2v-2a1 1 0 011-1zm5.999-7.82L3.757 5.998H14.24zM9 4.998a1 1 0 110-2 1 1 0 010 2zM8 16v1h2v-1z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="M9.395 16.269a31.947 31.947 0 002.333-2.53c1.402-1.702 2.432-3.362 2.936-4.872C14.886 8.2 15 7.574 15 7a6 6 0 00-6-6C5.636 1 3 3.602 3 7c0 .574.114 1.199.336 1.867.504 1.51 1.534 3.17 2.936 4.872A31.947 31.947 0 009 16.65l.395-.38zM9 18S2 12 2 7c0-4 3.134-7 7-7a7 7 0 017 7c0 5-7 11-7 11zm0-8a3 3 0 110-6 3 3 0 010 6zm0-1a2 2 0 100-4 2 2 0 000 4z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="M9 11.848l2-1.143V9H7v1.705zm-3-1.714V8h6v2.134l3-1.714V2H3v6.42zM16 4.75l1.53.956a1 1 0 01.47.848V15a2 2 0 01-2 2H2a2 2 0 01-2-2V6.554a1 1 0 01.47-.848L2 4.75V2a1 1 0 011-1h12a1 1 0 011 1zm0 1.18V9l-7 4-7-4V5.93l-1 .624V15a1 1 0 001 1h14a1 1 0 001-1V6.554zM6 4h6v1H6zM5 6h8v1H5z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="M9 1a8 8 0 110 16A8 8 0 019 1zM6.099 6.273H4.68v7.171H6.1zm4.559.025H7.665v7.146h2.91c.768 0 1.448-.148 2.042-.445s1.051-.715 1.373-1.255c.323-.54.484-1.16.484-1.86 0-.685-.16-1.303-.481-1.852a3.301 3.301 0 00-1.35-1.278c-.58-.304-1.242-.456-1.985-.456zm-.19 1.08c.788 0 1.415.223 1.88.67.466.447.698 1.055.698 1.826 0 .743-.23 1.344-.693 1.803-.462.459-1.075.688-1.839.688H9.046V7.378zM5.382 3.667a.786.786 0 00-.577.253.805.805 0 00-.25.58c0 .224.084.42.25.59a.818.818 0 001.177.004.81.81 0 00.25-.593c0-.234-.084-.432-.25-.593s-.366-.241-.6-.241z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="M1 7h6V1c0-.552.444-1 1-1 .552 0 1 .445 1 1v6h6c.552 0 1 .444 1 1 0 .552-.445 1-1 1H9v6c0 .552-.444 1-1 1-.552 0-1-.445-1-1V9H1c-.552 0-1-.444-1-1 0-.552.445-1 1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="M16.005 5H1.995A.997.997 0 001 6v6c0 .557.443 1 .995 1H3v-3h12v3h1.005c.55 0 .995-.447.995-1V6c0-.557-.443-1-.995-1zM14 4V1.998C14 1.45 13.548 1 12.991 1H5.01C4.452 1 4 1.45 4 1.998V4zm1 10v2.002A2.005 2.005 0 0112.991 18H5.01A1.998 1.998 0 013 16.002V14H1.995A1.992 1.992 0 010 12V6c0-1.105.893-2 1.995-2H3V1.998C3 .894 3.902 0 5.009 0h7.982C14.101 0 15 .898 15 1.998V4h1.005C17.107 4 18 4.887 18 6v6c0 1.105-.893 2-1.995 2zm-1-3H4v5.002c0 .555.447.998 1.009.998h7.982c.557 0 1.009-.45 1.009-.998zm-9 1h8v1H5zm0 2h5v1H5zm9-5a1 1 0 110-2 1 1 0 010 2z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 18 18"><path d="M16.78 15.736a.748.748 0 01.007 1.053.752.752 0 01-1.053-.007l-3.103-3.104A7.136 7.136 0 011 8.136a7.136 7.136 0 1112.677 4.496zm-8.644-1.483A6.116 6.116 0 108.135 2.02a6.116 6.116 0 000 12.233z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="M6.004 20A2 2 0 014 18.006V5.994C4 4.893 4.895 4 5.994 4h12.012C19.107 4 20 4.895 20 5.994v12.012A1.992 1.992 0 0117.997 20H15.04v-6.196h2.08l.31-2.415h-2.39V9.848c0-.7.194-1.176 1.196-1.176h1.28v-2.16l-.235-.025c-.344-.032-.95-.07-1.63-.07-1.843 0-3.105 1.125-3.105 3.191v1.781h-2.085v2.415h2.085V20z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="M18.877 6.874a3.772 3.772 0 001.625-2.096 7.237 7.237 0 01-2.347.917A3.64 3.64 0 0015.463 4.5c-2.04 0-3.693 1.696-3.693 3.786 0 .297.031.587.094.862C8.8 8.99 6.076 7.483 4.254 5.191c-.317.56-.5 1.211-.5 1.905 0 1.314.653 2.472 1.643 3.152a3.647 3.647 0 01-1.672-.473v.045c0 1.834 1.273 3.366 2.964 3.713a3.543 3.543 0 01-.975.133c-.236 0-.471-.023-.694-.067.471 1.507 1.832 2.604 3.45 2.633a7.297 7.297 0 01-4.59 1.62c-.298 0-.59-.016-.88-.051A10.279 10.279 0 008.661 19.5c6.793 0 10.506-5.771 10.506-10.775l-.009-.49A7.527 7.527 0 0021 6.275a7.267 7.267 0 01-2.122.596z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="M10.142 14.397l-.001-5.193 4.863 2.606zm9.963-7.927c-.685-.737-1.452-.74-1.804-.783-2.519-.187-6.297-.187-6.297-.187-.008 0-3.786 0-6.305.187-.352.043-1.119.046-1.804.783-.54.56-.715 1.834-.715 1.834S3 9.8 3 11.296v1.402c0 1.496.18 2.991.18 2.991s.176 1.274.715 1.835c.685.736 1.585.713 1.985.79 1.44.142 6.12.186 6.12.186s3.782-.006 6.301-.193c.352-.043 1.119-.047 1.804-.783.539-.561.715-1.835.715-1.835s.18-1.495.18-2.991v-1.402c0-1.496-.18-2.992-.18-2.992s-.176-1.273-.716-1.834z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="M12.5 8H6a3 3 0 00-3 3v1a3 3 0 003 3h1v-2h-.5a1.5 1.5 0 010-3H13a3 3 0 003-3V6a3 3 0 00-3-3h-2v2h1.5a1.5 1.5 0 010 3zM7 7V6H3.5a2.5 2.5 0 010-5h1.028a3 3 0 011.342.317l1.133.567A2 2 0 0111 2h2a4 4 0 014 4v1a4 4 0 01-4 4h-2v1h2a2 2 0 110 4h-2a2 2 0 11-4 0H6a4 4 0 01-4-4v-1a4 4 0 014-4zm0-2V2.943a2 2 0 01-.422-.154L5.422 2.21A2 2 0 004.528 2H3.5a1.5 1.5 0 000 3zm4 1v1h1.5a.5.5 0 100-1zm-1 1V2a1 1 0 10-2 0v5zm-2 4v5a1 1 0 002 0v-5zm3 2v2h2a1 1 0 000-2zm-4-1v-1h-.5a.5.5 0 100 1zM3.5 3h1a.5.5 0 010 1h-1a.5.5 0 010-1z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="M9 0a9 9 0 110 18A9 9 0 019 0zm3.486 4.982l-4.718 5.506L5.14 8.465a.991.991 0 00-1.423.133 1.06 1.06 0 00.13 1.463l3.407 2.733a1 1 0 001.387-.133l5.385-6.334a1.06 1.06 0 00-.116-1.464.991.991 0 00-1.424.119z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="M16.005 2A2 2 0 0118 4.006v9.988A2 2 0 0116.005 16l-4.006-.001L12 16h-1l-.001-.001h-5L6 16H5l-.001-.001L1.995 16A2 2 0 010 13.994V4.006A2 2 0 011.995 2zM4.999 7H1v6.994A1 1 0 001.995 15H5zm6 0h-5v8h5zm5.006-4h-4.006v3H17v1h-5.001v7.999l4.006.001A1 1 0 0017 13.994V4.006A1 1 0 0016.005 3zM3.5 12a.5.5 0 110 1h-1a.5.5 0 110-1zm12 0a.5.5 0 110 1h-2a.5.5 0 110-1zm-6 0a.5.5 0 110 1h-2a.5.5 0 110-1zm-6-2a.5.5 0 110 1h-1a.5.5 0 110-1zm12 0a.5.5 0 110 1h-2a.5.5 0 110-1zm-6 0a.5.5 0 110 1h-2a.5.5 0 110-1zm-6-2a.5.5 0 010 1h-1a.5.5 0 010-1zm12 0a.5.5 0 110 1h-2a.5.5 0 110-1zm-6 0a.5.5 0 010 1h-2a.5.5 0 010-1zm1.499-5h-5v3h5zm-6 0H1.995A1 1 0 001 4.006V6h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="M12 2c5.523 0 10 4.477 10 10s-4.477 10-10 10S2 17.523 2 12 6.477 2 12 2zm0 1a9 9 0 100 18 9 9 0 000-18zm4.22 5.366c.361-.437.999-.49 1.424-.119s.477 1.028.115 1.465l-6.093 6.944a1 1 0 01-1.405.097l-3.897-3.367c-.43-.368-.487-1.023-.13-1.464s.994-.5 1.423-.133l3.111 2.7z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="M6.768 9.211L3.657 6.628c-.429-.352-1.066-.295-1.423.127s-.3 1.049.13 1.4l3.915 3.24a1 1 0 001.375-.096l6.105-6.66a.984.984 0 00-.115-1.402 1.021 1.021 0 00-1.424.113z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="M1 13v1a1 1 0 001 1h14a1 1 0 001-1v-1h-1V3H2v10zm16-1h1v2a2 2 0 01-2 2H2a2 2 0 01-2-2v-2h1V3a1 1 0 011-1h14a1 1 0 011 1zm-1 0v1h-4.586l-1 1H7.586l-1-1H2v-1h5l1 1h2l1-1zM3 4h12v7H3zm1 1v5h10V5zm1 1h4v1H5zm0 2h4v1H5z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="M10.005 0c.55 0 1.318.323 1.707.712l4.576 4.576c.393.393.712 1.168.712 1.714v8.99A2.003 2.003 0 0115.006 18H2.994A1.994 1.994 0 011 16.005V1.995C1 .893 1.887 0 3 0zm0 1H3c-.557 0-1 .443-1 .995v14.01c0 .55.445.995.994.995h12.012c.546 0 .994-.45.994-1.009V7.002c0-.283-.215-.803-.419-1.007L11.005 1.42c-.204-.204-.719-.419-1-.419zM8.149 5.146a.5.5 0 01.705 0l2.591 2.59c.195.196.19.516.003.704a.49.49 0 01-.7-.007l-1.746-1.78V12.8a.501.501 0 01-1 0V6.653l-1.747 1.78a.497.497 0 01-.7.006.498.498 0 01.003-.702z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="M16.005 2A2 2 0 0118 4.006v9.988A2 2 0 0116.005 16H1.995A2 2 0 010 13.994V4.006A2 2 0 011.995 2zm0 1H1.995A1 1 0 001 4.006v9.988A1 1 0 001.995 15h14.01A1 1 0 0017 13.994V4.006A1 1 0 0016.005 3zm-8.31 2.25L12.3 7.99c.937.557.93 1.464 0 2.017l-4.604 2.74c-.937.557-1.696.165-1.696-.894v-5.71c0-1.05.767-1.446 1.696-.894zm-.674.96v5.575c0 .3-.108.245.166.085l4.584-2.675c.305-.178.305-.216 0-.394L7.187 6.126c-.267-.156-.166-.207-.166.084z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="M9 11.75a1.25 1.25 0 110 2.5 1.25 1.25 0 010-2.5zM9.413 4a1 1 0 011 1l-.003.083-.334 4A1 1 0 019.08 10h-.16a1 1 0 01-.996-.917l-.334-4a1 1 0 01.914-1.08l.041-.002zM9 18A9 9 0 119 0a9 9 0 010 18z" fill-rule="evenodd"/></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 00-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"/></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 01-1.554.429 2.718 2.718 0 001.19-1.502 5.456 5.456 0 01-1.72.657 2.71 2.71 0 00-1.979-.854 2.711 2.711 0 00-2.642 3.326 7.681 7.681 0 01-5.586-2.831 2.714 2.714 0 00.839 3.618 2.748 2.748 0 01-1.227-.339v.031a2.71 2.71 0 002.174 2.656 2.735 2.735 0 01-1.229.049 2.726 2.726 0 002.531 1.883 5.442 5.442 0 01-4.01 1.123 7.672 7.672 0 004.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 001.354-1.399z"/><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"/></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"/><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162m2.855-4.797a4.176 4.176 0 00-3.982-1.291.608.608 0 00-.465.72.604.604 0 00.72.466 2.968 2.968 0 012.827.92 3 3 0 01.625 2.918.602.602 0 00.39.762.603.603 0 00.763-.391v-.001a4.218 4.218 0 00-.878-4.103"/><path d="M485.041 276.276a2.037 2.037 0 00-1.938-.63.518.518 0 00-.396.621.517.517 0 00.617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 00.336.654.523.523 0 00.657-.336 2.038 2.038 0 00-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"/></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="M8 8.586l3.293-3.293a1 1 0 011.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414z" fill-rule="evenodd"/></symbol><symbol id="icon-download-rounded"><path d="M0 13c0-.556.449-1 1.002-1h9.996a.999.999 0 110 2H1.002A1.006 1.006 0 010 13zM7 1v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 010 1.417l-4.177 4.177a1.001 1.001 0 01-1.416 0L1.115 6.715a.991.991 0 01-.016-1.4 1 1 0 011.42.003L5 7.8V1c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"/></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"/><path d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"/><path d="M9.7 0L16 6.2V0z"/></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"/><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"/></symbol><symbol id="icon-rss" viewBox="0 0 18 18"><path d="M.975 6.016l.117.004c5.759.52 10.342 5.105 10.86 10.864a1 1 0 01-1.993.18A10.003 10.003 0 00.912 8.011a1 1 0 01.063-1.996zM2.5 12.97a2.5 2.5 0 110 5 2.5 2.5 0 010-5zM1.067 0c9.096.533 16.376 7.817 16.903 16.914a1 1 0 11-1.997.115C15.505 8.947 9.033 2.471.95 1.998A1 1 0 111.067.002z" fill-rule="evenodd"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-springer-collections" viewBox="3 3 32 32"><path fill-rule="evenodd" d="M25.583 30.125V7.121c0-.013-.003-.017.003-.017l-19.482.002v23.03c0 .97.79 1.76 1.76 1.76h18.193a3.525 3.525 0 01-.474-1.771zm-21.25.01V7.106c0-.984.792-1.772 1.769-1.772h19.484c.976 0 1.768.8 1.768 1.788v5.296h5.313V30.11c0 1.96-1.581 3.557-3.531 3.557H7.864a3.533 3.533 0 01-3.53-3.531zm23.021-15.948v15.938a1.77 1.77 0 103.542 0V14.187h-3.542zM9.646 10.646h12.396v7.083H9.646v-7.083zm1.77 1.77v3.542h8.855v-3.541h-8.854zM9.647 19.5h12.396v1.77H9.646V19.5zm0 3.542h12.396v1.77H9.646v-1.77zm0 3.541h12.396v1.771H9.646v-1.77z"/></symbol><symbol id="icon-springer-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"/></symbol><symbol id="icon-springer-info" viewBox="0 0 24 24"><path d="M12 0c6.627 0 12 5.373 12 12s-5.373 12-12 12S0 18.627 0 12 5.373 0 12 0zm.554 9.1h-1a1 1 0 00-1 1v8a1 1 0 001 1h1a1 1 0 001-1v-8a1 1 0 00-1-1zM12 5c-.464 0-.847.146-1.148.44-.301.292-.452.675-.452 1.15 0 .474.15.86.452 1.16.3.3.684.45 1.148.45.477 0 .863-.15 1.158-.45.295-.3.442-.686.442-1.16 0-.475-.147-.858-.442-1.15C12.863 5.145 12.477 5 12 5z"/></symbol><symbol id="icon-springer-tick-circle" viewBox="0 0 24 24"><path d="M12 24C5.373 24 0 18.627 0 12S5.373 0 12 0s12 5.373 12 12-5.373 12-12 12zM7.657 10.79a.991.991 0 00-1.423.133 1.06 1.06 0 00.13 1.464l3.897 3.367a1 1 0 001.405-.097l6.093-6.944a1.06 1.06 0 00-.115-1.465.991.991 0 00-1.424.12l-5.452 6.121-3.111-2.699z"/></symbol><symbol id="icon-updates" viewBox="0 0 18 18"><path d="M16.98 3.484h-.48c-2.52-.058-5.04 1.161-7.44 2.903-2.46-1.8-4.74-2.903-8.04-2.903-.3 0-.54.29-.54.58v9.813c0 .29.24.523.54.581 2.76.348 4.86 1.045 7.62 2.903.24.116.54.116.72 0 2.76-1.858 4.86-2.555 7.62-2.903.3-.058.54-.29.54-.58V4.064c0-.29-.24-.523-.54-.581zm-15.3 1.22c2.34 0 4.86 1.509 6.72 2.786v8.478c-2.34-1.394-4.38-2.09-6.72-2.439V4.703zm14.58 8.767c-2.34.348-4.38 1.045-6.72 2.439V7.374C12 5.632 14.1 4.645 16.26 4.645v8.826z"/><path d="M9 .058c-1.56 0-2.76 1.22-2.76 2.671C6.24 4.181 7.5 5.4 9 5.4c1.5 0 2.76-1.22 2.76-2.671 0-1.452-1.2-2.67-2.76-2.67zm0 4.413c-.96 0-1.8-.755-1.8-1.742C7.2 1.8 7.98.987 9 .987s1.8.755 1.8 1.742c0 .93-.84 1.742-1.8 1.742z"/></symbol><symbol id="icon-submit-closed" viewBox="0 0 18 18"><path d="M15 0a2 2 0 012 2v4.5a.5.5 0 11-1 0V2a1 1 0 00-.883-.993L15 1H6v3a2 2 0 01-2 2H1v10a1 1 0 00.883.993L2 17h4.5a.5.5 0 110 1H2a2 2 0 01-2-2V5.828a2 2 0 01.586-1.414L4.414.586A2 2 0 015.828 0zm-2.5 7a5.5 5.5 0 110 11 5.5 5.5 0 010-11zm0 1a4.5 4.5 0 100 9 4.5 4.5 0 000-9zm2.309 2.191a.653.653 0 010 .924l-1.386 1.384 1.386 1.386a.653.653 0 01-.924.924L12.5 13.423l-1.384 1.386a.653.653 0 01-.924-.924l1.385-1.386-1.385-1.384a.653.653 0 01.924-.924l1.384 1.385 1.386-1.385a.653.653 0 01.924 0zM5 1.413L1.413 5H4a1 1 0 001-1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 18 18"><path d="M15 0a2 2 0 012 2v5.5a.5.5 0 11-1 0V2a1 1 0 00-.883-.993L15 1H6v3a2 2 0 01-2 2H1v10a1 1 0 00.883.993L2 17h7.5a.5.5 0 110 1H2a2 2 0 01-2-2V5.828a2 2 0 01.586-1.414L4.414.586A2 2 0 015.828 0zm-.544 8.189l3.354 3.354c.251.251.254.66.004.91a.643.643 0 01-.907-.009l-2.26-2.304v7.223A.64.64 0 0114 18a.65.65 0 01-.647-.637l-.001-7.223-2.262 2.304a.634.634 0 01-.906.01.65.65 0 01.004-.91l3.354-3.355a.647.647 0 01.914 0zM13.5 5a.5.5 0 110 1h-6a.5.5 0 010-1zM5 1.413L1.413 5H4a1 1 0 001-1zM13.5 3a.5.5 0 110 1h-6a.5.5 0 010-1z"/></symbol><symbol id="icon-submit-upcoming" viewBox="0 0 18 18"><path d="M15 0a2 2 0 012 2v4.5a.5.5 0 11-1 0V2a1 1 0 00-.883-.993L15 1H6v3a2 2 0 01-2 2H1v10a1 1 0 00.883.993L2 17h4.5a.5.5 0 110 1H2a2 2 0 01-2-2V5.828a2 2 0 01.586-1.414L4.414.586A2 2 0 015.828 0zm-2.5 7a5.5 5.5 0 11-3.6 1.342l-1.516.022a.497.497 0 01-.5-.49c0-.243.178-.445.41-.487l.09-.008h2.663a.5.5 0 01.491.41l.008.089v2.664c0 .276-.23.499-.495.499a.488.488 0 01-.483-.41l-.007-.09.02-1.466A4.5 4.5 0 1012.5 8c-.77 0-.77-1-.001-1zm0 2.85a.65.65 0 01.643.554l.007.096v1.731l.81.81a.65.65 0 01.068.838l-.068.08a.65.65 0 01-.84.07l-.08-.07-.97-.97a.75.75 0 01-.21-.412l-.01-.118V10.5a.65.65 0 01.65-.65zM5 1.413L1.413 5H4a1 1 0 001-1z"/></symbol></svg>
</div>


        

        

    <div class="u-vh-full">
        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner">Skip to main content</span>
</a>

        
            
    <aside class="adsbox c-ad c-ad--728x90">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1"
                 data-gpt-unitpath="/270604982/bmc/environmentalevidencejournal/articles"
                 data-gpt-sizes="728x90,970x90"
                 data-gpt-targeting="pos=LB1;doi=10.1186/s13750-019-0171-5;kwrd=New plant breeding techniques,Gene editing,Targeted genome modification,Mutagenesis,Unintended effects,Off-target mutation,Evidence map,Evidence synthesis;pmc=U17009,L1904X,U16002,X33040,L22016,U17009;"
                 data-ad-type="LB1">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/environmentalevidencejournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s13750-019-0171-5&amp;kwrd=New plant breeding techniques,Gene editing,Targeted genome modification,Mutagenesis,Unintended effects,Off-target mutation,Evidence map,Evidence synthesis&amp;pmc=U17009,L1904X,U16002,X33040,L22016,U17009&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/environmentalevidencejournal/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/s13750-019-0171-5&amp;kwrd=New plant breeding techniques,Gene editing,Targeted genome modification,Mutagenesis,Unintended effects,Off-target mutation,Evidence map,Evidence synthesis&amp;pmc=U17009,L1904X,U16002,X33040,L22016,U17009&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            </div>
        </div>
    </aside>

        
        

         
    
        <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>
    
    
        <div id="top" class="u-position-relative">
    <header class="c-header" data-test="publisher-header">
        <div class="c-header__container">
            <div class="c-header__brand u-mr-48" itemscope itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <div class="c-logo">
    <a href="https://www.biomedcentral.com" itemprop="url">
        <img alt="BMC" itemprop="logo" width="76" height="18" role="img" src=/static/images/bmc/logos/logo-bmc-white-aj-5161b51dc0.svg>
        <div class="c-logo__strapline">
            <img alt="Part of Springer Nature" width="173" height="16" role="img" src=/static/images/bmc/logos/logo-bmc-white-strapline-sn-7ea0ab832c.svg>
        </div>
    </a>
</div>

            </div>
            <div class="c-header__navigation">
                <button
                    type="button"
                    class="c-header__link u-button-reset u-mr-24"
                    data-expander
                    data-expander-target="#publisher-header-search"
                    data-expander-autofocus="true"
                    data-test="header-search-button"
                    aria-controls="publisher-header-search"
                    aria-expanded="false">
                    <span class="u-display-flex u-align-items-center">
                        Search
                        <svg class="c-icon u-ml-8" width="14" height="14" aria-hidden="true" focusable="false">
                            <use xlink:href="#icon-search"></use>
                        </svg>
                    </span>
                </button>
                <nav>
                    <ul class="c-header__menu" data-enhanced-menu data-test="publisher-navigation">
                        
                            
                                <li class="c-header__item u-hide-at-lt-lg">
                                    <a class="c-header__link"
                                       href="//www.biomedcentral.com/journals">
                                        Explore journals
                                    </a>
                                </li>
                            
                                <li class="c-header__item u-hide-at-lt-lg">
                                    <a class="c-header__link"
                                       href="//www.biomedcentral.com/getpublished">
                                        Get published
                                    </a>
                                </li>
                            
                                <li class="c-header__item u-hide-at-lt-lg">
                                    <a class="c-header__link"
                                       href="//www.biomedcentral.com/about">
                                        About BMC
                                    </a>
                                </li>
                            
                        
                        <li class="c-header__item">
                            <a data-enhanced-account
                               class="c-header__link"
                               href="https://www.biomedcentral.com/account"
                               data-test="login-link">
                                My account
                            </a>
                        </li>
                    </ul>
                </nav>
            </div>
        </div>
    </header>
    <div class="c-popup-search u-js-hide" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit"
        data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label data-track-label="" data-test="global-search">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="u-display-flex">
            <input id="publisherSearch" class="c-form-field__input" data-search-input autocomplete="off" role="textbox" data-test="search-input" name="query" type="text" value=""/>
            <div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true" focusable="false">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


            </div>
        </div>
        <input type="hidden" name="searchType" value="publisherSearch"/>
    </form>
</div>

            </div>
        </div>
    </div>
</div>

    

        
            <header class="c-journal-header c-journal-header--environmental-evidence ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="/">
    
        
            <img class="c-journal-title__logo" src=https://environmentalevidencejournal.biomedcentral.com/images/13750/logo.svg alt="Environmental Evidence logo">
        
    
    <span class="c-journal-title__text ">Environmental Evidence</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="c-navbar__container">
                        
                            <div class="c-navbar__content">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" data-track="click" data-track-category="Home" data-track-action="Clicked journal navigation link" href='/'>Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" data-track="click" data-track-category="About" data-track-action="Clicked journal navigation link" href='/about'>About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" data-track="click" data-track-category="Articles" data-track-action="Clicked journal navigation link" href='/articles'>Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" data-track="click" data-track-category="Submission Guidelines" data-track-action="Clicked journal navigation link" href='/submission-guidelines'>Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--midnight-light-grey">
                    
                </div>
            </header>
            
        

        <div class="u-container u-mt-32 u-mb-32 c-article-container u-clearfix" id="main-content" data-component="article-container">

            <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
                
    
    <div class="c-pdf-download u-clear-both">
        <a href="//environmentalevidencejournal.biomedcentral.com/track/pdf/10.1186/s13750-019-0171-5" class="c-pdf-download__link" data-article-pdf="true"  data-readcube-pdf-url="true" data-test="pdf-link" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link">
            <span>Download PDF</span>
            <svg width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
        </a>
    </div>
    

                <article itemscope itemtype="http://schema.org/ScholarlyArticle" lang="en">
                    <div class="c-article-header">
                        

                        <ul class="c-article-identifiers" data-test="article-identifier">
                            
    
        <li class="c-article-identifiers__item" data-test="article-category">Systematic Map</li>
    
    
        <li class="c-article-identifiers__item">
            <span class="c-article-identifiers__open" data-test="open-access">Open Access</span>
        </li>
    
    

                            <li class="c-article-identifiers__item"><a href="#article-info" data-track="click" data-track-action="publication date" data-track-label="link">Published: <time datetime="2019-07-12" itemprop="datePublished">12 July 2019</time></a></li>
                        </ul>

                        <h1 class="c-article-title" data-test="article-title" data-article-title="" itemprop="name headline">What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map</h1>
                        <ul class="c-author-list js-etal-collapsed" data-etal="25" data-etal-small="3" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-1" data-corresp-id="c1">Dominik Modrzejewski<svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-email"></use></svg></a></span><span class="u-js-hide">
            <a class="js-orcid" itemprop="url" href="http://orcid.org/0000-0001-6545-7536"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6545-7536</a></span><sup class="u-js-hide"><a href="#Aff1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Julius Khn-Institute, Federal Research Centre for Cultivated Plants" /><meta itemprop="address" content="0000 0001 1089 3517, grid.13946.39, Institute for Biosafety in Plant Biotechnology, Julius Khn-Institute, Federal Research Centre for Cultivated Plants, Erwin-Baur-Strae 27, 06484, Quedlinburg, Germany" /></span></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-2">Frank Hartung</a></span><sup class="u-js-hide"><a href="#Aff1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Julius Khn-Institute, Federal Research Centre for Cultivated Plants" /><meta itemprop="address" content="0000 0001 1089 3517, grid.13946.39, Institute for Biosafety in Plant Biotechnology, Julius Khn-Institute, Federal Research Centre for Cultivated Plants, Erwin-Baur-Strae 27, 06484, Quedlinburg, Germany" /></span></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-3">Thorben Sprink</a></span><sup class="u-js-hide"><a href="#Aff1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Julius Khn-Institute, Federal Research Centre for Cultivated Plants" /><meta itemprop="address" content="0000 0001 1089 3517, grid.13946.39, Institute for Biosafety in Plant Biotechnology, Julius Khn-Institute, Federal Research Centre for Cultivated Plants, Erwin-Baur-Strae 27, 06484, Quedlinburg, Germany" /></span></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-4">Drthe Krause</a></span><sup class="u-js-hide"><a href="#Aff1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Julius Khn-Institute, Federal Research Centre for Cultivated Plants" /><meta itemprop="address" content="0000 0001 1089 3517, grid.13946.39, Institute for Biosafety in Plant Biotechnology, Julius Khn-Institute, Federal Research Centre for Cultivated Plants, Erwin-Baur-Strae 27, 06484, Quedlinburg, Germany" /></span></sup>, </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-5">Christian Kohl</a></span><sup class="u-js-hide"><a href="#Aff1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Julius Khn-Institute, Federal Research Centre for Cultivated Plants" /><meta itemprop="address" content="0000 0001 1089 3517, grid.13946.39, Institute for Biosafety in Plant Biotechnology, Julius Khn-Institute, Federal Research Centre for Cultivated Plants, Erwin-Baur-Strae 27, 06484, Quedlinburg, Germany" /></span></sup> &amp; </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-6">Ralf Wilhelm</a></span><sup class="u-js-hide"><a href="#Aff1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Julius Khn-Institute, Federal Research Centre for Cultivated Plants" /><meta itemprop="address" content="0000 0001 1089 3517, grid.13946.39, Institute for Biosafety in Plant Biotechnology, Julius Khn-Institute, Federal Research Centre for Cultivated Plants, Erwin-Baur-Strae 27, 06484, Quedlinburg, Germany" /></span></sup></li></ul>
                        <p class="c-article-info-details" data-container-section="info">
                            
    <a data-test="journal-link" href="/"><i data-test="journal-title">Environmental Evidence</i></a>

                            <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>8</b>, Articlenumber:<span data-test="article-number">27</span> (<span data-test="article-publication-year">2019</span>)
            <a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                        </p>
                        
    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">5487 <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">9 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">22 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/10.1186/s13750-019-0171-5/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    

                        
                        
                        
                    </div>

                    <section aria-labelledby="Abs1" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><h3 class="c-article__sub-heading" data-test="abstract-sub-heading">Background</h3><p>Within the last decades, genome-editing techniques such as CRISPR/Cas, TALENs, Zinc-Finger Nucleases, Meganucleases, Oligonucleotide-Directed Mutagenesis and base editing have been developed enabling a precise modification of DNA sequences. Such techniques provide options for simple, time-saving and cost-effective applications compared to other breeding techniques and hence genome editing has already been promoted for a wide range of plant species. Although the application of genome-editing induces less unintended modifications (off-targets) in the genome compared to classical mutagenesis techniques, off-target effects are a prominent point of criticism as they are supposed to cause unintended effects, e.g. genomic instability or cell death. To address these aspects, this map aims to answer the following question: What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects? This primary question will be considered by two secondary questions: One aims to overview the market-oriented traits being modified by genome-editing in plants and the other explores the occurrence of off-target effects.</p><h3 class="c-article__sub-heading" data-test="abstract-sub-heading">Methods</h3><p>A literature search in nine bibliographic databases, Google Scholar, and 47 web pages of companies and governmental agencies was conducted using predefined and tested search strings in English language. Articles were screened on title/abstract and full text level for relevance based on pre-defined inclusion criteria. The relevant information of included studies were mapped using a pre-defined data extraction strategy. Besides a descriptive summary of the relevant literature, a spreadsheet containing all extracted data is provided.</p><h3 class="c-article__sub-heading" data-test="abstract-sub-heading">Results</h3><p>Altogether, 555 relevant articles from journals, company web pages and web pages of governmental agencies were identified containing 1328 studies/applications of genome-editing in model plants and agricultural crops in the period January 1996 to May 2018. Most of the studies were conducted in China followed by the USA. Genome-editing was already applied in 68 different plants. Although most of the studies were basic research, 99 different market-oriented applications were identified in 28 different crops leading to plants with improved food and feed quality, agronomic value like growth characteristics or increased yield, tolerance to biotic and abiotic stress, herbicide tolerance or industrial benefits. 252 studies explored off-target effects. Most of the studies were conducted using CRISPR/Cas. Several studies firstly investigated whether sites in the genome show similarity to the target sequence and secondly analyzed these potential off-target sites by sequencing. In around 3% of the analyzed potential off-target sites, unintended mutations were detected. Only a few studies conducted off-target analyses using unbiased detection methods (e.g. whole genome sequencing). No off-target effects that could be correlated to the genome-editing process were identified in these studies.</p><h3 class="c-article__sub-heading" data-test="abstract-sub-heading">Conclusions</h3><p>The rapid adoption in plant breeding was demonstrated by a considerable number of market oriented applications (crops and traits) described in publications worldwide. Studies investigating off-target effects are very heterogeneous in their structure and design. Therefore, an in-depth assessment regarding their weight of evidence is mandatory.</p></div></div></section>
                    
    


                    <section aria-labelledby="Sec1"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Background</h2><div class="c-article-section__content" id="Sec1-content"><p>Technological progress in agriculture and plant breeding has contributed significantly to a stable food supply and formed the basis for high yields as well as the production of high-quality agricultural products [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Becker H. Pflanzenzchtung. 2nd ed. Stuttgart: UTB; 2011." href="/articles/10.1186/s13750-019-0171-5#ref-CR1" id="ref-link-section-d69214e446">1</a>]. However, in an ever-changing world, new challenges will be encountered within the next decades. Aside the demands of the growing global population and limited fossil resources, climate change is a driver of breeding efforts as it is associated with increased extreme weather events like droughts or floods as well as changing dynamics of pests and diseases. Agriculture needs to ensure and increase the world agricultural production to serve extended demands with limited environmental resources like soil and water [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Ronald P. Plant genetics, sustainable agriculture and global food security. Genetics. 2011;188:1120. &#xA;https://doi.org/10.1534/genetics.111.128553&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR2" id="ref-link-section-d69214e449">2</a>]. In contrast, intensification of agriculture causes considerable impact on nature, as naturally diverse landscapes are replaced by arable land for the cultivation of few plant species. Biodiversity is threatened through habitat loss and pesticide use, which in turn is considered to increase disease and pest pressure [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Becker H. Pflanzenzchtung. 2nd ed. Stuttgart: UTB; 2011." href="/articles/10.1186/s13750-019-0171-5#ref-CR1" id="ref-link-section-d69214e452">1</a>]. Additionally, these impacts of agriculture on the environment are becoming increasingly important in societal debates.</p><p>To meet all these challenges, improved crop varieties may be developed and integrated into a sustainable farming system considering their economic, environmental and social impacts [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Ronald P. Plant genetics, sustainable agriculture and global food security. Genetics. 2011;188:1120. &#xA;https://doi.org/10.1534/genetics.111.128553&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR2" id="ref-link-section-d69214e458">2</a>]. Examples for a sustainable farming system are the preservation of natural soil fertility through suitable crop rotations, fertilization and plant protection according to the principles of integrated cultivation or a balanced and species-appropriate animal husbandry [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="FAOCommittee on Agriculture (17th session). 06.03.2003. &#xA;http://www.fao.org/docrep/meeting/006/y8704e.htm&#xA;&#xA;. Accessed 25 Sept 2018." href="/articles/10.1186/s13750-019-0171-5#ref-CR3" id="ref-link-section-d69214e461">3</a>]. Additionally, plant breeding is of crucial importance to manage environmental impacts on cultivation systems by providing varieties resistant to plant diseases or pests, tolerant to abiotic stress and more broadly support a greener productionin time. This may reduce pesticide use and result in less intense management efforts (e.g. irrigation) [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Xie K, Wu S, Li Z, Zhou Y, Zhang D, Dong Z, et al. Map-based cloning and characterization of Zea mays male sterility33 (ZmMs33) gene, encoding a glycerol-3-phosphate acyltransferase. Theor Appl Genet. 2018;131:136378. &#xA;https://doi.org/10.1007/s00122-018-3083-9&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR4" id="ref-link-section-d69214e464">4</a>]. Further yield improvement can increase the yield per hectare and may open land management options e.g. balance with areas for nature conservation [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Becker H. Pflanzenzchtung. 2nd ed. Stuttgart: UTB; 2011." href="/articles/10.1186/s13750-019-0171-5#ref-CR1" id="ref-link-section-d69214e467">1</a>]. Nevertheless, there is evidence that increased yields may lead to a rebound-effect meaning that yield improvement conserves the rate of land clearance, but the effect is smaller than it could be [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Phalan B. What have we learned from the land sparing-sharing model? Sustainability. 2018;10:1760. &#xA;https://doi.org/10.3390/su10061760&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR5" id="ref-link-section-d69214e470">5</a>]. One example for the rebound-effect in agriculture is the Green Revolution [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Stevenson JR, Villoria N, Byerlee D, Kelley T, Maredia M. Green Revolution research saved an estimated 18 to 27 million hectares from being brought into agricultural production. Proc Natl Acad Sci USA. 2013;110:83638. &#xA;https://doi.org/10.1073/pnas.1208065110&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR6" id="ref-link-section-d69214e474">6</a>]. As a result of this revolution, yields increased, saving ecosystems from conversion to agricultural land. However, the effect was much smaller than expected. One explanation for this effect may be that increased productivity due to new technologies also increases the profitability of agriculture compared with alternative land use [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Stevenson JR, Villoria N, Byerlee D, Kelley T, Maredia M. Green Revolution research saved an estimated 18 to 27 million hectares from being brought into agricultural production. Proc Natl Acad Sci USA. 2013;110:83638. &#xA;https://doi.org/10.1073/pnas.1208065110&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR6" id="ref-link-section-d69214e477">6</a>].</p><p>Plant breeding essentially relies on the utilization of genetic variation within the breeding material that can be used for crossing and selection to develop improved varieties. New genetic variation occurs naturally by spontaneous mutations that enable some individual plants in a population to adapt to changing environmental conditions. However, as the mutation rate is fairly low and random, plant breeders and scientists have artificially induced mutations for several decades [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Songstad DD, Petolino JF, Voytas DF, Reichert NA. Genome editing of plants. Crit Rev Plant Sci. 2017;36:123. &#xA;https://doi.org/10.1080/07352689.2017.1281663&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR7" id="ref-link-section-d69214e483">7</a>]. The first generation of mutation breeding used chemical and physical mutagens to generate a plurality of nonspecific mutations. The increased mutation rate results in plants with a few positive, a lot of neutral and several negative characteristics. Thus, laborious backcrossing and selection steps are necessary in order to select for a target trait in a desired genetic background (maternal variety). Nevertheless, to date 3282 mutant varieties from 225 different plant species have been generated through undirected mutagenesis and officially but voluntarily registered [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="International Atomic Energy Agency (IAEA). Mutant variety database. &#xA;https://mvd.iaea.org/#!Search?page=1&amp;size=15&amp;sortby=Name&amp;sort=ASC&#xA;&#xA;. Accessed 25 Sept 2018." href="/articles/10.1186/s13750-019-0171-5#ref-CR8" id="ref-link-section-d69214e486">8</a>].</p><aside><div class="c-article-box" data-expandable-box-container="true"><div class="c-article-box__container" data-expandable-box="true" aria-hidden="true" id="box-Sec2"><h3 class="c-article-box__container-title u-h3 js-expandable-title" id="Sec2">Glossary: [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Becker H. Pflanzenzchtung. 2nd ed. Stuttgart: UTB; 2011." href="/articles/10.1186/s13750-019-0171-5#ref-CR1" id="ref-link-section-d69214e494">1</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Schlegel RHJ. Dictionary of plant breeding. 2nd ed. Boca Raton: CRC Press; 2009." href="/articles/10.1186/s13750-019-0171-5#ref-CR9" id="ref-link-section-d69214e497">9</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017." href="/articles/10.1186/s13750-019-0171-5#ref-CR10" id="ref-link-section-d69214e500">10</a>]</h3><div class="c-article-box__content"><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec3">Backcrossing</h4><p>Backcrossing is a crossing of a hybrid with one of its parents in order to achieve offspring that are genetically closer to the selected parent. This way, desired heterologous traits from the hybrid can be transferred into the genetic background of a parental line. Since crossing recombines all genes, many backcrosses are necessary to achieve considerable dilution of unwanted genes from the hybrid.</p><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec4">Mutagen</h4><p>A mutagen is an agent that increases the mutation rate within an organism or cell, e.g. X-rays, gamma-rays or chemicals like ethyl methane sulfonate (EMS).</p><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec5">Mutation breeding</h4><p>A plant breeding approach using mutagens to enhance genetic variation. The resulting random mutations can generate new gene variations with positive traits that can be selected for further breeding. However, several of these mutations are negative and diminish the viability of the plant.</p><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec6">Off-target effect due to genome editing</h4><p>Unintended cleavage and mutations at untargeted genomic sites with similar but not identical sequences compared to the target site.</p><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec7">Selection</h4><p>A process in breeding by which the breeder chooses only those individuals that show desired trait(s).</p><p>In 1983, the first recombinant DNA was delivered to plant cells using <i>Agrobacterium tumefaciens</i> [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Barton KA, Binns AN, Matzke AJM, Chilton M-D. Regeneration of intact tobacco plants containing full length copies of genetically engineered T-DNA, and transmission of T-DNA to R1 progeny. Cell. 1983;32:103343. &#xA;https://doi.org/10.1016/0092-8674(83)90288-X&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR11" id="ref-link-section-d69214e549">11</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Herrera-Estrella L, Depicker A, van Montagu M, Schell J. Expression of chimaeric genes transferred into plant cells using a Ti-plasmid-derived vector. Nature. 1983;303:20913. &#xA;https://doi.org/10.1038/303209a0&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR12" id="ref-link-section-d69214e552">12</a>]. From this time on, it is possible to work at a single gene level with genetic material from any organism, generating plants that cannot be bred conventionally. Nevertheless, the induced mutations using chemical and physical mutagens as well as the classical transgenic approach show limited efficiencies and unintended side effects due to the random targeting [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Govindan G, Ramalingam S. Programmable site-specific nucleases for targeted genome engineering in higher eukaryotes. J Cell Physiol. 2016;231:238092. &#xA;https://doi.org/10.1002/jcp.25367&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR13" id="ref-link-section-d69214e555">13</a>].</p><p>In recent years, genome-editing techniques have been developed enabling a more precise modification of DNA sequences in a site-directed manner [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017." href="/articles/10.1186/s13750-019-0171-5#ref-CR10" id="ref-link-section-d69214e561">10</a>]. To date, genome-editing comprises three molecular approaches that efficiently induce targeted alterations in genomes: (i) Site-directed nucleases (SDN), including Meganucleases (MN), Zinc-Finger Nucleases (ZFN), Transcription Activator-Like Effector Nucleases (TALENs) and Clustered Regularly Interspaced Short Palindromic Repeats/CRISPR associated protein (CRISPR/Cas), (ii) Oligonucleotide-directed mutagenesis (ODM) and (iii) base editing (BE). A detailed description of the single techniques is summarized in Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM1">1</a>. Site-directed nucleases induce double-strand breaks (DSBs) in the DNA which are subsequently repaired by the autochthonous cellular mechanisms. The type of repair can be categorized in three main types [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017." href="/articles/10.1186/s13750-019-0171-5#ref-CR10" id="ref-link-section-d69214e567">10</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="European Food Safety Authority (EFSA). Scientific opinion addressing the safety assessment of plants developed using Zinc Finger Nuclease 3 and other Site-Directed Nucleases with similar function. EFSA J. 2012;10:2943." href="#ref-CR14" id="ref-link-section-d69214e570">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="van de Wiel CCM, Schaart JG, Lotz LAP, Smulders MJM. New traits in crops produced by genome editing techniques based on deletions. Plant Biotechnol Rep. 2017;11:18. &#xA;https://doi.org/10.1007/s11816-017-0425-z&#xA;&#xA;." href="#ref-CR15" id="ref-link-section-d69214e570_1">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Podevin N, Davies HV, Hartung F, Nogu F, Casacuberta JM. Site-directed nucleases: a paradigm shift in predictable, knowledge-based plant breeding. Trends Biotechnol. 2013;31:37583. &#xA;https://doi.org/10.1016/j.tibtech.2013.03.004&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR16" id="ref-link-section-d69214e573">16</a>]:</p><ol class="u-list-style-none"><li><span class="u-custom-list-number">1.</span><p>No template is added and the DSB is repaired by autochthonous cellular mechanisms [in most cases non-homologous end joining (NHEJ)] resulting in small insertion-deletion (indel) mutations. This approach is defined as SDN1.</p></li><li><span class="u-custom-list-number">2.</span><p>A repair template is added which, except for a few nucleotides, is identical to the sequences in which the DSB is introduced. Then, the DSB is repaired via homology-directed repair (HDR), causing nucleotide substitution or, depending on the template used, targeted indels of a specific size. This approach is defined as SDN2.</p></li><li><span class="u-custom-list-number">3.</span><p>The repair template harbors a recombinant DNA sequence additional to the homologous sequences in which the DSB is made and the break is repaired via HDR, resulting in more complex alterations, i.e. the insertion of foreign genes. This approach is defined as SDN3.</p></li></ol>
<p>The Oligonucleotide-Directed Mutagenesis technique does not induce a DSB. Instead the introduced oligonucleotide binds to the targeted DNA sequences and this sequence is then modified by the cellular mechanism of mismatch repair [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017." href="/articles/10.1186/s13750-019-0171-5#ref-CR10" id="ref-link-section-d69214e597">10</a>]. Base editing is a recently developed approach enabling a targeted switch of DNA bases in a given frame into another without any DSB [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Komor AC, Kim YB, Packer MS, Zuris JA, Liu DR. Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Nature. 2016;533:4204. &#xA;https://doi.org/10.1038/nature17946&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR17" id="ref-link-section-d69214e600">17</a>].</p><p>Genome-editing offers substantial advantages compared to previous mutation breeding techniques and conventional genetic engineering in terms of speed and precision. It provides the opportunity to selectively mutate or modify one or a few genes (SDN1, SDN2). In addition, it is now possible to precisely modify or selectively replace (SDN3) entire genes from both closely as well as distantly related organisms [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017." href="/articles/10.1186/s13750-019-0171-5#ref-CR10" id="ref-link-section-d69214e607">10</a>]. By the use of conventional genetic engineering, traces of recombinant DNA, from the used gene shuttle (e.g. bacteria, virus or plasmid), persist in the modified organism leading to clearly characterized genetically modified organisms. In contrast, by applying genome-editing, it is possible to modify crops without inserting foreign DNA sequences at all [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Nationale Akademie der Wissenschaften Leopoldina, Deutsche Forschungsgemeinschaft, acatech  Deutsche Akademie der Technikwissenschaften, Union der deutschen Akademien der Wissenschaften. The opportunities and limits of genome editing. 2015." href="/articles/10.1186/s13750-019-0171-5#ref-CR18" id="ref-link-section-d69214e610">18</a>]. Therefore, some countries like USA, Canada, Brasilia, Argentina and others reduced the regulatory burden for plant breeders [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Bmeke O, Kahrmann J, Matthies A. Detaillierte bersicht zum regulatorischen Status der neuen molekularbiologischen Techniken (NMT) in ausgewhlten Drittstaaten. &#xA;https://www.bmel.de/DE/Landwirtschaft/Pflanzenbau/Gentechnik/_Texte/Neue_molekularbiologische_Techniken.html&#xA;&#xA;. Accessed 21 Feb 2019." href="/articles/10.1186/s13750-019-0171-5#ref-CR19" id="ref-link-section-d69214e613">19</a>]. Due to the simplicity, time-saving and cost-effective application of genome-editing, it has already been applied in a wide range of cultivars. Genome-editing has been used for:</p><ol class="u-list-style-none"><li><span class="u-custom-list-number">i.</span><p>Analyzing gene functions (e.g. effect of the RAV2 gene for salt stress in rice [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Duan Y-B, Li J, Qin R-Y, Xu R-F, Li H, Yang Y-C, et al. Identification of a regulatory element responsible for salt induction of rice OsRAV2 through ex situ and in situ promoter analysis. Plant Mol Biol. 2016;90:4962. &#xA;https://doi.org/10.1007/s11103-015-0393-z&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR20" id="ref-link-section-d69214e622">20</a>]).</p></li><li><span class="u-custom-list-number">ii.</span><p>Improvement of product quality (e.g. improved oil quality in soybean [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Demorest ZL, Coffman A, Baltes NJ, Stoddard TJ, Clasen BM, Luo S, et al. Direct stacking of sequence-specific nuclease-induced mutations to produce high oleic and low linolenic soybean oil. BMC Plant Biol. 2016;16:225. &#xA;https://doi.org/10.1186/s12870-016-0906-1&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR21" id="ref-link-section-d69214e630">21</a>]).</p></li><li><span class="u-custom-list-number">iii.</span><p>Development of disease resistant varieties (e.g. virus resistant cucumber [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Chandrasekaran J, Brumin M, Wolf D, Leibman D, Klap C, Pearlsman M, et al. Development of broad virus resistance in non-transgenic cucumber using CRISPR/Cas9 technology. Mol Plant Pathol. 2016;17:114053. &#xA;https://doi.org/10.1111/mpp.12375&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR22" id="ref-link-section-d69214e638">22</a>]).</p></li><li><span class="u-custom-list-number">iv.</span><p>Developing of herbicide tolerant varieties (e.g. resistance to the herbicide chlorsulfuron in canola [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Gocal GFW, Schpke C, Beetham PR. Oligo-mediated targeted gene editing. In: Zhang F, Puchta H, Thomson JG, editors. Advances in new technology for targeted modification of plant genomes. New York: Springer; 2015. p. 7389. &#xA;https://doi.org/10.1007/978-1-4939-2556-8_5&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR23" id="ref-link-section-d69214e646">23</a>]).</p></li><li><span class="u-custom-list-number">v.</span><p>Improved adaption to abiotic stress, (e.g. drought tolerance in maize [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Shi J, Gao H, Wang H, Lafitte HR, Archibald RL, Yang M, et al. ARGOS8 variants generated by CRISPRCas9 improve maize grain yield under field drought stress conditions. Plant Biotechnol J. 2017;15:20716. &#xA;https://doi.org/10.1111/pbi.12603&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR24" id="ref-link-section-d69214e654">24</a>]).</p></li></ol>
<p>Even in plants like hexaploid wheat, that were so far largely inaccessible for targeted genetic alterations, the simultaneous mutation of all six alleles was successfully performed [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Wang Y, Cheng X, Shan Q, Zhang Y, Liu J, Gao C, Qiu J-L. Simultaneous editing of three homoeoalleles in hexaploid bread wheat confers heritable resistance to powdery mildew. Nat Biotechnol. 2014;32:94751. &#xA;https://doi.org/10.1038/nbt.2969&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR25" id="ref-link-section-d69214e661">25</a>]. These successful applications are opening up new dimensions for the scientific plant breeding and agricultural community.</p><p>Compared to randomly induced mutations by chemicals or irradiation, the number of unintended mutations (off-target effects) is broadly reduced by genome-editing techniques [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017." href="/articles/10.1186/s13750-019-0171-5#ref-CR10" id="ref-link-section-d69214e667">10</a>]. Nevertheless, their application does not completely or per se exclude the occurrence of off-target effects. Off-target effects caused by genome editing most likely occur in DNA sequences that are similar (not identical) to the targeted one but are located at another site in the genome. Mainly, they occur due to the lack of exclusiveness and/or length of the recognition site [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017." href="/articles/10.1186/s13750-019-0171-5#ref-CR10" id="ref-link-section-d69214e670">10</a>]. When analyzing off-target effects, one distinguishes between biased and unbiased detection methods [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Martin F, Snchez-Hernndez S, Gutirrez-Guerrero A, Pinedo-Gomez J, Benabdellah K. Biased and unbiased methods for the detection of off-target cleavage by CRISPR/Cas9: an overview. Int J Mol Sci. 2016. &#xA;https://doi.org/10.3390/ijms17091507&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR26" id="ref-link-section-d69214e673">26</a>]. To date, the predominant approach for identifying off-target effects is the biased approach consisting of two steps: (i) Using sequence alignment programs, sites in the genome with high similarity to the target sequence are identified, which are designated as potential off-target sites. Several different tools like BLAST [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Altschul S, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J Mol Biol. 1990;215:40310. &#xA;https://doi.org/10.1006/jmbi.1990.9999&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR27" id="ref-link-section-d69214e676">27</a>], Cas-OFFinder [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Bae S, Park J, Kim J-S. Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases. Bioinformatics. 2014;30:14735. &#xA;https://doi.org/10.1093/bioinformatics/btu048&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR28" id="ref-link-section-d69214e679">28</a>] or CRISPR-P [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lei Y, Lu L, Liu H-Y, Li S, Xing F, Chen L-L. CRISPR-P: a web tool for synthetic single-guide RNA design of CRISPR-system in plants. Mol Plant. 2014;7:14946. &#xA;https://doi.org/10.1093/mp/ssu044&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR29" id="ref-link-section-d69214e683">29</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Liu H, Ding Y, Zhou Y, Jin W, Xie K, Chen L-L. CRISPR-P 2.0: an improved CRISPRCas9 tool for genome editing in plants. Mol Plant. 2017;10:5302. &#xA;https://doi.org/10.1016/j.molp.2017.01.003&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR30" id="ref-link-section-d69214e686">30</a>] are used to identify these potential off-target sites. (ii) The identified individual DNA sequences (potential off-target sites) are then analyzed for undesired mutations (off-target effects) using various detection methods like mismatch-sensitivity endonuclease assays, Sanger sequencing or targeted deep sequencing [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Tycko J, Myer VE, Hsu PD. Methods for optimizing CRISPRCas9 genome editing specificity. Mol Cell. 2016;63:35570. &#xA;https://doi.org/10.1016/j.molcel.2016.07.004&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR31" id="ref-link-section-d69214e689">31</a>]. All detection methods have their specific advantages and disadvantages which are addressed in several reviews (e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Martin F, Snchez-Hernndez S, Gutirrez-Guerrero A, Pinedo-Gomez J, Benabdellah K. Biased and unbiased methods for the detection of off-target cleavage by CRISPR/Cas9: an overview. Int J Mol Sci. 2016. &#xA;https://doi.org/10.3390/ijms17091507&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR26" id="ref-link-section-d69214e692">26</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Zischewski J, Fischer R, Bortesi L. Detection of on-target and off-target mutations generated by CRISPR/Cas9 and other sequence-specific nucleases. Biotechnol Adv. 2017;35:95104. &#xA;https://doi.org/10.1016/j.biotechadv.2016.12.003&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR32" id="ref-link-section-d69214e695">32</a>]). In contrast to the biased detection methods, unbiased ones are used to identify off-target effects in a completely unrestricted way. Therefore, it requires genome-wide sequencing to identify off-target mutations anywhere in the genome and de novo define off-target sites [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Martin F, Snchez-Hernndez S, Gutirrez-Guerrero A, Pinedo-Gomez J, Benabdellah K. Biased and unbiased methods for the detection of off-target cleavage by CRISPR/Cas9: an overview. Int J Mol Sci. 2016. &#xA;https://doi.org/10.3390/ijms17091507&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR26" id="ref-link-section-d69214e698">26</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Zischewski J, Fischer R, Bortesi L. Detection of on-target and off-target mutations generated by CRISPR/Cas9 and other sequence-specific nucleases. Biotechnol Adv. 2017;35:95104. &#xA;https://doi.org/10.1016/j.biotechadv.2016.12.003&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR32" id="ref-link-section-d69214e702">32</a>]. Depending on the detection method being used, the results of identifying off-target mutations vary widely. Although genome-editing techniques induce much less off-target effects compared to classical mutagenesis techniques, these are an important point of criticism as they may possibly cause genomic instability, cytotoxicity and cell death [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Agapito-Tenfen SZ, Wikmark O-G. Current status of emerging technologies for plant breeding. Biosafety and knowledge gaps of site directed nucleases and oligonucleotide-directed mutagenesis. Genk Centre for Biosafety: Troms; 2015." href="#ref-CR33" id="ref-link-section-d69214e705">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zhang X-H, Tee LY, Wang X-G, Huang Q-S, Yang S-H. Off-target effects in CRISPR/Cas9-mediated genome engineering. Mol Ther Nucleic Acids. 2015;4:e264. &#xA;https://doi.org/10.1038/mtna.2015.37&#xA;&#xA;." href="#ref-CR34" id="ref-link-section-d69214e705_1">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Kanchiswamy CN, Maffei M, Malnoy M, Velasco R, Kim J-S. Fine-tuning next-generation genome editing tools. Trends Biotechnol. 2016;34:56274. &#xA;https://doi.org/10.1016/j.tibtech.2016.03.007&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR35" id="ref-link-section-d69214e708">35</a>].</p><p>This systematic map facilitates an objective debate by informing interested stakeholder communities in a transparent and retraceable manner about the status of research, the progress of genome-editing in plants and the available evidence for the potential occurrence of associated off-target effects. Furthermore, risk assessors and decision makers are depending on the provision of a reliable body of evidence to support conclusions about potential risks being associated with the application of genome-editing. Thus, an overview of the available evidence on the occurrence of off-target effects could be of crucial importance.</p></div></div></div></aside><h3 class="c-article__sub-heading" id="Sec8">Stakeholder engagement</h3><p>The systematic map question, the secondary questions and the scope was designed by the review team reflecting discussions with policy makers, authorities, regulators and academia requesting a broad overview on the available evidence about the application of genome editing in plants and the potential occurrence of off-target effects. Throughout the review process there was no stakeholder engagement. As indicated in the systematic map protocol the results of the map were discussed on a conference with different stakeholders from this field, including besides others plant breeders, federal authorities, academia, farmer organizations and processing industry. Stakeholder remarks are taken into account when preparing a systematic review based on the results of this map.</p></div></div></section><section aria-labelledby="Sec9"><div class="c-article-section" id="Sec9-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">Objectives of the map</h2><div class="c-article-section__content" id="Sec9-content"><p>As genome-editing techniques are a promising tool to revolutionize plant breeding, they are of particular relevance to scientists, breeders, farmers but also to decision and policy makers with regards to the broader agricultural management and future challenges. Therefore, we wanted to provide a comprehensive and transparent overview of the available evidence base concerning the effects of genome-editing in plants. The main objectives were:</p><ul class="u-list-style-bullet"><li><p>Overview of the traits modified by genome-editing in model plants as well as in crops produced for agricultural production.</p></li><li><p>Overview of the available evidence about the occurrence of off-target effects due to the use of genome-editing techniques in model plants as well as in crops produced for agricultural production.</p></li><li><p>Identification of the geographical distribution of genome-editing activities in plants worldwide.</p></li><li><p>Identification of the volume of the available literature, evidence clusters and key characteristics of the evidence base to inform interested stakeholder communities.</p></li><li><p>Identification of knowledge gaps concerning the occurrence of off-target effects in order to inform decision makers which future research might be needed for a risk assessment.</p></li><li><p>Assessment whether a specific section of the available evidence base is suitable for an in-depth analyses by a systematic review.</p></li></ul>
<p>The primary question of the systematic map was: What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects?</p><p>To answer this primary question, it was subdivided into two secondary questions related to (1) the traits modified by genome-editing and (2) the occurrence of off-target effects due to the use of genome-editing.</p><h3 class="c-article__sub-heading" id="Sec10">Secondary question one</h3>
                  <blockquote class="c-blockquote"><div class="c-blockquote__body">
                    <p><i>What are the traits modified by genome</i>-<i>editing in model plants as well as in crops produced for agricultural production?</i></p>
                  </div></blockquote>
                <p>
                  <dl class="c-abbreviation_list"><dt class="c-abbreviation_list__term"><dfn>Population::</dfn></dt><dd class="c-abbreviation_list__description">
                        <p>Any model plant or crop produced for agricultural production.</p>
                      </dd><dt class="c-abbreviation_list__term"><dfn>Intervention::</dfn></dt><dd class="c-abbreviation_list__description">
                        <p>One of the following genome-editing techniques was used to induce an alteration in the plant genome: Clustered Regularly Interspaced Short Palindromic Repeats/CRISPR associated protein (CRISPR/Cas), Transcription Activator-Like Effector Nucleases (TALENs), Meganucleases (MN), Zinc-Finger Nucleases (ZFN), Oligonucleotide-Directed Mutagenesis (ODM), base editing (BE).</p>
                      </dd><dt class="c-abbreviation_list__term"><dfn>Outcome::</dfn></dt><dd class="c-abbreviation_list__description">
                        <p>The alteration of the genome (i.e. insertion, deletion or substitution of nucleotides) induced by the use of a genome-editing technique.</p>
                      </dd></dl>
                </p><h3 class="c-article__sub-heading" id="Sec11">Secondary question two</h3>
                  <blockquote class="c-blockquote"><div class="c-blockquote__body">
                    <p><i>What is the available evidence for the potential occurrence of associated off</i>-<i>target effects due to the use of genome</i>-<i>editing in model plants as well as in crops produced for agricultural production?</i></p>
                  </div></blockquote>
                <p>
                  <dl class="c-abbreviation_list"><dt class="c-abbreviation_list__term"><dfn>Population::</dfn></dt><dd class="c-abbreviation_list__description">
                        <p>Any model plant or crop produced for agricultural production.</p>
                      </dd><dt class="c-abbreviation_list__term"><dfn>Intervention::</dfn></dt><dd class="c-abbreviation_list__description">
                        <p>One of the following genome-editing techniques was used to induce an alteration in the plant genome: Clustered Regularly Interspaced Short Palindromic Repeats/CRISPR associated protein (CRISPR/Cas), Transcription Activator-Like Effector Nucleases (TALENs), Meganucleases (MN), Zinc-Finger Nucleases (ZFN), Oligonucleotide-Directed Mutagenesis (ODM), base editing (BE).</p>
                      </dd><dt class="c-abbreviation_list__term"><dfn>Outcome::</dfn></dt><dd class="c-abbreviation_list__description">
                        <p>The alteration of the genome (i.e. insertion, deletion or substitution of nucleotides) induced by the use of a genome-editing technique. Additionally, the occurrence of off-target effects was assessed.</p>
                      </dd></dl>
                </p></div></div></section><section aria-labelledby="Sec12"><div class="c-article-section" id="Sec12-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec12">Methods</h2><div class="c-article-section__content" id="Sec12-content"><p>The methods used to conduct this systematic map were based on the Collaboration for Environmental Evidence (CEE) systematic review guidelines [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Collaboration for Environmental Evidence. Guidelines for systematic review and evidence synthesis in environmental management, Version 4.2. Environmental Evidence. 2013. &#xA;http://www.environmentalevidence.org/Documents/Guidelines/Guidelines4.2.pd&#xA;&#xA;. Accessed 13 Sept 2018." href="/articles/10.1186/s13750-019-0171-5#ref-CR36" id="ref-link-section-d69214e889">36</a>]. Detailed information about the methods used to perform this systematic map are presented in the published protocol [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Modrzejewski D, Hartung F, Sprink T, Krause D, Kohl C, Schiemann J, Wilhelm R. What is the available evidence for the application of genome editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map protocol. Environ Evid. 2018;7:11. &#xA;https://doi.org/10.1186/s13750-018-0130-6&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR37" id="ref-link-section-d69214e892">37</a>]. A brief summary of these methods is provided here.</p><h3 class="c-article__sub-heading" id="Sec13">Search for articles</h3><p>The search string was composed of two parts: The first part defined the population of interest comprising less specific terms like crop, plant or seed as well as specific model plants and crops including their English and Latin names. The second part defined the intervention, i.e. the genome-editing technique applied to induce an alteration in the plant genome (CRISPR, TALENs, ZFN, MN, ODM or BE). To test the comprehensiveness of the search strategy a scoping search was carried out and the identified records where tested against an a priori defined test library with articles of known relevance. Details on search settings and subscriptions can be found in Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM2">2</a>. The following bibliographic databases were searched whereby the search string was adapted to the specific needs of each database to which it was applied to:</p><ul class="u-list-style-bullet"><li><p>Scopus</p></li><li><p>PubMed</p></li><li><p>Science direct</p></li><li><p>Agris</p></li><li><p>Web of Science (WoS)</p></li><li><p>Biological Abstracts</p></li><li><p>BIOSIS Previews</p></li><li><p>CAB Abstracts</p></li><li><p>SciELO Citation Index</p></li></ul>
<p>In addition, Google Scholar (<a href="https://scholar.google.com">https://scholar.google.com</a>) was searched using 30 different combinations of different (model) plants and genome-editing terms. The first 20 hits organized by relevance, of each search term were examined for relevance. Furthermore, a total amount of 47 web pages of companies working with genome-editing and the USDA database Am I regulated? (<a href="https://www.aphis.usda.gov/aphis/ourfocus/biotechnology/am-i-regulated">https://www.aphis.usda.gov/aphis/ourfocus/biotechnology/am-i-regulated</a>) were searched to identify grey literature. Finally, the bibliographies of 107 review articles identified by the literature search were screened to identify further relevant papers. In May 2018, the search string was applied in order to identify articles published after 1996, when the first study about a genome-editing technique was published.</p><h3 class="c-article__sub-heading" id="Sec14">Article screening and study eligibility criteria</h3><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec15">Screening process</h4><p>Before applying the selection criteria at title/abstract and then at full text level, a consistency check was conducted by all four participating reviewers aiming to determine the inter-reviewer agreement. The level of agreement was tested formally using a kappa test [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Cohen J. A coefficient of agreement for nominal scales. Educ Psychol Meas. 1960;20:3746. &#xA;https://doi.org/10.1177/001316446002000104&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR38" id="ref-link-section-d69214e957">38</a>]. 100 references retrieved by the search were randomly explored at title/abstract level leading to a kappa value of 0.48. After discussing all disagreements, a second consistency check was carried out using another randomly allocated 100 references resulting in a kappa value of 0.71. After title/abstract screening, potentially relevant articles were checked at full text level. A list of unobtainable articles (Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM3">3</a>) and articles excluded at full text level with the reason for exclusion (Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM4">4</a>) are provided. For conducting the consistency check as well as for the screening process at title/abstract and full text level the open-access and non-profit database CADIMA was used [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Kohl C, McIntosh EJ, Unger S, Haddaway NR, Kecke S, Schiemann J, Wilhelm R. Online tools supporting the conduct and reporting of systematic reviews and systematic maps: a case study on CADIMA and review of existing tools. Environ Evid. 2018;7:2420. &#xA;https://doi.org/10.1186/s13750-018-0115-5&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR39" id="ref-link-section-d69214e966">39</a>]. Two members of the review team are authors of a few articles retrieved by the review process. However, as none of these papers comprise primary data, their articles were excluded at title/abstract level. Nevertheless, the two coauthors routinely screened literature covering the issues addressed in this map.</p><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec16">Eligibility criteria</h4><p>An article had to meet all the following inclusion criteria in order to enter into the systematic map:<dl class="c-abbreviation_list"><dt class="c-abbreviation_list__term"><dfn>Eligible populations::</dfn></dt><dd class="c-abbreviation_list__description"><p>Any model plant or crop produced for agricultural production as well as higher fungi was used.</p></dd><dt class="c-abbreviation_list__term"><dfn>Eligible interventions::</dfn></dt><dd class="c-abbreviation_list__description"><p>At least one of the following genome-editing techniques was used to induce an alteration in the plant genome: Clustered Regularly Interspaced Short Palindromic Repeats/CRISPR associated protein (CRISPR/Cas), Transcription Activator-like Effector Nuclease (TALENs), Zinc-Finger Nuclease (ZFN), Meganuclease (MN), Oligonucleotide-Directed Mutagenesis (ODM), base editing (BE).</p></dd><dt class="c-abbreviation_list__term"><dfn>Eligible outcome for secondary question one::</dfn></dt><dd class="c-abbreviation_list__description"><p>An alteration in the plant genome was reported (insertion, deletion or substitution) due to the use of a genome-editing technique.</p></dd><dt class="c-abbreviation_list__term"><dfn>Eligible outcome for secondary question two::</dfn></dt><dd class="c-abbreviation_list__description"><p>The occurrence of off-target effects was assessed.</p></dd><dt class="c-abbreviation_list__term"><dfn>Eligible type of data::</dfn></dt><dd class="c-abbreviation_list__description"><p>Only those references were included which comprise primary data referring to the use of a genome-editing technique to induce a sequence alteration in the plant genome.</p></dd><dt class="c-abbreviation_list__term"><dfn>Eligible languages::</dfn></dt><dd class="c-abbreviation_list__description"><p>References in German and English languages were included. Articles in other languages were included when besides title and abstract, further parts of the article, like figures or tables, were in English or German and the provided information allowed for a definite judgment of their relevance.</p></dd></dl>
</p><h3 class="c-article__sub-heading" id="Sec17">Study validity assessment</h3><p>The aim of this systematic map was to provide a broad overview of the progress on genome-editing in plants as well as the examination of associated off-target effects. The validity of the included studies (critical appraisal) was not assessed. However, in order to facilitate the decision on the potential of a subsequent systematic review, data being indicative for the validity of an included study were extracted.</p><h3 class="c-article__sub-heading" id="Sec18">Data coding strategy</h3><p>Articles in which several genome-editing techniques were applied, different plants were used or different genes were addressed have been subdivided in distinct studies. While articles were screened for relevance at title/abstract and full text level, the relevant data were finally extracted at study level.</p><p>The data of each included study was extracted in one row in the excel file for the following superordinate categories:</p><ol class="u-list-style-none"><li><span class="u-custom-list-number">1.</span><p>Bibliographic information (reference type, authors, year of publication, title, abstract, keywords, periodical, issue number, page range, volume, DOI/ISBN, corresponding author and the name of the country the corresponding author is located).</p></li><li><span class="u-custom-list-number">2.</span><p>Information answering secondary question one about traits modified by genome-editing (genome-editing technique, plant species, sequence identifier, trait, type of alteration, progress in research, key topic).</p></li><li><span class="u-custom-list-number">3.</span><p>Information answering secondary question two about the occurrence of off-target effects due to the use of genome-editing (search for off-target effects, prediction of potential off-target effects (in silico), the prediction method used, identification of potential off-target sites, detection of off-target effects (biased/unbiased), detection method, amount of identified off-target effects (biased/unbiased).</p></li></ol>
<p>Data were extracted according to the systematic map protocol [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Modrzejewski D, Hartung F, Sprink T, Krause D, Kohl C, Schiemann J, Wilhelm R. What is the available evidence for the application of genome editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map protocol. Environ Evid. 2018;7:11. &#xA;https://doi.org/10.1186/s13750-018-0130-6&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR37" id="ref-link-section-d69214e1050">37</a>]. Data from a subset of 20 studies were extracted by two reviewers independently to assess the consistency of the extraction process across the reviewers. Discrepancies were discussed and clarified within the whole review team. Then, the data of the other studies were extracted by one reviewer and cross-checked by a second one to minimize the introduction of human error. In case of missing data, no information has been noted under the respective category. If data were in another language than English or German, it is stated as language.</p><h3 class="c-article__sub-heading" id="Sec19">Data mapping method</h3><p>The overview of research activities is provided in a narrative report and visualized in tables and figures. In addition, relevant studies and the extracted data are catalogued in Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM5">5</a> (to answer secondary question one) and Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM6">6</a> (to answer secondary question two) as well as provided in a searchable database that is freely accessible on the web page <a href="https://www.dialog-gea.de/de/service/repositorium">https://www.dialog-gea.de/de/service/repositorium</a>.</p><h3 class="c-article__sub-heading" id="Sec20">Deviations from the systematic map protocol</h3><p>The methods used in this map deviate from the protocol in the following aspects:</p><ol class="u-list-style-none"><li><span class="u-custom-list-number">i.</span><p>Contrary to what was stated in the protocol, the recently developed base editing method was not excluded. To identify all base editing studies the second part of the search string (intervention) was extended by the terms base-editing and base editing.</p></li><li><span class="u-custom-list-number">ii.</span><p>Articles in other languages than English or German were included if further parts than title and abstract of the article like figures or tables were in English or German and the provided information allowed for a definite judgment of their relevance.</p></li><li><span class="u-custom-list-number">iii.</span><p>In the course of data extraction we noticed that it was not possible to properly categorize the progress in research to the three classes indicated in the systematic map protocol [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Modrzejewski D, Hartung F, Sprink T, Krause D, Kohl C, Schiemann J, Wilhelm R. What is the available evidence for the application of genome editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map protocol. Environ Evid. 2018;7:11. &#xA;https://doi.org/10.1186/s13750-018-0130-6&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR37" id="ref-link-section-d69214e1095">37</a>]. Therefore, we decided to classify the studies as either basic research or market-oriented application. To be flagged as market-oriented, a study had to meet three criteria: </p><ol class="u-list-style-none"><li><span class="u-custom-list-number">1.</span><p>Genome-editing was applied in an agricultural crop.</p></li><li><span class="u-custom-list-number">2.</span><p>A trait was addressed that may be of interest for commercialization (market-oriented trait).</p></li><li><span class="u-custom-list-number">3.</span><p>The targeted trait is expressed in the edited plant grown.</p><p>All studies that did not meet all three criteria were classified as basic research.</p></li></ol>
</li><li><span class="u-custom-list-number">vi.</span><p>Contrary to what was stated in the protocol, no EndNote database of all studies included in the systematic map was attached to this systematic map. Instead, one excel file for each secondary question was provided as additional file containing all included studies and the extracted data.</p></li></ol>
</div></div></section><section aria-labelledby="Sec21"><div class="c-article-section" id="Sec21-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec21">Results</h2><div class="c-article-section__content" id="Sec21-content"><h3 class="c-article__sub-heading" id="Sec22">Review descriptive statistics</h3><p>Figure<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig1">1</a> presents the systematic mapping process of articles and studies in a flow diagram. From January 1996 until May 2018, in total 15,703 records were identified from ten bibliographic databases, Google Scholar, the targeted search on 47 company web pages, the USDA-database Am I regulated? and the screening of 107 review articles. After removing duplicates (n=9521), 6182 articles remained and were screened on title/abstract level. Main reasons for exclusion at this stage were the application of genome-editing in animals and the absence of primary data. 941 articles passed the inclusion criteria on title/abstract level or were rated as unclear and remained included for full text screening. The application of the inclusion criteria at full text level resulted in 524 relevant articles. Searching for grey literature on company websites and websites from governmental authorities identified another 31 further relevant documents and web pages that were considered during data extraction. Out of these 555 records, a total amount of 1328 studies were extracted and formed the basis for this systematic map. A list of all studies and the extracted data to answer secondary question one is provided in Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM5">5</a>. Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM6">6</a> provides a list of all studies and the extracted data to answer secondary question two. A ROSES reporting form is included in Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM7">7</a>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1"><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.1</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig1_HTML.png?as=webp"></source><img aria-describedby="figure-1-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig1_HTML.png" alt="figure1" loading="lazy" width="685" height="764" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>Flow diagram of the systematic mapping process explaining the selection of relevant articles and studies. This diagram follows ROSES guidance [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Haddaway NR, Macura B, Whaley P, Pullin AS. ROSES for Systematic Map Reports. Version 1.0. 2017. &#xA;https://doi.org/10.6084/m9.figshare.5897299&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR40" id="ref-link-section-d69214e1154">40</a>]</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/1" data-track-dest="link:Figure1 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<h3 class="c-article__sub-heading" id="Sec23">General overview of the application of genome-editing in model plants and crops</h3><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec24">Studies per year</h4><p>In the mapping period between January 1996 and May 2018, a total amount of 1328 studies were identified. As shown in Fig.<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig2">2</a>, the number of studies using TALENs, ZFN, ODM, MN and BE has remained on a relatively low level. In contrast, the number of studies on CRISPR/Cas has risen sharply soon after the system was applied for the first time in plants in 2013. Nearly 85% of the studies were published since 2015 indicating the rapid dissemination and development of these techniques within the last few years.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2"><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.2</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig2_HTML.png?as=webp"></source><img aria-describedby="figure-2-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig2_HTML.png" alt="figure2" loading="lazy" width="685" height="442" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p>The number of genome-editing applications published per year. *Only JanuaryMay 2018; <i>CRISPR/Cas</i> Clustered Regularly Interspaced Short Palindromic Repeats/CRISPR associated protein, <i>TALENs</i> Transcription Activator-Like Effector Nucleases, <i>ZFN</i> Zinc-Finger Nucleases, <i>ODM</i> Oligo-Directed Mutagenesis, <i>MN</i> Meganucleases, <i>BE</i> base editing</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/2" data-track-dest="link:Figure2 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<p>As shown in Table<a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/10.1186/s13750-019-0171-5#Tab1">1</a>, 26 studies used MN, 27 studies used ODM, 42 studies used BE, 73 studies used ZFN, 128 studies used TALENs and 1032 studies used CRISPR/Cas. When using CRISPR/Cas, the most frequently used nuclease was Cas9 (n=986) followed with a large distance by Cas12a (also known as Cpf1) (n=46).
</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b id="Tab1" data-test="table-caption">Table1 Heat map showing number of studies performed with the different genome-editing techniques (rows) and the year the studies were published (columns) (1996May 2018)</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/10.1186/s13750-019-0171-5/tables/1"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<h3 class="c-article__sub-heading" id="Sec25">Geographical distribution of genome-editing studies</h3><p>The number of studies per country was calculated based on the country the corresponding author is located at. In case of grey literature, the study was accounted based on the country the company is located at. Multiple assignments are possible if the corresponding author is affiliated with institutions in different countries or if more than one author was indicated being corresponding author. Therefore, the sum of studies accounted for different countries is higher (n=1494) than the total amount of studies identified (n=1328). Asia is the leading continent when applying and publicizing genome-editing in plants (n=784 studies, 53%) followed by North America (n=508 studies, 34%), Europe (n=189 studies, 13%), Australia (n=6 studies, &lt;1%), South America (n=4 studies, &lt;1%) and Africa (n=3 studies, &lt;1%) (Fig.<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig3">3</a>). In total, publications from 33 countries were identified. As shown in Fig.<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig3">3</a>, China has a substantial lead in the number of studies (n=599 studies, 40%) followed by the USA (n=487 studies, 33%), Japan (94 studies, 6%) and Germany (n=88 studies, 6%).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3"><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.3</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig3_HTML.png?as=webp"></source><img aria-describedby="figure-3-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig3_HTML.png" alt="figure3" loading="lazy" width="685" height="1011" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>Number of studies per country* in the systematic map database (grouped by continent; January 1996May 2018). *Identified by the corresponding author(s); South America: Grey; Australia: Black; North America: Orange; Europe: Blue; Asia: Green; Africa: Red</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/3" data-track-dest="link:Figure3 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec26">Genome-editing applications in model plants and crops</h3><p>Around two-third of the studies (n=907; 68%) were conducted on agricultural crops and one-third (n=421; 32%) on model organisms. However, it is worth noting that rice is an important crop plant but it is also used as model species because of its genome size and because embryogenic rice cultures can be easily prepared, transformed and rapidly regenerated into fertile plants. In total, 51 species and subspecies of different agricultural crops and 17 species/subspecies of model plants were under investigation, with the majority of studies focusing on rice (n=465) followed by the model organisms <i>Arabidopsis</i> (n=218) and tobacco (n=107). Besides to these, tomato (n=84) is most commonly studied followed by maize (n=77), wheat (n=63) and soybean (n=53) (Fig.<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig4">4</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4"><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.4</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig4_HTML.png?as=webp"></source><img aria-describedby="figure-4-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig4_HTML.png" alt="figure4" loading="lazy" width="685" height="1045" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>Total amount of genome-editing applications in crops and model plants (1996May 2018). *Several species or subspecies were used when applying genome-editing</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/4" data-track-dest="link:Figure4 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<h3 class="c-article__sub-heading" id="Sec27">Type of alteration in the plant genome</h3><p>As shown in Fig.<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig5">5</a>, the majority of studies (n=1223; 92%) describe the induction of point mutations or indels comparable to spontaneous mutations or undirected mutagenesis. This was mainly achieved with SDN1 (n=1154) where no repair template was added and the DSB was repaired by NHEJ. Additionally, the induction of point mutations (PM) using the ODM technique (n=27) or BE (n=42) leads to point mutations comparable to SDN1. Only 36 studies (3%) added a repair template that, except for a few nucleotides, was identical to the targeted sequence in which the DSB was introduced leading to a DSB repair via homology-directed repair (SDN2). In 68 studies (5%), a repair template was added that harbors a recombinant DNA sequence additional to the homologous sequences and the DSB was repaired via homology-directed repair (SDN3).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5"><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.5</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig5_HTML.png?as=webp"></source><img aria-describedby="figure-5-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig5_HTML.png" alt="figure5" loading="lazy" width="685" height="487" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p>Distribution of the type of alteration introduced by the application of genome-editing (January 1996May 2018). <i>SDN</i> Site directed nucleases, <i>PM</i> point mutation, <i>BE</i> base editing</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/5" data-track-dest="link:Figure5 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<h3 class="c-article__sub-heading" id="Sec28">Secondary question 1: What are the traits modified by genome-editing in model plants as well as in crops produced for agricultural production?</h3><p>In total, 193 studies were allocated as market-oriented applications. However, different scientists studied the same crop species and trait. Considering this, a total amount of 99 different applications in 28 different plant species remained. These market-oriented applications build the basis to answer secondary question 1. Figure<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig6">6</a> categorizes the market-oriented applications to different groups of traits. Most of the market-oriented applications (n=36) are related to an improved agronomic value (Table<a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/10.1186/s13750-019-0171-5#Tab2">2</a>), followed by 28 applications with an improved food and feed quality (Table<a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/10.1186/s13750-019-0171-5#Tab3">3</a>). For biotic stress tolerance, 16 different applications were identified (Table<a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/10.1186/s13750-019-0171-5#Tab4">4</a>), for herbicide tolerance eight applications (Table<a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/10.1186/s13750-019-0171-5#Tab5">5</a>), for industrial utilization six applications (Table<a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/10.1186/s13750-019-0171-5#Tab6">6</a>) and for abiotic stress tolerance five applications (Table<a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/10.1186/s13750-019-0171-5#Tab7">7</a>). For more detailed information about the analyzed traits, see the respective tables.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6"><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.6</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig6_HTML.png?as=webp"></source><img aria-describedby="figure-6-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig6_HTML.png" alt="figure6" loading="lazy" width="685" height="477" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p>Distribution of market-oriented applications of crops with nutritionally, agriculturally or industrially relevant traits (January 1996May 2018)</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/6" data-track-dest="link:Figure6 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-2"><figure><figcaption class="c-article-table__figcaption"><b id="Tab2" data-test="table-caption">Table2 Genome-editing in plants for modifying agronomically relevant traits (1996May 2018)</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/10.1186/s13750-019-0171-5/tables/2"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-3"><figure><figcaption class="c-article-table__figcaption"><b id="Tab3" data-test="table-caption">Table3 Genome-editing in plants for improved food and feed quality (1996May 2018)</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/10.1186/s13750-019-0171-5/tables/3"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-4"><figure><figcaption class="c-article-table__figcaption"><b id="Tab4" data-test="table-caption">Table4 Genome-editing in plants for increased resistance to biotic stress (1996May 2018)</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/10.1186/s13750-019-0171-5/tables/4"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-5"><figure><figcaption class="c-article-table__figcaption"><b id="Tab5" data-test="table-caption">Table5 Genome-editing for generating herbicide tolerant plants (1996May 2018)</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/10.1186/s13750-019-0171-5/tables/5"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-6"><figure><figcaption class="c-article-table__figcaption"><b id="Tab6" data-test="table-caption">Table6 Genome-editing in plants for industrial utilization (1996May 2018)</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/10.1186/s13750-019-0171-5/tables/6"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-7"><figure><figcaption class="c-article-table__figcaption"><b id="Tab7" data-test="table-caption">Table7 Genome-editing in plants to improve tolerance to abiotic stress (1996May 2018)</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/10.1186/s13750-019-0171-5/tables/7"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<p>As shown in Fig.<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig7">7</a> most of the market-oriented applications were applied in rice (n=29), followed by tomato (n=16), maize (n=10), potato (n=6), wheat (n=6), soybean (n=4) and canola (n=4). In 21 other agricultural relevant crops, one or two market-oriented applications were identified.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-7"><figure><figcaption><b id="Fig7" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.7</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/7" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig7_HTML.png?as=webp"></source><img aria-describedby="figure-7-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig7_HTML.png" alt="figure7" loading="lazy" width="685" height="521" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p>Total amount of genome-editing applications in crops with market-oriented traits (January 1996May 2018)</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/7" data-track-dest="link:Figure7 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<h3 class="c-article__sub-heading" id="Sec29">Secondary question 2: What is the available evidence for the potential occurrence of associated off-target effects due to the use of genome-editing in model plants as well as in crops produced for agricultural production?</h3><p>In total, 252 studies from 161 articles were identified in which the occurrence of off-target effects was assessed. Table<a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/10.1186/s13750-019-0171-5#Tab8">8</a> maps the number of analyzed off-target effects for different genome-editing techniques and different plant species. Most of the off-target analyses were conducted in CRISPR/Cas studies (n=228) followed by TALENs studies (n=9), BE studies (n=9) and ZFN studies (n=4). Solely in one ODM and in one MN study off-target effects were investigated. Most off-target effects were analyzed in rice (n=93), followed by tomato (n=28), <i>Arabidopsis</i> (n=23) and soybean (n=15) (Table<a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/10.1186/s13750-019-0171-5#Tab8">8</a>).</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-8"><figure><figcaption class="c-article-table__figcaption"><b id="Tab8" data-test="table-caption">Table8 Overview of off-target studies in relation to different genome-editing techniques and plant species (January 1996May 2018)</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/10.1186/s13750-019-0171-5/tables/8"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<h3 class="c-article__sub-heading" id="Sec30">Off-target effects considered for CRISPR/Cas-systems</h3><p>More than 90% of the studies, in which off-target effects were assessed, were conducted with CRISPR/Cas. Figure<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig8">8</a> provides an overview of the applied approaches to identify off-target effects. In total, 228 CRISPR/Cas studies dealt with the analysis of off-target effects. 205 studies predicted potential off-target sites and 195 of these identified potential off-target sites. Solely in 188 studies, these potential off-target sites were further analyzed for the occurrence of off-target effects using biased detection methods. In addition, 23 studies with already known potential off-target sites were assessed using biased detection methods. So, in total, 211 studies analyzed potential off-target sites using biased detection methods. Solely, nine studies searched for off-target mutations in a completely unrestricted way using unbiased detection methods. An overview of all identified CRISPR/Cas studies, in which off-target effects were addressed, is provided in Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM6">6</a> including all extracted data.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-8"><figure><figcaption><b id="Fig8" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.8</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/8" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig8_HTML.png?as=webp"></source><img aria-describedby="figure-8-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig8_HTML.png" alt="figure8" loading="lazy" width="685" height="679" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-8-desc"><p>Overview of the used approaches to assess off-target effects in CRISPR/Cas studies. All numbers represent the amount of studies conducted for different approaches. In some studies, several approaches were used to analyze off-targets</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/8" data-track-dest="link:Figure8 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<h3 class="c-article__sub-heading" id="Sec31">Prediction of potential off-target sites by CRISPR/Cas</h3><p>Different prediction tools can be used to identify potential off-target sites. All of them have in common that based on sequence alignment programs DNA-sequences are identified in which unintended mutations could occur due to high similarity between the targeted sequence and the potential off-target site. However, the prediction of a potential off-target site is not equated with a real off-target mutation. It has to be shown in a follow up step by verifying the potential off-target site using biased detection methods. 205 CRISPR/Cas studies searched for potential off-target sites. As shown in Fig.<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig9">9</a>, many different prediction tools were used to identify these sites. Mainly, three tools were used to predict potential off-target sites. BLAST was used 54 times, CRISPR-P 51 times and CasOFF-Finder 31 times. 12 other prediction tools were used in 31 studies (for detailed information see Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM6">6</a>). 41 studies did not provide any details about the used tool(s). Three studies used two different prediction tools to identify potential off-target sites.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-9"><figure><figcaption><b id="Fig9" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.9</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/9" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig9_HTML.png?as=webp"></source><img aria-describedby="figure-9-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig9_HTML.png" alt="figure9" loading="lazy" width="685" height="379" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-9-desc"><p>Tools used to predict potential off-target sites in 205 CRISPR/Cas-studies (1996May 2018). Three studies employed two tools</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/9" data-track-dest="link:Figure9 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<p>The number of predicted off-target sites varies widely between studies from zero to 4265. Several reasons for this broad heterogeneity exist, which were not extracted in detail within this map but will be elucidated in the discussion.</p><h3 class="c-article__sub-heading" id="Sec32">Detection of off-target effectsbiased</h3><p>Targeted sequencing to pre-selected sites was applied in 211 CRISPR/Cas studies to detect off-target effects. As shown in Fig.<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig8">8</a>, the predicted off-target sites were frequently analyzed for the occurrence of off-target effects using biased detection methods. In a few studies, potential off-target sites were already known and analyzed without using sequence alignment programs a priori. Figure<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig10">10</a> displays the different detection methods applied to identify off-target mutations. In the large majority of studies, off-target effects were detected using a PCR followed by sequencing (n=137). Only a few studies used the detection methods RE-PCR assay (n=16), targeted deep sequencing (n=15), enzyme mismatch cleavage assay (n=13) and CAPS analyses (n=13). In 11 studies, no information about the used detection method was provided.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-10"><figure><figcaption><b id="Fig10" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.10</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/10" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig10_HTML.png?as=webp"></source><img aria-describedby="figure-10-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig10_HTML.png" alt="figure10" loading="lazy" width="685" height="329" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-10-desc"><p>Biased detection methods used to identify off-target effects in CRISPR/Cas studies. Three studies employed two tools. <i>PCR</i> polymerase chain reaction, <i>REPCR</i> restriction enzymepolymerase chain reaction, <i>CAPS</i> cleaved amplified polymorphic sequences</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/10" data-track-dest="link:Figure10 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<p>Taking all CRISPR/Cas studies together, 1738 different potential off-target sites were analyzed using targeted sequencing. Off-target effects were identified in 55 of these sites, indicating that in around 3% of the analyzed sequences off-target mutations were detected. In another six studies, no information was provided about the amount of analyzed off-target sites but off-target mutations were not identified either.</p><p>Considering the different plant species, most of the CRISPR/Cas studies using biased detection methods were conducted in rice (n=77), followed by tomato (n=23), <i>Arabidopsis</i> (n=21), different moss species (n=13) and soybean (n=12) (Table<a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/10.1186/s13750-019-0171-5#Tab9">9</a>). In rice, a total amount of 291 potential off-target sites were analyzed and in 25 of these sites, off-target mutations were detected. In contrast, studies conducted in tomatoes solely reported the identification of one off-target mutation when analyzing 222 potential off-target sites.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-9"><figure><figcaption class="c-article-table__figcaption"><b id="Tab9" data-test="table-caption">Table9 Overview of the amount of studies analyzing potential off-target sites, the amount of analyzed potential off-target sites and the amount of identified off-target effects using biased detection methods in different plant species</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/10.1186/s13750-019-0171-5/tables/9"><span>Full size table</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<p>In one study, a different approach was chosen to assess the occurrence of off-target effects [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 175" title="Meng X, Hu X, Liu Q, Song X, Gao C, Li J, Wang K. Robust genome editing of CRISPRCas9 at NAG PAMs in rice. Sci China Life Sci. 2018;61:1225. &#xA;https://doi.org/10.1007/s11427-017-9247-9&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR175" id="ref-link-section-d69214e4427">175</a>]. In this study, a series of mismatches were introduced at the sgRNA followed by analyzing whether the targeted sequence was successfully mutated despite the mismatch(es) between the sgRNA and the targeted sequence. Additionally, in this study, the off-target patterns between two PAMs (NGG and NAG) were compared. 22 times the sgRNA was designed in a way that it contained one mismatch to the targeted sequence. 15 of these altered sgRNA induced a DSB at the targeted sequence. 14 times the sgRNA contained two mismatches to the targeted sequence and four of these sgRNA induced a DSB in the targeted sequence. Moreover, eight times the sgRNA contained three mismatches. In these cases, no mutation was identified in the targeted sequence [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 175" title="Meng X, Hu X, Liu Q, Song X, Gao C, Li J, Wang K. Robust genome editing of CRISPRCas9 at NAG PAMs in rice. Sci China Life Sci. 2018;61:1225. &#xA;https://doi.org/10.1007/s11427-017-9247-9&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR175" id="ref-link-section-d69214e4430">175</a>]. According to the prediction of off-target effects, the summary provided here does not allow any conclusions to be drawn due to broad heterogeneity.</p><h3 class="c-article__sub-heading" id="Sec33">Heterogeneity in CRISPR/Cas-studies regarding the evidence how to predict and detect potential off-target effects</h3><p>The number of potential off-target sites called identified by the respective authors varies widely between zero and 4265. Several reasons were identified that could explain this broad heterogeneity:</p><ol class="u-list-style-none"><li><span class="u-custom-list-number">i.</span><p>In total, potential off-target sites were investigated in over 30 different plant species and subspecies. The different genome sizes and the different number of chromosome sets of the individual plants vary widely which influence the number of potential off-target sites.</p></li><li><span class="u-custom-list-number">ii.</span><p>To identify potential off-target sites in CRISPR/Cas-studies 15 different prediction tools were applied.</p></li><li><span class="u-custom-list-number">iii.</span><p>For the detection of off-target effects, various methods have been used, but all of them show their specific advantages and disadvantages and could affect the occurrence of off-target effects [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Martin F, Snchez-Hernndez S, Gutirrez-Guerrero A, Pinedo-Gomez J, Benabdellah K. Biased and unbiased methods for the detection of off-target cleavage by CRISPR/Cas9: an overview. Int J Mol Sci. 2016. &#xA;https://doi.org/10.3390/ijms17091507&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR26" id="ref-link-section-d69214e4458">26</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Zischewski J, Fischer R, Bortesi L. Detection of on-target and off-target mutations generated by CRISPR/Cas9 and other sequence-specific nucleases. Biotechnol Adv. 2017;35:95104. &#xA;https://doi.org/10.1016/j.biotechadv.2016.12.003&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR32" id="ref-link-section-d69214e4461">32</a>].</p></li><li><span class="u-custom-list-number">iv.</span><p>The number of hypothetically tolerated mismatches between the target sequence and the potential off-target sites which is an exercise in combinatorics. Ali et al. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 176" title="Ali Z, Abul-faraj A, Piatek M, Mahfouz MM. Activity and specificity of TRV-mediated gene editing in plants. Plant Signal Behav. 2015;10:e1044191. &#xA;https://doi.org/10.1080/15592324.2015.1044191&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR176" id="ref-link-section-d69214e4469">176</a>] identified a total amount of 4265 potential off-target sites in the model organism <i>Nicotiana benthamiana</i> for a CRISPR/Cas9 sgRNA. To identify candidate off-target sites, the genome was screened allowing one to seven mismatches to the target sequence. Hence, the more mismatches are tolerated in prediction the higher is the number of potential off-target sites (in the paper: one, two or three mismatches tolerated: No potential off-target sites, four mismatches tolerated: One potential off-target site, five mismatches tolerated: 60 potential off-target sites, six mismatches tolerated: 515 potential off-target sites, seven mismatches tolerated: 3689 potential off-target sites). This indicates that the number of potential off-target sites strongly depends on the number of hypothetically tolerated mismatches. Throughout the available literature, the number of tolerated mismatches predetermined by the researchers was very heterogeneous ranging up to 13 mismatches [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 177" title="Shen C, Que Z, Xia Y, Tang N, Li D, He R, Cao M. Knock out of the annexin gene OsAnn3 via CRISPR/Cas9-mediated genome editing decreased cold tolerance in rice. J Plant Biol. 2017;60:53947. &#xA;https://doi.org/10.1007/s12374-016-0400-1&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR177" id="ref-link-section-d69214e4475">177</a>].</p></li><li><span class="u-custom-list-number">v.</span><p>Individual studies also deviate between different structural assumptions when determining potential off-target sites. In most studies, a potential off-target was only counted as such if a PAM followed the potential off-target site. However, in some studies, potential off-target sites were assigned as such without being followed by a PAM (e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Hu B, Li D, Liu X, Qi J, Gao D, Zhao S, et al. Engineering non-transgenic gynoecious cucumber using an improved transformation protocol and optimized CRISPR/Cas9 system. Mol Plant. 2017;10:15758. &#xA;https://doi.org/10.1016/j.molp.2017.09.005&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR44" id="ref-link-section-d69214e4483">44</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 178" title="Tian S, Jiang L, Gao Q, Zhang J, Zong M, Zhang H, et al. Efficient CRISPR/Cas9-based gene knockout in watermelon. Plant Cell Rep. 2017;36:399406. &#xA;https://doi.org/10.1007/s00299-016-2089-5&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR178" id="ref-link-section-d69214e4486">178</a>]), although at these sites no DSB can be induced with the specific CRISPR-nucleases used.</p></li><li><span class="u-custom-list-number">vi.</span><p>In some studies, the sgRNA was selected taking into account that no potential off-target effect should occur. Care was taken that, apart from the target sequence, no other sites in the genome possess a similar sequence that could result in an off-target mutation. Thus, the predicted number of potential off-target sites is lower or zero in these studies compared to studies, in which potential off-target sites were not considered a priori in the sequence selection.</p></li></ol>
<h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec34">Detection of off-target effectsunbiased</h4><p>Nine CRISPR/Cas studies were identified using whole genome sequencing (WGS) as an unbiased detection method to identify genome-wide off-target effects (see Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM6">6</a>). No off-target mutations were detected in any of these studies. One study compared both biased and unbiased detection methods. While using unbiased detection methods, no off-target effects were detected, but biased methods detected one off-target [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 179" title="Zhang H, Zhang J, Wei P, Zhang B, Gou F, Feng Z, et al. The CRISPR/Cas9 system produces specific and homozygous targeted gene editing in rice in one generation. Plant Biotechnol J. 2014;12:797807. &#xA;https://doi.org/10.1111/pbi.12200&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR179" id="ref-link-section-d69214e4505">179</a>]. An explanation for this could be that WGS is able to detect higher frequency off-target effects only, but lacks the sensibility required to detect off-target mutations in bulk population [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Martin F, Snchez-Hernndez S, Gutirrez-Guerrero A, Pinedo-Gomez J, Benabdellah K. Biased and unbiased methods for the detection of off-target cleavage by CRISPR/Cas9: an overview. Int J Mol Sci. 2016. &#xA;https://doi.org/10.3390/ijms17091507&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR26" id="ref-link-section-d69214e4508">26</a>].</p><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec35">Off-target effects considered for TALENs systems</h4><p>In the period until May 2018, nine TALENs studies were identified addressing off-target effects. Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM6">6</a> provides an overview of these studies including all extracted data according to the systematic map protocol. Figure<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig11">11</a> maps the different approaches used to analyze off-target effects. Five studies predicted potential off-target sites. Two times, the TAL Effector Nucleotide Targeter 2.0 was used and one time each of the tools PROGNONS, kmasker and Arabidopsis Information Resource PatMatch. The number of identified potential off-target sites varies widely between zero and 18. In one study, no precise information was given and it was just indicated that many potential off-target sites were identified. The four studies that identified potential off-target sites investigated these for the occurrence of off-target effects using biased detection methods. In addition, three studies with already known potential off-target sites assessed these sites. Different tools were used to examine whether off-target effects occurred. Two studies used enzyme mismatch cleavage assay and one study each PCR+Sequencing, PCR+CAPS and RE-PCR assay. In two studies, no detailed information about the applied detection method was provided. In total, 31 potential off-target sites were analyzed for the occurrence of off-target effects and one of these sites contained an off-target mutation. In one study, an unbiased search for off-target effects was conducted [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 180" title="Forner J, Pfeiffer A, Langenecker T, Manavella PA, Manavella P, Lohmann JU. Germline-transmitted genome editing in Arabidopsis thaliana using TAL-effector-nucleases. PLoS ONE. 2015;10:e0121056. &#xA;https://doi.org/10.1371/journal.pone.0121056&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR180" id="ref-link-section-d69214e4525">180</a>]. Using Whole Genome Sequencing (WGS), three off-target mutations were identified which were not present in the wild-type sample. However, the off-target sequences showed no similarity to the TALENs binding sites. Therefore, the occurrence of these mutations cannot be ruled out to be spontaneous ones or sequencing errors [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 180" title="Forner J, Pfeiffer A, Langenecker T, Manavella PA, Manavella P, Lohmann JU. Germline-transmitted genome editing in Arabidopsis thaliana using TAL-effector-nucleases. PLoS ONE. 2015;10:e0121056. &#xA;https://doi.org/10.1371/journal.pone.0121056&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR180" id="ref-link-section-d69214e4528">180</a>].</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-11"><figure><figcaption><b id="Fig11" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.11</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/11" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig11_HTML.png?as=webp"></source><img aria-describedby="figure-11-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig11_HTML.png" alt="figure11" loading="lazy" width="685" height="699" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-11-desc"><p>Overview of the used approaches to assess off-target effects in TALENs studies. All numbers represent the amount of studies conducted for different approaches</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/11" data-track-dest="link:Figure11 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
<h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec36">Off-target effects considered for Zinc-Finger Nucleases systems</h4><p>Four studies dealt with the analyses of off-target effects when applying ZFN in plants. Detailed information is provided in Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM6">6</a>. All studies identified putative off-target sites that are most closely related to the target sequences. Two times, the database PLantGDB was used to identify potential off-target sites and two times no detailed information was provided. In total, 10 potential off-target sites were detected which were then screened for the occurrence of off-target effects. No off-target mutations were detected in any of these analyzed, potential off-target sites.</p><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec37">Off-target effects considered for Meganuclease systems</h4><p>Solely one MN study was identified that analyzed off-target effects (Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM6">6</a>). In this study, a homologous sequence to the target one was identified that differs solely in two nucleotides. However, when screening this potential off-target site no modification was identified.</p><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec38">Off-target effects considered for Oligonucleotide-Directed Mutagenesis systems</h4><p>One study addressed the occurrence of off-target effects in rice using the ODM technique (Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM6">6</a>). Beside the targeted Acetolactate synthase (ALS) sequence, the whole coding region of ALS gene was sequenced but no further mutation was detected.</p><h4 class="c-article__sub-heading c-article__sub-heading--small c-article__sub-heading--light" id="Sec39">Off-target effects considered for base editing systems</h4><p>In total, nine BE studies analyzed the occurrence of off-target effects (Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM6">6</a>). However, one study could only rarely be evaluated due to language barriers [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 181" title="Ren B, Yan F, Kuang Y, Li N, Zhang D, Lin H, Zhou H. Specificity and inheritance of rBE3 and rBE4 endonuclease-induced gene modifications in rice. Chin J Biotechnol 2007;33(10):177685." href="/articles/10.1186/s13750-019-0171-5#ref-CR181" id="ref-link-section-d69214e4584">181</a>]. The approach used to identify off-target effects was similar in all studies (Fig.<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/10.1186/s13750-019-0171-5#Fig12">12</a>). In a first step, potential off-target sites were predicted using different tools (2 CRISPR-GE tool, 2 CRISPR-P, 1 CasOff-Finder, 3 no information). According to the other genome-editing techniques, the number of potential off-target sites varies widely between one and nine. In a second step, all predicted potential off-target sites were sequenced for the occurrence of off-target effects using PCR+sequencing method (n=6), targeted deep sequencing (n=2) or enzyme mismatch cleavage assay (n=1). One off-target mutation was identified.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-12"><figure><figcaption><b id="Fig12" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig.12</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/12" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig12_HTML.png?as=webp"></source><img aria-describedby="figure-12-desc" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_Fig12_HTML.png" alt="figure12" loading="lazy" width="597" height="1144" /></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-12-desc"><p>Overview of the used approaches to assess off-target effects in BE studies. All numbers represent the amount of studies conducted for different approaches</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/10.1186/s13750-019-0171-5/figures/12" data-track-dest="link:Figure12 Full size image" rel="nofollow"><span>Full size image</span><svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-chevron-right"></use></svg></a></div></figure></div>
</div></div></section><section aria-labelledby="Sec40"><div class="c-article-section" id="Sec40-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec40">Discussion</h2><div class="c-article-section__content" id="Sec40-content"><h3 class="c-article__sub-heading" id="Sec41">Market oriented applications of genome-editing</h3><p>This map documents the state of evidence for the application of genome-editing as a new tool for the modification of plant traits and the associated potential occurrence of off-target effects. The publication rate of primary research has risen sharply since the CRISPR/Cas technique was first applied in plants in 2013. It is worth mentioning that in total primary studies from 33 countries were identified but nearly three quarter of these studies originate from either China (40%) or the USA (33%). For comparison, Japan and Germany published around 6% of the studies each and no other country contributed more than 2% of the total. Summarizing the number of published studies by continent, more than 50% of the studies were conducted in an Asian country (53%), around one-third in North American countries (34%) and only 13% in European countries. As the genome-editing techniques, especially CRISPR/Cas, were just recently developed, the large majority of the existing applications represent basic research. Nevertheless, almost 100 different applications aimed to produce beneficial agricultural traits in 28 different agricultural crops. We determined that the majority of such market-oriented applications have been carried out in economically important crops such as maize, rice, wheat and soybeans, but less economically important crops such as cucumber, lettuce, peanut or grapefruit have been worked on as well. The market-oriented applications address several breeding objectives including yield improvement, improved growth characteristics, improved food and feed quality, tolerances to biotic and abiotic stress, herbicide tolerance and industrial utilization. This indicates that genome-editing is able to address beneficial traits for the agricultural value chain and could contribute to food security and the environmental management. To confirm this hypothesis a systematic review would be required to quantify such effects.</p><h3 class="c-article__sub-heading" id="Sec42">Off-target effects of genome-editing</h3><p>252 studies investigated the occurrence of off-target effects following a genome-editing application in plants. Most of these studies were performed with the genome-editing technique CRISPR/Cas (n=228), whereas only a few studies used TALENs (n=9), BE (n=9), ZFN (n=4), MN (n=1) or ODM (n=1). Reasons for these findings are that the total number of studies in which CRISPR/Cas was applied is much larger compared to the other genome-editing techniques. Additionally, CRISPR/Cas is more susceptible to off-target effects compared to other nuclease techniques such as ZFN and TALENs, as it works as a monomer, whereas ZFN and TALENs work as dimers [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 182" title="Khandagale K, Nadaf A. Genome editing for targeted improvement of plants. Plant Biotechnol Rep. 2016;10:32743. &#xA;https://doi.org/10.1007/s11816-016-0417-4&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR182" id="ref-link-section-d69214e4620">182</a>]. In addition, the sgRNA used in CRISPR/Cas-applications is able to tolerate several mismatches leading to the induction of a DSB at a site in the DNA that is similar but not identical to the targeted sequence [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Zischewski J, Fischer R, Bortesi L. Detection of on-target and off-target mutations generated by CRISPR/Cas9 and other sequence-specific nucleases. Biotechnol Adv. 2017;35:95104. &#xA;https://doi.org/10.1016/j.biotechadv.2016.12.003&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR32" id="ref-link-section-d69214e4623">32</a>]. In most CRISPR/Cas studies biased detection methods were used, meaning that potential off-target sites were first predicted using bioinformatics programs followed by targeted sequencing of these sites for the occurrence of off-target effects. The biased method suggests a very broad data basis, but as shown in the result section, individual studies are very heterogeneous in their structure and design. Heterogeneity was identified regarding the plant species, the CRISPR-variant, the prediction tools and detection methods used, the amount of tolerated mismatches and the chosen sgRNA. In order to allow any conclusions to be drawn about the occurrence of off-target effects a more in-depth analysis e.g. by a systematic review is mandatory.</p><h3 class="c-article__sub-heading" id="Sec43">Knowledge gaps</h3><p>Regarding secondary question 2, several topics have been identified representing knowledge gaps where no studies or only a small number of studies exist. A knowledge gap exists for the analysis of off-target effects in TALENs, ZFN, MN, ODM and BE. Only nine studies have been identified analyzing the occurrence of off-target effects in TALENs and BE studies. The amount of off-target studies for ZFN (n=4), ODM (n=1) and MN (n=1) were even lower. For a more in-depth off-target analysis regarding these techniques, further research and more primary studies are needed. However, the CRISPR/Cas technique is applied much more frequently because of efficiency, time saving and cost-effectiveness compared to TALENs, ZFN, ODM and MN [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 182" title="Khandagale K, Nadaf A. Genome editing for targeted improvement of plants. Plant Biotechnol Rep. 2016;10:32743. &#xA;https://doi.org/10.1007/s11816-016-0417-4&#xA;&#xA;." href="/articles/10.1186/s13750-019-0171-5#ref-CR182" id="ref-link-section-d69214e4634">182</a>]. Therefore, it is questionable what additional research effort is justified. BE differs as it has great potential to support plant breeding, though it is not broadly established yet. It is based on the CRISPR/Cas9 system but enables exchanging individual base pairs at specific sites without inducing a DSB. Our map shows that only a few BE studies addressed the occurrence of off-target effects so far. The likely reason is that BE is a young method in plants, and its first publication was in 2017. Further research about BE including the analysis of off-target effects is highly recommended. Another knowledge gap affecting all genome-editing techniques is the genome-wide analysis of off-target effects in a completely unrestricted way. Only nine studies addressed this aspect using the CRISPR/Cas technique and one using the TALENs technique. No study analyzed off-target effects using unbiased detection methods in ZFN, ODM, MN and BE studies.</p><h3 class="c-article__sub-heading" id="Sec44">Knowledge clusters</h3><p>The analysis of off-target effects in CRISPR/Cas studies using biased detection methods represents a knowledge cluster. This cluster is amenable for a critical appraisal serving the interest of researchers and decision-makers. The following parameters could affect the occurrence of off-targets and are worth to be analyzed in-depth:</p><ul class="u-list-style-bullet"><li><p>Genome composition, size and ploidy level of different plant species.</p></li><li><p>Amount of tolerated mismatches at the potential off-target sites.</p></li><li><p>Quality of available biased detection methods.</p></li><li><p>CRISPR variants (e.g. Nickase).</p></li><li><p>Chosen sgRNA.</p></li><li><p>Methodology of SDN delivery.</p></li></ul>
<h3 class="c-article__sub-heading" id="Sec45">Limitations of the systematic map</h3><p>References were only searched in German and English language. Publications in an Asian language, in which only the abstract was available in English, were also identified but could not be included in this map for the extraction of detailed data. Since most of the studies were found in scientific journals, a bias in the pool of articles found is that publications from countries that probably use genome editing but do not publish in either English or German language are not represented. For example, it can be assumed that in South American countries, considerably more research is done than the identified literature suggests. Identifying grey literature as well as manually identifying scientific journals publishing in the local language(s) was also not possible due to language barriers.</p><p>Additionally, the full text of 104 articles, that have been rated as relevant on title/abstract level, were un-retrievable and were therefore not included in the systematic map (see Additional file <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/10.1186/s13750-019-0171-5#MOESM3">3</a>).</p><p>It has been demonstrated that the individual studies addressing off-target effects differ widely in design and implementation. Therefore, no reliable conclusions about the occurrence of off-target effects can be drawn based on the results of this map. A critical appraisal of the individual studies in form of a systematic review is recommended.</p></div></div></section><section aria-labelledby="Sec46"><div class="c-article-section" id="Sec46-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec46">Conclusions</h2><div class="c-article-section__content" id="Sec46-content"><h3 class="c-article__sub-heading" id="Sec47">Implications for policy/management</h3><p>This systematic map identified substantial bodies of evidence regarding the applications of different genome-editing techniques in plants as well as the occurrence of off-target effects. Until May 2018, almost 100 market-oriented applications were identified including improved food and feed quality, yield improvement, altered growth characteristics, resistance against biotic and abiotic stress, herbicide tolerance and industrial utilization. The wide range of different applications addressing all parts in the agricultural value chain and the application in many different plant species indicates that genome-editing became a promising tool to breed varieties that are better adapted to the needs of agriculture and to enable a valuable contribution to food security and the environment.</p><p>A decisive factor that impacts the use of genome-editing is the acceptance by consumers and retailers for products derived from innovative plant breeding methods. A prominent point of criticism in this context is the occurrence of off-target effects. Since plant breeding is the contextual background, it is worth to compare the occurrence of off-target effects in the context of naturally occurring mutations and routinely used breeding techniques like regular crossing or undirected mutagenesis using chemical mutagens or irradiation [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017." href="/articles/10.1186/s13750-019-0171-5#ref-CR10" id="ref-link-section-d69214e4700">10</a>].</p><h3 class="c-article__sub-heading" id="Sec48">Implication for research</h3><p>Results of this systematic map determined that different approaches were used to analyze off-target effects depending on the plant species and with regard to the CRISPR-variant, the prediction tools and detection methods used, the amount of tolerated mismatches and the chosen sgRNA. A critical appraisal in the course of a systematic review could help to identify parameters in order to further reduce the occurrence of off-target effects. The identified knowledge cluster for the detection of off-target effects in CRISPR-studies using biased detection methods would be suitable to address this.</p><p>The results of this map further identified a knowledge gap regarding the analysis of off-target effects using unbiased detection tools. To increase this data basis and to evaluate whether an unbiased off-target analysis has an added value compared to biased of target analysis, more studies should assess this aspect, in future.</p></div></div></section>
                    <section aria-labelledby="availability-of-data-and-materials"><div class="c-article-section" id="availability-of-data-and-materials-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="availability-of-data-and-materials">Availability of data and materials</h2><div class="c-article-section__content" id="availability-of-data-and-materials-content">
              
              <p>The datasets generated and/or analyzed during the current study are available at the ELSA-GEA homepage <a href="https://www.dialog-gea.de/de/service/repositorium">https://www.dialog-gea.de/de/service/repositorium</a>.</p>
            </div></div></section><section aria-labelledby="Bib1"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references"><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><meta itemprop="author" content="H. Becker, " /><meta itemprop="datePublished" content="2011" /><meta itemprop="headline" content="Becker H. Pflanzenzchtung. 2nd ed. Stuttgart: UTB; 2011." /><span class="c-article-references__counter">1.</span><p class="c-article-references__text" id="ref-CR1">Becker H. Pflanzenzchtung. 2nd ed. Stuttgart: UTB; 2011.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 1 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Pflanzenz%C3%BCchtung&amp;publication_year=2011&amp;author=Becker%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="P. Ronald, " /><meta itemprop="datePublished" content="2011" /><meta itemprop="headline" content="Ronald P. Plant genetics, sustainable agriculture and global food security. Genetics. 2011;188:1120. https://" /><span class="c-article-references__counter">2.</span><p class="c-article-references__text" id="ref-CR2">Ronald P. Plant genetics, sustainable agriculture and global food security. Genetics. 2011;188:1120. <a href="https://doi.org/10.1534/genetics.111.128553">https://doi.org/10.1534/genetics.111.128553</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1534%2Fgenetics.111.128553" aria-label="View reference 2">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 2 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Plant%20genetics%2C%20sustainable%20agriculture%20and%20global%20food%20security&amp;journal=Genetics&amp;doi=10.1534%2Fgenetics.111.128553&amp;volume=188&amp;pages=11-20&amp;publication_year=2011&amp;author=Ronald%2CP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="FAOCommittee on Agriculture (17th session). 06.03.2003. http://www.fao.org/docrep/meeting/006/y8704e.htm. Acc" /><span class="c-article-references__counter">3.</span><p class="c-article-references__text" id="ref-CR3">FAOCommittee on Agriculture (17th session). 06.03.2003. <a href="http://www.fao.org/docrep/meeting/006/y8704e.htm">http://www.fao.org/docrep/meeting/006/y8704e.htm</a>. Accessed 25 Sept 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="K. Xie, S. Wu, Z. Li, Y. Zhou, D. Zhang, Z. Dong, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Xie K, Wu S, Li Z, Zhou Y, Zhang D, Dong Z, et al. Map-based cloning and characterization of Zea mays male ste" /><span class="c-article-references__counter">4.</span><p class="c-article-references__text" id="ref-CR4">Xie K, Wu S, Li Z, Zhou Y, Zhang D, Dong Z, et al. Map-based cloning and characterization of Zea mays male sterility33 (ZmMs33) gene, encoding a glycerol-3-phosphate acyltransferase. Theor Appl Genet. 2018;131:136378. <a href="https://doi.org/10.1007/s00122-018-3083-9">https://doi.org/10.1007/s00122-018-3083-9</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXkslagt70%253D" aria-label="View reference 4 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs00122-018-3083-9" aria-label="View reference 4">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 4 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Map-based%20cloning%20and%20characterization%20of%20Zea%20mays%20male%20sterility33%20%28ZmMs33%29%20gene%2C%20encoding%20a%20glycerol-3-phosphate%20acyltransferase&amp;journal=Theor%20Appl%20Genet&amp;doi=10.1007%2Fs00122-018-3083-9&amp;volume=131&amp;pages=1363-1378&amp;publication_year=2018&amp;author=Xie%2CK&amp;author=Wu%2CS&amp;author=Li%2CZ&amp;author=Zhou%2CY&amp;author=Zhang%2CD&amp;author=Dong%2CZ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="B. Phalan, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Phalan B. What have we learned from the land sparing-sharing model? Sustainability. 2018;10:1760. https://doi." /><span class="c-article-references__counter">5.</span><p class="c-article-references__text" id="ref-CR5">Phalan B. What have we learned from the land sparing-sharing model? Sustainability. 2018;10:1760. <a href="https://doi.org/10.3390/su10061760">https://doi.org/10.3390/su10061760</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3390%2Fsu10061760" aria-label="View reference 5">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 5 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=What%20have%20we%20learned%20from%20the%20land%20sparing-sharing%20model%3F&amp;journal=Sustainability&amp;doi=10.3390%2Fsu10061760&amp;volume=10&amp;publication_year=2018&amp;author=Phalan%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="JR. Stevenson, N. Villoria, D. Byerlee, T. Kelley, M. Maredia, " /><meta itemprop="datePublished" content="2013" /><meta itemprop="headline" content="Stevenson JR, Villoria N, Byerlee D, Kelley T, Maredia M. Green Revolution research saved an estimated 18 to 2" /><span class="c-article-references__counter">6.</span><p class="c-article-references__text" id="ref-CR6">Stevenson JR, Villoria N, Byerlee D, Kelley T, Maredia M. Green Revolution research saved an estimated 18 to 27 million hectares from being brought into agricultural production. Proc Natl Acad Sci USA. 2013;110:83638. <a href="https://doi.org/10.1073/pnas.1208065110">https://doi.org/10.1073/pnas.1208065110</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.1208065110" aria-label="View reference 6">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 6 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Green%20Revolution%20research%20saved%20an%20estimated%2018%20to%2027%20million%20hectares%20from%20being%20brought%20into%20agricultural%20production&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;doi=10.1073%2Fpnas.1208065110&amp;volume=110&amp;pages=8363-8368&amp;publication_year=2013&amp;author=Stevenson%2CJR&amp;author=Villoria%2CN&amp;author=Byerlee%2CD&amp;author=Kelley%2CT&amp;author=Maredia%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="DD. Songstad, JF. Petolino, DF. Voytas, NA. Reichert, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Songstad DD, Petolino JF, Voytas DF, Reichert NA. Genome editing of plants. Crit Rev Plant Sci. 2017;36:123. " /><span class="c-article-references__counter">7.</span><p class="c-article-references__text" id="ref-CR7">Songstad DD, Petolino JF, Voytas DF, Reichert NA. Genome editing of plants. Crit Rev Plant Sci. 2017;36:123. <a href="https://doi.org/10.1080/07352689.2017.1281663">https://doi.org/10.1080/07352689.2017.1281663</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1080%2F07352689.2017.1281663" aria-label="View reference 7">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 7 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20editing%20of%20plants&amp;journal=Crit%20Rev%20Plant%20Sci&amp;doi=10.1080%2F07352689.2017.1281663&amp;volume=36&amp;pages=1-23&amp;publication_year=2017&amp;author=Songstad%2CDD&amp;author=Petolino%2CJF&amp;author=Voytas%2CDF&amp;author=Reichert%2CNA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="International Atomic Energy Agency (IAEA). Mutant variety database. https://mvd.iaea.org/#!Search?page=1&amp;size=" /><span class="c-article-references__counter">8.</span><p class="c-article-references__text" id="ref-CR8">International Atomic Energy Agency (IAEA). Mutant variety database. <a href="https://mvd.iaea.org/#!Search%3fpage%3d1%26size%3d15%26sortby%3dName%26sort%3dASC">https://mvd.iaea.org/#!Search?page=1&amp;size=15&amp;sortby=Name&amp;sort=ASC</a>. Accessed 25 Sept 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><meta itemprop="author" content="RHJ. Schlegel, " /><meta itemprop="datePublished" content="2009" /><meta itemprop="headline" content="Schlegel RHJ. Dictionary of plant breeding. 2nd ed. Boca Raton: CRC Press; 2009." /><span class="c-article-references__counter">9.</span><p class="c-article-references__text" id="ref-CR9">Schlegel RHJ. Dictionary of plant breeding. 2nd ed. Boca Raton: CRC Press; 2009.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 9 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Dictionary%20of%20plant%20breeding&amp;publication_year=2009&amp;author=Schlegel%2CRHJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017." /><span class="c-article-references__counter">10.</span><p class="c-article-references__text" id="ref-CR10">SAM High Level Group of Scientific Advisors. New techniques in agricultural biotechnology. 2017.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="KA. Barton, AN. Binns, AJM. Matzke, M-D. Chilton, " /><meta itemprop="datePublished" content="1983" /><meta itemprop="headline" content="Barton KA, Binns AN, Matzke AJM, Chilton M-D. Regeneration of intact tobacco plants containing full length cop" /><span class="c-article-references__counter">11.</span><p class="c-article-references__text" id="ref-CR11">Barton KA, Binns AN, Matzke AJM, Chilton M-D. Regeneration of intact tobacco plants containing full length copies of genetically engineered T-DNA, and transmission of T-DNA to R1 progeny. Cell. 1983;32:103343. <a href="https://doi.org/10.1016/0092-8674(83)90288-X">https://doi.org/10.1016/0092-8674(83)90288-X</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADyaL3sXktVWiu7Y%253D" aria-label="View reference 11 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2F0092-8674%2883%2990288-X" aria-label="View reference 11">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 11 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Regeneration%20of%20intact%20tobacco%20plants%20containing%20full%20length%20copies%20of%20genetically%20engineered%20T-DNA%2C%20and%20transmission%20of%20T-DNA%20to%20R1%20progeny&amp;journal=Cell&amp;doi=10.1016%2F0092-8674%2883%2990288-X&amp;volume=32&amp;pages=1033-1043&amp;publication_year=1983&amp;author=Barton%2CKA&amp;author=Binns%2CAN&amp;author=Matzke%2CAJM&amp;author=Chilton%2CM-D">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="L. Herrera-Estrella, A. Depicker, M. Montagu, J. Schell, " /><meta itemprop="datePublished" content="1983" /><meta itemprop="headline" content="Herrera-Estrella L, Depicker A, van Montagu M, Schell J. Expression of chimaeric genes transferred into plant " /><span class="c-article-references__counter">12.</span><p class="c-article-references__text" id="ref-CR12">Herrera-Estrella L, Depicker A, van Montagu M, Schell J. Expression of chimaeric genes transferred into plant cells using a Ti-plasmid-derived vector. Nature. 1983;303:20913. <a href="https://doi.org/10.1038/303209a0">https://doi.org/10.1038/303209a0</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADyaL3sXktFOgtrc%253D" aria-label="View reference 12 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2F303209a0" aria-label="View reference 12">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 12 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Expression%20of%20chimaeric%20genes%20transferred%20into%20plant%20cells%20using%20a%20Ti-plasmid-derived%20vector&amp;journal=Nature&amp;doi=10.1038%2F303209a0&amp;volume=303&amp;pages=209-213&amp;publication_year=1983&amp;author=Herrera-Estrella%2CL&amp;author=Depicker%2CA&amp;author=Montagu%2CM&amp;author=Schell%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="G. Govindan, S. Ramalingam, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Govindan G, Ramalingam S. Programmable site-specific nucleases for targeted genome engineering in higher eukar" /><span class="c-article-references__counter">13.</span><p class="c-article-references__text" id="ref-CR13">Govindan G, Ramalingam S. Programmable site-specific nucleases for targeted genome engineering in higher eukaryotes. J Cell Physiol. 2016;231:238092. <a href="https://doi.org/10.1002/jcp.25367">https://doi.org/10.1002/jcp.25367</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XktV2gu7w%253D" aria-label="View reference 13 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1002%2Fjcp.25367" aria-label="View reference 13">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 13 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Programmable%20site-specific%20nucleases%20for%20targeted%20genome%20engineering%20in%20higher%20eukaryotes&amp;journal=J%20Cell%20Physiol&amp;doi=10.1002%2Fjcp.25367&amp;volume=231&amp;pages=2380-2392&amp;publication_year=2016&amp;author=Govindan%2CG&amp;author=Ramalingam%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="datePublished" content="2012" /><meta itemprop="headline" content="European Food Safety Authority (EFSA). Scientific opinion addressing the safety assessment of plants developed" /><span class="c-article-references__counter">14.</span><p class="c-article-references__text" id="ref-CR14">European Food Safety Authority (EFSA). Scientific opinion addressing the safety assessment of plants developed using Zinc Finger Nuclease 3 and other Site-Directed Nucleases with similar function. EFSA J. 2012;10:2943.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.2903%2Fj.efsa.2012.2943" aria-label="View reference 14">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 14 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Scientific%20opinion%20addressing%20the%20safety%20assessment%20of%20plants%20developed%20using%20Zinc%20Finger%20Nuclease%203%20and%20other%20Site-Directed%20Nucleases%20with%20similar%20function&amp;journal=EFSA%20J&amp;volume=10&amp;publication_year=2012">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="CCM. Wiel, JG. Schaart, LAP. Lotz, MJM. Smulders, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="van de Wiel CCM, Schaart JG, Lotz LAP, Smulders MJM. New traits in crops produced by genome editing techniques" /><span class="c-article-references__counter">15.</span><p class="c-article-references__text" id="ref-CR15">van de Wiel CCM, Schaart JG, Lotz LAP, Smulders MJM. New traits in crops produced by genome editing techniques based on deletions. Plant Biotechnol Rep. 2017;11:18. <a href="https://doi.org/10.1007/s11816-017-0425-z">https://doi.org/10.1007/s11816-017-0425-z</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs11816-017-0425-z" aria-label="View reference 15">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 15 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=New%20traits%20in%20crops%20produced%20by%20genome%20editing%20techniques%20based%20on%20deletions&amp;journal=Plant%20Biotechnol%20Rep&amp;doi=10.1007%2Fs11816-017-0425-z&amp;volume=11&amp;pages=1-8&amp;publication_year=2017&amp;author=Wiel%2CCCM&amp;author=Schaart%2CJG&amp;author=Lotz%2CLAP&amp;author=Smulders%2CMJM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="N. Podevin, HV. Davies, F. Hartung, F. Nogu, JM. Casacuberta, " /><meta itemprop="datePublished" content="2013" /><meta itemprop="headline" content="Podevin N, Davies HV, Hartung F, Nogu F, Casacuberta JM. Site-directed nucleases: a paradigm shift in predict" /><span class="c-article-references__counter">16.</span><p class="c-article-references__text" id="ref-CR16">Podevin N, Davies HV, Hartung F, Nogu F, Casacuberta JM. Site-directed nucleases: a paradigm shift in predictable, knowledge-based plant breeding. Trends Biotechnol. 2013;31:37583. <a href="https://doi.org/10.1016/j.tibtech.2013.03.004">https://doi.org/10.1016/j.tibtech.2013.03.004</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXmtV2iur4%253D" aria-label="View reference 16 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.tibtech.2013.03.004" aria-label="View reference 16">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 16 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Site-directed%20nucleases%3A%20a%20paradigm%20shift%20in%20predictable%2C%20knowledge-based%20plant%20breeding&amp;journal=Trends%20Biotechnol&amp;doi=10.1016%2Fj.tibtech.2013.03.004&amp;volume=31&amp;pages=375-383&amp;publication_year=2013&amp;author=Podevin%2CN&amp;author=Davies%2CHV&amp;author=Hartung%2CF&amp;author=Nogu%C3%A9%2CF&amp;author=Casacuberta%2CJM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="AC. Komor, YB. Kim, MS. Packer, JA. Zuris, DR. Liu, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Komor AC, Kim YB, Packer MS, Zuris JA, Liu DR. Programmable editing of a target base in genomic DNA without do" /><span class="c-article-references__counter">17.</span><p class="c-article-references__text" id="ref-CR17">Komor AC, Kim YB, Packer MS, Zuris JA, Liu DR. Programmable editing of a target base in genomic DNA without double-stranded DNA cleavage. Nature. 2016;533:4204. <a href="https://doi.org/10.1038/nature17946">https://doi.org/10.1038/nature17946</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XmsVehsr8%253D" aria-label="View reference 17 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnature17946" aria-label="View reference 17">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 17 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Programmable%20editing%20of%20a%20target%20base%20in%20genomic%20DNA%20without%20double-stranded%20DNA%20cleavage&amp;journal=Nature&amp;doi=10.1038%2Fnature17946&amp;volume=533&amp;pages=420-424&amp;publication_year=2016&amp;author=Komor%2CAC&amp;author=Kim%2CYB&amp;author=Packer%2CMS&amp;author=Zuris%2CJA&amp;author=Liu%2CDR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Nationale Akademie der Wissenschaften Leopoldina, Deutsche Forschungsgemeinschaft, acatech  Deutsche Akademie" /><span class="c-article-references__counter">18.</span><p class="c-article-references__text" id="ref-CR18">Nationale Akademie der Wissenschaften Leopoldina, Deutsche Forschungsgemeinschaft, acatech  Deutsche Akademie der Technikwissenschaften, Union der deutschen Akademien der Wissenschaften. The opportunities and limits of genome editing. 2015.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Bmeke O, Kahrmann J, Matthies A. Detaillierte bersicht zum regulatorischen Status der neuen molekularbiologi" /><span class="c-article-references__counter">19.</span><p class="c-article-references__text" id="ref-CR19">Bmeke O, Kahrmann J, Matthies A. Detaillierte bersicht zum regulatorischen Status der neuen molekularbiologischen Techniken (NMT) in ausgewhlten Drittstaaten. <a href="https://www.bmel.de/DE/Landwirtschaft/Pflanzenbau/Gentechnik/_Texte/Neue_molekularbiologische_Techniken.html">https://www.bmel.de/DE/Landwirtschaft/Pflanzenbau/Gentechnik/_Texte/Neue_molekularbiologische_Techniken.html</a>. Accessed 21 Feb 2019.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y-B. Duan, J. Li, R-Y. Qin, R-F. Xu, H. Li, Y-C. Yang, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Duan Y-B, Li J, Qin R-Y, Xu R-F, Li H, Yang Y-C, et al. Identification of a regulatory element responsible for" /><span class="c-article-references__counter">20.</span><p class="c-article-references__text" id="ref-CR20">Duan Y-B, Li J, Qin R-Y, Xu R-F, Li H, Yang Y-C, et al. Identification of a regulatory element responsible for salt induction of rice OsRAV2 through ex situ and in situ promoter analysis. Plant Mol Biol. 2016;90:4962. <a href="https://doi.org/10.1007/s11103-015-0393-z">https://doi.org/10.1007/s11103-015-0393-z</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXhs1OrtbzK" aria-label="View reference 20 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs11103-015-0393-z" aria-label="View reference 20">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 20 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20a%20regulatory%20element%20responsible%20for%20salt%20induction%20of%20rice%20OsRAV2%20through%20ex%20situ%20and%20in%20situ%20promoter%20analysis&amp;journal=Plant%20Mol%20Biol&amp;doi=10.1007%2Fs11103-015-0393-z&amp;volume=90&amp;pages=49-62&amp;publication_year=2016&amp;author=Duan%2CY-B&amp;author=Li%2CJ&amp;author=Qin%2CR-Y&amp;author=Xu%2CR-F&amp;author=Li%2CH&amp;author=Yang%2CY-C">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="ZL. Demorest, A. Coffman, NJ. Baltes, TJ. Stoddard, BM. Clasen, S. Luo, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Demorest ZL, Coffman A, Baltes NJ, Stoddard TJ, Clasen BM, Luo S, et al. Direct stacking of sequence-specific " /><span class="c-article-references__counter">21.</span><p class="c-article-references__text" id="ref-CR21">Demorest ZL, Coffman A, Baltes NJ, Stoddard TJ, Clasen BM, Luo S, et al. Direct stacking of sequence-specific nuclease-induced mutations to produce high oleic and low linolenic soybean oil. BMC Plant Biol. 2016;16:225. <a href="https://doi.org/10.1186/s12870-016-0906-1">https://doi.org/10.1186/s12870-016-0906-1</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXpsVelsQ%253D%253D" aria-label="View reference 21 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2Fs12870-016-0906-1" aria-label="View reference 21">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 21 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Direct%20stacking%20of%20sequence-specific%20nuclease-induced%20mutations%20to%20produce%20high%20oleic%20and%20low%20linolenic%20soybean%20oil&amp;journal=BMC%20Plant%20Biol&amp;doi=10.1186%2Fs12870-016-0906-1&amp;volume=16&amp;publication_year=2016&amp;author=Demorest%2CZL&amp;author=Coffman%2CA&amp;author=Baltes%2CNJ&amp;author=Stoddard%2CTJ&amp;author=Clasen%2CBM&amp;author=Luo%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Chandrasekaran, M. Brumin, D. Wolf, D. Leibman, C. Klap, M. Pearlsman, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Chandrasekaran J, Brumin M, Wolf D, Leibman D, Klap C, Pearlsman M, et al. Development of broad virus resistan" /><span class="c-article-references__counter">22.</span><p class="c-article-references__text" id="ref-CR22">Chandrasekaran J, Brumin M, Wolf D, Leibman D, Klap C, Pearlsman M, et al. Development of broad virus resistance in non-transgenic cucumber using CRISPR/Cas9 technology. Mol Plant Pathol. 2016;17:114053. <a href="https://doi.org/10.1111/mpp.12375">https://doi.org/10.1111/mpp.12375</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28Xht1yls7jM" aria-label="View reference 22 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fmpp.12375" aria-label="View reference 22">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 22 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20of%20broad%20virus%20resistance%20in%20non-transgenic%20cucumber%20using%20CRISPR%2FCas9%20technology&amp;journal=Mol%20Plant%20Pathol&amp;doi=10.1111%2Fmpp.12375&amp;volume=17&amp;pages=1140-1153&amp;publication_year=2016&amp;author=Chandrasekaran%2CJ&amp;author=Brumin%2CM&amp;author=Wolf%2CD&amp;author=Leibman%2CD&amp;author=Klap%2CC&amp;author=Pearlsman%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><meta itemprop="author" content="GFW. Gocal, C. Schpke, PR. Beetham, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Gocal GFW, Schpke C, Beetham PR. Oligo-mediated targeted gene editing. In: Zhang F, Puchta H, Thomson JG, edi" /><span class="c-article-references__counter">23.</span><p class="c-article-references__text" id="ref-CR23">Gocal GFW, Schpke C, Beetham PR. Oligo-mediated targeted gene editing. In: Zhang F, Puchta H, Thomson JG, editors. Advances in new technology for targeted modification of plant genomes. New York: Springer; 2015. p. 7389. <a href="https://doi.org/10.1007/978-1-4939-2556-8_5">https://doi.org/10.1007/978-1-4939-2556-8_5</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 23 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Advances%20in%20new%20technology%20for%20targeted%20modification%20of%20plant%20genomes&amp;pages=73-89&amp;publication_year=2015&amp;author=Gocal%2CGFW&amp;author=Sch%C3%B6pke%2CC&amp;author=Beetham%2CPR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Shi, H. Gao, H. Wang, HR. Lafitte, RL. Archibald, M. Yang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Shi J, Gao H, Wang H, Lafitte HR, Archibald RL, Yang M, et al. ARGOS8 variants generated by CRISPRCas9 improv" /><span class="c-article-references__counter">24.</span><p class="c-article-references__text" id="ref-CR24">Shi J, Gao H, Wang H, Lafitte HR, Archibald RL, Yang M, et al. ARGOS8 variants generated by CRISPRCas9 improve maize grain yield under field drought stress conditions. Plant Biotechnol J. 2017;15:20716. <a href="https://doi.org/10.1111/pbi.12603">https://doi.org/10.1111/pbi.12603</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhtlCgtbg%253D" aria-label="View reference 24 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12603" aria-label="View reference 24">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 24 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=ARGOS8%20variants%20generated%20by%20CRISPR%E2%80%93Cas9%20improve%20maize%20grain%20yield%20under%20field%20drought%20stress%20conditions&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12603&amp;volume=15&amp;pages=207-216&amp;publication_year=2017&amp;author=Shi%2CJ&amp;author=Gao%2CH&amp;author=Wang%2CH&amp;author=Lafitte%2CHR&amp;author=Archibald%2CRL&amp;author=Yang%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Wang, X. Cheng, Q. Shan, Y. Zhang, J. Liu, C. Gao, J-L. Qiu, " /><meta itemprop="datePublished" content="2014" /><meta itemprop="headline" content="Wang Y, Cheng X, Shan Q, Zhang Y, Liu J, Gao C, Qiu J-L. Simultaneous editing of three homoeoalleles in hexapl" /><span class="c-article-references__counter">25.</span><p class="c-article-references__text" id="ref-CR25">Wang Y, Cheng X, Shan Q, Zhang Y, Liu J, Gao C, Qiu J-L. Simultaneous editing of three homoeoalleles in hexaploid bread wheat confers heritable resistance to powdery mildew. Nat Biotechnol. 2014;32:94751. <a href="https://doi.org/10.1038/nbt.2969">https://doi.org/10.1038/nbt.2969</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jJ" aria-label="View reference 25 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnbt.2969" aria-label="View reference 25">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 25 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Simultaneous%20editing%20of%20three%20homoeoalleles%20in%20hexaploid%20bread%20wheat%20confers%20heritable%20resistance%20to%20powdery%20mildew&amp;journal=Nat%20Biotechnol&amp;doi=10.1038%2Fnbt.2969&amp;volume=32&amp;pages=947-951&amp;publication_year=2014&amp;author=Wang%2CY&amp;author=Cheng%2CX&amp;author=Shan%2CQ&amp;author=Zhang%2CY&amp;author=Liu%2CJ&amp;author=Gao%2CC&amp;author=Qiu%2CJ-L">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="F. Martin, S. Snchez-Hernndez, A. Gutirrez-Guerrero, J. Pinedo-Gomez, K. Benabdellah, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Martin F, Snchez-Hernndez S, Gutirrez-Guerrero A, Pinedo-Gomez J, Benabdellah K. Biased and unbiased method" /><span class="c-article-references__counter">26.</span><p class="c-article-references__text" id="ref-CR26">Martin F, Snchez-Hernndez S, Gutirrez-Guerrero A, Pinedo-Gomez J, Benabdellah K. Biased and unbiased methods for the detection of off-target cleavage by CRISPR/Cas9: an overview. Int J Mol Sci. 2016. <a href="https://doi.org/10.3390/ijms17091507">https://doi.org/10.3390/ijms17091507</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3390%2Fijms17091507" aria-label="View reference 26">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 26 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Biased%20and%20unbiased%20methods%20for%20the%20detection%20of%20off-target%20cleavage%20by%20CRISPR%2FCas9%3A%20an%20overview&amp;journal=Int%20J%20Mol%20Sci&amp;doi=10.3390%2Fijms17091507&amp;publication_year=2016&amp;author=Martin%2CF&amp;author=S%C3%A1nchez-Hern%C3%A1ndez%2CS&amp;author=Guti%C3%A9rrez-Guerrero%2CA&amp;author=Pinedo-Gomez%2CJ&amp;author=Benabdellah%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Altschul, W. Gish, W. Miller, EW. Myers, DJ. Lipman, " /><meta itemprop="datePublished" content="1990" /><meta itemprop="headline" content="Altschul S, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J Mol Biol. 1990;215:403" /><span class="c-article-references__counter">27.</span><p class="c-article-references__text" id="ref-CR27">Altschul S, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J Mol Biol. 1990;215:40310. <a href="https://doi.org/10.1006/jmbi.1990.9999">https://doi.org/10.1006/jmbi.1990.9999</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3MXitVGmsA%253D%253D" aria-label="View reference 27 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1006%2Fjmbi.1990.9999" aria-label="View reference 27">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 27 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Basic%20local%20alignment%20search%20tool&amp;journal=J%20Mol%20Biol&amp;doi=10.1006%2Fjmbi.1990.9999&amp;volume=215&amp;pages=403-410&amp;publication_year=1990&amp;author=Altschul%2CS&amp;author=Gish%2CW&amp;author=Miller%2CW&amp;author=Myers%2CEW&amp;author=Lipman%2CDJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Bae, J. Park, J-S. Kim, " /><meta itemprop="datePublished" content="2014" /><meta itemprop="headline" content="Bae S, Park J, Kim J-S. Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target si" /><span class="c-article-references__counter">28.</span><p class="c-article-references__text" id="ref-CR28">Bae S, Park J, Kim J-S. Cas-OFFinder: a fast and versatile algorithm that searches for potential off-target sites of Cas9 RNA-guided endonucleases. Bioinformatics. 2014;30:14735. <a href="https://doi.org/10.1093/bioinformatics/btu048">https://doi.org/10.1093/bioinformatics/btu048</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXnslOitrs%253D" aria-label="View reference 28 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu048" aria-label="View reference 28">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 28 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Cas-OFFinder%3A%20a%20fast%20and%20versatile%20algorithm%20that%20searches%20for%20potential%20off-target%20sites%20of%20Cas9%20RNA-guided%20endonucleases&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu048&amp;volume=30&amp;pages=1473-1475&amp;publication_year=2014&amp;author=Bae%2CS&amp;author=Park%2CJ&amp;author=Kim%2CJ-S">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Lei, L. Lu, H-Y. Liu, S. Li, F. Xing, L-L. Chen, " /><meta itemprop="datePublished" content="2014" /><meta itemprop="headline" content="Lei Y, Lu L, Liu H-Y, Li S, Xing F, Chen L-L. CRISPR-P: a web tool for synthetic single-guide RNA design of CR" /><span class="c-article-references__counter">29.</span><p class="c-article-references__text" id="ref-CR29">Lei Y, Lu L, Liu H-Y, Li S, Xing F, Chen L-L. CRISPR-P: a web tool for synthetic single-guide RNA design of CRISPR-system in plants. Mol Plant. 2014;7:14946. <a href="https://doi.org/10.1093/mp/ssu044">https://doi.org/10.1093/mp/ssu044</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhsleitL3P" aria-label="View reference 29 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fmp%2Fssu044" aria-label="View reference 29">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 29 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-P%3A%20a%20web%20tool%20for%20synthetic%20single-guide%20RNA%20design%20of%20CRISPR-system%20in%20plants&amp;journal=Mol%20Plant&amp;doi=10.1093%2Fmp%2Fssu044&amp;volume=7&amp;pages=1494-1496&amp;publication_year=2014&amp;author=Lei%2CY&amp;author=Lu%2CL&amp;author=Liu%2CH-Y&amp;author=Li%2CS&amp;author=Xing%2CF&amp;author=Chen%2CL-L">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="H. Liu, Y. Ding, Y. Zhou, W. Jin, K. Xie, L-L. Chen, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Liu H, Ding Y, Zhou Y, Jin W, Xie K, Chen L-L. CRISPR-P 2.0: an improved CRISPRCas9 tool for genome editing i" /><span class="c-article-references__counter">30.</span><p class="c-article-references__text" id="ref-CR30">Liu H, Ding Y, Zhou Y, Jin W, Xie K, Chen L-L. CRISPR-P 2.0: an improved CRISPRCas9 tool for genome editing in plants. Mol Plant. 2017;10:5302. <a href="https://doi.org/10.1016/j.molp.2017.01.003">https://doi.org/10.1016/j.molp.2017.01.003</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhslejurg%253D" aria-label="View reference 30 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molp.2017.01.003" aria-label="View reference 30">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 30 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-P%202.0%3A%20an%20improved%20CRISPR%E2%80%93Cas9%20tool%20for%20genome%20editing%20in%20plants&amp;journal=Mol%20Plant&amp;doi=10.1016%2Fj.molp.2017.01.003&amp;volume=10&amp;pages=530-532&amp;publication_year=2017&amp;author=Liu%2CH&amp;author=Ding%2CY&amp;author=Zhou%2CY&amp;author=Jin%2CW&amp;author=Xie%2CK&amp;author=Chen%2CL-L">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Tycko, VE. Myer, PD. Hsu, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Tycko J, Myer VE, Hsu PD. Methods for optimizing CRISPRCas9 genome editing specificity. Mol Cell. 2016;63:355" /><span class="c-article-references__counter">31.</span><p class="c-article-references__text" id="ref-CR31">Tycko J, Myer VE, Hsu PD. Methods for optimizing CRISPRCas9 genome editing specificity. Mol Cell. 2016;63:35570. <a href="https://doi.org/10.1016/j.molcel.2016.07.004">https://doi.org/10.1016/j.molcel.2016.07.004</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhtlWjtbjE" aria-label="View reference 31 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molcel.2016.07.004" aria-label="View reference 31">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 31 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Methods%20for%20optimizing%20CRISPR%E2%80%93Cas9%20genome%20editing%20specificity&amp;journal=Mol%20Cell&amp;doi=10.1016%2Fj.molcel.2016.07.004&amp;volume=63&amp;pages=355-370&amp;publication_year=2016&amp;author=Tycko%2CJ&amp;author=Myer%2CVE&amp;author=Hsu%2CPD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Zischewski, R. Fischer, L. Bortesi, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Zischewski J, Fischer R, Bortesi L. Detection of on-target and off-target mutations generated by CRISPR/Cas9 a" /><span class="c-article-references__counter">32.</span><p class="c-article-references__text" id="ref-CR32">Zischewski J, Fischer R, Bortesi L. Detection of on-target and off-target mutations generated by CRISPR/Cas9 and other sequence-specific nucleases. Biotechnol Adv. 2017;35:95104. <a href="https://doi.org/10.1016/j.biotechadv.2016.12.003">https://doi.org/10.1016/j.biotechadv.2016.12.003</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXjvFarug%253D%253D" aria-label="View reference 32 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.biotechadv.2016.12.003" aria-label="View reference 32">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 32 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Detection%20of%20on-target%20and%20off-target%20mutations%20generated%20by%20CRISPR%2FCas9%20and%20other%20sequence-specific%20nucleases&amp;journal=Biotechnol%20Adv&amp;doi=10.1016%2Fj.biotechadv.2016.12.003&amp;volume=35&amp;pages=95-104&amp;publication_year=2017&amp;author=Zischewski%2CJ&amp;author=Fischer%2CR&amp;author=Bortesi%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><meta itemprop="author" content="SZ. Agapito-Tenfen, O-G. Wikmark, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Agapito-Tenfen SZ, Wikmark O-G. Current status of emerging technologies for plant breeding. Biosafety and know" /><span class="c-article-references__counter">33.</span><p class="c-article-references__text" id="ref-CR33">Agapito-Tenfen SZ, Wikmark O-G. Current status of emerging technologies for plant breeding. Biosafety and knowledge gaps of site directed nucleases and oligonucleotide-directed mutagenesis. Genk Centre for Biosafety: Troms; 2015.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 33 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Current%20status%20of%20emerging%20technologies%20for%20plant%20breeding.%20Biosafety%20and%20knowledge%20gaps%20of%20site%20directed%20nucleases%20and%20oligonucleotide-directed%20mutagenesis&amp;publication_year=2015&amp;author=Agapito-Tenfen%2CSZ&amp;author=Wikmark%2CO-G">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X-H. Zhang, LY. Tee, X-G. Wang, Q-S. Huang, S-H. Yang, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Zhang X-H, Tee LY, Wang X-G, Huang Q-S, Yang S-H. Off-target effects in CRISPR/Cas9-mediated genome engineerin" /><span class="c-article-references__counter">34.</span><p class="c-article-references__text" id="ref-CR34">Zhang X-H, Tee LY, Wang X-G, Huang Q-S, Yang S-H. Off-target effects in CRISPR/Cas9-mediated genome engineering. Mol Ther Nucleic Acids. 2015;4:e264. <a href="https://doi.org/10.1038/mtna.2015.37">https://doi.org/10.1038/mtna.2015.37</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXhvVCqtLjJ" aria-label="View reference 34 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fmtna.2015.37" aria-label="View reference 34">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 34 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Off-target%20effects%20in%20CRISPR%2FCas9-mediated%20genome%20engineering&amp;journal=Mol%20Ther%20Nucleic%20Acids&amp;doi=10.1038%2Fmtna.2015.37&amp;volume=4&amp;publication_year=2015&amp;author=Zhang%2CX-H&amp;author=Tee%2CLY&amp;author=Wang%2CX-G&amp;author=Huang%2CQ-S&amp;author=Yang%2CS-H">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="CN. Kanchiswamy, M. Maffei, M. Malnoy, R. Velasco, J-S. Kim, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Kanchiswamy CN, Maffei M, Malnoy M, Velasco R, Kim J-S. Fine-tuning next-generation genome editing tools. Tren" /><span class="c-article-references__counter">35.</span><p class="c-article-references__text" id="ref-CR35">Kanchiswamy CN, Maffei M, Malnoy M, Velasco R, Kim J-S. Fine-tuning next-generation genome editing tools. Trends Biotechnol. 2016;34:56274. <a href="https://doi.org/10.1016/j.tibtech.2016.03.007">https://doi.org/10.1016/j.tibtech.2016.03.007</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XlvValtbw%253D" aria-label="View reference 35 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.tibtech.2016.03.007" aria-label="View reference 35">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 35 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Fine-tuning%20next-generation%20genome%20editing%20tools&amp;journal=Trends%20Biotechnol&amp;doi=10.1016%2Fj.tibtech.2016.03.007&amp;volume=34&amp;pages=562-574&amp;publication_year=2016&amp;author=Kanchiswamy%2CCN&amp;author=Maffei%2CM&amp;author=Malnoy%2CM&amp;author=Velasco%2CR&amp;author=Kim%2CJ-S">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Collaboration for Environmental Evidence. Guidelines for systematic review and evidence synthesis in environme" /><span class="c-article-references__counter">36.</span><p class="c-article-references__text" id="ref-CR36">Collaboration for Environmental Evidence. Guidelines for systematic review and evidence synthesis in environmental management, Version 4.2. Environmental Evidence. 2013. <a href="http://www.environmentalevidence.org/Documents/Guidelines/Guidelines4.2.pd">http://www.environmentalevidence.org/Documents/Guidelines/Guidelines4.2.pd</a>. Accessed 13 Sept 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="D. Modrzejewski, F. Hartung, T. Sprink, D. Krause, C. Kohl, J. Schiemann, R. Wilhelm, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Modrzejewski D, Hartung F, Sprink T, Krause D, Kohl C, Schiemann J, Wilhelm R. What is the available evidence " /><span class="c-article-references__counter">37.</span><p class="c-article-references__text" id="ref-CR37">Modrzejewski D, Hartung F, Sprink T, Krause D, Kohl C, Schiemann J, Wilhelm R. What is the available evidence for the application of genome editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map protocol. Environ Evid. 2018;7:11. <a href="https://doi.org/10.1186/s13750-018-0130-6">https://doi.org/10.1186/s13750-018-0130-6</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2Fs13750-018-0130-6" aria-label="View reference 37">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 37 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=What%20is%20the%20available%20evidence%20for%20the%20application%20of%20genome%20editing%20as%20a%20new%20tool%20for%20plant%20trait%20modification%20and%20the%20potential%20occurrence%20of%20associated%20off-target%20effects%3A%20a%20systematic%20map%20protocol&amp;journal=Environ%20Evid&amp;doi=10.1186%2Fs13750-018-0130-6&amp;volume=7&amp;publication_year=2018&amp;author=Modrzejewski%2CD&amp;author=Hartung%2CF&amp;author=Sprink%2CT&amp;author=Krause%2CD&amp;author=Kohl%2CC&amp;author=Schiemann%2CJ&amp;author=Wilhelm%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Cohen, " /><meta itemprop="datePublished" content="1960" /><meta itemprop="headline" content="Cohen J. A coefficient of agreement for nominal scales. Educ Psychol Meas. 1960;20:3746. https://doi.org/10.1" /><span class="c-article-references__counter">38.</span><p class="c-article-references__text" id="ref-CR38">Cohen J. A coefficient of agreement for nominal scales. Educ Psychol Meas. 1960;20:3746. <a href="https://doi.org/10.1177/001316446002000104">https://doi.org/10.1177/001316446002000104</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1177%2F001316446002000104" aria-label="View reference 38">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 38 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20coefficient%20of%20agreement%20for%20nominal%20scales&amp;journal=Educ%20Psychol%20Meas&amp;doi=10.1177%2F001316446002000104&amp;volume=20&amp;pages=37-46&amp;publication_year=1960&amp;author=Cohen%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="C. Kohl, EJ. McIntosh, S. Unger, NR. Haddaway, S. Kecke, J. Schiemann, R. Wilhelm, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Kohl C, McIntosh EJ, Unger S, Haddaway NR, Kecke S, Schiemann J, Wilhelm R. Online tools supporting the conduc" /><span class="c-article-references__counter">39.</span><p class="c-article-references__text" id="ref-CR39">Kohl C, McIntosh EJ, Unger S, Haddaway NR, Kecke S, Schiemann J, Wilhelm R. Online tools supporting the conduct and reporting of systematic reviews and systematic maps: a case study on CADIMA and review of existing tools. Environ Evid. 2018;7:2420. <a href="https://doi.org/10.1186/s13750-018-0115-5">https://doi.org/10.1186/s13750-018-0115-5</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2Fs13750-018-0115-5" aria-label="View reference 39">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 39 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Online%20tools%20supporting%20the%20conduct%20and%20reporting%20of%20systematic%20reviews%20and%20systematic%20maps%3A%20a%20case%20study%20on%20CADIMA%20and%20review%20of%20existing%20tools&amp;journal=Environ%20Evid&amp;doi=10.1186%2Fs13750-018-0115-5&amp;volume=7&amp;publication_year=2018&amp;author=Kohl%2CC&amp;author=McIntosh%2CEJ&amp;author=Unger%2CS&amp;author=Haddaway%2CNR&amp;author=Kecke%2CS&amp;author=Schiemann%2CJ&amp;author=Wilhelm%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Haddaway NR, Macura B, Whaley P, Pullin AS. ROSES for Systematic Map Reports. Version 1.0. 2017. https://doi.o" /><span class="c-article-references__counter">40.</span><p class="c-article-references__text" id="ref-CR40">Haddaway NR, Macura B, Whaley P, Pullin AS. ROSES for Systematic Map Reports. Version 1.0. 2017. <a href="https://doi.org/10.6084/m9.figshare.5897299">https://doi.org/10.6084/m9.figshare.5897299</a>.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Braatz, H-J. Harloff, M. Mascher, N. Stein, A. Himmelbach, C. Jung, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Braatz J, Harloff H-J, Mascher M, Stein N, Himmelbach A, Jung C. CRISPRCas9 targeted mutagenesis leads to sim" /><span class="c-article-references__counter">41.</span><p class="c-article-references__text" id="ref-CR41">Braatz J, Harloff H-J, Mascher M, Stein N, Himmelbach A, Jung C. CRISPRCas9 targeted mutagenesis leads to simultaneous modification of different homoeologous gene copies in polyploid oilseed rape (<i>Brassica napus</i>). Plant Physiol. 2017;174:9. <a href="https://doi.org/10.1104/pp.17.00426">https://doi.org/10.1104/pp.17.00426</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXht1Cru7%252FM" aria-label="View reference 41 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1104%2Fpp.17.00426" aria-label="View reference 41">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 41 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%E2%80%93Cas9%20targeted%20mutagenesis%20leads%20to%20simultaneous%20modification%20of%20different%20homoeologous%20gene%20copies%20in%20polyploid%20oilseed%20rape%20%28Brassica%20napus%29&amp;journal=Plant%20Physiol&amp;doi=10.1104%2Fpp.17.00426&amp;volume=174&amp;publication_year=2017&amp;author=Braatz%2CJ&amp;author=Harloff%2CH-J&amp;author=Mascher%2CM&amp;author=Stein%2CN&amp;author=Himmelbach%2CA&amp;author=Jung%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Yang, K. Zhu, H. Li, S. Han, Q. Meng, SU. Khan, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Yang Y, Zhu K, Li H, Han S, Meng Q, Khan SU, et al. Precise editing of CLAVATA genes in Brassica napus L. regu" /><span class="c-article-references__counter">42.</span><p class="c-article-references__text" id="ref-CR42">Yang Y, Zhu K, Li H, Han S, Meng Q, Khan SU, et al. Precise editing of CLAVATA genes in <i>Brassica napus</i> L. regulates multilocular silique development. Plant Biotechnol J. 2018;16:132235. <a href="https://doi.org/10.1111/pbi.12872">https://doi.org/10.1111/pbi.12872</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhtFKhtLzN" aria-label="View reference 42 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12872" aria-label="View reference 42">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 42 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Precise%20editing%20of%20CLAVATA%20genes%20in%20Brassica%20napus%20L.%20regulates%20multilocular%20silique%20development&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12872&amp;volume=16&amp;pages=1322-1335&amp;publication_year=2018&amp;author=Yang%2CY&amp;author=Zhu%2CK&amp;author=Li%2CH&amp;author=Han%2CS&amp;author=Meng%2CQ&amp;author=Khan%2CSU">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Wang, Z. Meng, C. Liang, Z. Meng, Y. Wang, G. Sun, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Wang Y, Meng Z, Liang C, Meng Z, Wang Y, Sun G, et al. Increased lateral root formation by CRISPR/Cas9-mediate" /><span class="c-article-references__counter">43.</span><p class="c-article-references__text" id="ref-CR43">Wang Y, Meng Z, Liang C, Meng Z, Wang Y, Sun G, et al. Increased lateral root formation by CRISPR/Cas9-mediated editing of arginase genes in cotton. Sci China Life Sci. 2017;60:5247. <a href="https://doi.org/10.1007/s11427-017-9031-y">https://doi.org/10.1007/s11427-017-9031-y</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXotVOkt70%253D" aria-label="View reference 43 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs11427-017-9031-y" aria-label="View reference 43">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 43 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Increased%20lateral%20root%20formation%20by%20CRISPR%2FCas9-mediated%20editing%20of%20arginase%20genes%20in%20cotton&amp;journal=Sci%20China%20Life%20Sci&amp;doi=10.1007%2Fs11427-017-9031-y&amp;volume=60&amp;pages=524-527&amp;publication_year=2017&amp;author=Wang%2CY&amp;author=Meng%2CZ&amp;author=Liang%2CC&amp;author=Meng%2CZ&amp;author=Wang%2CY&amp;author=Sun%2CG">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="B. Hu, D. Li, X. Liu, J. Qi, D. Gao, S. Zhao, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Hu B, Li D, Liu X, Qi J, Gao D, Zhao S, et al. Engineering non-transgenic gynoecious cucumber using an improve" /><span class="c-article-references__counter">44.</span><p class="c-article-references__text" id="ref-CR44">Hu B, Li D, Liu X, Qi J, Gao D, Zhao S, et al. Engineering non-transgenic gynoecious cucumber using an improved transformation protocol and optimized CRISPR/Cas9 system. Mol Plant. 2017;10:15758. <a href="https://doi.org/10.1016/j.molp.2017.09.005">https://doi.org/10.1016/j.molp.2017.09.005</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhslems7vE" aria-label="View reference 44 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molp.2017.09.005" aria-label="View reference 44">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 44 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Engineering%20non-transgenic%20gynoecious%20cucumber%20using%20an%20improved%20transformation%20protocol%20and%20optimized%20CRISPR%2FCas9%20system&amp;journal=Mol%20Plant&amp;doi=10.1016%2Fj.molp.2017.09.005&amp;volume=10&amp;pages=1575-1578&amp;publication_year=2017&amp;author=Hu%2CB&amp;author=Li%2CD&amp;author=Liu%2CX&amp;author=Qi%2CJ&amp;author=Gao%2CD&amp;author=Zhao%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="LD. Bertier, M. Ron, H. Huo, KJ. Bradford, AB. Britt, RW. Michelmore, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Bertier LD, Ron M, Huo H, Bradford KJ, Britt AB, Michelmore RW. High-resolution analysis of the efficiency, he" /><span class="c-article-references__counter">45.</span><p class="c-article-references__text" id="ref-CR45">Bertier LD, Ron M, Huo H, Bradford KJ, Britt AB, Michelmore RW. High-resolution analysis of the efficiency, heritability, and editing outcomes of CRISPR/Cas9-induced modifications of NCED4 in lettuce (<i>Lactuca sativa</i>). G3. 2018;8:151321. <a href="https://doi.org/10.1534/g3.117.300396">https://doi.org/10.1534/g3.117.300396</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXhsVSgtLfI" aria-label="View reference 45 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1534%2Fg3.117.300396" aria-label="View reference 45">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 45 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=High-resolution%20analysis%20of%20the%20efficiency%2C%20heritability%2C%20and%20editing%20outcomes%20of%20CRISPR%2FCas9-induced%20modifications%20of%20NCED4%20in%20lettuce%20%28Lactuca%20sativa%29&amp;journal=G3&amp;doi=10.1534%2Fg3.117.300396&amp;volume=8&amp;pages=1513-1521&amp;publication_year=2018&amp;author=Bertier%2CLD&amp;author=Ron%2CM&amp;author=Huo%2CH&amp;author=Bradford%2CKJ&amp;author=Britt%2CAB&amp;author=Michelmore%2CRW">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2015. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">46.</span><p class="c-article-references__text" id="ref-CR46">United States Department of Agriculture (USDA). 2015. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-013-01air.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-013-01air.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="C. Huang, H. Sun, D. Xu, Q. Chen, Y. Liang, X. Wang, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Huang C, Sun H, Xu D, Chen Q, Liang Y, Wang X, et al. ZmCCT9 enhances maize adaptation to higher latitudes. Pr" /><span class="c-article-references__counter">47.</span><p class="c-article-references__text" id="ref-CR47">Huang C, Sun H, Xu D, Chen Q, Liang Y, Wang X, et al. ZmCCT9 enhances maize adaptation to higher latitudes. Proc Natl Acad Sci USA. 2018;115:E33441. <a href="https://doi.org/10.1073/pnas.1718058115">https://doi.org/10.1073/pnas.1718058115</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXitVeqtbfO" aria-label="View reference 47 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.1718058115" aria-label="View reference 47">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 47 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=ZmCCT9%20enhances%20maize%20adaptation%20to%20higher%20latitudes&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;doi=10.1073%2Fpnas.1718058115&amp;volume=115&amp;pages=E334-E341&amp;publication_year=2018&amp;author=Huang%2CC&amp;author=Sun%2CH&amp;author=Xu%2CD&amp;author=Chen%2CQ&amp;author=Liang%2CY&amp;author=Wang%2CX">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Svitashev, JK. Young, C. Schwartz, H. Gao, SC. Falco, AM. Cigan, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Svitashev S, Young JK, Schwartz C, Gao H, Falco SC, Cigan AM. Targeted mutagenesis, precise gene editing, and " /><span class="c-article-references__counter">48.</span><p class="c-article-references__text" id="ref-CR48">Svitashev S, Young JK, Schwartz C, Gao H, Falco SC, Cigan AM. Targeted mutagenesis, precise gene editing, and site-specific gene insertion in maize using Cas9 and guide RNA. Plant Physiol. 2015;169:93145. <a href="https://doi.org/10.1104/pp.15.00793">https://doi.org/10.1104/pp.15.00793</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XisVSmt7Y%253D" aria-label="View reference 48 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1104%2Fpp.15.00793" aria-label="View reference 48">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 48 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20mutagenesis%2C%20precise%20gene%20editing%2C%20and%20site-specific%20gene%20insertion%20in%20maize%20using%20Cas9%20and%20guide%20RNA&amp;journal=Plant%20Physiol&amp;doi=10.1104%2Fpp.15.00793&amp;volume=169&amp;pages=931-945&amp;publication_year=2015&amp;author=Svitashev%2CS&amp;author=Young%2CJK&amp;author=Schwartz%2CC&amp;author=Gao%2CH&amp;author=Falco%2CSC&amp;author=Cigan%2CAM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Svitashev, C. Schwartz, B. Lenderts, JK. Young, Cigan A. Mark, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Svitashev S, Schwartz C, Lenderts B, Young JK, Mark Cigan A. Genome editing in maize directed by CRISPRCas9 r" /><span class="c-article-references__counter">49.</span><p class="c-article-references__text" id="ref-CR49">Svitashev S, Schwartz C, Lenderts B, Young JK, Mark Cigan A. Genome editing in maize directed by CRISPRCas9 ribonucleoprotein complexes. Nat Commun. 2016;7:17. <a href="https://doi.org/10.1038/ncomms13274">https://doi.org/10.1038/ncomms13274</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhvVygu77N" aria-label="View reference 49 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fncomms13274" aria-label="View reference 49">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 49 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20editing%20in%20maize%20directed%20by%20CRISPR%E2%80%93Cas9%20ribonucleoprotein%20complexes&amp;journal=Nat%20Commun&amp;doi=10.1038%2Fncomms13274&amp;volume=7&amp;pages=1-7&amp;publication_year=2016&amp;author=Svitashev%2CS&amp;author=Schwartz%2CC&amp;author=Lenderts%2CB&amp;author=Young%2CJK&amp;author=Mark%2CCigan%20A">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Li, H. Zhang, X. Si, Y. Tian, K. Chen, J. Liu, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Li J, Zhang H, Si X, Tian Y, Chen K, Liu J, et al. Generation of thermosensitive male-sterile maize by targete" /><span class="c-article-references__counter">50.</span><p class="c-article-references__text" id="ref-CR50">Li J, Zhang H, Si X, Tian Y, Chen K, Liu J, et al. Generation of thermosensitive male-sterile maize by targeted knockout of the ZmTMS5 gene. J Genet Genom. 2017;44:4658. <a href="https://doi.org/10.1016/j.jgg.2017.02.002">https://doi.org/10.1016/j.jgg.2017.02.002</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.jgg.2017.02.002" aria-label="View reference 50">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 50 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Generation%20of%20thermosensitive%20male-sterile%20maize%20by%20targeted%20knockout%20of%20the%20ZmTMS5%20gene&amp;journal=J%20Genet%20Genom&amp;doi=10.1016%2Fj.jgg.2017.02.002&amp;volume=44&amp;pages=465-468&amp;publication_year=2017&amp;author=Li%2CJ&amp;author=Zhang%2CH&amp;author=Si%2CX&amp;author=Tian%2CY&amp;author=Chen%2CK&amp;author=Liu%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="T. Kelliher, D. Starr, L. Richbourg, S. Chintamanani, B. Delzer, ML. Nuccio, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Kelliher T, Starr D, Richbourg L, Chintamanani S, Delzer B, Nuccio ML, et al. MATRILINEAL, a sperm-specific ph" /><span class="c-article-references__counter">51.</span><p class="c-article-references__text" id="ref-CR51">Kelliher T, Starr D, Richbourg L, Chintamanani S, Delzer B, Nuccio ML, et al. MATRILINEAL, a sperm-specific phospholipase, triggers maize haploid induction. Nature. 2017;542:1059. <a href="https://doi.org/10.1038/nature20827">https://doi.org/10.1038/nature20827</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhsFGlt7w%253D" aria-label="View reference 51 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnature20827" aria-label="View reference 51">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 51 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=MATRILINEAL%2C%20a%20sperm-specific%20phospholipase%2C%20triggers%20maize%20haploid%20induction&amp;journal=Nature&amp;doi=10.1038%2Fnature20827&amp;volume=542&amp;pages=105-109&amp;publication_year=2017&amp;author=Kelliher%2CT&amp;author=Starr%2CD&amp;author=Richbourg%2CL&amp;author=Chintamanani%2CS&amp;author=Delzer%2CB&amp;author=Nuccio%2CML">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="BM. Clasen, TJ. Stoddard, S. Luo, ZL. Demorest, J. Li, F. Cedrone, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Clasen BM, Stoddard TJ, Luo S, Demorest ZL, Li J, Cedrone F, et al. Improving cold storage and processing trai" /><span class="c-article-references__counter">52.</span><p class="c-article-references__text" id="ref-CR52">Clasen BM, Stoddard TJ, Luo S, Demorest ZL, Li J, Cedrone F, et al. Improving cold storage and processing traits in potato through targeted gene knockout. Plant Biotechnol J. 2016;14:16976. <a href="https://doi.org/10.1111/pbi.12370">https://doi.org/10.1111/pbi.12370</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXitV2ksrzI" aria-label="View reference 52 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12370" aria-label="View reference 52">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 52 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Improving%20cold%20storage%20and%20processing%20traits%20in%20potato%20through%20targeted%20gene%20knockout&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12370&amp;volume=14&amp;pages=169-176&amp;publication_year=2016&amp;author=Clasen%2CBM&amp;author=Stoddard%2CTJ&amp;author=Luo%2CS&amp;author=Demorest%2CZL&amp;author=Li%2CJ&amp;author=Cedrone%2CF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="M. Li, X. Li, Z. Zhou, P. Wu, M. Fang, X. Pan, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Li M, Li X, Zhou Z, Wu P, Fang M, Pan X, et al. Reassessment of the four yield-related genes Gn1a, DEP1, GS3, " /><span class="c-article-references__counter">53.</span><p class="c-article-references__text" id="ref-CR53">Li M, Li X, Zhou Z, Wu P, Fang M, Pan X, et al. Reassessment of the four yield-related genes Gn1a, DEP1, GS3, and IPA1 in rice using a CRISPR/Cas9 system. Front Plant Sci. 2016;7:113. <a href="https://doi.org/10.3389/fpls.2016.00377">https://doi.org/10.3389/fpls.2016.00377</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3389%2Ffpls.2016.00377" aria-label="View reference 53">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 53 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Reassessment%20of%20the%20four%20yield-related%20genes%20Gn1a%2C%20DEP1%2C%20GS3%2C%20and%20IPA1%20in%20rice%20using%20a%20CRISPR%2FCas9%20system&amp;journal=Front%20Plant%20Sci&amp;doi=10.3389%2Ffpls.2016.00377&amp;volume=7&amp;pages=1-13&amp;publication_year=2016&amp;author=Li%2CM&amp;author=Li%2CX&amp;author=Zhou%2CZ&amp;author=Wu%2CP&amp;author=Fang%2CM&amp;author=Pan%2CX">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="L. Shen, C. Wang, Y. Fu, J. Wang, Q. Liu, X. Zhang, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Shen L, Wang C, Fu Y, Wang J, Liu Q, Zhang X, et al. QTL editing confers opposing yield performance in differe" /><span class="c-article-references__counter">54.</span><p class="c-article-references__text" id="ref-CR54">Shen L, Wang C, Fu Y, Wang J, Liu Q, Zhang X, et al. QTL editing confers opposing yield performance in different rice varieties. J Integr Plant Biol. 2016. <a href="https://doi.org/10.1111/jipb.12501">https://doi.org/10.1111/jipb.12501</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fjipb.12501" aria-label="View reference 54">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 54 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=QTL%20editing%20confers%20opposing%20yield%20performance%20in%20different%20rice%20varieties&amp;journal=J%20Integr%20Plant%20Biol&amp;doi=10.1111%2Fjipb.12501&amp;publication_year=2016&amp;author=Shen%2CL&amp;author=Wang%2CC&amp;author=Fu%2CY&amp;author=Wang%2CJ&amp;author=Liu%2CQ&amp;author=Zhang%2CX">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="R. Xu, Y. Yang, R. Qin, H. Li, C. Qiu, L. Li, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Xu R, Yang Y, Qin R, Li H, Qiu C, Li L, et al. Rapid improvement of grain weight via highly efficient CRISPR/C" /><span class="c-article-references__counter">55.</span><p class="c-article-references__text" id="ref-CR55">Xu R, Yang Y, Qin R, Li H, Qiu C, Li L, et al. Rapid improvement of grain weight via highly efficient CRISPR/Cas9-mediated multiplex genome editing in rice. J Genet Genom. 2016;43:52932. <a href="https://doi.org/10.1016/j.jgg.2016.07.003">https://doi.org/10.1016/j.jgg.2016.07.003</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.jgg.2016.07.003" aria-label="View reference 55">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 55 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20improvement%20of%20grain%20weight%20via%20highly%20efficient%20CRISPR%2FCas9-mediated%20multiplex%20genome%20editing%20in%20rice&amp;journal=J%20Genet%20Genom&amp;doi=10.1016%2Fj.jgg.2016.07.003&amp;volume=43&amp;pages=529-532&amp;publication_year=2016&amp;author=Xu%2CR&amp;author=Yang%2CY&amp;author=Qin%2CR&amp;author=Li%2CH&amp;author=Qiu%2CC&amp;author=Li%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Z. Hu, S-J. Lu, M-J. Wang, H. He, L. Sun, H. Wang, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Hu Z, Lu S-J, Wang M-J, He H, Sun L, Wang H, et al. A novel QTL q TGW3 encodes the GSK3/SHAGGY-like kinase OsG" /><span class="c-article-references__counter">56.</span><p class="c-article-references__text" id="ref-CR56">Hu Z, Lu S-J, Wang M-J, He H, Sun L, Wang H, et al. A novel QTL q TGW3 encodes the GSK3/SHAGGY-like kinase OsGSK5/OsSK41 that interacts with OsARF4 to negatively regulate grain size and weight in rice. Mol Plant. 2018;11:73649. <a href="https://doi.org/10.1016/j.molp.2018.03.005">https://doi.org/10.1016/j.molp.2018.03.005</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXntFWqtbs%253D" aria-label="View reference 56 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molp.2018.03.005" aria-label="View reference 56">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 56 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20novel%20QTL%20q%20TGW3%20encodes%20the%20GSK3%2FSHAGGY-like%20kinase%20OsGSK5%2FOsSK41%20that%20interacts%20with%20OsARF4%20to%20negatively%20regulate%20grain%20size%20and%20weight%20in%20rice&amp;journal=Mol%20Plant&amp;doi=10.1016%2Fj.molp.2018.03.005&amp;volume=11&amp;pages=736-749&amp;publication_year=2018&amp;author=Hu%2CZ&amp;author=Lu%2CS-J&amp;author=Wang%2CM-J&amp;author=He%2CH&amp;author=Sun%2CL&amp;author=Wang%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="L. Shen, J. Li, Y. Fu, J. Wang, Y. Hua, X. Jiao, C. Yan, K. Wang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Shen L, Li J, Fu Y, Wang J, Hua Y, Jiao X, Yan C, Wang K. Orientation improvement of grain length and grain nu" /><span class="c-article-references__counter">57.</span><p class="c-article-references__text" id="ref-CR57">Shen L, Li J, Fu Y, Wang J, Hua Y, Jiao X, Yan C, Wang K. Orientation improvement of grain length and grain number in rice by using CRISPR/Cas9 system. Chin J Rice Sci. 2017. <a href="https://doi.org/10.16819/j.1001-7216.2017.7029">https://doi.org/10.16819/j.1001-7216.2017.7029</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.16819%2Fj.1001-7216.2017.7029" aria-label="View reference 57">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 57 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Orientation%20improvement%20of%20grain%20length%20and%20grain%20number%20in%20rice%20by%20using%20CRISPR%2FCas9%20system&amp;journal=Chin%20J%20Rice%20Sci&amp;doi=10.16819%2Fj.1001-7216.2017.7029&amp;publication_year=2017&amp;author=Shen%2CL&amp;author=Li%2CJ&amp;author=Fu%2CY&amp;author=Wang%2CJ&amp;author=Hua%2CY&amp;author=Jiao%2CX&amp;author=Yan%2CC&amp;author=Wang%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X. Ji, F. Li, Y. Yan, HZ. Sun, J. Zhang, JZ. Li, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Ji X, Li F, Yan Y, Sun HZ, Zhang J, Li JZ, et al. CRISPR/Cas9 system-based editing of phytochrome-interacting " /><span class="c-article-references__counter">58.</span><p class="c-article-references__text" id="ref-CR58">Ji X, Li F, Yan Y, Sun HZ, Zhang J, Li JZ, et al. CRISPR/Cas9 system-based editing of phytochrome-interacting factor OsPIL15. Sci Agric Sin. 2017. <a href="https://doi.org/10.3864/j.issn.0578-1752.2017.15.002">https://doi.org/10.3864/j.issn.0578-1752.2017.15.002</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3864%2Fj.issn.0578-1752.2017.15.002" aria-label="View reference 58">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 58 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FCas9%20system-based%20editing%20of%20phytochrome-interacting%20factor%20OsPIL15&amp;journal=Sci%20Agric%20Sin&amp;doi=10.3864%2Fj.issn.0578-1752.2017.15.002&amp;publication_year=2017&amp;author=Ji%2CX&amp;author=Li%2CF&amp;author=Yan%2CY&amp;author=Sun%2CHZ&amp;author=Zhang%2CJ&amp;author=Li%2CJZ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="L. Shen, Y. Hua, Y. Fu, J. Li, Q. Liu, X. Jiao, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Shen L, Hua Y, Fu Y, Li J, Liu Q, Jiao X, et al. Rapid generation of genetic diversity by multiplex CRISPR/Cas" /><span class="c-article-references__counter">59.</span><p class="c-article-references__text" id="ref-CR59">Shen L, Hua Y, Fu Y, Li J, Liu Q, Jiao X, et al. Rapid generation of genetic diversity by multiplex CRISPR/Cas9 genome editing in rice. Sci China Life Sci. 2017;60:50615. <a href="https://doi.org/10.1007/s11427-017-9008-8">https://doi.org/10.1007/s11427-017-9008-8</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXlt1elt7o%253D" aria-label="View reference 59 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs11427-017-9008-8" aria-label="View reference 59">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 59 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20generation%20of%20genetic%20diversity%20by%20multiplex%20CRISPR%2FCas9%20genome%20editing%20in%20rice&amp;journal=Sci%20China%20Life%20Sci&amp;doi=10.1007%2Fs11427-017-9008-8&amp;volume=60&amp;pages=506-515&amp;publication_year=2017&amp;author=Shen%2CL&amp;author=Hua%2CY&amp;author=Fu%2CY&amp;author=Li%2CJ&amp;author=Liu%2CQ&amp;author=Jiao%2CX">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="K. Lu, B. Wu, J. Wang, W. Zhu, H. Nie, J. Qian, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Lu K, Wu B, Wang J, Zhu W, Nie H, Qian J, et al. Blocking amino acid transporter OsAAP3 improves grain yield b" /><span class="c-article-references__counter">60.</span><p class="c-article-references__text" id="ref-CR60">Lu K, Wu B, Wang J, Zhu W, Nie H, Qian J, et al. Blocking amino acid transporter OsAAP3 improves grain yield by promoting outgrowth buds and increasing tiller number in rice. Plant Biotechnol J. 2018;50:1416. <a href="https://doi.org/10.1111/pbi.12907">https://doi.org/10.1111/pbi.12907</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhslSrsrjP" aria-label="View reference 60 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12907" aria-label="View reference 60">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 60 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Blocking%20amino%20acid%20transporter%20OsAAP3%20improves%20grain%20yield%20by%20promoting%20outgrowth%20buds%20and%20increasing%20tiller%20number%20in%20rice&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12907&amp;volume=50&amp;publication_year=2018&amp;author=Lu%2CK&amp;author=Wu%2CB&amp;author=Wang%2CJ&amp;author=Zhu%2CW&amp;author=Nie%2CH&amp;author=Qian%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Liao, Q. Bai, P. Xu, T. Wu, D. Guo, Y. Peng, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Liao Y, Bai Q, Xu P, Wu T, Guo D, Peng Y, et al. Mutation in rice abscisic acid2 results in cell death, enhanc" /><span class="c-article-references__counter">61.</span><p class="c-article-references__text" id="ref-CR61">Liao Y, Bai Q, Xu P, Wu T, Guo D, Peng Y, et al. Mutation in rice abscisic acid2 results in cell death, enhanced disease-resistance, altered seed dormancy and development. Front Plant Sci. 2018;9:1248. <a href="https://doi.org/10.3389/fpls.2018.00405">https://doi.org/10.3389/fpls.2018.00405</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3389%2Ffpls.2018.00405" aria-label="View reference 61">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 61 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Mutation%20in%20rice%20abscisic%20acid2%20results%20in%20cell%20death%2C%20enhanced%20disease-resistance%2C%20altered%20seed%20dormancy%20and%20development&amp;journal=Front%20Plant%20Sci&amp;doi=10.3389%2Ffpls.2018.00405&amp;volume=9&amp;publication_year=2018&amp;author=Liao%2CY&amp;author=Bai%2CQ&amp;author=Xu%2CP&amp;author=Wu%2CT&amp;author=Guo%2CD&amp;author=Peng%2CY">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X. Li, W. Zhou, Y. Ren, X. Tian, T. Lv, Z. Wang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Li X, Zhou W, Ren Y, Tian X, Lv T, Wang Z, et al. High-efficiency breeding of early-maturing rice cultivars vi" /><span class="c-article-references__counter">62.</span><p class="c-article-references__text" id="ref-CR62">Li X, Zhou W, Ren Y, Tian X, Lv T, Wang Z, et al. High-efficiency breeding of early-maturing rice cultivars via CRISPR/Cas9-mediated genome editing. J Genet Genom. 2017;44:1758. <a href="https://doi.org/10.1016/j.jgg.2017.02.001">https://doi.org/10.1016/j.jgg.2017.02.001</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.jgg.2017.02.001" aria-label="View reference 62">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 62 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=High-efficiency%20breeding%20of%20early-maturing%20rice%20cultivars%20via%20CRISPR%2FCas9-mediated%20genome%20editing&amp;journal=J%20Genet%20Genom&amp;doi=10.1016%2Fj.jgg.2017.02.001&amp;volume=44&amp;pages=175-178&amp;publication_year=2017&amp;author=Li%2CX&amp;author=Zhou%2CW&amp;author=Ren%2CY&amp;author=Tian%2CX&amp;author=Lv%2CT&amp;author=Wang%2CZ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S-K. Lee, J-S. Eom, S-K. Hwang, D. Shin, G. An, TW. Okita, J-S. Jeon, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Lee S-K, Eom J-S, Hwang S-K, Shin D, An G, Okita TW, Jeon J-S. Plastidic phosphoglucomutase and ADP-glucose py" /><span class="c-article-references__counter">63.</span><p class="c-article-references__text" id="ref-CR63">Lee S-K, Eom J-S, Hwang S-K, Shin D, An G, Okita TW, Jeon J-S. Plastidic phosphoglucomutase and ADP-glucose pyrophosphorylase mutants impair starch synthesis in rice pollen grains and cause male sterility. J Exp Bot. 2016;67:555769. <a href="https://doi.org/10.1093/jxb/erw324">https://doi.org/10.1093/jxb/erw324</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhvFalurbM" aria-label="View reference 63 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fjxb%2Ferw324" aria-label="View reference 63">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 63 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Plastidic%20phosphoglucomutase%20and%20ADP-glucose%20pyrophosphorylase%20mutants%20impair%20starch%20synthesis%20in%20rice%20pollen%20grains%20and%20cause%20male%20sterility&amp;journal=J%20Exp%20Bot&amp;doi=10.1093%2Fjxb%2Ferw324&amp;volume=67&amp;pages=5557-5569&amp;publication_year=2016&amp;author=Lee%2CS-K&amp;author=Eom%2CJ-S&amp;author=Hwang%2CS-K&amp;author=Shin%2CD&amp;author=An%2CG&amp;author=Okita%2CTW&amp;author=Jeon%2CJ-S">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Q. Li, D. Zhang, M. Chen, W. Liang, J. Wei, Y. Qi, Z. Yuan, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Li Q, Zhang D, Chen M, Liang W, Wei J, Qi Y, Yuan Z. Development of japonica photo-sensitive genic male steril" /><span class="c-article-references__counter">64.</span><p class="c-article-references__text" id="ref-CR64">Li Q, Zhang D, Chen M, Liang W, Wei J, Qi Y, Yuan Z. Development of japonica photo-sensitive genic male sterile rice lines by editing carbon starved anther using CRISPR/Cas9. J Genet Genom. 2016;43:4159. <a href="https://doi.org/10.1016/j.jgg.2016.04.011">https://doi.org/10.1016/j.jgg.2016.04.011</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.jgg.2016.04.011" aria-label="View reference 64">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 64 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20of%20japonica%20photo-sensitive%20genic%20male%20sterile%20rice%20lines%20by%20editing%20carbon%20starved%20anther%20using%20CRISPR%2FCas9&amp;journal=J%20Genet%20Genom&amp;doi=10.1016%2Fj.jgg.2016.04.011&amp;volume=43&amp;pages=415-419&amp;publication_year=2016&amp;author=Li%2CQ&amp;author=Zhang%2CD&amp;author=Chen%2CM&amp;author=Liang%2CW&amp;author=Wei%2CJ&amp;author=Qi%2CY&amp;author=Yuan%2CZ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Xie, B. Niu, Y. Long, G. Li, J. Tang, Y. Zhang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Xie Y, Niu B, Long Y, Li G, Tang J, Zhang Y, et al. Suppression or knockout of SaF/SaM overcomes the Sa-mediat" /><span class="c-article-references__counter">65.</span><p class="c-article-references__text" id="ref-CR65">Xie Y, Niu B, Long Y, Li G, Tang J, Zhang Y, et al. Suppression or knockout of SaF/SaM overcomes the Sa-mediated hybrid male sterility in rice. J Integr Plant Biol. 2017;59:66979. <a href="https://doi.org/10.1111/jipb.12564">https://doi.org/10.1111/jipb.12564</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhsFSmsr7E" aria-label="View reference 65 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fjipb.12564" aria-label="View reference 65">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 65 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Suppression%20or%20knockout%20of%20SaF%2FSaM%20overcomes%20the%20Sa-mediated%20hybrid%20male%20sterility%20in%20rice&amp;journal=J%20Integr%20Plant%20Biol&amp;doi=10.1111%2Fjipb.12564&amp;volume=59&amp;pages=669-679&amp;publication_year=2017&amp;author=Xie%2CY&amp;author=Niu%2CB&amp;author=Long%2CY&amp;author=Li%2CG&amp;author=Tang%2CJ&amp;author=Zhang%2CY">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="H. Zhou, M. He, J. Li, L. Chen, Z. Huang, S. Zheng, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Zhou H, He M, Li J, Chen L, Huang Z, Zheng S, et al. Development of commercial thermo-sensitive genic male ste" /><span class="c-article-references__counter">66.</span><p class="c-article-references__text" id="ref-CR66">Zhou H, He M, Li J, Chen L, Huang Z, Zheng S, et al. Development of commercial thermo-sensitive genic male sterile rice accelerates hybrid rice breeding using the CRISPR/Cas9-mediated TMS5 editing system. Sci Rep. 2016;6:112. <a href="https://doi.org/10.1038/srep37395">https://doi.org/10.1038/srep37395</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XitFSmt7bK" aria-label="View reference 66 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fsrep37395" aria-label="View reference 66">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 66 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20of%20commercial%20thermo-sensitive%20genic%20male%20sterile%20rice%20accelerates%20hybrid%20rice%20breeding%20using%20the%20CRISPR%2FCas9-mediated%20TMS5%20editing%20system&amp;journal=Sci%20Rep&amp;doi=10.1038%2Fsrep37395&amp;volume=6&amp;pages=1-12&amp;publication_year=2016&amp;author=Zhou%2CH&amp;author=He%2CM&amp;author=Li%2CJ&amp;author=Chen%2CL&amp;author=Huang%2CZ&amp;author=Zheng%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="T. Zou, Z. He, L. Qu, M. Liu, J. Zeng, Y. Liang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Zou T, He Z, Qu L, Liu M, Zeng J, Liang Y, et al. Knockout of OsACOS12 caused male sterility in rice. Mol Bree" /><span class="c-article-references__counter">67.</span><p class="c-article-references__text" id="ref-CR67">Zou T, He Z, Qu L, Liu M, Zeng J, Liang Y, et al. Knockout of OsACOS12 caused male sterility in rice. Mol Breed. 2017;37:437. <a href="https://doi.org/10.1007/s11032-017-0722-9">https://doi.org/10.1007/s11032-017-0722-9</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhsFWisr7P" aria-label="View reference 67 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs11032-017-0722-9" aria-label="View reference 67">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 67 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Knockout%20of%20OsACOS12%20caused%20male%20sterility%20in%20rice&amp;journal=Mol%20Breed&amp;doi=10.1007%2Fs11032-017-0722-9&amp;volume=37&amp;publication_year=2017&amp;author=Zou%2CT&amp;author=He%2CZ&amp;author=Qu%2CL&amp;author=Liu%2CM&amp;author=Zeng%2CJ&amp;author=Liang%2CY">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="T. Zou, Q. Xiao, W. Li, T. Luo, G. Yuan, Z. He, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Zou T, Xiao Q, Li W, Luo T, Yuan G, He Z, et al. OsLAP6/OsPKS1, an orthologue of Arabidopsis PKSA/LAP6, is cri" /><span class="c-article-references__counter">68.</span><p class="c-article-references__text" id="ref-CR68">Zou T, Xiao Q, Li W, Luo T, Yuan G, He Z, et al. OsLAP6/OsPKS1, an orthologue of <i>Arabidopsis</i> PKSA/LAP6, is critical for proper pollen exine formation. Rice. 2017;10:615. <a href="https://doi.org/10.1186/s12284-017-0191-0">https://doi.org/10.1186/s12284-017-0191-0</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2Fs12284-017-0191-0" aria-label="View reference 68">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 68 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=OsLAP6%2FOsPKS1%2C%20an%20orthologue%20of%20Arabidopsis%20PKSA%2FLAP6%2C%20is%20critical%20for%20proper%20pollen%20exine%20formation&amp;journal=Rice&amp;doi=10.1186%2Fs12284-017-0191-0&amp;volume=10&amp;publication_year=2017&amp;author=Zou%2CT&amp;author=Xiao%2CQ&amp;author=Li%2CW&amp;author=Luo%2CT&amp;author=Yuan%2CG&amp;author=He%2CZ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="L. Liu, C. Zheng, B. Kuang, L. Wei, L. Yan, T. Wang, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Liu L, Zheng C, Kuang B, Wei L, Yan L, Wang T. Receptor-like kinase RUPO interacts with potassium transporters" /><span class="c-article-references__counter">69.</span><p class="c-article-references__text" id="ref-CR69">Liu L, Zheng C, Kuang B, Wei L, Yan L, Wang T. Receptor-like kinase RUPO interacts with potassium transporters to regulate pollen tube growth and integrity in rice. PLoS Genet. 2016;12:e1006085. <a href="https://doi.org/10.1371/journal.pgen.1006085">https://doi.org/10.1371/journal.pgen.1006085</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhvVaqtLzL" aria-label="View reference 69 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pgen.1006085" aria-label="View reference 69">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 69 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Receptor-like%20kinase%20RUPO%20interacts%20with%20potassium%20transporters%20to%20regulate%20pollen%20tube%20growth%20and%20integrity%20in%20rice&amp;journal=PLoS%20Genet&amp;doi=10.1371%2Fjournal.pgen.1006085&amp;volume=12&amp;publication_year=2016&amp;author=Liu%2CL&amp;author=Zheng%2CC&amp;author=Kuang%2CB&amp;author=Wei%2CL&amp;author=Yan%2CL&amp;author=Wang%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="L. Ma, F. Zhu, Z. Li, J. Zhang, X. Li, J. Dong, T. Wang, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Ma L, Zhu F, Li Z, Zhang J, Li X, Dong J, Wang T. TALEN-based mutagenesis of lipoxygenase LOX3 enhances the st" /><span class="c-article-references__counter">70.</span><p class="c-article-references__text" id="ref-CR70">Ma L, Zhu F, Li Z, Zhang J, Li X, Dong J, Wang T. TALEN-based mutagenesis of lipoxygenase LOX3 enhances the storage tolerance of rice (<i>Oryza sativa</i>) seeds. PLoS ONE. 2015;10:e0143877. <a href="https://doi.org/10.1371/journal.pone.0143877">https://doi.org/10.1371/journal.pone.0143877</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXitVykt7fN" aria-label="View reference 70 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0143877" aria-label="View reference 70">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 70 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=TALEN-based%20mutagenesis%20of%20lipoxygenase%20LOX3%20enhances%20the%20storage%20tolerance%20of%20rice%20%28Oryza%20sativa%29%20seeds&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0143877&amp;volume=10&amp;publication_year=2015&amp;author=Ma%2CL&amp;author=Zhu%2CF&amp;author=Li%2CZ&amp;author=Zhang%2CJ&amp;author=Li%2CX&amp;author=Dong%2CJ&amp;author=Wang%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="W. Qian, C. Wu, Y. Fu, G. Hu, Z. He, W. Liu, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Qian W, Wu C, Fu Y, Hu G, He Z, Liu W. Novel rice mutants overexpressing the brassinosteroid catabolic gene CY" /><span class="c-article-references__counter">71.</span><p class="c-article-references__text" id="ref-CR71">Qian W, Wu C, Fu Y, Hu G, He Z, Liu W. Novel rice mutants overexpressing the brassinosteroid catabolic gene CYP734A4. Plant Mol Biol. 2017;93:197208. <a href="https://doi.org/10.1007/s11103-016-0558-4">https://doi.org/10.1007/s11103-016-0558-4</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhvVShsr%252FM" aria-label="View reference 71 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs11103-016-0558-4" aria-label="View reference 71">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 71 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Novel%20rice%20mutants%20overexpressing%20the%20brassinosteroid%20catabolic%20gene%20CYP734A4&amp;journal=Plant%20Mol%20Biol&amp;doi=10.1007%2Fs11103-016-0558-4&amp;volume=93&amp;pages=197-208&amp;publication_year=2017&amp;author=Qian%2CW&amp;author=Wu%2CC&amp;author=Fu%2CY&amp;author=Hu%2CG&amp;author=He%2CZ&amp;author=Liu%2CW">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Yuan, S. Chen, W. Jiao, L. Wang, L. Wang, W. Ye, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Yuan J, Chen S, Jiao W, Wang L, Wang L, Ye W, et al. Both maternally and paternally imprinted genes regulate s" /><span class="c-article-references__counter">72.</span><p class="c-article-references__text" id="ref-CR72">Yuan J, Chen S, Jiao W, Wang L, Wang L, Ye W, et al. Both maternally and paternally imprinted genes regulate seed development in rice. New Phytol. 2017;216:37387. <a href="https://doi.org/10.1111/nph.14510">https://doi.org/10.1111/nph.14510</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhsFKjs7zL" aria-label="View reference 72 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fnph.14510" aria-label="View reference 72">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 72 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Both%20maternally%20and%20paternally%20imprinted%20genes%20regulate%20seed%20development%20in%20rice&amp;journal=New%20Phytol&amp;doi=10.1111%2Fnph.14510&amp;volume=216&amp;pages=373-387&amp;publication_year=2017&amp;author=Yuan%2CJ&amp;author=Chen%2CS&amp;author=Jiao%2CW&amp;author=Wang%2CL&amp;author=Wang%2CL&amp;author=Ye%2CW">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Lu, J-K. Zhu, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Lu Y, Zhu J-K. Precise editing of a target base in the rice genome using a modified CRISPR/Cas9 system. Mol Pl" /><span class="c-article-references__counter">73.</span><p class="c-article-references__text" id="ref-CR73">Lu Y, Zhu J-K. Precise editing of a target base in the rice genome using a modified CRISPR/Cas9 system. Mol Plant. 2017;10:5235. <a href="https://doi.org/10.1016/j.molp.2016.11.013">https://doi.org/10.1016/j.molp.2016.11.013</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XitFWlsLvJ" aria-label="View reference 73 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molp.2016.11.013" aria-label="View reference 73">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 73 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Precise%20editing%20of%20a%20target%20base%20in%20the%20rice%20genome%20using%20a%20modified%20CRISPR%2FCas9%20system&amp;journal=Mol%20Plant&amp;doi=10.1016%2Fj.molp.2016.11.013&amp;volume=10&amp;pages=523-525&amp;publication_year=2017&amp;author=Lu%2CY&amp;author=Zhu%2CJ-K">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Wang, L. Geng, M. Yuan, J. Wei, C. Jin, M. Li, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Wang Y, Geng L, Yuan M, Wei J, Jin C, Li M, et al. Deletion of a target gene in Indica rice via CRISPR/Cas9. P" /><span class="c-article-references__counter">74.</span><p class="c-article-references__text" id="ref-CR74">Wang Y, Geng L, Yuan M, Wei J, Jin C, Li M, et al. Deletion of a target gene in Indica rice via CRISPR/Cas9. Plant Cell Rep. 2017;36:133343. <a href="https://doi.org/10.1007/s00299-017-2158-4">https://doi.org/10.1007/s00299-017-2158-4</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXpt1Oqs7g%253D" aria-label="View reference 74 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs00299-017-2158-4" aria-label="View reference 74">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 74 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Deletion%20of%20a%20target%20gene%20in%20Indica%20rice%20via%20CRISPR%2FCas9&amp;journal=Plant%20Cell%20Rep&amp;doi=10.1007%2Fs00299-017-2158-4&amp;volume=36&amp;pages=1333-1343&amp;publication_year=2017&amp;author=Wang%2CY&amp;author=Geng%2CL&amp;author=Yuan%2CM&amp;author=Wei%2CJ&amp;author=Jin%2CC&amp;author=Li%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Huang, Y. Guo, Y. Liu, F. Zhang, Z. Wang, H. Wang, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Huang Y, Guo Y, Liu Y, Zhang F, Wang Z, Wang H, et al. 9-cis-Epoxycarotenoid dioxygenase 3 regulates plant gro" /><span class="c-article-references__counter">75.</span><p class="c-article-references__text" id="ref-CR75">Huang Y, Guo Y, Liu Y, Zhang F, Wang Z, Wang H, et al. 9-<i>cis</i>-Epoxycarotenoid dioxygenase 3 regulates plant growth and enhances multi-abiotic stress tolerance in rice. Front Plant Sci. 2018;9:1248. <a href="https://doi.org/10.3389/fpls.2018.00162">https://doi.org/10.3389/fpls.2018.00162</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3389%2Ffpls.2018.00162" aria-label="View reference 75">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 75 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=9-cis-Epoxycarotenoid%20dioxygenase%203%20regulates%20plant%20growth%20and%20enhances%20multi-abiotic%20stress%20tolerance%20in%20rice&amp;journal=Front%20Plant%20Sci&amp;doi=10.3389%2Ffpls.2018.00162&amp;volume=9&amp;publication_year=2018&amp;author=Huang%2CY&amp;author=Guo%2CY&amp;author=Liu%2CY&amp;author=Zhang%2CF&amp;author=Wang%2CZ&amp;author=Wang%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Cai, L. Chen, X. Liu, C. Guo, S. Sun, C. Wu, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Cai Y, Chen L, Liu X, Guo C, Sun S, Wu C, et al. CRISPR/Cas9-mediated targeted mutagenesis of GmFT2a delays fl" /><span class="c-article-references__counter">76.</span><p class="c-article-references__text" id="ref-CR76">Cai Y, Chen L, Liu X, Guo C, Sun S, Wu C, et al. CRISPR/Cas9-mediated targeted mutagenesis of GmFT2a delays flowering time in soya bean. Plant Biotechnol J. 2017. <a href="https://doi.org/10.1111/pbi.12758">https://doi.org/10.1111/pbi.12758</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12758" aria-label="View reference 76">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 76 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FCas9-mediated%20targeted%20mutagenesis%20of%20GmFT2a%20delays%20flowering%20time%20in%20soya%20bean&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12758&amp;publication_year=2017&amp;author=Cai%2CY&amp;author=Chen%2CL&amp;author=Liu%2CX&amp;author=Guo%2CC&amp;author=Sun%2CS&amp;author=Wu%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Liu, P. Merrick, Z. Zhang, C. Ji, B. Yang, S-Z. Fei, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Liu Y, Merrick P, Zhang Z, Ji C, Yang B, Fei S-Z. Targeted mutagenesis in tetraploid switchgrass (Panicum virg" /><span class="c-article-references__counter">77.</span><p class="c-article-references__text" id="ref-CR77">Liu Y, Merrick P, Zhang Z, Ji C, Yang B, Fei S-Z. Targeted mutagenesis in tetraploid switchgrass (<i>Panicum virgatum</i> L.) using CRISPR/Cas9. Plant Biotechnol J. 2018;16:38193. <a href="https://doi.org/10.1111/pbi.12778">https://doi.org/10.1111/pbi.12778</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXitVWgs7w%253D" aria-label="View reference 77 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12778" aria-label="View reference 77">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 77 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20mutagenesis%20in%20tetraploid%20switchgrass%20%28Panicum%20virgatum%20L.%29%20using%20CRISPR%2FCas9&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12778&amp;volume=16&amp;pages=381-393&amp;publication_year=2018&amp;author=Liu%2CY&amp;author=Merrick%2CP&amp;author=Zhang%2CZ&amp;author=Ji%2CC&amp;author=Yang%2CB&amp;author=Fei%2CS-Z">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Ito, A. Nishizawa-Yokoi, M. Endo, M. Mikami, S. Toki, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Ito Y, Nishizawa-Yokoi A, Endo M, Mikami M, Toki S. CRISPR/Cas9-mediated mutagenesis of the RIN locus that reg" /><span class="c-article-references__counter">78.</span><p class="c-article-references__text" id="ref-CR78">Ito Y, Nishizawa-Yokoi A, Endo M, Mikami M, Toki S. CRISPR/Cas9-mediated mutagenesis of the RIN locus that regulates tomato fruit ripening. Biochem Biophys Res Commun. 2015;467:7682. <a href="https://doi.org/10.1016/j.bbrc.2015.09.117">https://doi.org/10.1016/j.bbrc.2015.09.117</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXhsF2gtrfJ" aria-label="View reference 78 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.bbrc.2015.09.117" aria-label="View reference 78">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 78 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FCas9-mediated%20mutagenesis%20of%20the%20RIN%20locus%20that%20regulates%20tomato%20fruit%20ripening&amp;journal=Biochem%20Biophys%20Res%20Commun&amp;doi=10.1016%2Fj.bbrc.2015.09.117&amp;volume=467&amp;pages=76-82&amp;publication_year=2015&amp;author=Ito%2CY&amp;author=Nishizawa-Yokoi%2CA&amp;author=Endo%2CM&amp;author=Mikami%2CM&amp;author=Toki%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="VS. Lor, CG. Starker, DF. Voytas, D. Weiss, NE. Olszewski, " /><meta itemprop="datePublished" content="2014" /><meta itemprop="headline" content="Lor VS, Starker CG, Voytas DF, Weiss D, Olszewski NE. Targeted mutagenesis of the tomato PROCERA gene using tr" /><span class="c-article-references__counter">79.</span><p class="c-article-references__text" id="ref-CR79">Lor VS, Starker CG, Voytas DF, Weiss D, Olszewski NE. Targeted mutagenesis of the tomato PROCERA gene using transcription activator-like effector nucleases. Plant Physiol. 2014;166:128891. <a href="https://doi.org/10.1104/pp.114.247593">https://doi.org/10.1104/pp.114.247593</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXitlOnsg%253D%253D" aria-label="View reference 79 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1104%2Fpp.114.247593" aria-label="View reference 79">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 79 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20mutagenesis%20of%20the%20tomato%20PROCERA%20gene%20using%20transcription%20activator-like%20effector%20nucleases&amp;journal=Plant%20Physiol&amp;doi=10.1104%2Fpp.114.247593&amp;volume=166&amp;pages=1288-1291&amp;publication_year=2014&amp;author=Lor%2CVS&amp;author=Starker%2CCG&amp;author=Voytas%2CDF&amp;author=Weiss%2CD&amp;author=Olszewski%2CNE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Soyk, NA. Mller, SJ. Park, I. Schmalenbach, K. Jiang, R. Hayama, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Soyk S, Mller NA, Park SJ, Schmalenbach I, Jiang K, Hayama R, et al. Variation in the flowering gene SELF PRU" /><span class="c-article-references__counter">80.</span><p class="c-article-references__text" id="ref-CR80">Soyk S, Mller NA, Park SJ, Schmalenbach I, Jiang K, Hayama R, et al. Variation in the flowering gene SELF PRUNING 5G promotes day-neutrality and early yield in tomato. Nat Genet. 2017;49:1628. <a href="https://doi.org/10.1038/ng.3733">https://doi.org/10.1038/ng.3733</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XitVSrurnN" aria-label="View reference 80 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fng.3733" aria-label="View reference 80">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 80 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Variation%20in%20the%20flowering%20gene%20SELF%20PRUNING%205G%20promotes%20day-neutrality%20and%20early%20yield%20in%20tomato&amp;journal=Nat%20Genet&amp;doi=10.1038%2Fng.3733&amp;volume=49&amp;pages=162-168&amp;publication_year=2017&amp;author=Soyk%2CS&amp;author=M%C3%BCller%2CNA&amp;author=Park%2CSJ&amp;author=Schmalenbach%2CI&amp;author=Jiang%2CK&amp;author=Hayama%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2018. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">81.</span><p class="c-article-references__text" id="ref-CR81">United States Department of Agriculture (USDA). 2018. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/18-051-01_air_response_signed.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/18-051-01_air_response_signed.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="D. Rodrguez-Leal, ZH. Lemmon, J. Man, ME. Bartlett, ZB. Lippman, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Rodrguez-Leal D, Lemmon ZH, Man J, Bartlett ME, Lippman ZB. Engineering quantitative trait variation for crop" /><span class="c-article-references__counter">82.</span><p class="c-article-references__text" id="ref-CR82">Rodrguez-Leal D, Lemmon ZH, Man J, Bartlett ME, Lippman ZB. Engineering quantitative trait variation for crop improvement by genome editing. Cell. 2017;171(470480):e8. <a href="https://doi.org/10.1016/j.cell.2017.08.030">https://doi.org/10.1016/j.cell.2017.08.030</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhsFWgu7%252FI" aria-label="View reference 82 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.cell.2017.08.030" aria-label="View reference 82">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 82 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Engineering%20quantitative%20trait%20variation%20for%20crop%20improvement%20by%20genome%20editing&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.08.030&amp;volume=171&amp;issue=470%E2%80%93480&amp;publication_year=2017&amp;author=Rodr%C3%ADguez-Leal%2CD&amp;author=Lemmon%2CZH&amp;author=Man%2CJ&amp;author=Bartlett%2CME&amp;author=Lippman%2CZB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Filler Hayut, C. Melamed Bessudo, AA. Levy, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Filler Hayut S, Melamed Bessudo C, Levy AA. Targeted recombination between homologous chromosomes for precise " /><span class="c-article-references__counter">83.</span><p class="c-article-references__text" id="ref-CR83">Filler Hayut S, Melamed Bessudo C, Levy AA. Targeted recombination between homologous chromosomes for precise breeding in tomato. Nat Commun. 2017;8:15605. <a href="https://doi.org/10.1038/ncomms15605">https://doi.org/10.1038/ncomms15605</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXovVGrsrs%253D" aria-label="View reference 83 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fncomms15605" aria-label="View reference 83">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 83 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20recombination%20between%20homologous%20chromosomes%20for%20precise%20breeding%20in%20tomato&amp;journal=Nat%20Commun&amp;doi=10.1038%2Fncomms15605&amp;volume=8&amp;publication_year=2017&amp;author=Filler%20Hayut%2CS&amp;author=Melamed%20Bessudo%2CC&amp;author=Levy%2CAA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="T. Dahan-Meir, S. Filler-Hayut, C. Melamed-Bessudo, S. Bocobza, H. Czosnek, A. Aharoni, AA. Levy, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Dahan-Meir T, Filler-Hayut S, Melamed-Bessudo C, Bocobza S, Czosnek H, Aharoni A, Levy AA. Efficient in planta" /><span class="c-article-references__counter">84.</span><p class="c-article-references__text" id="ref-CR84">Dahan-Meir T, Filler-Hayut S, Melamed-Bessudo C, Bocobza S, Czosnek H, Aharoni A, Levy AA. Efficient in planta gene targeting in tomato using geminiviral replicons and the CRISPR/Cas9 system. Plant J. 2018;95:516. <a href="https://doi.org/10.1111/tpj.13932">https://doi.org/10.1111/tpj.13932</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXpvVektbs%253D" aria-label="View reference 84 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Ftpj.13932" aria-label="View reference 84">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 84 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20in%20planta%20gene%20targeting%20in%20tomato%20using%20geminiviral%20replicons%20and%20the%20CRISPR%2FCas9%20system&amp;journal=Plant%20J&amp;doi=10.1111%2Ftpj.13932&amp;volume=95&amp;pages=5-16&amp;publication_year=2018&amp;author=Dahan-Meir%2CT&amp;author=Filler-Hayut%2CS&amp;author=Melamed-Bessudo%2CC&amp;author=Bocobza%2CS&amp;author=Czosnek%2CH&amp;author=Aharoni%2CA&amp;author=Levy%2CAA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="L. Deng, H. Wang, C. Sun, Q. Li, H. Jiang, M. Du, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Deng L, Wang H, Sun C, Li Q, Jiang H, Du M, et al. Efficient generation of pink-fruited tomatoes using CRISPR/" /><span class="c-article-references__counter">85.</span><p class="c-article-references__text" id="ref-CR85">Deng L, Wang H, Sun C, Li Q, Jiang H, Du M, et al. Efficient generation of pink-fruited tomatoes using CRISPR/Cas9 system. J Genet Genom. 2018;45:514. <a href="https://doi.org/10.1016/j.jgg.2017.10.002">https://doi.org/10.1016/j.jgg.2017.10.002</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.jgg.2017.10.002" aria-label="View reference 85">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 85 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20generation%20of%20pink-fruited%20tomatoes%20using%20CRISPR%2FCas9%20system&amp;journal=J%20Genet%20Genom&amp;doi=10.1016%2Fj.jgg.2017.10.002&amp;volume=45&amp;pages=51-54&amp;publication_year=2018&amp;author=Deng%2CL&amp;author=Wang%2CH&amp;author=Sun%2CC&amp;author=Li%2CQ&amp;author=Jiang%2CH&amp;author=Du%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="W. Wang, Q. Pan, F. He, A. Akhunova, S. Chao, H. Trick, E. Akhunov, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Wang W, Pan Q, He F, Akhunova A, Chao S, Trick H, Akhunov E. Transgenerational CRISPRCas9 activity facilitate" /><span class="c-article-references__counter">86.</span><p class="c-article-references__text" id="ref-CR86">Wang W, Pan Q, He F, Akhunova A, Chao S, Trick H, Akhunov E. Transgenerational CRISPRCas9 activity facilitates multiplex gene editing in allopolyploid wheat. CRISPR J. 2018;1:6574. <a href="https://doi.org/10.1089/crispr.2017.0010">https://doi.org/10.1089/crispr.2017.0010</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXhsVKgt77M" aria-label="View reference 86 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1089%2Fcrispr.2017.0010" aria-label="View reference 86">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 86 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Transgenerational%20CRISPR%E2%80%93Cas9%20activity%20facilitates%20multiplex%20gene%20editing%20in%20allopolyploid%20wheat&amp;journal=CRISPR%20J&amp;doi=10.1089%2Fcrispr.2017.0010&amp;volume=1&amp;pages=65-74&amp;publication_year=2018&amp;author=Wang%2CW&amp;author=Pan%2CQ&amp;author=He%2CF&amp;author=Akhunova%2CA&amp;author=Chao%2CS&amp;author=Trick%2CH&amp;author=Akhunov%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Zhang, D. Li, D. Zhang, X. Zhao, X. Cao, L. Dong, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Zhang Y, Li D, Zhang D, Zhao X, Cao X, Dong L, et al. Analysis of the functions of TaGW2 homoeologs in wheat g" /><span class="c-article-references__counter">87.</span><p class="c-article-references__text" id="ref-CR87">Zhang Y, Li D, Zhang D, Zhao X, Cao X, Dong L, et al. Analysis of the functions of TaGW2 homoeologs in wheat grain weight and protein content traits. Plant J. 2018;94:85766. <a href="https://doi.org/10.1111/tpj.13903">https://doi.org/10.1111/tpj.13903</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXot1Cqsrg%253D" aria-label="View reference 87 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Ftpj.13903" aria-label="View reference 87">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 87 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20the%20functions%20of%20TaGW2%20homoeologs%20in%20wheat%20grain%20weight%20and%20protein%20content%20traits&amp;journal=Plant%20J&amp;doi=10.1111%2Ftpj.13903&amp;volume=94&amp;pages=857-866&amp;publication_year=2018&amp;author=Zhang%2CY&amp;author=Li%2CD&amp;author=Zhang%2CD&amp;author=Zhao%2CX&amp;author=Cao%2CX&amp;author=Dong%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Zhou, G. Wang, Z. Liu, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Zhou J, Wang G, Liu Z. Efficient genome editing of wild strawberry genes, vector development and validation. P" /><span class="c-article-references__counter">88.</span><p class="c-article-references__text" id="ref-CR88">Zhou J, Wang G, Liu Z. Efficient genome editing of wild strawberry genes, vector development and validation. Plant Biotechnol J. 2018;166:1292. <a href="https://doi.org/10.1111/pbi.12922">https://doi.org/10.1111/pbi.12922</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhvFSjurbM" aria-label="View reference 88 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12922" aria-label="View reference 88">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 88 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20genome%20editing%20of%20wild%20strawberry%20genes%2C%20vector%20development%20and%20validation&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12922&amp;volume=166&amp;publication_year=2018&amp;author=Zhou%2CJ&amp;author=Wang%2CG&amp;author=Liu%2CZ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2017. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">89.</span><p class="c-article-references__text" id="ref-CR89">United States Department of Agriculture (USDA). 2017. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-038-02_air_inquiry_cbidel.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-038-02_air_inquiry_cbidel.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="ME. Ozseyhan, J. Kang, X. Mu, C. Lu, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Ozseyhan ME, Kang J, Mu X, Lu C. Mutagenesis of the FAE1 genes significantly changes fatty acid composition in" /><span class="c-article-references__counter">90.</span><p class="c-article-references__text" id="ref-CR90">Ozseyhan ME, Kang J, Mu X, Lu C. Mutagenesis of the FAE1 genes significantly changes fatty acid composition in seeds of <i>Camelina sativa</i>. Plant Physiol Biochem. 2018;123:17. <a href="https://doi.org/10.1016/j.plaphy.2017.11.021">https://doi.org/10.1016/j.plaphy.2017.11.021</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhvFWmu77P" aria-label="View reference 90 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.plaphy.2017.11.021" aria-label="View reference 90">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 90 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Mutagenesis%20of%20the%20FAE1%20genes%20significantly%20changes%20fatty%20acid%20composition%20in%20seeds%20of%20Camelina%20sativa&amp;journal=Plant%20Physiol%20Biochem&amp;doi=10.1016%2Fj.plaphy.2017.11.021&amp;volume=123&amp;pages=1-7&amp;publication_year=2018&amp;author=Ozseyhan%2CME&amp;author=Kang%2CJ&amp;author=Mu%2CX&amp;author=Lu%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="WZ. Jiang, IM. Henry, PG. Lynagh, L. Comai, EB. Cahoon, DP. Weeks, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Jiang WZ, Henry IM, Lynagh PG, Comai L, Cahoon EB, Weeks DP. Significant enhancement of fatty acid composition" /><span class="c-article-references__counter">91.</span><p class="c-article-references__text" id="ref-CR91">Jiang WZ, Henry IM, Lynagh PG, Comai L, Cahoon EB, Weeks DP. Significant enhancement of fatty acid composition in seeds of the allohexaploid, <i>Camelina sativa</i>, using CRISPR/Cas9 gene editing. Plant Biotechnol J. 2017;15:64857. <a href="https://doi.org/10.1111/pbi.12663">https://doi.org/10.1111/pbi.12663</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXmsVSls78%253D" aria-label="View reference 91 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12663" aria-label="View reference 91">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 91 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Significant%20enhancement%20of%20fatty%20acid%20composition%20in%20seeds%20of%20the%20allohexaploid%2C%20Camelina%20sativa%2C%20using%20CRISPR%2FCas9%20gene%20editing&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12663&amp;volume=15&amp;pages=648-657&amp;publication_year=2017&amp;author=Jiang%2CWZ&amp;author=Henry%2CIM&amp;author=Lynagh%2CPG&amp;author=Comai%2CL&amp;author=Cahoon%2CEB&amp;author=Weeks%2CDP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="C. Morineau, Y. Bellec, F. Tellier, L. Gissot, Z. Kelemen, F. Nogu, J-D. Faure, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Morineau C, Bellec Y, Tellier F, Gissot L, Kelemen Z, Nogu F, Faure J-D. Selective gene dosage by CRISPRCas9" /><span class="c-article-references__counter">92.</span><p class="c-article-references__text" id="ref-CR92">Morineau C, Bellec Y, Tellier F, Gissot L, Kelemen Z, Nogu F, Faure J-D. Selective gene dosage by CRISPRCas9 genome editing in hexaploid <i>Camelina sativa</i>. Plant Biotechnol J. 2017;15:72939. <a href="https://doi.org/10.1111/pbi.12671">https://doi.org/10.1111/pbi.12671</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXnsV2ht7w%253D" aria-label="View reference 92 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12671" aria-label="View reference 92">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 92 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Selective%20gene%20dosage%20by%20CRISPR%E2%80%93Cas9%20genome%20editing%20in%20hexaploid%20Camelina%20sativa&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12671&amp;volume=15&amp;pages=729-739&amp;publication_year=2017&amp;author=Morineau%2CC&amp;author=Bellec%2CY&amp;author=Tellier%2CF&amp;author=Gissot%2CL&amp;author=Kelemen%2CZ&amp;author=Nogu%C3%A9%2CF&amp;author=Faure%2CJ-D">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="JA. Aznar-Moreno, TP. Durrett, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Aznar-Moreno JA, Durrett TP. Simultaneous targeting of multiple gene homeologs to alter seed oil production in" /><span class="c-article-references__counter">93.</span><p class="c-article-references__text" id="ref-CR93">Aznar-Moreno JA, Durrett TP. Simultaneous targeting of multiple gene homeologs to alter seed oil production in <i>Camelina sativa</i>. Plant Cell Physiol. 2017;58:12607. <a href="https://doi.org/10.1093/pcp/pcx058">https://doi.org/10.1093/pcp/pcx058</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhsFWms7jL" aria-label="View reference 93 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1093%2Fpcp%2Fpcx058" aria-label="View reference 93">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 93 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Simultaneous%20targeting%20of%20multiple%20gene%20homeologs%20to%20alter%20seed%20oil%20production%20in%20Camelina%20sativa&amp;journal=Plant%20Cell%20Physiol&amp;doi=10.1093%2Fpcp%2Fpcx058&amp;volume=58&amp;pages=1260-1267&amp;publication_year=2017&amp;author=Aznar-Moreno%2CJA&amp;author=Durrett%2CTP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="A. Okuzaki, T. Ogawa, C. Koizuka, K. Kaneko, M. Inaba, J. Imamura, N. Koizuka, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Okuzaki A, Ogawa T, Koizuka C, Kaneko K, Inaba M, Imamura J, Koizuka N. CRISPR/Cas9-mediated genome editing of" /><span class="c-article-references__counter">94.</span><p class="c-article-references__text" id="ref-CR94">Okuzaki A, Ogawa T, Koizuka C, Kaneko K, Inaba M, Imamura J, Koizuka N. CRISPR/Cas9-mediated genome editing of the fatty acid desaturase 2 gene in <i>Brassica napus</i>. Plant Physiol Biochem. 2018. <a href="https://doi.org/10.1016/j.plaphy.2018.04.025">https://doi.org/10.1016/j.plaphy.2018.04.025</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.plaphy.2018.04.025" aria-label="View reference 94">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 94 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FCas9-mediated%20genome%20editing%20of%20the%20fatty%20acid%20desaturase%202%20gene%20in%20Brassica%20napus&amp;journal=Plant%20Physiol%20Biochem&amp;doi=10.1016%2Fj.plaphy.2018.04.025&amp;publication_year=2018&amp;author=Okuzaki%2CA&amp;author=Ogawa%2CT&amp;author=Koizuka%2CC&amp;author=Kaneko%2CK&amp;author=Inaba%2CM&amp;author=Imamura%2CJ&amp;author=Koizuka%2CN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2015. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">95.</span><p class="c-article-references__text" id="ref-CR95">United States Department of Agriculture (USDA). 2015. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-352-01_air_inquiry_cbidel.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-352-01_air_inquiry_cbidel.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X. Qi, L. Dong, C. Liu, L. Mao, F. Liu, X. Zhang, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Qi X, Dong L, Liu C, Mao L, Liu F, Zhang X, et al. Systematic identification of endogenous RNA polymerase III " /><span class="c-article-references__counter">96.</span><p class="c-article-references__text" id="ref-CR96">Qi X, Dong L, Liu C, Mao L, Liu F, Zhang X, et al. Systematic identification of endogenous RNA polymerase III promoters for efficient RNA guide-based genome editing technologies in maize. Crop J. 2018;6:31420. <a href="https://doi.org/10.1016/j.cj.2018.02.005">https://doi.org/10.1016/j.cj.2018.02.005</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.cj.2018.02.005" aria-label="View reference 96">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 96 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20identification%20of%20endogenous%20RNA%20polymerase%20III%20promoters%20for%20efficient%20RNA%20guide-based%20genome%20editing%20technologies%20in%20maize&amp;journal=Crop%20J&amp;doi=10.1016%2Fj.cj.2018.02.005&amp;volume=6&amp;pages=314-320&amp;publication_year=2018&amp;author=Qi%2CX&amp;author=Dong%2CL&amp;author=Liu%2CC&amp;author=Mao%2CL&amp;author=Liu%2CF&amp;author=Zhang%2CX">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2015. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">97.</span><p class="c-article-references__text" id="ref-CR97">United States Department of Agriculture (USDA). 2015. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-078-02_air_inquiry.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-078-02_air_inquiry.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2010. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">98.</span><p class="c-article-references__text" id="ref-CR98">United States Department of Agriculture (USDA). 2010. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/DOW_Email_%20to_Susan_%20Kohler_032010.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/DOW_Email_%20to_Susan_%20Kohler_032010.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="VK. Shukla, Y. Doyon, JC. Miller, RC. DeKelver, EA. Moehle, SE. Worden, " /><meta itemprop="datePublished" content="2009" /><meta itemprop="headline" content="Shukla VK, Doyon Y, Miller JC, DeKelver RC, Moehle EA, Worden SE, et al. Precise genome modification in the cr" /><span class="c-article-references__counter">99.</span><p class="c-article-references__text" id="ref-CR99">Shukla VK, Doyon Y, Miller JC, DeKelver RC, Moehle EA, Worden SE, et al. Precise genome modification in the crop species Zea mays using zinc-finger nucleases. Nature. 2009;459:43741. <a href="https://doi.org/10.1038/nature07992">https://doi.org/10.1038/nature07992</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXlt1Srs70%253D" aria-label="View reference 99 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnature07992" aria-label="View reference 99">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 99 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Precise%20genome%20modification%20in%20the%20crop%20species%20Zea%20mays%20using%20zinc-finger%20nucleases&amp;journal=Nature&amp;doi=10.1038%2Fnature07992&amp;volume=459&amp;pages=437-441&amp;publication_year=2009&amp;author=Shukla%2CVK&amp;author=Doyon%2CY&amp;author=Miller%2CJC&amp;author=DeKelver%2CRC&amp;author=Moehle%2CEA&amp;author=Worden%2CSE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2015. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">100.</span><p class="c-article-references__text" id="ref-CR100">United States Department of Agriculture (USDA). 2015. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-321-01_air_inquiry.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-321-01_air_inquiry.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Alagoz, T. Gurkok, B. Zhang, T. Unver, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Alagoz Y, Gurkok T, Zhang B, Unver T. Manipulating the biosynthesis of bioactive compound alkaloids for next-g" /><span class="c-article-references__counter">101.</span><p class="c-article-references__text" id="ref-CR101">Alagoz Y, Gurkok T, Zhang B, Unver T. Manipulating the biosynthesis of bioactive compound alkaloids for next-generation metabolic engineering in opium poppy using CRISPRCas 9 genome editing technology. Sci Rep. 2016;6:309108. <a href="https://doi.org/10.1038/srep30910">https://doi.org/10.1038/srep30910</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhtlagsrzL" aria-label="View reference 101 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fsrep30910" aria-label="View reference 101">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 101 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Manipulating%20the%20biosynthesis%20of%20bioactive%20compound%20alkaloids%20for%20next-generation%20metabolic%20engineering%20in%20opium%20poppy%20using%20CRISPR%E2%80%93Cas%209%20genome%20editing%20technology&amp;journal=Sci%20Rep&amp;doi=10.1038%2Fsrep30910&amp;volume=6&amp;pages=30910-30918&amp;publication_year=2016&amp;author=Alagoz%2CY&amp;author=Gurkok%2CT&amp;author=Zhang%2CB&amp;author=Unver%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Wen, H. Liu, X. Li, X. Chen, Y. Hong, H. Li, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Wen S, Liu H, Li X, Chen X, Hong Y, Li H, et al. TALEN-mediated targeted mutagenesis of fatty acid desaturase " /><span class="c-article-references__counter">102.</span><p class="c-article-references__text" id="ref-CR102">Wen S, Liu H, Li X, Chen X, Hong Y, Li H, et al. TALEN-mediated targeted mutagenesis of fatty acid desaturase 2 (FAD2) in peanut (<i>Arachis hypogaea</i> L.) promotes the accumulation of oleic acid. Plant Mol Biol. 2018;97:17785. <a href="https://doi.org/10.1007/s11103-018-0731-z">https://doi.org/10.1007/s11103-018-0731-z</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXosVWrtbg%253D" aria-label="View reference 102 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs11103-018-0731-z" aria-label="View reference 102">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 102 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=TALEN-mediated%20targeted%20mutagenesis%20of%20fatty%20acid%20desaturase%202%20%28FAD2%29%20in%20peanut%20%28Arachis%20hypogaea%20L.%29%20promotes%20the%20accumulation%20of%20oleic%20acid&amp;journal=Plant%20Mol%20Biol&amp;doi=10.1007%2Fs11103-018-0731-z&amp;volume=97&amp;pages=177-185&amp;publication_year=2018&amp;author=Wen%2CS&amp;author=Liu%2CH&amp;author=Li%2CX&amp;author=Chen%2CX&amp;author=Hong%2CY&amp;author=Li%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2016. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">103.</span><p class="c-article-references__text" id="ref-CR103">United States Department of Agriculture (USDA). 2016. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/16-090-01_air_inquiry_cbidel.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/16-090-01_air_inquiry_cbidel.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2016. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">104.</span><p class="c-article-references__text" id="ref-CR104">United States Department of Agriculture (USDA). 2016. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/16-320-01_air_inquiry.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/16-320-01_air_inquiry.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Sawai, K. Ohyama, S. Yasumoto, H. Seki, T. Sakuma, T. Yamamoto, " /><meta itemprop="datePublished" content="2014" /><meta itemprop="headline" content="Sawai S, Ohyama K, Yasumoto S, Seki H, Sakuma T, Yamamoto T, et al. Sterol side chain reductase 2 is a key enz" /><span class="c-article-references__counter">105.</span><p class="c-article-references__text" id="ref-CR105">Sawai S, Ohyama K, Yasumoto S, Seki H, Sakuma T, Yamamoto T, et al. Sterol side chain reductase 2 is a key enzyme in the biosynthesis of cholesterol, the common precursor of toxic steroidal glycoalkaloids in potato. Plant Cell. 2014;26:376374. <a href="https://doi.org/10.1105/tpc.114.130096">https://doi.org/10.1105/tpc.114.130096</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhvFGhsL3O" aria-label="View reference 105 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1105%2Ftpc.114.130096" aria-label="View reference 105">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 105 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Sterol%20side%20chain%20reductase%202%20is%20a%20key%20enzyme%20in%20the%20biosynthesis%20of%20cholesterol%2C%20the%20common%20precursor%20of%20toxic%20steroidal%20glycoalkaloids%20in%20potato&amp;journal=Plant%20Cell&amp;doi=10.1105%2Ftpc.114.130096&amp;volume=26&amp;pages=3763-3774&amp;publication_year=2014&amp;author=Sawai%2CS&amp;author=Ohyama%2CK&amp;author=Yasumoto%2CS&amp;author=Seki%2CH&amp;author=Sakuma%2CT&amp;author=Yamamoto%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="M. Nakayasu, R. Akiyama, HJ. Lee, K. Osakabe, Y. Osakabe, B. Watanabe, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Nakayasu M, Akiyama R, Lee HJ, Osakabe K, Osakabe Y, Watanabe B, et al. Generation of -solanine-free hairy ro" /><span class="c-article-references__counter">106.</span><p class="c-article-references__text" id="ref-CR106">Nakayasu M, Akiyama R, Lee HJ, Osakabe K, Osakabe Y, Watanabe B, et al. Generation of -solanine-free hairy roots of potato by CRISPR/Cas9 mediated genome editing of the St16DOX gene. Plant Physiol Biochem. 2018. <a href="https://doi.org/10.1016/j.plaphy.2018.04.026">https://doi.org/10.1016/j.plaphy.2018.04.026</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.plaphy.2018.04.026" aria-label="View reference 106">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 106 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Generation%20of%20%CE%B1-solanine-free%20hairy%20roots%20of%20potato%20by%20CRISPR%2FCas9%20mediated%20genome%20editing%20of%20the%20St16DOX%20gene&amp;journal=Plant%20Physiol%20Biochem&amp;doi=10.1016%2Fj.plaphy.2018.04.026&amp;publication_year=2018&amp;author=Nakayasu%2CM&amp;author=Akiyama%2CR&amp;author=Lee%2CHJ&amp;author=Osakabe%2CK&amp;author=Osakabe%2CY&amp;author=Watanabe%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Q. Shan, Y. Zhang, K. Chen, K. Zhang, C. Gao, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Shan Q, Zhang Y, Chen K, Zhang K, Gao C. Creation of fragrant rice by targeted knockout of the OsBADH2 gene us" /><span class="c-article-references__counter">107.</span><p class="c-article-references__text" id="ref-CR107">Shan Q, Zhang Y, Chen K, Zhang K, Gao C. Creation of fragrant rice by targeted knockout of the OsBADH2 gene using TALEN technology. Plant Biotechnol J. 2015;13:791800. <a href="https://doi.org/10.1111/pbi.12312">https://doi.org/10.1111/pbi.12312</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXhtFGmu7vI" aria-label="View reference 107 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12312" aria-label="View reference 107">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 107 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Creation%20of%20fragrant%20rice%20by%20targeted%20knockout%20of%20the%20OsBADH2%20gene%20using%20TALEN%20technology&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12312&amp;volume=13&amp;pages=791-800&amp;publication_year=2015&amp;author=Shan%2CQ&amp;author=Zhang%2CY&amp;author=Chen%2CK&amp;author=Zhang%2CK&amp;author=Gao%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Sun, G. Jiao, Z. Liu, X. Zhang, J. Li, X. Guo, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Sun Y, Jiao G, Liu Z, Zhang X, Li J, Guo X, et al. Generation of high-amylose rice through CRISPR/Cas9-mediate" /><span class="c-article-references__counter">108.</span><p class="c-article-references__text" id="ref-CR108">Sun Y, Jiao G, Liu Z, Zhang X, Li J, Guo X, et al. Generation of high-amylose rice through CRISPR/Cas9-mediated targeted mutagenesis of starch branching enzymes. Front Plant Sci. 2017;8:115. <a href="https://doi.org/10.3389/fpls.2017.00298">https://doi.org/10.3389/fpls.2017.00298</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3389%2Ffpls.2017.00298" aria-label="View reference 108">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 108 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Generation%20of%20high-amylose%20rice%20through%20CRISPR%2FCas9-mediated%20targeted%20mutagenesis%20of%20starch%20branching%20enzymes&amp;journal=Front%20Plant%20Sci&amp;doi=10.3389%2Ffpls.2017.00298&amp;volume=8&amp;pages=1-15&amp;publication_year=2017&amp;author=Sun%2CY&amp;author=Jiao%2CG&amp;author=Liu%2CZ&amp;author=Zhang%2CX&amp;author=Li%2CJ&amp;author=Guo%2CX">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Ye, P. Li, T. Xu, L. Zeng, D. Cheng, M. Yang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Ye Y, Li P, Xu T, Zeng L, Cheng D, Yang M, et al. OsPT4 contributes to arsenate uptake and transport in rice. " /><span class="c-article-references__counter">109.</span><p class="c-article-references__text" id="ref-CR109">Ye Y, Li P, Xu T, Zeng L, Cheng D, Yang M, et al. OsPT4 contributes to arsenate uptake and transport in rice. Front Plant Sci. 2017;8:311. <a href="https://doi.org/10.3389/fpls.2017.02197">https://doi.org/10.3389/fpls.2017.02197</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3389%2Ffpls.2017.02197" aria-label="View reference 109">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 109 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=OsPT4%20contributes%20to%20arsenate%20uptake%20and%20transport%20in%20rice&amp;journal=Front%20Plant%20Sci&amp;doi=10.3389%2Ffpls.2017.02197&amp;volume=8&amp;publication_year=2017&amp;author=Ye%2CY&amp;author=Li%2CP&amp;author=Xu%2CT&amp;author=Zeng%2CL&amp;author=Cheng%2CD&amp;author=Yang%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="F-Z. Wang, M-X. Chen, L-J. Yu, L-J. Xie, L-B. Yuan, H. Qi, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Wang F-Z, Chen M-X, Yu L-J, Xie L-J, Yuan L-B, Qi H, et al. OsARM1, an R2R3 MYB transcription factor, is invol" /><span class="c-article-references__counter">110.</span><p class="c-article-references__text" id="ref-CR110">Wang F-Z, Chen M-X, Yu L-J, Xie L-J, Yuan L-B, Qi H, et al. OsARM1, an R2R3 MYB transcription factor, is involved in regulation of the response to arsenic stress in rice. Front Plant Sci. 2017;8:1868. <a href="https://doi.org/10.3389/fpls.2017.01868">https://doi.org/10.3389/fpls.2017.01868</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3389%2Ffpls.2017.01868" aria-label="View reference 110">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 110 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=OsARM1%2C%20an%20R2R3%20MYB%20transcription%20factor%2C%20is%20involved%20in%20regulation%20of%20the%20response%20to%20arsenic%20stress%20in%20rice&amp;journal=Front%20Plant%20Sci&amp;doi=10.3389%2Ffpls.2017.01868&amp;volume=8&amp;publication_year=2017&amp;author=Wang%2CF-Z&amp;author=Chen%2CM-X&amp;author=Yu%2CL-J&amp;author=Xie%2CL-J&amp;author=Yuan%2CL-B&amp;author=Qi%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="K. Abe, E. Araki, Y. Suzuki, S. Toki, H. Saika, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Abe K, Araki E, Suzuki Y, Toki S, Saika H. Production of high oleic/low linoleic rice by genome editing. Plant" /><span class="c-article-references__counter">111.</span><p class="c-article-references__text" id="ref-CR111">Abe K, Araki E, Suzuki Y, Toki S, Saika H. Production of high oleic/low linoleic rice by genome editing. Plant Physiol Biochem. 2018. <a href="https://doi.org/10.1016/j.plaphy.2018.04.033">https://doi.org/10.1016/j.plaphy.2018.04.033</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.plaphy.2018.04.033" aria-label="View reference 111">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 111 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Production%20of%20high%20oleic%2Flow%20linoleic%20rice%20by%20genome%20editing&amp;journal=Plant%20Physiol%20Biochem&amp;doi=10.1016%2Fj.plaphy.2018.04.033&amp;publication_year=2018&amp;author=Abe%2CK&amp;author=Araki%2CE&amp;author=Suzuki%2CY&amp;author=Toki%2CS&amp;author=Saika%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="M. Nieves-Cordones, S. Mohamed, K. Tanoi, NI. Kobayashi, K. Takagi, A. Vernet, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Nieves-Cordones M, Mohamed S, Tanoi K, Kobayashi NI, Takagi K, Vernet A, et al. Production of low-Cs+ rice pla" /><span class="c-article-references__counter">112.</span><p class="c-article-references__text" id="ref-CR112">Nieves-Cordones M, Mohamed S, Tanoi K, Kobayashi NI, Takagi K, Vernet A, et al. Production of low-Cs+ rice plants by inactivation of the K+ transporter OsHAK1 with the CRISPRCas system. Plant J. 2017;92:4356. <a href="https://doi.org/10.1111/tpj.13632">https://doi.org/10.1111/tpj.13632</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhtlSrur7N" aria-label="View reference 112 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Ftpj.13632" aria-label="View reference 112">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 112 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Production%20of%20low-Cs%2B%20rice%20plants%20by%20inactivation%20of%20the%20K%2B%20transporter%20OsHAK1%20with%20the%20CRISPR%E2%80%93Cas%20system&amp;journal=Plant%20J&amp;doi=10.1111%2Ftpj.13632&amp;volume=92&amp;pages=43-56&amp;publication_year=2017&amp;author=Nieves-Cordones%2CM&amp;author=Mohamed%2CS&amp;author=Tanoi%2CK&amp;author=Kobayashi%2CNI&amp;author=Takagi%2CK&amp;author=Vernet%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="L. Tang, B. Mao, Y. Li, Q. Lv, L. Zhang, C. Chen, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Tang L, Mao B, Li Y, Lv Q, Zhang L, Chen C, et al. Knockout of OsNramp5 using the CRISPR/Cas9 system produces " /><span class="c-article-references__counter">113.</span><p class="c-article-references__text" id="ref-CR113">Tang L, Mao B, Li Y, Lv Q, Zhang L, Chen C, et al. Knockout of OsNramp5 using the CRISPR/Cas9 system produces low Cd-accumulating indica rice without compromising yield. Sci Rep. 2017;7:14438. <a href="https://doi.org/10.1038/s41598-017-14832-9">https://doi.org/10.1038/s41598-017-14832-9</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhsVyks7fJ" aria-label="View reference 113 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fs41598-017-14832-9" aria-label="View reference 113">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 113 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Knockout%20of%20OsNramp5%20using%20the%20CRISPR%2FCas9%20system%20produces%20low%20Cd-accumulating%20indica%20rice%20without%20compromising%20yield&amp;journal=Sci%20Rep&amp;doi=10.1038%2Fs41598-017-14832-9&amp;volume=7&amp;publication_year=2017&amp;author=Tang%2CL&amp;author=Mao%2CB&amp;author=Li%2CY&amp;author=Lv%2CQ&amp;author=Zhang%2CL&amp;author=Chen%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Zhang, H. Zhang, JR. Botella, J-K. Zhu, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Zhang J, Zhang H, Botella JR, Zhu J-K. Generation of new glutinous rice by CRISPR/Cas9-targeted mutagenesis of" /><span class="c-article-references__counter">114.</span><p class="c-article-references__text" id="ref-CR114">Zhang J, Zhang H, Botella JR, Zhu J-K. Generation of new glutinous rice by CRISPR/Cas9-targeted mutagenesis of the Waxy gene in elite rice varieties. J Integr Plant Biol. 2018;60:36975. <a href="https://doi.org/10.1111/jipb.12620">https://doi.org/10.1111/jipb.12620</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXptlOruro%253D" aria-label="View reference 114 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fjipb.12620" aria-label="View reference 114">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 114 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Generation%20of%20new%20glutinous%20rice%20by%20CRISPR%2FCas9-targeted%20mutagenesis%20of%20the%20Waxy%20gene%20in%20elite%20rice%20varieties&amp;journal=J%20Integr%20Plant%20Biol&amp;doi=10.1111%2Fjipb.12620&amp;volume=60&amp;pages=369-375&amp;publication_year=2018&amp;author=Zhang%2CJ&amp;author=Zhang%2CH&amp;author=Botella%2CJR&amp;author=Zhu%2CJ-K">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Z. Zhou, H. Tan, Q. Li, J. Chen, S. Gao, Y. Wang, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Zhou Z, Tan H, Li Q, Chen J, Gao S, Wang Y, et al. CRISPR/Cas9-mediated efficient targeted mutagenesis of RAS " /><span class="c-article-references__counter">115.</span><p class="c-article-references__text" id="ref-CR115">Zhou Z, Tan H, Li Q, Chen J, Gao S, Wang Y, et al. CRISPR/Cas9-mediated efficient targeted mutagenesis of RAS in <i>Salvia miltiorrhiza</i>. Phytochemistry. 2018;148:6370. <a href="https://doi.org/10.1016/j.phytochem.2018.01.015">https://doi.org/10.1016/j.phytochem.2018.01.015</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXisVamtrY%253D" aria-label="View reference 115 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.phytochem.2018.01.015" aria-label="View reference 115">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 115 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FCas9-mediated%20efficient%20targeted%20mutagenesis%20of%20RAS%20in%20Salvia%20miltiorrhiza&amp;journal=Phytochemistry&amp;doi=10.1016%2Fj.phytochem.2018.01.015&amp;volume=148&amp;pages=63-70&amp;publication_year=2018&amp;author=Zhou%2CZ&amp;author=Tan%2CH&amp;author=Li%2CQ&amp;author=Chen%2CJ&amp;author=Gao%2CS&amp;author=Wang%2CY">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2014. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">116.</span><p class="c-article-references__text" id="ref-CR116">United States Department of Agriculture (USDA). 2014. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/cellectis_air_fad2k0_soy_cbidel.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/cellectis_air_fad2k0_soy_cbidel.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2015. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">117.</span><p class="c-article-references__text" id="ref-CR117">United States Department of Agriculture (USDA). 2015. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-071-01air.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-071-01air.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="W. Haun, A. Coffman, BM. Clasen, ZL. Demorest, A. Lowy, E. Ray, " /><meta itemprop="datePublished" content="2014" /><meta itemprop="headline" content="Haun W, Coffman A, Clasen BM, Demorest ZL, Lowy A, Ray E, et al. Improved soybean oil quality by targeted muta" /><span class="c-article-references__counter">118.</span><p class="c-article-references__text" id="ref-CR118">Haun W, Coffman A, Clasen BM, Demorest ZL, Lowy A, Ray E, et al. Improved soybean oil quality by targeted mutagenesis of the fatty acid desaturase 2 gene family. Plant Biotechnol J. 2014;12:93440. <a href="https://doi.org/10.1111/pbi.12201">https://doi.org/10.1111/pbi.12201</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhsVSrtL%252FF" aria-label="View reference 118 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12201" aria-label="View reference 118">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 118 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20soybean%20oil%20quality%20by%20targeted%20mutagenesis%20of%20the%20fatty%20acid%20desaturase%202%20gene%20family&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12201&amp;volume=12&amp;pages=934-940&amp;publication_year=2014&amp;author=Haun%2CW&amp;author=Coffman%2CA&amp;author=Clasen%2CBM&amp;author=Demorest%2CZL&amp;author=Lowy%2CA&amp;author=Ray%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="C. Klap, E. Yeshayahou, AM. Bolger, T. Arazi, SK. Gupta, S. Shabtai, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Klap C, Yeshayahou E, Bolger AM, Arazi T, Gupta SK, Shabtai S, et al. Tomato facultative parthenocarpy results" /><span class="c-article-references__counter">119.</span><p class="c-article-references__text" id="ref-CR119">Klap C, Yeshayahou E, Bolger AM, Arazi T, Gupta SK, Shabtai S, et al. Tomato facultative parthenocarpy results from SlAGAMOUS-LIKE 6 loss of function. Plant Biotechnol J. 2017;15:63447. <a href="https://doi.org/10.1111/pbi.12662">https://doi.org/10.1111/pbi.12662</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXmsVSls74%253D" aria-label="View reference 119 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12662" aria-label="View reference 119">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 119 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Tomato%20facultative%20parthenocarpy%20results%20from%20SlAGAMOUS-LIKE%206%20loss%20of%20function&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12662&amp;volume=15&amp;pages=634-647&amp;publication_year=2017&amp;author=Klap%2CC&amp;author=Yeshayahou%2CE&amp;author=Bolger%2CAM&amp;author=Arazi%2CT&amp;author=Gupta%2CSK&amp;author=Shabtai%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="R. Ueta, C. Abe, T. Watanabe, SS. Sugano, R. Ishihara, H. Ezura, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Ueta R, Abe C, Watanabe T, Sugano SS, Ishihara R, Ezura H, et al. Rapid breeding of parthenocarpic tomato plan" /><span class="c-article-references__counter">120.</span><p class="c-article-references__text" id="ref-CR120">Ueta R, Abe C, Watanabe T, Sugano SS, Ishihara R, Ezura H, et al. Rapid breeding of parthenocarpic tomato plants using CRISPR/Cas9. Sci Rep. 2017;7:507. <a href="https://doi.org/10.1038/s41598-017-00501-4">https://doi.org/10.1038/s41598-017-00501-4</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhs1KrtrjO" aria-label="View reference 120 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fs41598-017-00501-4" aria-label="View reference 120">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 120 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20breeding%20of%20parthenocarpic%20tomato%20plants%20using%20CRISPR%2FCas9&amp;journal=Sci%20Rep&amp;doi=10.1038%2Fs41598-017-00501-4&amp;volume=7&amp;publication_year=2017&amp;author=Ueta%2CR&amp;author=Abe%2CC&amp;author=Watanabe%2CT&amp;author=Sugano%2CSS&amp;author=Ishihara%2CR&amp;author=Ezura%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Lee, S. Nonaka, M. Takayama, H. Ezura, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Lee J, Nonaka S, Takayama M, Ezura H. Utilization of a genome-edited tomato (Solanum lycopersicum) with high g" /><span class="c-article-references__counter">121.</span><p class="c-article-references__text" id="ref-CR121">Lee J, Nonaka S, Takayama M, Ezura H. Utilization of a genome-edited tomato (<i>Solanum lycopersicum</i>) with high gamma aminobutyric acid content in hybrid breeding. J Agric Food Chem. 2018;66:96371. <a href="https://doi.org/10.1021/acs.jafc.7b05171">https://doi.org/10.1021/acs.jafc.7b05171</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXlt1agsQ%253D%253D" aria-label="View reference 121 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1021%2Facs.jafc.7b05171" aria-label="View reference 121">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 121 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Utilization%20of%20a%20genome-edited%20tomato%20%28Solanum%20lycopersicum%29%20with%20high%20gamma%20aminobutyric%20acid%20content%20in%20hybrid%20breeding&amp;journal=J%20Agric%20Food%20Chem&amp;doi=10.1021%2Facs.jafc.7b05171&amp;volume=66&amp;pages=963-971&amp;publication_year=2018&amp;author=Lee%2CJ&amp;author=Nonaka%2CS&amp;author=Takayama%2CM&amp;author=Ezura%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Nonaka, C. Arai, M. Takayama, C. Matsukura, H. Ezura, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Nonaka S, Arai C, Takayama M, Matsukura C, Ezura H. Efficient increase of -aminobutyric acid (GABA) content i" /><span class="c-article-references__counter">122.</span><p class="c-article-references__text" id="ref-CR122">Nonaka S, Arai C, Takayama M, Matsukura C, Ezura H. Efficient increase of -aminobutyric acid (GABA) content in tomato fruits by targeted mutagenesis. Sci Rep. 2017;7:7057. <a href="https://doi.org/10.1038/s41598-017-06400-y">https://doi.org/10.1038/s41598-017-06400-y</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhtlWnurzF" aria-label="View reference 122 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fs41598-017-06400-y" aria-label="View reference 122">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 122 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20increase%20of%20%C9%A3-aminobutyric%20acid%20%28GABA%29%20content%20in%20tomato%20fruits%20by%20targeted%20mutagenesis&amp;journal=Sci%20Rep&amp;doi=10.1038%2Fs41598-017-06400-y&amp;volume=7&amp;publication_year=2017&amp;author=Nonaka%2CS&amp;author=Arai%2CC&amp;author=Takayama%2CM&amp;author=Matsukura%2CC&amp;author=Ezura%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X. Li, Y. Wang, S. Chen, H. Tian, D. Fu, B. Zhu, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Li X, Wang Y, Chen S, Tian H, Fu D, Zhu B, et al. Lycopene is enriched in tomato fruit by CRISPR/Cas9-mediated" /><span class="c-article-references__counter">123.</span><p class="c-article-references__text" id="ref-CR123">Li X, Wang Y, Chen S, Tian H, Fu D, Zhu B, et al. Lycopene is enriched in tomato fruit by CRISPR/Cas9-mediated multiplex genome editing. Front Plant Sci. 2018;9:179. <a href="https://doi.org/10.3389/fpls.2018.00559">https://doi.org/10.3389/fpls.2018.00559</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3389%2Ffpls.2018.00559" aria-label="View reference 123">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 123 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Lycopene%20is%20enriched%20in%20tomato%20fruit%20by%20CRISPR%2FCas9-mediated%20multiplex%20genome%20editing&amp;journal=Front%20Plant%20Sci&amp;doi=10.3389%2Ffpls.2018.00559&amp;volume=9&amp;publication_year=2018&amp;author=Li%2CX&amp;author=Wang%2CY&amp;author=Chen%2CS&amp;author=Tian%2CH&amp;author=Fu%2CD&amp;author=Zhu%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Q-H. Yu, B. Wang, N. Li, Y. Tang, S. Yang, T. Yang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Yu Q-H, Wang B, Li N, Tang Y, Yang S, Yang T, et al. CRISPR/Cas9-induced targeted mutagenesis and gene replace" /><span class="c-article-references__counter">124.</span><p class="c-article-references__text" id="ref-CR124">Yu Q-H, Wang B, Li N, Tang Y, Yang S, Yang T, et al. CRISPR/Cas9-induced targeted mutagenesis and gene replacement to generate long-shelf life tomato lines. Sci Rep. 2017;7:818. <a href="https://doi.org/10.1038/s41598-017-12262-1">https://doi.org/10.1038/s41598-017-12262-1</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhtl2ks77F" aria-label="View reference 124 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fs41598-017-12262-1" aria-label="View reference 124">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 124 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FCas9-induced%20targeted%20mutagenesis%20and%20gene%20replacement%20to%20generate%20long-shelf%20life%20tomato%20lines&amp;journal=Sci%20Rep&amp;doi=10.1038%2Fs41598-017-12262-1&amp;volume=7&amp;publication_year=2017&amp;author=Yu%2CQ-H&amp;author=Wang%2CB&amp;author=Li%2CN&amp;author=Tang%2CY&amp;author=Yang%2CS&amp;author=Yang%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2017. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">125.</span><p class="c-article-references__text" id="ref-CR125">United States Department of Agriculture (USDA). 2017. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-038-01_air_inquiry_cbidel.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-038-01_air_inquiry_cbidel.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Snchez-Len, J. Gil-Humanes, CV. Ozuna, MJ. Gimnez, C. Sousa, DF. Voytas, F. Barro, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Snchez-Len S, Gil-Humanes J, Ozuna CV, Gimnez MJ, Sousa C, Voytas DF, Barro F. Low-gluten, nontransgenic wh" /><span class="c-article-references__counter">126.</span><p class="c-article-references__text" id="ref-CR126">Snchez-Len S, Gil-Humanes J, Ozuna CV, Gimnez MJ, Sousa C, Voytas DF, Barro F. Low-gluten, nontransgenic wheat engineered with CRISPR/Cas9. Plant Biotechnol J. 2017. <a href="https://doi.org/10.1111/pbi.12837">https://doi.org/10.1111/pbi.12837</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12837" aria-label="View reference 126">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 126 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Low-gluten%2C%20nontransgenic%20wheat%20engineered%20with%20CRISPR%2FCas9&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12837&amp;publication_year=2017&amp;author=S%C3%A1nchez-Le%C3%B3n%2CS&amp;author=Gil-Humanes%2CJ&amp;author=Ozuna%2CCV&amp;author=Gim%C3%A9nez%2CMJ&amp;author=Sousa%2CC&amp;author=Voytas%2CDF&amp;author=Barro%2CF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="AS. Fister, L. Landherr, SN. Maximova, MJ. Guiltinan, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Fister AS, Landherr L, Maximova SN, Guiltinan MJ. Transient expression of CRISPR/Cas9 machinery targeting TcNP" /><span class="c-article-references__counter">127.</span><p class="c-article-references__text" id="ref-CR127">Fister AS, Landherr L, Maximova SN, Guiltinan MJ. Transient expression of CRISPR/Cas9 machinery targeting TcNPR3 enhances defense response in Theobroma cacao. Front Plant Sci. 2018;9:47. <a href="https://doi.org/10.3389/fpls.2018.00268">https://doi.org/10.3389/fpls.2018.00268</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3389%2Ffpls.2018.00268" aria-label="View reference 127">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 127 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Transient%20expression%20of%20CRISPR%2FCas9%20machinery%20targeting%20TcNPR3%20enhances%20defense%20response%20in%20Theobroma%20cacao&amp;journal=Front%20Plant%20Sci&amp;doi=10.3389%2Ffpls.2018.00268&amp;volume=9&amp;publication_year=2018&amp;author=Fister%2CAS&amp;author=Landherr%2CL&amp;author=Maximova%2CSN&amp;author=Guiltinan%2CMJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="H. Jia, V. Orbovic, JB. Jones, N. Wang, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Jia H, Orbovic V, Jones JB, Wang N. Modification of the PthA4 effector binding elements in Type I CsLOB1 promo" /><span class="c-article-references__counter">128.</span><p class="c-article-references__text" id="ref-CR128">Jia H, Orbovic V, Jones JB, Wang N. Modification of the PthA4 effector binding elements in Type I CsLOB1 promoter using Cas9/sgRNA to produce transgenic Duncan grapefruit alleviating XccpthA4:dCsLOB1.3 infection. Plant Biotechnol J. 2016;14:1291301. <a href="https://doi.org/10.1111/pbi.12495">https://doi.org/10.1111/pbi.12495</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XlvV2js78%253D" aria-label="View reference 128 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12495" aria-label="View reference 128">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 128 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Modification%20of%20the%20PthA4%20effector%20binding%20elements%20in%20Type%20I%20CsLOB1%20promoter%20using%20Cas9%2FsgRNA%20to%20produce%20transgenic%20Duncan%20grapefruit%20alleviating%20Xcc%CE%94pthA4%3AdCsLOB1.3%20infection&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12495&amp;volume=14&amp;pages=1291-1301&amp;publication_year=2016&amp;author=Jia%2CH&amp;author=Orbovic%2CV&amp;author=Jones%2CJB&amp;author=Wang%2CN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="H. Jia, Y. Zhang, V. Orbovi, J. Xu, FF. White, JB. Jones, N. Wang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Jia H, Zhang Y, Orbovi V, Xu J, White FF, Jones JB, Wang N. Genome editing of the disease susceptibility gene" /><span class="c-article-references__counter">129.</span><p class="c-article-references__text" id="ref-CR129">Jia H, Zhang Y, Orbovi V, Xu J, White FF, Jones JB, Wang N. Genome editing of the disease susceptibility gene CsLOB1 in citrus confers resistance to citrus canker. Plant Biotechnol J. 2017;15:81723. <a href="https://doi.org/10.1111/pbi.12677">https://doi.org/10.1111/pbi.12677</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhtVWjsrfE" aria-label="View reference 129 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12677" aria-label="View reference 129">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 129 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20editing%20of%20the%20disease%20susceptibility%20gene%20CsLOB1%20in%20citrus%20confers%20resistance%20to%20citrus%20canker&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12677&amp;volume=15&amp;pages=817-823&amp;publication_year=2017&amp;author=Jia%2CH&amp;author=Zhang%2CY&amp;author=Orbovi%C4%87%2CV&amp;author=Xu%2CJ&amp;author=White%2CFF&amp;author=Jones%2CJB&amp;author=Wang%2CN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X. Wang, M. Tu, D. Wang, J. Liu, Y. Li, Z. Li, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Wang X, Tu M, Wang D, Liu J, Li Y, Li Z, et al. CRISPR/Cas9-mediated efficient targeted mutagenesis in grape i" /><span class="c-article-references__counter">130.</span><p class="c-article-references__text" id="ref-CR130">Wang X, Tu M, Wang D, Liu J, Li Y, Li Z, et al. CRISPR/Cas9-mediated efficient targeted mutagenesis in grape in the first generation. Plant Biotechnol J. 2018;16:84455. <a href="https://doi.org/10.1111/pbi.12832">https://doi.org/10.1111/pbi.12832</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXmsVahurs%253D" aria-label="View reference 130 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12832" aria-label="View reference 130">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 130 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FCas9-mediated%20efficient%20targeted%20mutagenesis%20in%20grape%20in%20the%20first%20generation&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12832&amp;volume=16&amp;pages=844-855&amp;publication_year=2018&amp;author=Wang%2CX&amp;author=Tu%2CM&amp;author=Wang%2CD&amp;author=Liu%2CJ&amp;author=Li%2CY&amp;author=Li%2CZ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2017. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">131.</span><p class="c-article-references__text" id="ref-CR131">United States Department of Agriculture (USDA). 2017. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-076-01_air_inquiry_cbidel.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-076-01_air_inquiry_cbidel.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="A. Peng, S. Chen, T. Lei, L. Xu, Y. He, L. Wu, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Peng A, Chen S, Lei T, Xu L, He Y, Wu L, et al. Engineering canker-resistant plants through CRISPR/Cas9-target" /><span class="c-article-references__counter">132.</span><p class="c-article-references__text" id="ref-CR132">Peng A, Chen S, Lei T, Xu L, He Y, Wu L, et al. Engineering canker-resistant plants through CRISPR/Cas9-targeted editing of the susceptibility gene CsLOB1 promoter in citrus. Plant Biotechnol J. 2017;15:150919. <a href="https://doi.org/10.1111/pbi.12733">https://doi.org/10.1111/pbi.12733</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhvV2nsLrE" aria-label="View reference 132 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12733" aria-label="View reference 132">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 132 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Engineering%20canker-resistant%20plants%20through%20CRISPR%2FCas9-targeted%20editing%20of%20the%20susceptibility%20gene%20CsLOB1%20promoter%20in%20citrus&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12733&amp;volume=15&amp;pages=1509-1519&amp;publication_year=2017&amp;author=Peng%2CA&amp;author=Chen%2CS&amp;author=Lei%2CT&amp;author=Xu%2CL&amp;author=He%2CY&amp;author=Wu%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="F. Wang, C. Wang, P. Liu, C. Lei, W. Hao, Y. Gao, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Wang F, Wang C, Liu P, Lei C, Hao W, Gao Y, et al. Enhanced rice blast resistance by CRISPR/Cas9-targeted muta" /><span class="c-article-references__counter">133.</span><p class="c-article-references__text" id="ref-CR133">Wang F, Wang C, Liu P, Lei C, Hao W, Gao Y, et al. Enhanced rice blast resistance by CRISPR/Cas9-targeted mutagenesis of the ERF transcription factor gene OsERF922. PLoS ONE. 2016;11:e0154027. <a href="https://doi.org/10.1371/journal.pone.0154027">https://doi.org/10.1371/journal.pone.0154027</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhsVSgtL%252FE" aria-label="View reference 133 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0154027" aria-label="View reference 133">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 133 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Enhanced%20rice%20blast%20resistance%20by%20CRISPR%2FCas9-targeted%20mutagenesis%20of%20the%20ERF%20transcription%20factor%20gene%20OsERF922&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0154027&amp;volume=11&amp;publication_year=2016&amp;author=Wang%2CF&amp;author=Wang%2CC&amp;author=Liu%2CP&amp;author=Lei%2CC&amp;author=Hao%2CW&amp;author=Gao%2CY">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Zhou, Z. Peng, J. Long, D. Sosso, B. Liu, J-S. Eom, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Zhou J, Peng Z, Long J, Sosso D, Liu B, Eom J-S, et al. Gene targeting by the TAL effector PthXo2 reveals cryp" /><span class="c-article-references__counter">134.</span><p class="c-article-references__text" id="ref-CR134">Zhou J, Peng Z, Long J, Sosso D, Liu B, Eom J-S, et al. Gene targeting by the TAL effector PthXo2 reveals cryptic resistance gene for bacterial blight of rice. Plant J. 2015;82:63243. <a href="https://doi.org/10.1111/tpj.12838">https://doi.org/10.1111/tpj.12838</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXotVWntbk%253D" aria-label="View reference 134 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Ftpj.12838" aria-label="View reference 134">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 134 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20targeting%20by%20the%20TAL%20effector%20PthXo2%20reveals%20cryptic%20resistance%20gene%20for%20bacterial%20blight%20of%20rice&amp;journal=Plant%20J&amp;doi=10.1111%2Ftpj.12838&amp;volume=82&amp;pages=632-643&amp;publication_year=2015&amp;author=Zhou%2CJ&amp;author=Peng%2CZ&amp;author=Long%2CJ&amp;author=Sosso%2CD&amp;author=Liu%2CB&amp;author=Eom%2CJ-S">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Blanvillain-Baufum, M. Reschke, M. Sol, F. Auguy, H. Doucoure, B. Szurek, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Blanvillain-Baufum S, Reschke M, Sol M, Auguy F, Doucoure H, Szurek B, et al. Targeted promoter editing for " /><span class="c-article-references__counter">135.</span><p class="c-article-references__text" id="ref-CR135">Blanvillain-Baufum S, Reschke M, Sol M, Auguy F, Doucoure H, Szurek B, et al. Targeted promoter editing for rice resistance to <i>Xanthomonas oryzae</i> pv. oryzae reveals differential activities for SWEET14-inducing TAL effectors. Plant Biotechnol J. 2017;15:30617. <a href="https://doi.org/10.1111/pbi.12613">https://doi.org/10.1111/pbi.12613</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXivVOnsr0%253D" aria-label="View reference 135 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12613" aria-label="View reference 135">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 135 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20promoter%20editing%20for%20rice%20resistance%20to%20Xanthomonas%20oryzae%20pv.%20oryzae%20reveals%20differential%20activities%20for%20SWEET14-inducing%20TAL%20effectors&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12613&amp;volume=15&amp;pages=306-317&amp;publication_year=2017&amp;author=Blanvillain-Baufum%C3%A9%2CS&amp;author=Reschke%2CM&amp;author=Sol%C3%A9%2CM&amp;author=Auguy%2CF&amp;author=Doucoure%2CH&amp;author=Szurek%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="T. Li, B. Liu, MH. Spalding, DP. Weeks, B. Yang, " /><meta itemprop="datePublished" content="2012" /><meta itemprop="headline" content="Li T, Liu B, Spalding MH, Weeks DP, Yang B. High-efficiency TALEN-based gene editing produces disease-resistan" /><span class="c-article-references__counter">136.</span><p class="c-article-references__text" id="ref-CR136">Li T, Liu B, Spalding MH, Weeks DP, Yang B. High-efficiency TALEN-based gene editing produces disease-resistant rice. Nat Biotechnol. 2012;30:3902. <a href="https://doi.org/10.1038/nbt.2199">https://doi.org/10.1038/nbt.2199</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XmsFyjs7g%253D" aria-label="View reference 136 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnbt.2199" aria-label="View reference 136">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 136 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=High-efficiency%20TALEN-based%20gene%20editing%20produces%20disease-resistant%20rice&amp;journal=Nat%20Biotechnol&amp;doi=10.1038%2Fnbt.2199&amp;volume=30&amp;pages=390-392&amp;publication_year=2012&amp;author=Li%2CT&amp;author=Liu%2CB&amp;author=Spalding%2CMH&amp;author=Weeks%2CDP&amp;author=Yang%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Wang, D. Tian, K. Gu, X. Yang, L. Wang, X. Zeng, Z. Yin, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Wang J, Tian D, Gu K, Yang X, Wang L, Zeng X, Yin Z. Induction of Xa10-like genes in rice cultivar nipponbare " /><span class="c-article-references__counter">137.</span><p class="c-article-references__text" id="ref-CR137">Wang J, Tian D, Gu K, Yang X, Wang L, Zeng X, Yin Z. Induction of Xa10-like genes in rice cultivar nipponbare confers disease resistance to rice bacterial blight. Mol Plant Microbe Interact. 2017;30:46677. <a href="https://doi.org/10.1094/MPMI-11-16-0229-R">https://doi.org/10.1094/MPMI-11-16-0229-R</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXht1aisbzN" aria-label="View reference 137 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1094%2FMPMI-11-16-0229-R" aria-label="View reference 137">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 137 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Induction%20of%20Xa10-like%20genes%20in%20rice%20cultivar%20nipponbare%20confers%20disease%20resistance%20to%20rice%20bacterial%20blight&amp;journal=Mol%20Plant%20Microbe%20Interact&amp;doi=10.1094%2FMPMI-11-16-0229-R&amp;volume=30&amp;pages=466-477&amp;publication_year=2017&amp;author=Wang%2CJ&amp;author=Tian%2CD&amp;author=Gu%2CK&amp;author=Yang%2CX&amp;author=Wang%2CL&amp;author=Zeng%2CX&amp;author=Yin%2CZ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="C. Xie, G. Zhang, Y. Zhang, X. Song, H. Guo, X. Chen, R. Fang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Xie C, Zhang G, Zhang Y, Song X, Guo H, Chen X, Fang R. SRWD1, a novel target gene of DELLA and WRKY proteins," /><span class="c-article-references__counter">138.</span><p class="c-article-references__text" id="ref-CR138">Xie C, Zhang G, Zhang Y, Song X, Guo H, Chen X, Fang R. SRWD1, a novel target gene of DELLA and WRKY proteins, participates in the development and immune response of rice (<i>Oryza sativa</i> L.). Sci Bull. 2017;62:163948. <a href="https://doi.org/10.1016/j.scib.2017.12.002">https://doi.org/10.1016/j.scib.2017.12.002</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhtVGhs77P" aria-label="View reference 138 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.scib.2017.12.002" aria-label="View reference 138">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 138 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=SRWD1%2C%20a%20novel%20target%20gene%20of%20DELLA%20and%20WRKY%20proteins%2C%20participates%20in%20the%20development%20and%20immune%20response%20of%20rice%20%28Oryza%20sativa%20L.%29&amp;journal=Sci%20Bull&amp;doi=10.1016%2Fj.scib.2017.12.002&amp;volume=62&amp;pages=1639-1648&amp;publication_year=2017&amp;author=Xie%2CC&amp;author=Zhang%2CG&amp;author=Zhang%2CY&amp;author=Song%2CX&amp;author=Guo%2CH&amp;author=Chen%2CX&amp;author=Fang%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X. Zhou, H. Liao, M. Chern, J. Yin, Y. Chen, J. Wang, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Zhou X, Liao H, Chern M, Yin J, Chen Y, Wang J, et al. Loss of function of a rice TPR-domain RNA-binding prote" /><span class="c-article-references__counter">139.</span><p class="c-article-references__text" id="ref-CR139">Zhou X, Liao H, Chern M, Yin J, Chen Y, Wang J, et al. Loss of function of a rice TPR-domain RNA-binding protein confers broad-spectrum disease resistance. Proc Natl Acad Sci USA. 2018;115:31749. <a href="https://doi.org/10.1073/pnas.1705927115">https://doi.org/10.1073/pnas.1705927115</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXht1Chs7bP" aria-label="View reference 139 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.1705927115" aria-label="View reference 139">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 139 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Loss%20of%20function%20of%20a%20rice%20TPR-domain%20RNA-binding%20protein%20confers%20broad-spectrum%20disease%20resistance&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;doi=10.1073%2Fpnas.1705927115&amp;volume=115&amp;pages=3174-3179&amp;publication_year=2018&amp;author=Zhou%2CX&amp;author=Liao%2CH&amp;author=Chern%2CM&amp;author=Yin%2CJ&amp;author=Chen%2CY&amp;author=Wang%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2014. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">140.</span><p class="c-article-references__text" id="ref-CR140">United States Department of Agriculture (USDA). 2014. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/air_isu_ting_rice.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/air_isu_ting_rice.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="L. Cai, Y. Cao, Z. Xu, W. Ma, M. Zakria, L. Zou, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Cai L, Cao Y, Xu Z, Ma W, Zakria M, Zou L, et al. A transcription activator-like effector Tal7 of Xanthomonas " /><span class="c-article-references__counter">141.</span><p class="c-article-references__text" id="ref-CR141">Cai L, Cao Y, Xu Z, Ma W, Zakria M, Zou L, et al. A transcription activator-like effector Tal7 of <i>Xanthomonas oryzae</i> pv. oryzicola activates rice gene Os09g29100 to suppress rice immunity. Sci Rep. 2017;7:5089. <a href="https://doi.org/10.1038/s41598-017-04800-8">https://doi.org/10.1038/s41598-017-04800-8</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhtlGgsrfO" aria-label="View reference 141 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fs41598-017-04800-8" aria-label="View reference 141">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 141 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20transcription%20activator-like%20effector%20Tal7%20of%20Xanthomonas%20oryzae%20pv.%20oryzicola%20activates%20rice%20gene%20Os09g29100%20to%20suppress%20rice%20immunity&amp;journal=Sci%20Rep&amp;doi=10.1038%2Fs41598-017-04800-8&amp;volume=7&amp;publication_year=2017&amp;author=Cai%2CL&amp;author=Cao%2CY&amp;author=Xu%2CZ&amp;author=Ma%2CW&amp;author=Zakria%2CM&amp;author=Zou%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="A. Macovei, NR. Sevilla, C. Cantos, GB. Jonson, I. Slamet-Loedin, T. ermk, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Macovei A, Sevilla NR, Cantos C, Jonson GB, Slamet-Loedin I, ermk T, et al. Novel alleles of rice eIF4G gene" /><span class="c-article-references__counter">142.</span><p class="c-article-references__text" id="ref-CR142">Macovei A, Sevilla NR, Cantos C, Jonson GB, Slamet-Loedin I, ermk T, et al. Novel alleles of rice eIF4G generated by CRISPR/Cas9-targeted mutagenesis confer resistance to Rice tungro spherical virus. Plant Biotechnol J. 2018;47:417. <a href="https://doi.org/10.1111/pbi.12927">https://doi.org/10.1111/pbi.12927</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhvFSjurnI" aria-label="View reference 142 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12927" aria-label="View reference 142">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 142 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Novel%20alleles%20of%20rice%20eIF4G%20generated%20by%20CRISPR%2FCas9-targeted%20mutagenesis%20confer%20resistance%20to%20Rice%20tungro%20spherical%20virus&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12927&amp;volume=47&amp;publication_year=2018&amp;author=Macovei%2CA&amp;author=Sevilla%2CNR&amp;author=Cantos%2CC&amp;author=Jonson%2CGB&amp;author=Slamet-Loedin%2CI&amp;author=%C4%8Cerm%C3%A1k%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="V. Nekrasov, C. Wang, J. Win, C. Lanz, D. Weigel, S. Kamoun, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Nekrasov V, Wang C, Win J, Lanz C, Weigel D, Kamoun S. Rapid generation of a transgene-free powdery mildew res" /><span class="c-article-references__counter">143.</span><p class="c-article-references__text" id="ref-CR143">Nekrasov V, Wang C, Win J, Lanz C, Weigel D, Kamoun S. Rapid generation of a transgene-free powdery mildew resistant tomato by genome deletion. Sci Rep. 2017;7:482. <a href="https://doi.org/10.1038/s41598-017-00578-x">https://doi.org/10.1038/s41598-017-00578-x</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhs1yksbnM" aria-label="View reference 143 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fs41598-017-00578-x" aria-label="View reference 143">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 143 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20generation%20of%20a%20transgene-free%20powdery%20mildew%20resistant%20tomato%20by%20genome%20deletion&amp;journal=Sci%20Rep&amp;doi=10.1038%2Fs41598-017-00578-x&amp;volume=7&amp;publication_year=2017&amp;author=Nekrasov%2CV&amp;author=Wang%2CC&amp;author=Win%2CJ&amp;author=Lanz%2CC&amp;author=Weigel%2CD&amp;author=Kamoun%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Mahfouz M, Tashkandi M, Ali Z, Aljedaani F, Shami A. Engineering resistance against tomato yellow leaf curl vi" /><span class="c-article-references__counter">144.</span><p class="c-article-references__text" id="ref-CR144">Mahfouz M, Tashkandi M, Ali Z, Aljedaani F, Shami A. Engineering resistance against tomato yellow leaf curl virus via the CRISPR/Cas9 system in tomato. 2017. <a href="https://doi.org/10.1101/237735">https://doi.org/10.1101/237735</a>.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="Toledo Thomazella DP de, Brail Q, Dahlbeck D, Staskawicz BJ. CRISPRCas9 mediated mutagenesis of a DMR6 orthol" /><span class="c-article-references__counter">145.</span><p class="c-article-references__text" id="ref-CR145">Toledo Thomazella DP de, Brail Q, Dahlbeck D, Staskawicz BJ. CRISPRCas9 mediated mutagenesis of a DMR6 ortholog in tomato confers broad-spectrum disease resistance. 2016:123. <a href="https://doi.org/10.1101/064824">https://doi.org/10.1101/064824</a>.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Zhang, Y. Bai, G. Wu, S. Zou, Y. Chen, C. Gao, D. Tang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Zhang Y, Bai Y, Wu G, Zou S, Chen Y, Gao C, Tang D. Simultaneous modification of three homoeologs of TaEDR1 by" /><span class="c-article-references__counter">146.</span><p class="c-article-references__text" id="ref-CR146">Zhang Y, Bai Y, Wu G, Zou S, Chen Y, Gao C, Tang D. Simultaneous modification of three homoeologs of TaEDR1 by genome editing enhances powdery mildew resistance in wheat. Plant J. 2017;91:71424. <a href="https://doi.org/10.1111/tpj.13599">https://doi.org/10.1111/tpj.13599</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXpslWjtLY%253D" aria-label="View reference 146 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Ftpj.13599" aria-label="View reference 146">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 146 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Simultaneous%20modification%20of%20three%20homoeologs%20of%20TaEDR1%20by%20genome%20editing%20enhances%20powdery%20mildew%20resistance%20in%20wheat&amp;journal=Plant%20J&amp;doi=10.1111%2Ftpj.13599&amp;volume=91&amp;pages=714-724&amp;publication_year=2017&amp;author=Zhang%2CY&amp;author=Bai%2CY&amp;author=Wu%2CG&amp;author=Zou%2CS&amp;author=Chen%2CY&amp;author=Gao%2CC&amp;author=Tang%2CD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2015. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">147.</span><p class="c-article-references__text" id="ref-CR147">United States Department of Agriculture (USDA). 2015. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-238-01_air_inquiry_cbidel.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/15-238-01_air_inquiry_cbidel.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="R. Ruiter, I. Brande, E. Stals, S. Delaur, M. Cornelissen, K. DHalluin, " /><meta itemprop="datePublished" content="2003" /><meta itemprop="headline" content="Ruiter R, van den Brande I, Stals E, Delaur S, Cornelissen M, DHalluin K. Spontaneous mutation frequency in " /><span class="c-article-references__counter">148.</span><p class="c-article-references__text" id="ref-CR148">Ruiter R, van den Brande I, Stals E, Delaur S, Cornelissen M, DHalluin K. Spontaneous mutation frequency in plants obscures the effect of chimeraplasty. Plant Mol Biol. 2003;53:67589. <a href="https://doi.org/10.1023/b:plan.0000019111.96107.01">https://doi.org/10.1023/b:plan.0000019111.96107.01</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXhvFCgtrs%253D" aria-label="View reference 148 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1023%2Fb%3Aplan.0000019111.96107.01" aria-label="View reference 148">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 148 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Spontaneous%20mutation%20frequency%20in%20plants%20obscures%20the%20effect%20of%20chimeraplasty&amp;journal=Plant%20Mol%20Biol&amp;doi=10.1023%2Fb%3Aplan.0000019111.96107.01&amp;volume=53&amp;pages=675-689&amp;publication_year=2003&amp;author=Ruiter%2CR&amp;author=Brande%2CI&amp;author=Stals%2CE&amp;author=Delaur%C3%A9%2CS&amp;author=Cornelissen%2CM&amp;author=D%E2%80%99Halluin%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="AW. Hummel, RD. Chauhan, T. Cermak, AM. Mutka, A. Vijayaraghavan, A. Boyher, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Hummel AW, Chauhan RD, Cermak T, Mutka AM, Vijayaraghavan A, Boyher A, et al. Allele exchange at the EPSPS loc" /><span class="c-article-references__counter">149.</span><p class="c-article-references__text" id="ref-CR149">Hummel AW, Chauhan RD, Cermak T, Mutka AM, Vijayaraghavan A, Boyher A, et al. Allele exchange at the EPSPS locus confers glyphosate tolerance in cassava. Plant Biotechnol J. 2018;16:127582. <a href="https://doi.org/10.1111/pbi.12868">https://doi.org/10.1111/pbi.12868</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhtFKhtL%252FK" aria-label="View reference 149 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12868" aria-label="View reference 149">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 149 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Allele%20exchange%20at%20the%20EPSPS%20locus%20confers%20glyphosate%20tolerance%20in%20cassava&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12868&amp;volume=16&amp;pages=1275-1282&amp;publication_year=2018&amp;author=Hummel%2CAW&amp;author=Chauhan%2CRD&amp;author=Cermak%2CT&amp;author=Mutka%2CAM&amp;author=Vijayaraghavan%2CA&amp;author=Boyher%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="K. DHalluin, C. Vanderstraeten, J. Hulle, J. Rosolowska, I. Brande, A. Pennewaert, " /><meta itemprop="datePublished" content="2013" /><meta itemprop="headline" content="DHalluin K, Vanderstraeten C, van Hulle J, Rosolowska J, van den Brande I, Pennewaert A, et al. Targeted mole" /><span class="c-article-references__counter">150.</span><p class="c-article-references__text" id="ref-CR150">DHalluin K, Vanderstraeten C, van Hulle J, Rosolowska J, van den Brande I, Pennewaert A, et al. Targeted molecular trait stacking in cotton through targeted double-strand break induction. Plant Biotechnol J. 2013;11:93341. <a href="https://doi.org/10.1111/pbi.12085">https://doi.org/10.1111/pbi.12085</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhsFaltbzF" aria-label="View reference 150 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12085" aria-label="View reference 150">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 150 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20molecular%20trait%20stacking%20in%20cotton%20through%20targeted%20double-strand%20break%20induction&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12085&amp;volume=11&amp;pages=933-941&amp;publication_year=2013&amp;author=D%E2%80%99Halluin%2CK&amp;author=Vanderstraeten%2CC&amp;author=Hulle%2CJ&amp;author=Rosolowska%2CJ&amp;author=Brande%2CI&amp;author=Pennewaert%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="NJ. Sauer, J. Narvez-Vsquez, J. Mozoruk, RB. Miller, ZJ. Warburg, MJ. Woodward, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Sauer NJ, Narvez-Vsquez J, Mozoruk J, Miller RB, Warburg ZJ, Woodward MJ, et al. Oligonucleotide-mediated ge" /><span class="c-article-references__counter">151.</span><p class="c-article-references__text" id="ref-CR151">Sauer NJ, Narvez-Vsquez J, Mozoruk J, Miller RB, Warburg ZJ, Woodward MJ, et al. Oligonucleotide-mediated genome editing provides precision and function to engineered nucleases and antibiotics in plants. Plant Physiol. 2016;170:191728. <a href="https://doi.org/10.1104/pp.15.01696">https://doi.org/10.1104/pp.15.01696</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhsVygurzF" aria-label="View reference 151 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1104%2Fpp.15.01696" aria-label="View reference 151">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 151 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Oligonucleotide-mediated%20genome%20editing%20provides%20precision%20and%20function%20to%20engineered%20nucleases%20and%20antibiotics%20in%20plants&amp;journal=Plant%20Physiol&amp;doi=10.1104%2Fpp.15.01696&amp;volume=170&amp;pages=1917-1928&amp;publication_year=2016&amp;author=Sauer%2CNJ&amp;author=Narv%C3%A1ez-V%C3%A1squez%2CJ&amp;author=Mozoruk%2CJ&amp;author=Miller%2CRB&amp;author=Warburg%2CZJ&amp;author=Woodward%2CMJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="WM. Ainley, L. Sastry-Dent, ME. Welter, MG. Murray, B. Zeitler, R. Amora, " /><meta itemprop="datePublished" content="2013" /><meta itemprop="headline" content="Ainley WM, Sastry-Dent L, Welter ME, Murray MG, Zeitler B, Amora R, et al. Trait stacking via targeted genome " /><span class="c-article-references__counter">152.</span><p class="c-article-references__text" id="ref-CR152">Ainley WM, Sastry-Dent L, Welter ME, Murray MG, Zeitler B, Amora R, et al. Trait stacking via targeted genome editing. Plant Biotechnol J. 2013;11:112634. <a href="https://doi.org/10.1111/pbi.12107">https://doi.org/10.1111/pbi.12107</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXhvVarsb7J" aria-label="View reference 152 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12107" aria-label="View reference 152">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 152 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Trait%20stacking%20via%20targeted%20genome%20editing&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12107&amp;volume=11&amp;pages=1126-1134&amp;publication_year=2013&amp;author=Ainley%2CWM&amp;author=Sastry-Dent%2CL&amp;author=Welter%2CME&amp;author=Murray%2CMG&amp;author=Zeitler%2CB&amp;author=Amora%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="T. Zhu, DJ. Peterson, L. Tagliani, G. Clair, CL. Baszczynski, B. Bowen, " /><meta itemprop="datePublished" content="1999" /><meta itemprop="headline" content="Zhu T, Peterson DJ, Tagliani L, Clair G, Baszczynski CL, Bowen B. Targeted manipulation of maize genes in vivo" /><span class="c-article-references__counter">153.</span><p class="c-article-references__text" id="ref-CR153">Zhu T, Peterson DJ, Tagliani L, Clair G, Baszczynski CL, Bowen B. Targeted manipulation of maize genes in vivo using chimeric RNA/DNA oligonucleotides. Proc Natl Acad Sci. 1999;96:876873. <a href="https://doi.org/10.1073/pnas.96.15.8768">https://doi.org/10.1073/pnas.96.15.8768</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADyaK1MXkslOkur8%253D" aria-label="View reference 153 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.96.15.8768" aria-label="View reference 153">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 153 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20manipulation%20of%20maize%20genes%20in%20vivo%20using%20chimeric%20RNA%2FDNA%20oligonucleotides&amp;journal=Proc%20Natl%20Acad%20Sci&amp;doi=10.1073%2Fpnas.96.15.8768&amp;volume=96&amp;pages=8768-8773&amp;publication_year=1999&amp;author=Zhu%2CT&amp;author=Peterson%2CDJ&amp;author=Tagliani%2CL&amp;author=Clair%2CG&amp;author=Baszczynski%2CCL&amp;author=Bowen%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="T. Zhu, K. Mettenburg, DJ. Peterson, L. Tagliani, CL. Baszczynski, " /><meta itemprop="datePublished" content="2000" /><meta itemprop="headline" content="Zhu T, Mettenburg K, Peterson DJ, Tagliani L, Baszczynski CL. Engineering herbicide-resistant maize using chim" /><span class="c-article-references__counter">154.</span><p class="c-article-references__text" id="ref-CR154">Zhu T, Mettenburg K, Peterson DJ, Tagliani L, Baszczynski CL. Engineering herbicide-resistant maize using chimeric RNA/DNA oligonucleotides. Nat Biotechnol. 2000;18:5558. <a href="https://doi.org/10.1038/75435">https://doi.org/10.1038/75435</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3cXjsVOgtrY%253D" aria-label="View reference 154 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2F75435" aria-label="View reference 154">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 154 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Engineering%20herbicide-resistant%20maize%20using%20chimeric%20RNA%2FDNA%20oligonucleotides&amp;journal=Nat%20Biotechnol&amp;doi=10.1038%2F75435&amp;volume=18&amp;pages=555-558&amp;publication_year=2000&amp;author=Zhu%2CT&amp;author=Mettenburg%2CK&amp;author=Peterson%2CDJ&amp;author=Tagliani%2CL&amp;author=Baszczynski%2CCL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="NM. Butler, NJ. Baltes, DF. Voytas, DS. Douches, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Butler NM, Baltes NJ, Voytas DF, Douches DS. Geminivirus-mediated genome editing in potato (Solanum tuberosum " /><span class="c-article-references__counter">155.</span><p class="c-article-references__text" id="ref-CR155">Butler NM, Baltes NJ, Voytas DF, Douches DS. Geminivirus-mediated genome editing in potato (<i>Solanum tuberosum</i> L.) using sequence-specific nucleases. Front Plant Sci. 2016;7:113. <a href="https://doi.org/10.3389/fpls.2016.01045">https://doi.org/10.3389/fpls.2016.01045</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3389%2Ffpls.2016.01045" aria-label="View reference 155">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 155 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Geminivirus-mediated%20genome%20editing%20in%20potato%20%28Solanum%20tuberosum%20L.%29%20using%20sequence-specific%20nucleases&amp;journal=Front%20Plant%20Sci&amp;doi=10.3389%2Ffpls.2016.01045&amp;volume=7&amp;pages=1-13&amp;publication_year=2016&amp;author=Butler%2CNM&amp;author=Baltes%2CNJ&amp;author=Voytas%2CDF&amp;author=Douches%2CDS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Li, X. Meng, Y. Zong, K. Chen, H. Zhang, J. Liu, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Li J, Meng X, Zong Y, Chen K, Zhang H, Liu J, et al. Gene replacements and insertions in rice by intron target" /><span class="c-article-references__counter">156.</span><p class="c-article-references__text" id="ref-CR156">Li J, Meng X, Zong Y, Chen K, Zhang H, Liu J, et al. Gene replacements and insertions in rice by intron targeting using CRISPRCas9. Nat Plants. 2016;2:16. <a href="https://doi.org/10.1038/nplants.2016.139">https://doi.org/10.1038/nplants.2016.139</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhsFWisLnE" aria-label="View reference 156 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnplants.2016.139" aria-label="View reference 156">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 156 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20replacements%20and%20insertions%20in%20rice%20by%20intron%20targeting%20using%20CRISPR%E2%80%93Cas9&amp;journal=Nat%20Plants&amp;doi=10.1038%2Fnplants.2016.139&amp;volume=2&amp;pages=1-6&amp;publication_year=2016&amp;author=Li%2CJ&amp;author=Meng%2CX&amp;author=Zong%2CY&amp;author=Chen%2CK&amp;author=Zhang%2CH&amp;author=Liu%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Y. Sun, X. Zhang, C. Wu, Y. He, Y. Ma, H. Hou, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Sun Y, Zhang X, Wu C, He Y, Ma Y, Hou H, et al. Engineering herbicide-resistant rice plants through CRISPR/Cas" /><span class="c-article-references__counter">157.</span><p class="c-article-references__text" id="ref-CR157">Sun Y, Zhang X, Wu C, He Y, Ma Y, Hou H, et al. Engineering herbicide-resistant rice plants through CRISPR/Cas9-mediated homologous recombination of acetolactate synthase. Mol Plant. 2016;9:62831. <a href="https://doi.org/10.1016/j.molp.2016.01.001">https://doi.org/10.1016/j.molp.2016.01.001</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhvVOjur4%253D" aria-label="View reference 157 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.molp.2016.01.001" aria-label="View reference 157">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 157 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Engineering%20herbicide-resistant%20rice%20plants%20through%20CRISPR%2FCas9-mediated%20homologous%20recombination%20of%20acetolactate%20synthase&amp;journal=Mol%20Plant&amp;doi=10.1016%2Fj.molp.2016.01.001&amp;volume=9&amp;pages=628-631&amp;publication_year=2016&amp;author=Sun%2CY&amp;author=Zhang%2CX&amp;author=Wu%2CC&amp;author=He%2CY&amp;author=Ma%2CY&amp;author=Hou%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="M. Wang, Y. Liu, C. Zhang, J. Liu, X. Liu, L. Wang, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Wang M, Liu Y, Zhang C, Liu J, Liu X, Wang L, et al. Gene editing by co-transformation of TALEN and chimeric R" /><span class="c-article-references__counter">158.</span><p class="c-article-references__text" id="ref-CR158">Wang M, Liu Y, Zhang C, Liu J, Liu X, Wang L, et al. Gene editing by co-transformation of TALEN and chimeric RNA/DNA oligonucleotides on the rice OsEPSPS gene and the inheritance of mutations. PLoS ONE. 2015;10:e0122755. <a href="https://doi.org/10.1371/journal.pone.0122755">https://doi.org/10.1371/journal.pone.0122755</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XhsVCksLnI" aria-label="View reference 158 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0122755" aria-label="View reference 158">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 158 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20editing%20by%20co-transformation%20of%20TALEN%20and%20chimeric%20RNA%2FDNA%20oligonucleotides%20on%20the%20rice%20OsEPSPS%20gene%20and%20the%20inheritance%20of%20mutations&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0122755&amp;volume=10&amp;publication_year=2015&amp;author=Wang%2CM&amp;author=Liu%2CY&amp;author=Zhang%2CC&amp;author=Liu%2CJ&amp;author=Liu%2CX&amp;author=Wang%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="A. Okuzaki, K. Toriyama, " /><meta itemprop="datePublished" content="2004" /><meta itemprop="headline" content="Okuzaki A, Toriyama K. Chimeric RNA/DNA oligonucleotide-directed gene targeting in rice. Plant Cell Rep. 2004;" /><span class="c-article-references__counter">159.</span><p class="c-article-references__text" id="ref-CR159">Okuzaki A, Toriyama K. Chimeric RNA/DNA oligonucleotide-directed gene targeting in rice. Plant Cell Rep. 2004;22:50912. <a href="https://doi.org/10.1007/s00299-003-0698-2">https://doi.org/10.1007/s00299-003-0698-2</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXmvVGhsw%253D%253D" aria-label="View reference 159 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs00299-003-0698-2" aria-label="View reference 159">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 159 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Chimeric%20RNA%2FDNA%20oligonucleotide-directed%20gene%20targeting%20in%20rice&amp;journal=Plant%20Cell%20Rep&amp;doi=10.1007%2Fs00299-003-0698-2&amp;volume=22&amp;pages=509-512&amp;publication_year=2004&amp;author=Okuzaki%2CA&amp;author=Toriyama%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Z. Shimatani, S. Kashojiya, M. Takayama, R. Terada, T. Arazoe, H. Ishii, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Shimatani Z, Kashojiya S, Takayama M, Terada R, Arazoe T, Ishii H, et al. Targeted base editing in rice and to" /><span class="c-article-references__counter">160.</span><p class="c-article-references__text" id="ref-CR160">Shimatani Z, Kashojiya S, Takayama M, Terada R, Arazoe T, Ishii H, et al. Targeted base editing in rice and tomato using a CRISPRCas9 cytidine deaminase fusion. Nat Biotechnol. 2017;35:4413. <a href="https://doi.org/10.1038/nbt.3833">https://doi.org/10.1038/nbt.3833</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXkvFahtLw%253D" aria-label="View reference 160 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1038%2Fnbt.3833" aria-label="View reference 160">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 160 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20base%20editing%20in%20rice%20and%20tomato%20using%20a%20CRISPR%E2%80%93Cas9%20cytidine%20deaminase%20fusion&amp;journal=Nat%20Biotechnol&amp;doi=10.1038%2Fnbt.3833&amp;volume=35&amp;pages=441-443&amp;publication_year=2017&amp;author=Shimatani%2CZ&amp;author=Kashojiya%2CS&amp;author=Takayama%2CM&amp;author=Terada%2CR&amp;author=Arazoe%2CT&amp;author=Ishii%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Z. Shimatani, U. Fujikura, H. Ishii, Y. Matsui, M. Suzuki, Y. Ueke, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Shimatani Z, Fujikura U, Ishii H, Matsui Y, Suzuki M, Ueke Y, et al. Inheritance of co-edited genes by CRISPR-" /><span class="c-article-references__counter">161.</span><p class="c-article-references__text" id="ref-CR161">Shimatani Z, Fujikura U, Ishii H, Matsui Y, Suzuki M, Ueke Y, et al. Inheritance of co-edited genes by CRISPR-based targeted nucleotide substitutions in rice. Plant Physiol Biochem. 2018. <a href="https://doi.org/10.1016/j.plaphy.2018.04.028">https://doi.org/10.1016/j.plaphy.2018.04.028</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fj.plaphy.2018.04.028" aria-label="View reference 161">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 161 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Inheritance%20of%20co-edited%20genes%20by%20CRISPR-based%20targeted%20nucleotide%20substitutions%20in%20rice&amp;journal=Plant%20Physiol%20Biochem&amp;doi=10.1016%2Fj.plaphy.2018.04.028&amp;publication_year=2018&amp;author=Shimatani%2CZ&amp;author=Fujikura%2CU&amp;author=Ishii%2CH&amp;author=Matsui%2CY&amp;author=Suzuki%2CM&amp;author=Ueke%2CY">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="H. Butt, A. Eid, Z. Ali, MAM. Atia, MM. Mokhtar, N. Hassan, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Butt H, Eid A, Ali Z, Atia MAM, Mokhtar MM, Hassan N, et al. Efficient CRISPR/Cas9-mediated genome editing usi" /><span class="c-article-references__counter">162.</span><p class="c-article-references__text" id="ref-CR162">Butt H, Eid A, Ali Z, Atia MAM, Mokhtar MM, Hassan N, et al. Efficient CRISPR/Cas9-mediated genome editing using a chimeric single-guide RNA molecule. Front Plant Sci. 2017;8:1441. <a href="https://doi.org/10.3389/fpls.2017.01441">https://doi.org/10.3389/fpls.2017.01441</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.3389%2Ffpls.2017.01441" aria-label="View reference 162">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 162 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20CRISPR%2FCas9-mediated%20genome%20editing%20using%20a%20chimeric%20single-guide%20RNA%20molecule&amp;journal=Front%20Plant%20Sci&amp;doi=10.3389%2Ffpls.2017.01441&amp;volume=8&amp;publication_year=2017&amp;author=Butt%2CH&amp;author=Eid%2CA&amp;author=Ali%2CZ&amp;author=Atia%2CMAM&amp;author=Mokhtar%2CMM&amp;author=Hassan%2CN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Z. Li, Z-B. Liu, A. Xing, BP. Moon, JP. Koellhoffer, L. Huang, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Li Z, Liu Z-B, Xing A, Moon BP, Koellhoffer JP, Huang L, et al. Cas9-guide RNA directed genome editing in soyb" /><span class="c-article-references__counter">163.</span><p class="c-article-references__text" id="ref-CR163">Li Z, Liu Z-B, Xing A, Moon BP, Koellhoffer JP, Huang L, et al. Cas9-guide RNA directed genome editing in soybean. Plant Physiol. 2015;169:96070. <a href="https://doi.org/10.1104/pp.15.00783">https://doi.org/10.1104/pp.15.00783</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XisVSmu7k%253D" aria-label="View reference 163 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1104%2Fpp.15.00783" aria-label="View reference 163">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 163 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Cas9-guide%20RNA%20directed%20genome%20editing%20in%20soybean&amp;journal=Plant%20Physiol&amp;doi=10.1104%2Fpp.15.00783&amp;volume=169&amp;pages=960-970&amp;publication_year=2015&amp;author=Li%2CZ&amp;author=Liu%2CZ-B&amp;author=Xing%2CA&amp;author=Moon%2CBP&amp;author=Koellhoffer%2CJP&amp;author=Huang%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="D. Chilcoat, Z-B. Liu, J. Sander, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Chilcoat D, Liu Z-B, Sander J. Use of CRISPR/Cas9 for crop improvement in maize and soybean. Prog Mol Biol Tra" /><span class="c-article-references__counter">164.</span><p class="c-article-references__text" id="ref-CR164">Chilcoat D, Liu Z-B, Sander J. Use of CRISPR/Cas9 for crop improvement in maize and soybean. Prog Mol Biol Transl Sci. 2017;149:2746. <a href="https://doi.org/10.1016/bs.pmbts.2017.04.005">https://doi.org/10.1016/bs.pmbts.2017.04.005</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BB3cXhsVOjsLY%253D" aria-label="View reference 164 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1016%2Fbs.pmbts.2017.04.005" aria-label="View reference 164">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 164 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Use%20of%20CRISPR%2FCas9%20for%20crop%20improvement%20in%20maize%20and%20soybean&amp;journal=Prog%20Mol%20Biol%20Transl%20Sci&amp;doi=10.1016%2Fbs.pmbts.2017.04.005&amp;volume=149&amp;pages=27-46&amp;publication_year=2017&amp;author=Chilcoat%2CD&amp;author=Liu%2CZ-B&amp;author=Sander%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2018. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">165.</span><p class="c-article-references__text" id="ref-CR165">United States Department of Agriculture (USDA). 2018. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/18-036-01_a1_air_inquiry_cbidel.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/18-036-01_a1_air_inquiry_cbidel.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X. Zhou, TB. Jacobs, L-J. Xue, SA. Harding, C-J. Tsai, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Zhou X, Jacobs TB, Xue L-J, Harding SA, Tsai C-J. Exploiting SNPs for biallelic CRISPR mutations in the outcro" /><span class="c-article-references__counter">166.</span><p class="c-article-references__text" id="ref-CR166">Zhou X, Jacobs TB, Xue L-J, Harding SA, Tsai C-J. Exploiting SNPs for biallelic CRISPR mutations in the outcrossing woody perennial <i>Populus</i> reveals 4-coumarate:CoA ligase specificity and redundancy. New Phytol. 2015;208:298301. <a href="https://doi.org/10.1111/nph.13470">https://doi.org/10.1111/nph.13470</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXhsFGrur3M" aria-label="View reference 166 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fnph.13470" aria-label="View reference 166">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 166 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Exploiting%20SNPs%20for%20biallelic%20CRISPR%20mutations%20in%20the%20outcrossing%20woody%20perennial%20Populus%20reveals%204-coumarate%3ACoA%20ligase%20specificity%20and%20redundancy&amp;journal=New%20Phytol&amp;doi=10.1111%2Fnph.13470&amp;volume=208&amp;pages=298-301&amp;publication_year=2015&amp;author=Zhou%2CX&amp;author=Jacobs%2CTB&amp;author=Xue%2CL-J&amp;author=Harding%2CSA&amp;author=Tsai%2CC-J">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="M. Andersson, H. Turesson, A. Nicolia, A-S. Flt, M. Samuelsson, P. Hofvander, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Andersson M, Turesson H, Nicolia A, Flt A-S, Samuelsson M, Hofvander P. Efficient targeted multiallelic mutag" /><span class="c-article-references__counter">167.</span><p class="c-article-references__text" id="ref-CR167">Andersson M, Turesson H, Nicolia A, Flt A-S, Samuelsson M, Hofvander P. Efficient targeted multiallelic mutagenesis in tetraploid potato (<i>Solanum tuberosum</i>) by transient CRISPRCas9 expression in protoplasts. Plant Cell Rep. 2017;36:11728. <a href="https://doi.org/10.1007/s00299-016-2062-3">https://doi.org/10.1007/s00299-016-2062-3</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28Xhs1Wnt7nO" aria-label="View reference 167 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs00299-016-2062-3" aria-label="View reference 167">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 167 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20targeted%20multiallelic%20mutagenesis%20in%20tetraploid%20potato%20%28Solanum%20tuberosum%29%20by%20transient%20CRISPR%E2%80%93Cas9%20expression%20in%20protoplasts&amp;journal=Plant%20Cell%20Rep&amp;doi=10.1007%2Fs00299-016-2062-3&amp;volume=36&amp;pages=117-128&amp;publication_year=2017&amp;author=Andersson%2CM&amp;author=Turesson%2CH&amp;author=Nicolia%2CA&amp;author=F%C3%A4lt%2CA-S&amp;author=Samuelsson%2CM&amp;author=Hofvander%2CP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="JH. Jung, F. Altpeter, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Jung JH, Altpeter F. TALEN mediated targeted mutagenesis of the caffeic acid O-methyltransferase in highly pol" /><span class="c-article-references__counter">168.</span><p class="c-article-references__text" id="ref-CR168">Jung JH, Altpeter F. TALEN mediated targeted mutagenesis of the caffeic acid <i>O</i>-methyltransferase in highly polyploid sugarcane improves cell wall composition for production of bioethanol. Plant Mol Biol. 2016;92:13142. <a href="https://doi.org/10.1007/s11103-016-0499-y">https://doi.org/10.1007/s11103-016-0499-y</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XpvFSjtbY%253D" aria-label="View reference 168 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs11103-016-0499-y" aria-label="View reference 168">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 168 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=TALEN%20mediated%20targeted%20mutagenesis%20of%20the%20caffeic%20acid%20O-methyltransferase%20in%20highly%20polyploid%20sugarcane%20improves%20cell%20wall%20composition%20for%20production%20of%20bioethanol&amp;journal=Plant%20Mol%20Biol&amp;doi=10.1007%2Fs11103-016-0499-y&amp;volume=92&amp;pages=131-142&amp;publication_year=2016&amp;author=Jung%2CJH&amp;author=Altpeter%2CF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="B. Kannan, JH. Jung, GW. Moxley, S-M. Lee, F. Altpeter, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Kannan B, Jung JH, Moxley GW, Lee S-M, Altpeter F. TALEN-mediated targeted mutagenesis of more than 100 COMT c" /><span class="c-article-references__counter">169.</span><p class="c-article-references__text" id="ref-CR169">Kannan B, Jung JH, Moxley GW, Lee S-M, Altpeter F. TALEN-mediated targeted mutagenesis of more than 100 COMT copies/alleles in highly polyploid sugarcane improves saccharification efficiency without compromising biomass yield. Plant Biotechnol J. 2018;16:85666. <a href="https://doi.org/10.1111/pbi.12833">https://doi.org/10.1111/pbi.12833</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXmsVahurw%253D" aria-label="View reference 169 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12833" aria-label="View reference 169">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 169 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=TALEN-mediated%20targeted%20mutagenesis%20of%20more%20than%20100%20COMT%20copies%2Falleles%20in%20highly%20polyploid%20sugarcane%20improves%20saccharification%20efficiency%20without%20compromising%20biomass%20yield&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12833&amp;volume=16&amp;pages=856-866&amp;publication_year=2018&amp;author=Kannan%2CB&amp;author=Jung%2CJH&amp;author=Moxley%2CGW&amp;author=Lee%2CS-M&amp;author=Altpeter%2CF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J-J. Park, CG. Yoo, A. Flanagan, Y. Pu, S. Debnath, Y. Ge, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Park J-J, Yoo CG, Flanagan A, Pu Y, Debnath S, Ge Y, et al. Defined tetra-allelic gene disruption of the 4-cou" /><span class="c-article-references__counter">170.</span><p class="c-article-references__text" id="ref-CR170">Park J-J, Yoo CG, Flanagan A, Pu Y, Debnath S, Ge Y, et al. Defined tetra-allelic gene disruption of the 4-coumarate: coenzyme A ligase 1 (Pv4CL1) gene by CRISPR/Cas9 in switchgrass results in lignin reduction and improved sugar release. Biotechnol Biofuels. 2017;10:284. <a href="https://doi.org/10.1186/s13068-017-0972-0">https://doi.org/10.1186/s13068-017-0972-0</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1MXis1Oht7s%253D" aria-label="View reference 170 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1186%2Fs13068-017-0972-0" aria-label="View reference 170">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 170 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Defined%20tetra-allelic%20gene%20disruption%20of%20the%204-coumarate%3A%20coenzyme%20A%20ligase%201%20%28Pv4CL1%29%20gene%20by%20CRISPR%2FCas9%20in%20switchgrass%20results%20in%20lignin%20reduction%20and%20improved%20sugar%20release&amp;journal=Biotechnol%20Biofuels&amp;doi=10.1186%2Fs13068-017-0972-0&amp;volume=10&amp;publication_year=2017&amp;author=Park%2CJ-J&amp;author=Yoo%2CCG&amp;author=Flanagan%2CA&amp;author=Pu%2CY&amp;author=Debnath%2CS&amp;author=Ge%2CY">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2011. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">171.</span><p class="c-article-references__text" id="ref-CR171">United States Department of Agriculture (USDA). 2011. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-126-01_air_inquiry.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-126-01_air_inquiry.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="E. Njuguna, G. Coussens, S. Aesaert, P. Neyt, S. Anami, M. Lijsebettens, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Njuguna E, Coussens G, Aesaert S, Neyt P, Anami S, van Lijsebettens M. Modulation of energy homeostasis in mai" /><span class="c-article-references__counter">172.</span><p class="c-article-references__text" id="ref-CR172">Njuguna E, Coussens G, Aesaert S, Neyt P, Anami S, van Lijsebettens M. Modulation of energy homeostasis in maize and <i>Arabidopsis</i> to develop lines tolerant to drought, genotoxic and oxidative stresses. Afrika Focus. 2018. <a href="https://doi.org/10.21825/af.v30i2.8080">https://doi.org/10.21825/af.v30i2.8080</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.21825%2Faf.v30i2.8080" aria-label="View reference 172">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 172 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Modulation%20of%20energy%20homeostasis%20in%20maize%20and%20Arabidopsis%20to%20develop%20lines%20tolerant%20to%20drought%2C%20genotoxic%20and%20oxidative%20stresses&amp;journal=Afrika%20Focus&amp;doi=10.21825%2Faf.v30i2.8080&amp;publication_year=2018&amp;author=Njuguna%2CE&amp;author=Coussens%2CG&amp;author=Aesaert%2CS&amp;author=Neyt%2CP&amp;author=Anami%2CS&amp;author=Lijsebettens%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="headline" content="United States Department of Agriculture (USDA). 2017. https://www.aphis.usda.gov/biotechnology/downloads/reg_l" /><span class="c-article-references__counter">173.</span><p class="c-article-references__text" id="ref-CR173">United States Department of Agriculture (USDA). 2017. <a href="https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-219-01_air_inquiry.pdf">https://www.aphis.usda.gov/biotechnology/downloads/reg_loi/17-219-01_air_inquiry.pdf</a>. Accessed 25 Aug 2018.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="D. Kim, B. Alptekin, H. Budak, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Kim D, Alptekin B, Budak H. CRISPR/Cas9 genome editing in wheat. Funct Integr Genom. 2018;18:3141. https://do" /><span class="c-article-references__counter">174.</span><p class="c-article-references__text" id="ref-CR174">Kim D, Alptekin B, Budak H. CRISPR/Cas9 genome editing in wheat. Funct Integr Genom. 2018;18:3141. <a href="https://doi.org/10.1007/s10142-017-0572-x">https://doi.org/10.1007/s10142-017-0572-x</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhsFSrsr7L" aria-label="View reference 174 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs10142-017-0572-x" aria-label="View reference 174">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 174 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FCas9%20genome%20editing%20in%20wheat&amp;journal=Funct%20Integr%20Genom&amp;doi=10.1007%2Fs10142-017-0572-x&amp;volume=18&amp;pages=31-41&amp;publication_year=2018&amp;author=Kim%2CD&amp;author=Alptekin%2CB&amp;author=Budak%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="X. Meng, X. Hu, Q. Liu, X. Song, C. Gao, J. Li, K. Wang, " /><meta itemprop="datePublished" content="2018" /><meta itemprop="headline" content="Meng X, Hu X, Liu Q, Song X, Gao C, Li J, Wang K. Robust genome editing of CRISPRCas9 at NAG PAMs in rice. Sc" /><span class="c-article-references__counter">175.</span><p class="c-article-references__text" id="ref-CR175">Meng X, Hu X, Liu Q, Song X, Gao C, Li J, Wang K. Robust genome editing of CRISPRCas9 at NAG PAMs in rice. Sci China Life Sci. 2018;61:1225. <a href="https://doi.org/10.1007/s11427-017-9247-9">https://doi.org/10.1007/s11427-017-9247-9</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC1cXhvVChtg%253D%253D" aria-label="View reference 175 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs11427-017-9247-9" aria-label="View reference 175">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 175 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20genome%20editing%20of%20CRISPR%E2%80%93Cas9%20at%20NAG%20PAMs%20in%20rice&amp;journal=Sci%20China%20Life%20Sci&amp;doi=10.1007%2Fs11427-017-9247-9&amp;volume=61&amp;pages=122-125&amp;publication_year=2018&amp;author=Meng%2CX&amp;author=Hu%2CX&amp;author=Liu%2CQ&amp;author=Song%2CX&amp;author=Gao%2CC&amp;author=Li%2CJ&amp;author=Wang%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="Z. Ali, A. Abul-faraj, M. Piatek, MM. Mahfouz, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Ali Z, Abul-faraj A, Piatek M, Mahfouz MM. Activity and specificity of TRV-mediated gene editing in plants. Pl" /><span class="c-article-references__counter">176.</span><p class="c-article-references__text" id="ref-CR176">Ali Z, Abul-faraj A, Piatek M, Mahfouz MM. Activity and specificity of TRV-mediated gene editing in plants. Plant Signal Behav. 2015;10:e1044191. <a href="https://doi.org/10.1080/15592324.2015.1044191">https://doi.org/10.1080/15592324.2015.1044191</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXhvVCis7rM" aria-label="View reference 176 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1080%2F15592324.2015.1044191" aria-label="View reference 176">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 176 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Activity%20and%20specificity%20of%20TRV-mediated%20gene%20editing%20in%20plants&amp;journal=Plant%20Signal%20Behav&amp;doi=10.1080%2F15592324.2015.1044191&amp;volume=10&amp;publication_year=2015&amp;author=Ali%2CZ&amp;author=Abul-faraj%2CA&amp;author=Piatek%2CM&amp;author=Mahfouz%2CMM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="C. Shen, Z. Que, Y. Xia, N. Tang, D. Li, R. He, M. Cao, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Shen C, Que Z, Xia Y, Tang N, Li D, He R, Cao M. Knock out of the annexin gene OsAnn3 via CRISPR/Cas9-mediated" /><span class="c-article-references__counter">177.</span><p class="c-article-references__text" id="ref-CR177">Shen C, Que Z, Xia Y, Tang N, Li D, He R, Cao M. Knock out of the annexin gene OsAnn3 via CRISPR/Cas9-mediated genome editing decreased cold tolerance in rice. J Plant Biol. 2017;60:53947. <a href="https://doi.org/10.1007/s12374-016-0400-1">https://doi.org/10.1007/s12374-016-0400-1</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2sXhvFKhu7bF" aria-label="View reference 177 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs12374-016-0400-1" aria-label="View reference 177">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 177 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Knock%20out%20of%20the%20annexin%20gene%20OsAnn3%20via%20CRISPR%2FCas9-mediated%20genome%20editing%20decreased%20cold%20tolerance%20in%20rice&amp;journal=J%20Plant%20Biol&amp;doi=10.1007%2Fs12374-016-0400-1&amp;volume=60&amp;pages=539-547&amp;publication_year=2017&amp;author=Shen%2CC&amp;author=Que%2CZ&amp;author=Xia%2CY&amp;author=Tang%2CN&amp;author=Li%2CD&amp;author=He%2CR&amp;author=Cao%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="S. Tian, L. Jiang, Q. Gao, J. Zhang, M. Zong, H. Zhang, " /><meta itemprop="datePublished" content="2017" /><meta itemprop="headline" content="Tian S, Jiang L, Gao Q, Zhang J, Zong M, Zhang H, et al. Efficient CRISPR/Cas9-based gene knockout in watermel" /><span class="c-article-references__counter">178.</span><p class="c-article-references__text" id="ref-CR178">Tian S, Jiang L, Gao Q, Zhang J, Zong M, Zhang H, et al. Efficient CRISPR/Cas9-based gene knockout in watermelon. Plant Cell Rep. 2017;36:399406. <a href="https://doi.org/10.1007/s00299-016-2089-5">https://doi.org/10.1007/s00299-016-2089-5</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28XitFWntb%252FP" aria-label="View reference 178 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs00299-016-2089-5" aria-label="View reference 178">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 178 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20CRISPR%2FCas9-based%20gene%20knockout%20in%20watermelon&amp;journal=Plant%20Cell%20Rep&amp;doi=10.1007%2Fs00299-016-2089-5&amp;volume=36&amp;pages=399-406&amp;publication_year=2017&amp;author=Tian%2CS&amp;author=Jiang%2CL&amp;author=Gao%2CQ&amp;author=Zhang%2CJ&amp;author=Zong%2CM&amp;author=Zhang%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="H. Zhang, J. Zhang, P. Wei, B. Zhang, F. Gou, Z. Feng, " /><meta itemprop="datePublished" content="2014" /><meta itemprop="headline" content="Zhang H, Zhang J, Wei P, Zhang B, Gou F, Feng Z, et al. The CRISPR/Cas9 system produces specific and homozygou" /><span class="c-article-references__counter">179.</span><p class="c-article-references__text" id="ref-CR179">Zhang H, Zhang J, Wei P, Zhang B, Gou F, Feng Z, et al. The CRISPR/Cas9 system produces specific and homozygous targeted gene editing in rice in one generation. Plant Biotechnol J. 2014;12:797807. <a href="https://doi.org/10.1111/pbi.12200">https://doi.org/10.1111/pbi.12200</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXht1Wmsr7N" aria-label="View reference 179 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1111%2Fpbi.12200" aria-label="View reference 179">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 179 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20CRISPR%2FCas9%20system%20produces%20specific%20and%20homozygous%20targeted%20gene%20editing%20in%20rice%20in%20one%20generation&amp;journal=Plant%20Biotechnol%20J&amp;doi=10.1111%2Fpbi.12200&amp;volume=12&amp;pages=797-807&amp;publication_year=2014&amp;author=Zhang%2CH&amp;author=Zhang%2CJ&amp;author=Wei%2CP&amp;author=Zhang%2CB&amp;author=Gou%2CF&amp;author=Feng%2CZ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="J. Forner, A. Pfeiffer, T. Langenecker, PA. Manavella, P. Manavella, JU. Lohmann, " /><meta itemprop="datePublished" content="2015" /><meta itemprop="headline" content="Forner J, Pfeiffer A, Langenecker T, Manavella PA, Manavella P, Lohmann JU. Germline-transmitted genome editin" /><span class="c-article-references__counter">180.</span><p class="c-article-references__text" id="ref-CR180">Forner J, Pfeiffer A, Langenecker T, Manavella PA, Manavella P, Lohmann JU. Germline-transmitted genome editing in <i>Arabidopsis thaliana</i> using TAL-effector-nucleases. PLoS ONE. 2015;10:e0121056. <a href="https://doi.org/10.1371/journal.pone.0121056">https://doi.org/10.1371/journal.pone.0121056</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2MXhs1Sjt7jK" aria-label="View reference 180 on CAS">CAS</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0121056" aria-label="View reference 180">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 180 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Germline-transmitted%20genome%20editing%20in%20Arabidopsis%20thaliana%20using%20TAL-effector-nucleases&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0121056&amp;volume=10&amp;publication_year=2015&amp;author=Forner%2CJ&amp;author=Pfeiffer%2CA&amp;author=Langenecker%2CT&amp;author=Manavella%2CPA&amp;author=Manavella%2CP&amp;author=Lohmann%2CJU">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="B. Ren, F. Yan, Y. Kuang, N. Li, D. Zhang, H. Lin, H. Zhou, " /><meta itemprop="datePublished" content="2007" /><meta itemprop="headline" content="Ren B, Yan F, Kuang Y, Li N, Zhang D, Lin H, Zhou H. Specificity and inheritance of rBE3 and rBE4 endonuclease" /><span class="c-article-references__counter">181.</span><p class="c-article-references__text" id="ref-CR181">Ren B, Yan F, Kuang Y, Li N, Zhang D, Lin H, Zhou H. Specificity and inheritance of rBE3 and rBE4 endonuclease-induced gene modifications in rice. Chin J Biotechnol 2007;33(10):177685.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 181 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Specificity%20and%20inheritance%20of%20rBE3%20and%20rBE4%20endonuclease-induced%20gene%20modifications%20in%20rice&amp;journal=Chin%20J%20Biotechnol&amp;volume=33&amp;issue=10&amp;pages=1776-85&amp;publication_year=2007&amp;author=Ren%2CB&amp;author=Yan%2CF&amp;author=Kuang%2CY&amp;author=Li%2CN&amp;author=Zhang%2CD&amp;author=Lin%2CH&amp;author=Zhou%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><meta itemprop="author" content="K. Khandagale, A. Nadaf, " /><meta itemprop="datePublished" content="2016" /><meta itemprop="headline" content="Khandagale K, Nadaf A. Genome editing for targeted improvement of plants. Plant Biotechnol Rep. 2016;10:32743" /><span class="c-article-references__counter">182.</span><p class="c-article-references__text" id="ref-CR182">Khandagale K, Nadaf A. Genome editing for targeted improvement of plants. Plant Biotechnol Rep. 2016;10:32743. <a href="https://doi.org/10.1007/s11816-016-0417-4">https://doi.org/10.1007/s11816-016-0417-4</a>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="outbound reference" data-track-label="link" href="https://doi.org/10.1007%2Fs11816-016-0417-4" aria-label="View reference 182">Article</a>
    <a data-track="click" data-track-action="outbound reference" data-track-label="link" aria-label="Search for reference 182 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20editing%20for%20targeted%20improvement%20of%20plants&amp;journal=Plant%20Biotechnol%20Rep&amp;doi=10.1007%2Fs11816-016-0417-4&amp;volume=10&amp;pages=327-343&amp;publication_year=2016&amp;author=Khandagale%2CK&amp;author=Nadaf%2CA">
                    Google Scholar</a>
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" href="/articles/10.1186/s13750-019-0171-5-references.ris">Download references<svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p></div></div></div></section><section aria-labelledby="Ack1"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>The authors wish to thank Maren Fischer for carefully reading the manuscript.</p></div></div></section><section aria-labelledby="Fun"><div class="c-article-section" id="Fun-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Fun">Funding</h2><div class="c-article-section__content" id="Fun-content"><p>This systematic map is part of the ELSA-GEA project funded by the Federal Ministry of Education and Research (BMBF) (01GP1613B).</p></div></div></section><section aria-labelledby="author-information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Institute for Biosafety in Plant Biotechnology, Julius Khn-Institute, Federal Research Centre for Cultivated Plants, Erwin-Baur-Strae 27, 06484, Quedlinburg, Germany</p><p class="c-article-author-affiliation__authors-list">Dominik Modrzejewski,Frank Hartung,Thorben Sprink,Drthe Krause,Christian Kohl&amp;Ralf Wilhelm</p></li></ol><div class="js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-1"><span class="c-article-authors-search__title u-h3 js-search-name">Dominik Modrzejewski</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23&#34;Dominik+Modrzejewski&#34;" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Dominik+Modrzejewski" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Dominik+Modrzejewski%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-2"><span class="c-article-authors-search__title u-h3 js-search-name">Frank Hartung</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23&#34;Frank+Hartung&#34;" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Frank+Hartung" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Frank+Hartung%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-3"><span class="c-article-authors-search__title u-h3 js-search-name">Thorben Sprink</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23&#34;Thorben+Sprink&#34;" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Thorben+Sprink" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Thorben+Sprink%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-4"><span class="c-article-authors-search__title u-h3 js-search-name">Drthe Krause</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23&#34;D%C3%B6rthe+Krause&#34;" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=D%C3%B6rthe+Krause" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22D%C3%B6rthe+Krause%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-5"><span class="c-article-authors-search__title u-h3 js-search-name">Christian Kohl</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23&#34;Christian+Kohl&#34;" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christian+Kohl" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christian+Kohl%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li><li id="auth-6"><span class="c-article-authors-search__title u-h3 js-search-name">Ralf Wilhelm</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="https://www.biomedcentral.com/search?query=author%23&#34;Ralf+Wilhelm&#34;" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ralf+Wilhelm" data-track="click" data-track-action="author link - pubmed" data-track-label="link">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ralf+Wilhelm%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>The review process was coordinated by DM. DM, FH, TS and DK conducted the screening of articles and data extraction. DM drafted the report. FH, TS, CK and RW assisted in editing and revising the draft. All authors read and approved the final manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" rel="nofollow" href="/articles/10.1186/s13750-019-0171-5/email/correspondent/c1/new">Dominik Modrzejewski</a>.</p></div></div></section><section aria-labelledby="ethics"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading">Ethics approval and consent to participate</h3>
                <p>Not applicable.</p>
              
              
                <h3 class="c-article__sub-heading">Consent for publication</h3>
                <p>Not applicable.</p>
              
              
                <h3 class="c-article__sub-heading">Competing interests</h3>
                <p>The reviewers Frank Hartung (FH) and Thorben Sprink (TS) are authors of a few research studies included in the review process. However, as none of these studies comprise primary data, their articles were excluded at title/abstract level.</p>
              
            </div></div></section><section aria-labelledby="additional-information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><h3 class="c-article__sub-heading">Publisher's Note</h3><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section aria-labelledby="Sec49"><div class="c-article-section" id="Sec49-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec49">Additional files</h2><div class="c-article-section__content" id="Sec49-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div>
                  
                  
                  
                  
                  
                  
                  
                <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_MOESM1_ESM.docx" data-supp-info-image=""><b>Additional file 1.</b> Detailed description of the genome-editing techniques.</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_MOESM2_ESM.xlsx" data-supp-info-image=""><b>Additional file 2.</b> List of peer-reviewed and grey literature sources searched.</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_MOESM3_ESM.xlsx" data-supp-info-image=""><b>Additional file 3.</b> List of un-retrievable articles.</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_MOESM4_ESM.xlsx" data-supp-info-image=""><b>Additional file 4.</b> List of articles excluded at full text assessment with the reason for exclusion.</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_MOESM5_ESM.xlsx" data-supp-info-image=""><b>Additional file 5.</b> List of relevant studies and the systematic map database to answer secondary question 1.</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_MOESM6_ESM.xlsx" data-supp-info-image=""><b>Additional file 6.</b> List of relevant studies and the systematic map database to answer secondary question 2.</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-track-label="link" data-test="supp-info-link" href="https://static-content.springer.com/esm/art%3A10.1186%2Fs13750-019-0171-5/MediaObjects/13750_2019_171_MOESM7_ESM.xlsx" data-supp-info-image=""><b>Additional file 7.</b> ROSES Reporting Form.</a></h3></div></div></div></div></section><section aria-labelledby="rightslink"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b> This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license" itemprop="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<a href="http://creativecommons.org/publicdomain/zero/1.0/" rel="license" itemprop="license">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=What%20is%20the%20available%20evidence%20for%20the%20range%20of%20applications%20of%20genome-editing%20as%20a%20new%20tool%20for%20plant%20trait%20modification%20and%20the%20potential%20occurrence%20of%20associated%20off-target%20effects%3A%20a%20systematic%20map&amp;author=Dominik%20Modrzejewski%20et%20al&amp;contentID=10.1186%2Fs13750-019-0171-5&amp;publication=2047-2382&amp;publicationDate=2019-07-12&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY%20%2B%20CC0">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1186/s13750-019-0171-5" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1186/s13750-019-0171-5" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img width="57" height="81" alt="Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" /></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Modrzejewski, D., Hartung, F., Sprink, T. <i>et al.</i> What is the available evidence for the range of applications of genome-editing as a new tool for plant trait modification and the potential occurrence of associated off-target effects: a systematic map.
                    <i>Environ Evid</i> <b>8, </b>27 (2019). https://doi.org/10.1186/s13750-019-0171-5</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" href="/articles/10.1186/s13750-019-0171-5.ris">Download citation<svg width="16" height="16" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#global-icon-download"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2019-03-28">28 March 2019</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2019-06-27">27 June 2019</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2019-07-12">12 July 2019</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--doi"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value"><a href="https://doi.org/10.1186/s13750-019-0171-5" data-track="click" data-track-action="view doi" data-track-label="link" itemprop="sameAs">https://doi.org/10.1186/s13750-019-0171-5</a></span></p></li></ul><div data-component="share-box"></div><h3 class="c-article__sub-heading">Keywords</h3><ul class="c-article-subject-list"><li class="c-article-subject-list__subject"><span itemprop="about">New plant breeding techniques</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Gene editing</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Targeted genome modification</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Mutagenesis</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Unintended effects</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Off-target mutation</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Evidence map</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Evidence synthesis</span></li></ul><div data-component="article-info-list"></div></div></div></div></div></section>

                    


                </article>
            </main>

            <div class="c-article-extras u-text-sm u-hide-print" data-track-component="reading companion">
                <aside>
                    
    
    <div class="c-pdf-download u-clear-both">
        <a href="//environmentalevidencejournal.biomedcentral.com/track/pdf/10.1186/s13750-019-0171-5" class="c-pdf-download__link" data-article-pdf="true"  data-readcube-pdf-url="true" data-test="pdf-link" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link">
            <span>Download PDF</span>
            <svg width="16" height="16" class="u-icon"><use xlink:href="#global-icon-download"/></svg>
        </a>
    </div>
    

                    

                    
        
    

                    <div class="c-reading-companion">
                        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
                            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                                <div class="js-ad">
    <aside class="adsbox c-ad c-ad--300x250">
        <div class="c-ad__inner" >
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-MPU1"
                 data-gpt-unitpath="/270604982/bmc/environmentalevidencejournal/articles"
                 data-gpt-sizes="300x250"
                 data-gpt-targeting="pos=MPU1;doi=10.1186/s13750-019-0171-5;kwrd=New plant breeding techniques,Gene editing,Targeted genome modification,Mutagenesis,Unintended effects,Off-target mutation,Evidence map,Evidence synthesis;pmc=U17009,L1904X,U16002,X33040,L22016,U17009;"
                 data-ad-type="MPU1">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/environmentalevidencejournal/articles&amp;sz=300x250&amp;pos=MPU1&amp;doi=10.1186/s13750-019-0171-5&amp;kwrd=New plant breeding techniques,Gene editing,Targeted genome modification,Mutagenesis,Unintended effects,Off-target mutation,Evidence map,Evidence synthesis&amp;pmc=U17009,L1904X,U16002,X33040,L22016,U17009&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/environmentalevidencejournal/articles&amp;sz=300x250&amp;pos=MPU1&amp;doi=10.1186/s13750-019-0171-5&amp;kwrd=New plant breeding techniques,Gene editing,Targeted genome modification,Mutagenesis,Unintended effects,Off-target mutation,Evidence map,Evidence synthesis&amp;pmc=U17009,L1904X,U16002,X33040,L22016,U17009&amp;"
                             alt="Advertisement"
                             width="300"
                             height="250">
                    </a>
                </noscript>
            </div>
        </div>
    </aside>
</div>
                            </div>
                            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
                            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
                        </div>
                    </div>
                </aside>
            </div>

        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            <img class="c-journal-title__logo" src="https://environmentalevidencejournal.biomedcentral.com/images/13750/logo.svg"/>
                            <span class="c-journal-title__text">Environmental Evidence</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 2047-2382</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title c-journal-footer__contact-title--adjust-for-logo">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/enev" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class='tracker' style='display:none' src='/track/article/10.1186/s13750-019-0171-5' alt=""/>

         
    <footer>
        
            <div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between" data-test="publisher-footer-menu">
            <div class="u-display-flex">
                
                    
                        <ul class="c-list-group c-list-group--sm u-mr-24 u-mb-16">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-mr-24 u-mb-16">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-mr-24 u-mb-16">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://support.biomedcentral.com/support/home">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="//www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
                    
                
            </div>
            <div class="u-mb-24">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="u-display-flex u-list-reset" data-test="footer-social-links">
                    
                        <li class="u-mt-8 u-mr-8">
                            <a href="https://twitter.com/biomedcentral"
                               class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-twitter-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-mt-8 u-mr-8">
                            <a href="https://www.facebook.com/BioMedCentral"
                               class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-facebook-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-mt-8 u-mr-8">
                            <a href="http://www.weibo.com/biomedcentral"
                               class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-weibo-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="//www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="//www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        
        <div class="c-corporate-footer">
    <div class="u-container">
        <img src=/static/images/logo-springernature-44af1f90df.svg class="c-corporate-footer__logo" alt="Springer Nature" itemprop="logo" role="img">
        <p class="c-corporate-footer__legal" data-test="copyright"> &#169; 2020 BioMed Central Ltd unless otherwise stated. Part of
            <a class="c-corporate-footer__link" href="https://www.springernature.com" itemscope itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

        
    </footer>

    </div>

    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>



        
    <svg class="u-hide hide">
        <symbol id="global-icon-chevron-right" viewBox="0 0 16 16">
            <path d="M7.782 7L5.3 4.518c-.393-.392-.4-1.022-.02-1.403a1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L7.782 9l1.013-.998z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-download" viewBox="0 0 16 16">
            <path d="M2 14c0-.556.449-1 1.002-1h9.996a.999.999 0 110 2H3.002A1.006 1.006 0 012 14zM9 2v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 010 1.417l-4.177 4.177a1.001 1.001 0 01-1.416 0L3.115 7.715a.991.991 0 01-.016-1.4 1 1 0 011.42.003L7 8.8V2c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-email" viewBox="0 0 18 18">
            <path d="M1.995 2h14.01A2 2 0 0118 4.006v9.988A2 2 0 0116.005 16H1.995A2 2 0 010 13.994V4.006A2 2 0 011.995 2zM1 13.994A1 1 0 001.995 15h14.01A1 1 0 0017 13.994V4.006A1 1 0 0016.005 3H1.995A1 1 0 001 4.006zM9 11L2 7V5.557l7 4 7-4V7z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-institution" viewBox="0 0 18 18">
            <path d="M14 8a1 1 0 011 1v6h1.5a.5.5 0 01.5.5v.5h.5a.5.5 0 01.5.5V18H0v-1.5a.5.5 0 01.5-.5H1v-.5a.5.5 0 01.5-.5H3V9a1 1 0 112 0v6h8V9a1 1 0 011-1zM6 8l2 1v4l-2 1zm6 0v6l-2-1V9zM9.573.401l7.036 4.925A.92.92 0 0116.081 7H1.92a.92.92 0 01-.528-1.674L8.427.401a1 1 0 011.146 0zM9 2.441L5.345 5h7.31z" fill-rule="evenodd"/>
        </symbol>
        <symbol id="global-icon-search" viewBox="0 0 22 22">
            <path fill-rule="evenodd" d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z"/>
        </symbol>
    </svg>

    </body>
</html>



